# PubMed-GitHub

***update 2024-02-07***

A collection of GitHub repositories that appear in PubMed abstract.

## Top 20 Authors

| Rank | Author | Count | Stars | Score |
| ---: | -----: | ----: | ----: | ----: |
| 1 | [brentp](https://github.com/brentp) | 9 | 2364 | 70.41 |
| 2 | [lh3](https://github.com/lh3) | 10 | 2607 | 70.25 |
| 3 | [ncbi](https://github.com/ncbi) | 10 | 1996 | 69.34 |
| 4 | [theislab](https://github.com/theislab) | 7 | 2959 | 55.90 |
| 5 | [bcgsc](https://github.com/bcgsc) | 8 | 957 | 53.49 |
| 6 | [OpenGene](https://github.com/OpenGene) | 7 | 2329 | 50.83 |
| 7 | [microsoft](https://github.com/microsoft) | 5 | 15936 | 43.34 |
| 8 | [Illumina](https://github.com/Illumina) | 5 | 1087 | 37.69 |
| 9 | [Ha0Tang](https://github.com/Ha0Tang) | 4 | 1383 | 32.69 |
| 10 | [broadinstitute](https://github.com/broadinstitute) | 4 | 2272 | 32.39 |
| 11 | [rrwick](https://github.com/rrwick) | 4 | 1272 | 31.63 |
| 12 | [EpistasisLab](https://github.com/EpistasisLab) | 3 | 10388 | 30.49 |
| 13 | [tanghaibao](https://github.com/tanghaibao) | 3 | 1414 | 24.57 |
| 14 | [galaxyproject](https://github.com/galaxyproject) | 3 | 1503 | 24.54 |
| 15 | [MannLabs](https://github.com/MannLabs) | 4 | 271 | 24.27 |
| 16 | [marbl](https://github.com/marbl) | 3 | 825 | 24.19 |
| 17 | [gpertea](https://github.com/gpertea) | 3 | 821 | 24.09 |
| 18 | [zju3dv](https://github.com/zju3dv) | 3 | 1090 | 24.02 |
| 19 | [NVlabs](https://github.com/NVlabs) | 2 | 12281 | 24.00 |
| 20 | [MolecularAI](https://github.com/MolecularAI) | 3 | 929 | 23.86 |


***Note: only count repositories with stars not less than top 1000 threshold (53).***

## Top 1000 Repositories

***Format: description, language, created date, updated date.***

1. [CSSEGISandData/COVID-19](https://github.com/CSSEGISandData/COVID-19) (29199 stars)
Novel Coronavirus (COVID-19) Cases, provided by JHU CSSE (None, 2020-02-04, 2024-02-07)
2. [facebookresearch/Detectron](https://github.com/facebookresearch/Detectron) (26089 stars)
FAIR's research platform for object detection research, implementing popular algorithms like Mask R-CNN and RetinaNet. (Python, 2017-10-05, 2024-02-07)
3. [kubernetes/charts](https://github.com/kubernetes/charts) (15542 stars)
âš ï¸(OBSOLETE) Curated applications for Kubernetes (Go, 2015-11-24, 2024-02-06)
4. [microsoft/Bringing-Old-Photos-Back-to-Life](https://github.com/microsoft/Bringing-Old-Photos-Back-to-Life) (14210 stars)
Bringing Old Photo Back to Life (CVPR 2020 oral) (Python, 2020-06-24, 2024-02-07)
5. [flairNLP/flair](https://github.com/flairNLP/flair) (13388 stars)
A very simple framework for state-of-the-art Natural Language Processing (NLP) (Python, 2018-06-11, 2024-02-06)
6. [NVlabs/stylegan2](https://github.com/NVlabs/stylegan2) (10713 stars)
StyleGAN2 - Official TensorFlow Implementation (Python, 2019-11-26, 2024-02-07)
7. [EpistasisLab/tpot](https://github.com/EpistasisLab/tpot) (9410 stars)
A Python Automated Machine Learning tool that optimizes machine learning pipelines using genetic programming. (Python, 2015-11-03, 2024-02-06)
8. [pachyderm/pachyderm](https://github.com/pachyderm/pachyderm) (6042 stars)
Data-Centric Pipelines and Data Versioning (Go, 2014-09-04, 2024-02-06)
9. [owid/covid-19-data](https://github.com/owid/covid-19-data) (5618 stars)
Data on COVID-19 (coronavirus) cases, deaths, hospitalizations, tests â€¢ All countries â€¢ Updated daily by Our World in Data (Python, 2020-03-09, 2024-02-04)
10. [tyiannak/pyAudioAnalysis](https://github.com/tyiannak/pyAudioAnalysis) (5578 stars)
Python Audio Analysis Library: Feature Extraction, Classification, Segmentation and Applications (Python, 2014-08-27, 2024-02-05)
11. [mdbloice/Augmentor](https://github.com/mdbloice/Augmentor) (5006 stars)
Image augmentation library in Python for machine learning. (Python, 2016-03-01, 2024-02-02)
12. [deepchem/deepchem](https://github.com/deepchem/deepchem) (4897 stars)
Democratizing Deep-Learning for Drug Discovery, Quantum Chemistry, Materials Science and Biology (Python, 2015-09-24, 2024-02-06)
13. [google/deepvariant](https://github.com/google/deepvariant) (3014 stars)
DeepVariant is an analysis pipeline that uses a deep neural network to call genetic variants from next-generation DNA sequencing data. (Python, 2017-11-23, 2024-02-07)
14. [mahyarnajibi/SNIPER](https://github.com/mahyarnajibi/SNIPER) (2680 stars)
SNIPER / AutoFocus is an efficient multi-scale object detection training / inference algorithm (Python, 2018-01-13, 2024-01-24)
15. [MrGiovanni/UNetPlusPlus](https://github.com/MrGiovanni/UNetPlusPlus) (2101 stars)
[IEEE TMI] Official Implementation for UNet++ (Python, 2018-04-16, 2024-02-07)
16. [voxelmorph/voxelmorph](https://github.com/voxelmorph/voxelmorph) (2078 stars)
Unsupervised Learning for Image Registration (Python, 2018-05-14, 2024-02-07)
17. [jiupinjia/SkyAR](https://github.com/jiupinjia/SkyAR) (1975 stars)
Official Pytorch implementation of the preprint paper "Castle in the Sky: Dynamic Sky Replacement and Harmonization in Videos", in arXiv:2010.11800. (Python, 2020-10-22, 2024-02-05)
18. [OpenGene/fastp](https://github.com/OpenGene/fastp) (1690 stars)
An ultra-fast all-in-one FASTQ preprocessor (QC/adapters/trimming/filtering/splitting/merging...) (C++, 2017-10-31, 2024-02-07)
19. [theislab/Scanpy](https://github.com/theislab/Scanpy) (1690 stars)
Single-cell analysis in Python. Scales to >1M cells. (Python, 2017-01-29, 2024-02-06)
20. [lh3/minimap2](https://github.com/lh3/minimap2) (1615 stars)
A versatile pairwise aligner for genomic and spliced nucleotide sequences (C, 2017-07-18, 2024-02-06)
21. [sokrypton/ColabFold](https://github.com/sokrypton/ColabFold) (1584 stars)
Making Protein folding accessible to all! (Jupyter Notebook, 2021-07-19, 2024-02-06)
22. [NVlabs/PWC-Net](https://github.com/NVlabs/PWC-Net) (1568 stars)
PWC-Net: CNNs for Optical Flow Using Pyramid, Warping, and Cost Volume, CVPR 2018 (Oral) (Python, 2018-06-13, 2024-01-29)
23. [broadinstitute/gatk](https://github.com/broadinstitute/gatk) (1557 stars)
Official code repository for GATK versions 4 and up (Java, 2014-12-02, 2024-02-06)
24. [samtools/samtools](https://github.com/samtools/samtools) (1504 stars)
Tools (written in C using htslib) for manipulating next-generation sequencing data (C, 2012-03-09, 2024-02-07)
25. [microsoft/Cream](https://github.com/microsoft/Cream) (1479 stars)
This is a collection of our NAS and Vision Transformer work. (Python, 2020-10-12, 2024-02-07)
26. [SmartDataAnalytics/PyKEEN](https://github.com/SmartDataAnalytics/PyKEEN) (1449 stars)
ðŸ¤– A Python library for learning and evaluating knowledge graph embeddings (Python, 2020-02-24, 2024-02-06)
27. [pykeen/pykeen](https://github.com/pykeen/pykeen) (1449 stars)
ðŸ¤– A Python library for learning and evaluating knowledge graph embeddings (Python, 2020-02-24, 2024-02-06)
28. [speedinghzl/CCNet](https://github.com/speedinghzl/CCNet) (1376 stars)
CCNet: Criss-Cross Attention for Semantic Segmentation (TPAMI 2020 & ICCV 2019). (Python, 2018-11-26, 2024-02-06)
29. [galaxyproject/galaxy](https://github.com/galaxyproject/galaxy) (1219 stars)
Data intensive science for everyone. (Python, 2015-02-23, 2024-02-06)
30. [cvg/pixel-perfect-sfm](https://github.com/cvg/pixel-perfect-sfm) (1188 stars)
Pixel-Perfect Structure-from-Motion with Featuremetric Refinement (ICCV 2021, Best Student Paper Award) (C++, 2021-08-17, 2024-02-05)
31. [xvjiarui/GCNet](https://github.com/xvjiarui/GCNet) (1162 stars)
GCNet: Non-local Networks Meet Squeeze-Excitation Networks and Beyond (Python, 2019-04-25, 2024-01-30)
32. [glample/tagger](https://github.com/glample/tagger) (1149 stars)
Named Entity Recognition Tool (Python, 2016-03-10, 2024-01-26)
33. [shenwei356/seqkit](https://github.com/shenwei356/seqkit) (1141 stars)
A cross-platform and ultrafast toolkit for FASTA/Q file manipulation (Go, 2016-02-28, 2024-02-05)
34. [ewels/MultiQC](https://github.com/ewels/MultiQC) (1128 stars)
Aggregate results from bioinformatics analyses across many samples into a single report. (JavaScript, 2015-08-04, 2024-02-06)
35. [kemaloksuz/ObjectDetectionImbalance](https://github.com/kemaloksuz/ObjectDetectionImbalance) (1107 stars)
Lists the papers related to imbalance problems in object detection [TPAMI] (None, 2019-08-26, 2024-02-07)
36. [YosefLab/scVI](https://github.com/YosefLab/scVI) (1072 stars)
Deep probabilistic analysis of single-cell omics data (Python, 2017-09-06, 2024-02-06)
37. [weihaox/awesome-gan-inversion](https://github.com/weihaox/awesome-gan-inversion) (1072 stars)
[TPAMI 2022] GAN Inversion: A Survey (TeX, 2021-03-07, 2024-02-04)
38. [fahadshamshad/awesome-transformers-in-medical-imaging](https://github.com/fahadshamshad/awesome-transformers-in-medical-imaging) (1038 stars)
A collection of resources on applications of Transformers in Medical Imaging. (None, 2021-12-03, 2024-02-06)
39. [vgteam/vg](https://github.com/vgteam/vg) (1026 stars)
tools for working with genome variation graphs (C++, 2014-10-02, 2024-02-06)
40. [ncbi/sra-tools](https://github.com/ncbi/sra-tools) (1020 stars)
SRA Tools (C, 2014-08-21, 2024-02-06)
41. [vlawhern/arl-eegmodels](https://github.com/vlawhern/arl-eegmodels) (1011 stars)
This is the Army Research Laboratory (ARL) EEGModels Project: A Collection of Convolutional Neural Network (CNN) models for EEG signal classification, using Keras and Tensorflow (Python, 2018-05-16, 2024-02-03)
42. [VisDrone/VisDrone-Dataset](https://github.com/VisDrone/VisDrone-Dataset) (982 stars)
The dataset for drone based detection and tracking is released, including both image/video, and annotations. (None, 2019-08-28, 2024-02-07)
43. [OpenDriveLab/Birds-eye-view-Perception](https://github.com/OpenDriveLab/Birds-eye-view-Perception) (957 stars)
[IEEE T-PAMI] Awesome BEV perception research and cookbook for all level audience in autonomous diriving (Python, 2022-08-20, 2024-02-05)
44. [agemagician/ProtTrans](https://github.com/agemagician/ProtTrans) (956 stars)
ProtTrans is providing state of the art pretrained language models for proteins. ProtTrans was trained on thousands of GPUs from Summit and hundreds of Google TPUs using Transformers Models. (Jupyter Notebook, 2020-05-11, 2024-02-06)
45. [THUYimingLi/backdoor-learning-resources](https://github.com/THUYimingLi/backdoor-learning-resources) (944 stars)
A list of backdoor learning resources (None, 2020-06-13, 2024-02-06)
46. [tensorflow/gan](https://github.com/tensorflow/gan) (895 stars)
Tooling for GANs in TensorFlow (Jupyter Notebook, 2019-04-29, 2024-01-28)
47. [xieenze/PolarMask](https://github.com/xieenze/PolarMask) (864 stars)
Code for 'PolarMask: Single Shot Instance Segmentation with Polar Representation' (Python, 2019-09-29, 2024-01-04)
48. [kexinhuang12345/DeepPurpose](https://github.com/kexinhuang12345/DeepPurpose) (863 stars)
A Deep Learning Toolkit for DTI, Drug Property, PPI, DDI, Protein Function Prediction (Bioinformatics) (Jupyter Notebook, 2020-03-19, 2024-02-06)
49. [PeikeLi/Self-Correction-Human-Parsing](https://github.com/PeikeLi/Self-Correction-Human-Parsing) (850 stars)
An out-of-box human parsing representation extractor. (Jupyter Notebook, 2019-08-27, 2024-02-05)
50. [pachterlab/gget](https://github.com/pachterlab/gget) (827 stars)
ðŸ§¬ gget enables efficient querying of genomic reference databases (Python, 2022-05-04, 2024-02-02)
51. [easezyc/deep-transfer-learning](https://github.com/easezyc/deep-transfer-learning) (813 stars)
A collection of implementations of deep domain adaptation algorithms (Python, 2018-07-06, 2024-01-29)
52. [jdblischak/workflowr](https://github.com/jdblischak/workflowr) (793 stars)
Organize your project into a research website (R, 2016-12-08, 2024-02-06)
53. [InsightSoftwareConsortium/SimpleITK-Notebooks](https://github.com/InsightSoftwareConsortium/SimpleITK-Notebooks) (783 stars)
Jupyter notebooks for learning how to use SimpleITK (Jupyter Notebook, 2013-01-07, 2024-02-05)
54. [RainCloudPlots/RainCloudPlots](https://github.com/RainCloudPlots/RainCloudPlots) (776 stars)
Code and tutorials to visualise your data that is both beautiful *and* statistically valid (HTML, 2018-08-08, 2024-02-05)
55. [zju3dv/pvnet](https://github.com/zju3dv/pvnet) (775 stars)
Code for "PVNet: Pixel-wise Voting Network for 6DoF Pose Estimation" CVPR 2019 oral (Jupyter Notebook, 2018-12-27, 2024-02-01)
56. [EpistasisLab/pmlb](https://github.com/EpistasisLab/pmlb) (768 stars)
PMLB: A large, curated repository of benchmark datasets for evaluating supervised machine learning algorithms. (Python, 2016-11-11, 2024-02-05)
57. [arose/nglview](https://github.com/arose/nglview) (761 stars)
Jupyter widget to interactively view molecular structures and trajectories (Jupyter Notebook, 2015-11-09, 2024-02-05)
58. [molecularsets/moses](https://github.com/molecularsets/moses) (757 stars)
Molecular Sets (MOSES): A Benchmarking Platform for Molecular Generation Models (Python, 2018-11-28, 2024-02-04)
59. [tengteng95/Pose-Transfer](https://github.com/tengteng95/Pose-Transfer) (727 stars)
Code for the paper Progressive Pose Attention for Person Image Generation in CVPR19 (Oral). (Python, 2019-03-03, 2024-01-20)
60. [HuiZeng/Image-Adaptive-3DLUT](https://github.com/HuiZeng/Image-Adaptive-3DLUT) (710 stars)
Learning Image-adaptive 3D Lookup Tables for High Performance Photo Enhancement in Real-time (Python, 2020-07-08, 2024-02-06)
61. [MrGiovanni/ModelsGenesis](https://github.com/MrGiovanni/ModelsGenesis) (706 stars)
[MICCAI 2019] [MEDIA 2020] Models Genesis (Jupyter Notebook, 2019-07-24, 2024-02-01)
62. [tanghaibao/goatools](https://github.com/tanghaibao/goatools) (692 stars)
Python library to handle Gene Ontology (GO) terms (Python, 2010-01-15, 2024-02-05)
63. [deeptools/pyGenomeTracks](https://github.com/deeptools/pyGenomeTracks) (690 stars)
python module to plot beautiful and highly customizable genome browser tracks (Python, 2017-10-27, 2024-02-03)
64. [wtjiang98/PSGAN](https://github.com/wtjiang98/PSGAN) (683 stars)
PyTorch code for "PSGAN: Pose and Expression Robust Spatial-Aware GAN for Customizable Makeup Transfer" (CVPR 2020 Oral) (Python, 2020-03-19, 2024-02-02)
65. [taoliu/MACS](https://github.com/taoliu/MACS) (671 stars)
MACS -- Model-based Analysis of ChIP-Seq (Cython, 2011-03-02, 2024-02-05)
66. [tanghaibao/jcvi](https://github.com/tanghaibao/jcvi) (668 stars)
Python library to facilitate genome assembly, annotation, and comparative genomics (Python, 2010-12-01, 2024-02-04)
67. [awslabs/dgl-lifesci](https://github.com/awslabs/dgl-lifesci) (661 stars)
Python package for graph neural networks in chemistry and biology (Python, 2020-04-23, 2024-02-06)
68. [beoutbreakprepared/nCoV2019](https://github.com/beoutbreakprepared/nCoV2019) (658 stars)
Location for summaries and analysis of data related to n-CoV 2019, first reported in Wuhan, China (HTML, 2020-01-24, 2024-01-04)
69. [ablab/spades](https://github.com/ablab/spades) (642 stars)
SPAdes Genome Assembler (C++, 2016-06-01, 2024-02-04)
70. [volkamerlab/TeachOpenCADD](https://github.com/volkamerlab/TeachOpenCADD) (633 stars)
TeachOpenCADD: a teaching platform for computer-aided drug design (CADD) using open source packages and data (Jupyter Notebook, 2018-11-09, 2024-02-05)
71. [arose/ngl](https://github.com/arose/ngl) (629 stars)
WebGL protein viewer (TypeScript, 2014-09-01, 2024-02-03)
72. [brentp/mosdepth](https://github.com/brentp/mosdepth) (626 stars)
fast BAM/CRAM depth calculation for WGS, exome, or targeted sequencing (Nim, 2017-08-24, 2024-02-06)
73. [naver/biobert-pretrained](https://github.com/naver/biobert-pretrained) (625 stars)
BioBERT: a pre-trained biomedical language representation model for biomedical text mining (None, 2019-01-28, 2024-01-25)
74. [torognes/vsearch](https://github.com/torognes/vsearch) (613 stars)
Versatile open-source tool for microbiome analysis (C++, 2014-05-16, 2024-02-06)
75. [Ha0Tang/AttentionGAN](https://github.com/Ha0Tang/AttentionGAN) (610 stars)
AttentionGAN for Unpaired Image-to-Image Translation & Multi-Domain Image-to-Image Translation (Python, 2019-07-09, 2024-01-28)
76. [davidemms/OrthoFinder](https://github.com/davidemms/OrthoFinder) (607 stars)
Phylogenetic orthology inference for comparative genomics (Python, 2015-06-29, 2024-02-06)
77. [samtools/BCFtools](https://github.com/samtools/BCFtools) (594 stars)
This is the official development repository for BCFtools. See installation instructions and other documentation here http://samtools.github.io/bcftools/howtos/install.html (C, 2013-07-12, 2024-02-07)
78. [igvteam/igv](https://github.com/igvteam/igv) (593 stars)
Integrative Genomics Viewer. Fast, efficient, scalable visualization tool for genomics data and annotations (Java, 2012-02-11, 2024-02-06)
79. [KlugerLab/FIt-SNE](https://github.com/KlugerLab/FIt-SNE) (575 stars)
Fast Fourier Transform-accelerated Interpolation-based  t-SNE (FIt-SNE) (C++, 2017-12-25, 2024-01-27)
80. [ialbert/biostar-central](https://github.com/ialbert/biostar-central) (569 stars)
Biostar Q&A (Python, 2011-03-22, 2024-02-04)
81. [NeuroTechX/moabb](https://github.com/NeuroTechX/moabb) (567 stars)
Mother of All BCI Benchmarks (Python, 2017-05-15, 2024-02-07)
82. [biojava/biojava](https://github.com/biojava/biojava) (566 stars)
:book::microscope::coffee: BioJava is an open-source project dedicated to providing a Java library for processing biological data. (Java, 2013-04-03, 2024-02-06)
83. [cszn/DPIR](https://github.com/cszn/DPIR) (562 stars)
Plug-and-Play Image Restoration with Deep Denoiser Prior (IEEE TPAMI 2021) (PyTorch) (Python, 2020-08-08, 2024-01-29)
84. [AbnerHqC/GaitSet](https://github.com/AbnerHqC/GaitSet) (554 stars)
A flexible, effective and fast cross-view gait recognition network (Python, 2018-11-16, 2024-01-29)
85. [RenYurui/Global-Flow-Local-Attention](https://github.com/RenYurui/Global-Flow-Local-Attention) (550 stars)
The source code for paper "Deep Image Spatial Transformation for Person Image Generation" (Jupyter Notebook, 2020-02-28, 2024-01-12)
86. [broadinstitute/keras-rcnn](https://github.com/broadinstitute/keras-rcnn) (548 stars)
Keras package for region-based convolutional neural networks (RCNNs) (Python, 2017-04-25, 2024-01-05)
87. [rrwick/Bandage](https://github.com/rrwick/Bandage) (546 stars)
a Bioinformatics Application for Navigating De novo Assembly Graphs Easily (C++, 2015-02-14, 2024-01-28)
88. [molstar/molstar](https://github.com/molstar/molstar) (545 stars)
A comprehensive macromolecular library (TypeScript, 2018-01-17, 2024-02-06)
89. [ThomasYeoLab/CBIG](https://github.com/ThomasYeoLab/CBIG) (532 stars)
None (MATLAB, 2016-07-13, 2024-01-31)
90. [jianzongwu/Awesome-Open-Vocabulary](https://github.com/jianzongwu/Awesome-Open-Vocabulary) (527 stars)
A Survey on Open Vocabulary Learning (None, 2023-06-08, 2024-02-06)
91. [ZhaoZhibin/DL-based-Intelligent-Diagnosis-Benchmark](https://github.com/ZhaoZhibin/DL-based-Intelligent-Diagnosis-Benchmark) (515 stars)
Source codes for the paper "Deep Learning Algorithms for Rotating Machinery Intelligent Diagnosis: An Open Source Benchmark Study" (Python, 2019-12-28, 2024-02-05)
92. [jerryji1993/DNABERT](https://github.com/jerryji1993/DNABERT) (515 stars)
DNABERT: pre-trained Bidirectional Encoder Representations from Transformers model for DNA-language in genome (Python, 2020-06-11, 2024-02-01)
93. [gnina/gnina](https://github.com/gnina/gnina) (513 stars)
A deep learning framework for molecular docking (C++, 2015-11-04, 2024-02-04)
94. [raoyongming/DynamicViT](https://github.com/raoyongming/DynamicViT) (512 stars)
[NeurIPS 2021] [T-PAMI] DynamicViT: Efficient Vision Transformers with Dynamic Token Sparsification (Jupyter Notebook, 2021-06-03, 2024-02-06)
95. [MolecularAI/aizynthfinder](https://github.com/MolecularAI/aizynthfinder) (509 stars)
A tool for retrosynthetic planning (Python, 2020-06-11, 2024-02-06)
96. [pachterlab/ffq](https://github.com/pachterlab/ffq) (509 stars)
A tool to find sequencing data and metadata from public databases. (Python, 2020-03-25, 2024-02-02)
97. [skovaka/UNCALLED](https://github.com/skovaka/UNCALLED) (509 stars)
Raw nanopore signal mapper that enables real-time targeted sequencing (C++, 2017-07-19, 2024-01-04)
98. [rrwick/Unicycler](https://github.com/rrwick/Unicycler) (505 stars)
hybrid assembly pipeline for bacterial genomes (C++, 2016-03-14, 2024-02-06)
99. [mcahny/Deep-Video-Inpainting](https://github.com/mcahny/Deep-Video-Inpainting) (489 stars)
Official pytorch implementation for "Deep Video Inpainting" (CVPR 2019) (Python, 2019-05-22, 2024-02-06)
100. [fritzsedlazeck/Sniffles](https://github.com/fritzsedlazeck/Sniffles) (486 stars)
Structural variation caller using third generation sequencing (Python, 2015-10-25, 2024-02-06)
101. [gamcil/clinker](https://github.com/gamcil/clinker) (486 stars)
Gene cluster comparison figure generator (Python, 2019-06-21, 2024-02-05)
102. [ruanjue/wtdbg2](https://github.com/ruanjue/wtdbg2) (485 stars)
Redbean: A fuzzy Bruijn graph approach to long noisy reads assembly (C, 2017-09-29, 2024-01-23)
103. [biojs/biojs](https://github.com/biojs/biojs) (480 stars)
ðŸ”¬A library of JavaScript components to represent biological data (None, 2013-08-02, 2024-01-27)
104. [theislab/anndata](https://github.com/theislab/anndata) (479 stars)
Annotated data. (Python, 2017-08-11, 2024-02-05)
105. [zqfang/GSEApy](https://github.com/zqfang/GSEApy) (479 stars)
Gene Set Enrichment Analysis in Python (Python, 2016-01-09, 2024-02-06)
106. [callmeray/PointMVSNet](https://github.com/callmeray/PointMVSNet) (478 stars)
Code for  "Point-based Multi-view Stereo Network" (ICCV 2019 Oral) & "Visibility-aware Point-based Multi-view Stereo Network" (TPAMI) (Python, 2019-08-04, 2024-02-06)
107. [owkin/PyDESeq2](https://github.com/owkin/PyDESeq2) (477 stars)
A Python implementation of the DESeq2 pipeline for bulk RNA-seq DEA. (Python, 2022-11-22, 2024-02-05)
108. [Martinsos/edlib](https://github.com/Martinsos/edlib) (476 stars)
Lightweight, super fast C/C++ (& Python) library for sequence alignment using edit (Levenshtein) distance. (C++, 2014-01-19, 2024-02-07)
109. [soedinglab/hh-suite](https://github.com/soedinglab/hh-suite) (475 stars)
Remote protein homology detection suite. (C, 2015-05-04, 2024-02-07)
110. [CERN/TIGRE](https://github.com/CERN/TIGRE) (474 stars)
TIGRE: Tomographic Iterative GPU-based Reconstruction Toolbox (MATLAB, 2016-06-13, 2024-02-05)
111. [ncoudray/DeepPATH](https://github.com/ncoudray/DeepPATH) (473 stars)
Classification of Lung cancer slide images using deep-learning (Python, 2017-05-23, 2024-02-01)
112. [cfzd/Ultra-Fast-Lane-Detection-v2](https://github.com/cfzd/Ultra-Fast-Lane-Detection-v2) (470 stars)
Ultra Fast Deep Lane Detection With Hybrid Anchor Driven Ordinal Classification (TPAMI 2022) (Python, 2022-03-14, 2024-01-29)
113. [XingangPan/deep-generative-prior](https://github.com/XingangPan/deep-generative-prior) (469 stars)
Code for deep generative prior (ECCV2020 oral) (Python, 2020-03-30, 2024-02-06)
114. [dddzg/up-detr](https://github.com/dddzg/up-detr) (468 stars)
[TPAMI 2022 & CVPR2021 Oral] UP-DETR: Unsupervised Pre-training for Object Detection with Transformers (Python, 2021-03-03, 2024-01-24)
115. [Ildaron/ironbci](https://github.com/Ildaron/ironbci) (467 stars)
Open-Source Brain-Computer Interface, ADS1299 and STM32 (C, 2021-07-25, 2024-01-28)
116. [BMIRDS/deepslide](https://github.com/BMIRDS/deepslide) (465 stars)
Code for the Nature Scientific Reports paper "Pathologist-level classification of histologic patterns on resected lung adenocarcinoma slides with deep neural networks." A sliding window framework for classification of high resolution whole-slide images, often microscopy or histopathology images. (Python, 2018-12-26, 2024-01-31)
117. [ryanlayer/samplot](https://github.com/ryanlayer/samplot) (465 stars)
Plot structural variant signals from many BAMs and CRAMs (Python, 2017-08-14, 2024-01-12)
118. [immunogenomics/harmony](https://github.com/immunogenomics/harmony) (464 stars)
Fast, sensitive and accurate integration of single-cell data with Harmony (R, 2018-06-12, 2024-02-01)
119. [sunwj/CAR](https://github.com/sunwj/CAR) (464 stars)
Content adaptive resampler for image downscaling (C++, 2019-07-05, 2024-01-29)
120. [menyifang/ADGAN](https://github.com/menyifang/ADGAN) (462 stars)
The Implementation of paper "Controllable Person Image Synthesis with Attribute-Decomposed GAN" CVPR 2020 (Oral); Pose and Appearance Attributes Transfer; (Python, 2020-03-22, 2024-02-03)
121. [etal/cnvkit](https://github.com/etal/cnvkit) (458 stars)
Copy number variant detection from targeted DNA sequencing (Python, 2014-07-07, 2024-02-06)
122. [JudasDie/SOTS](https://github.com/JudasDie/SOTS) (456 stars)
Single object tracking and segmentation. (Python, 2020-09-25, 2024-01-29)
123. [Ha0Tang/SelectionGAN](https://github.com/Ha0Tang/SelectionGAN) (455 stars)
[CVPR 2019 Oral] Multi-Channel Attention Selection GAN with Cascaded Semantic Guidance for Cross-View Image Translation (Python, 2019-04-13, 2024-01-27)
124. [lindenb/jvarkit](https://github.com/lindenb/jvarkit) (454 stars)
Java utilities for Bioinformatics (Java, 2013-05-06, 2024-01-30)
125. [seqan/seqan](https://github.com/seqan/seqan) (446 stars)
SeqAn's official repository. (C++, 2012-10-25, 2024-02-06)
126. [Maclory/SPSR](https://github.com/Maclory/SPSR) (439 stars)
Pytorch implementation of Structure-Preserving Super Resolution with Gradient Guidance (CVPR 2020 & TPAMI 2021) (Python, 2020-03-16, 2024-01-23)
127. [OpenMS/OpenMS](https://github.com/OpenMS/OpenMS) (439 stars)
The codebase of the OpenMS project (C++, 2014-01-09, 2024-01-31)
128. [aimagelab/mammoth](https://github.com/aimagelab/mammoth) (437 stars)
An Extendible (General) Continual Learning Framework based on Pytorch - official codebase of Dark Experience for General Continual Learning (Python, 2020-10-05, 2024-02-07)
129. [sallymmx/ActionCLIP](https://github.com/sallymmx/ActionCLIP) (437 stars)
This is the official implement of paper "ActionCLIP: A New Paradigm for Action Recognition" (Python, 2021-08-27, 2024-02-01)
130. [sinAshish/Multi-Scale-Attention](https://github.com/sinAshish/Multi-Scale-Attention) (437 stars)
Code for our paper "Multi-scale Guided Attention for Medical Image Segmentation" (Python, 2019-06-06, 2024-01-29)
131. [benjjneb/dada2](https://github.com/benjjneb/dada2) (434 stars)
Accurate sample inference from amplicon data with single nucleotide resolution (R, 2014-12-17, 2024-02-06)
132. [facebookresearch/LaMCTS](https://github.com/facebookresearch/LaMCTS) (434 stars)
The release codes of LA-MCTS with its application to Neural Architecture Search. (Python, 2020-07-30, 2024-02-02)
133. [HazyResearch/hyena-dna](https://github.com/HazyResearch/hyena-dna) (432 stars)
Official implementation for HyenaDNA, a long-range genomic foundation model built with Hyena (Assembly, 2023-06-23, 2024-02-06)
134. [weiyithu/NerfingMVS](https://github.com/weiyithu/NerfingMVS) (430 stars)
[ICCV 2021 Oral] NerfingMVS: Guided Optimization of Neural Radiance Fields for Indoor Multi-view Stereo (Python, 2021-08-11, 2024-01-31)
135. [AlanLuSun/High-quality-ellipse-detection](https://github.com/AlanLuSun/High-quality-ellipse-detection) (426 stars)
A high-quality ellipse detector based on arc-support line segments which can both accurately and efficiently detect ellipses in images. (C++, 2018-09-27, 2024-02-02)
136. [aiqm/torchani](https://github.com/aiqm/torchani) (426 stars)
Accurate Neural Network Potential on PyTorch (Python, 2018-04-02, 2024-02-05)
137. [gangweiX/ACVNet](https://github.com/gangweiX/ACVNet) (423 stars)
[CVPR 2022] ACVNet: Attention Concatenation Volume for Accurate and Efficient Stereo Matching (Python, 2021-08-05, 2024-02-06)
138. [saeyslab/nichenetr](https://github.com/saeyslab/nichenetr) (414 stars)
NicheNet: predict active ligand-target links between interacting cells (R, 2018-02-05, 2024-01-29)
139. [puzzlepaint/surfelmeshing](https://github.com/puzzlepaint/surfelmeshing) (412 stars)
Real-time surfel-based mesh reconstruction from RGB-D video. (C++, 2018-09-19, 2024-02-02)
140. [pezmaster31/bamtools](https://github.com/pezmaster31/bamtools) (404 stars)
C++ API & command-line toolkit for working with BAM data (C++, 2010-05-21, 2024-01-29)
141. [ecogenomics/gtdbtk](https://github.com/ecogenomics/gtdbtk) (398 stars)
GTDB-Tk: a toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes. (Python, 2016-11-29, 2024-01-31)
142. [shijieS/SST](https://github.com/shijieS/SST) (395 stars)
Single Shot Tracker (Python, 2018-04-13, 2024-01-04)
143. [aristoteleo/dynamo-release](https://github.com/aristoteleo/dynamo-release) (384 stars)
Inclusive model of expression dynamics with conventional or metabolic labeling based scRNA-seq / multiomics, vector field reconstruction and differential geometry analyses (Python, 2019-07-05, 2024-02-04)
144. [oddt/oddt](https://github.com/oddt/oddt) (380 stars)
Open Drug Discovery Toolkit (Python, 2014-08-20, 2024-01-31)
145. [Illumina/manta](https://github.com/Illumina/manta) (379 stars)
Structural variant and indel caller for mapped sequencing data (C++, 2013-05-30, 2024-02-06)
146. [wenguanwang/SODsurvey](https://github.com/wenguanwang/SODsurvey) (379 stars)
Salient Object Detection in the Deep Learning Era: An In-Depth Survey (None, 2019-04-08, 2024-01-11)
147. [danini/graph-cut-ransac](https://github.com/danini/graph-cut-ransac) (378 stars)
The Graph-Cut RANSAC algorithm proposed in paper: Daniel Barath and Jiri Matas; Graph-Cut RANSAC, Conference on Computer Vision and Pattern Recognition, 2018. It is available at http://openaccess.thecvf.com/content_cvpr_2018/papers/Barath_Graph-Cut_RANSAC_CVPR_2018_paper.pdf (C++, 2017-09-07, 2024-02-02)
148. [agshumate/Liftoff](https://github.com/agshumate/Liftoff) (375 stars)
An accurate GFF3/GTF lift over pipeline (Python, 2019-11-05, 2024-02-02)
149. [danini/magsac](https://github.com/danini/magsac) (372 stars)
The MAGSAC algorithm for robust model fitting without using an inlier-outlier threshold (C++, 2019-04-01, 2024-01-29)
150. [prody/ProDy](https://github.com/prody/ProDy) (372 stars)
A Python Package for Protein Dynamics Analysis (Python, 2010-10-31, 2024-02-01)
151. [cabani/MaskedFace-Net](https://github.com/cabani/MaskedFace-Net) (371 stars)
MaskedFace-Net is a dataset of human faces with a correctly and incorrectly worn mask based on the dataset Flickr-Faces-HQ (FFHQ). (None, 2020-08-13, 2024-01-04)
152. [oschwengers/bakta](https://github.com/oschwengers/bakta) (371 stars)
Rapid & standardized annotation of bacterial genomes, MAGs & plasmids (Python, 2020-01-15, 2024-02-06)
153. [raoyongming/GFNet](https://github.com/raoyongming/GFNet) (371 stars)
[NeurIPS 2021] [T-PAMI] Global Filter Networks for Image Classification (Jupyter Notebook, 2021-07-01, 2024-01-21)
154. [dluvizon/deephar](https://github.com/dluvizon/deephar) (367 stars)
Deep human action recognition and pose estimation (Python, 2018-06-13, 2024-01-19)
155. [marbl/mash](https://github.com/marbl/mash) (366 stars)
Fast genome and metagenome distance estimation using MinHash (C++, 2015-08-24, 2024-02-03)
156. [VDIGPKU/CBNetV2](https://github.com/VDIGPKU/CBNetV2) (365 stars)
None (Python, 2021-06-19, 2024-02-05)
157. [digitalmethodsinitiative/dmi-tcat](https://github.com/digitalmethodsinitiative/dmi-tcat) (365 stars)
Digital Methods Initiative - Twitter Capture and Analysis Toolset (PHP, 2013-04-10, 2024-02-04)
158. [Juyong/3DFace](https://github.com/Juyong/3DFace) (363 stars)
This repository contains the dataset including the pair of 2D face image and its corresponding 3D face geometry model. (None, 2017-08-01, 2024-01-17)
159. [ZhangGongjie/Meta-DETR](https://github.com/ZhangGongjie/Meta-DETR) (357 stars)
[T-PAMI  2022]  Meta-DETR for Few-Shot Object Detection: Official PyTorch Implementation (Python, 2021-03-22, 2024-01-30)
160. [nservant/HiC-Pro](https://github.com/nservant/HiC-Pro) (355 stars)
HiC-Pro: An optimized and flexible pipeline for Hi-C data processing (Python, 2015-01-30, 2024-02-06)
161. [bxlab/metaWRAP](https://github.com/bxlab/metaWRAP) (351 stars)
MetaWRAP - a flexible pipeline for genome-resolved metagenomic data analysis (Shell, 2017-06-06, 2024-01-23)
162. [wmayner/pyphi](https://github.com/wmayner/pyphi) (348 stars)
A toolbox for integrated information theory. (Python, 2014-04-08, 2024-01-16)
163. [lishen/end2end-all-conv](https://github.com/lishen/end2end-all-conv) (347 stars)
Deep Learning to Improve Breast Cancer Detection on Screening Mammography (Jupyter Notebook, 2017-08-18, 2024-02-04)
164. [amkozlov/raxml-ng](https://github.com/amkozlov/raxml-ng) (346 stars)
RAxML Next Generation: faster, easier-to-use and more flexible (C++, 2016-12-08, 2024-02-06)
165. [brentp/cyvcf2](https://github.com/brentp/cyvcf2) (345 stars)
cython + htslib == fast VCF and BCF processing (Cython, 2015-08-12, 2024-01-30)
166. [swz30/MIRNetv2](https://github.com/swz30/MIRNetv2) (345 stars)
[TPAMI 2022] Learning Enriched Features for Fast Image Restoration and Enhancement. Results on Defocus Deblurring, Denoising, Super-resolution, and image enhancement (Python, 2021-08-04, 2024-02-03)
167. [mlpen/Nystromformer](https://github.com/mlpen/Nystromformer) (344 stars)
None (Python, 2020-12-11, 2024-01-23)
168. [lxtGH/SFSegNets](https://github.com/lxtGH/SFSegNets) (342 stars)
Implementation of Our ECCV-2020-oral paper and its extension: Semantic Flow for Fast and Accurate Scene Parsing (Python, 2020-08-07, 2024-01-11)
169. [brentp/vcfanno](https://github.com/brentp/vcfanno) (341 stars)
annotate a VCF with other VCFs/BEDs/tabixed files (Go, 2015-04-29, 2024-01-29)
170. [HikariTJU/LD](https://github.com/HikariTJU/LD) (339 stars)
Localization Distillation for Object Detection (CVPR 2022, TPAMI 2023) (Python, 2021-01-20, 2024-02-07)
171. [fbreitwieser/pavian](https://github.com/fbreitwieser/pavian) (339 stars)
ðŸŒˆ Interactive analysis of metagenomics data (R, 2016-10-19, 2024-02-01)
172. [lxtGH/GALD-DGCNet](https://github.com/lxtGH/GALD-DGCNet) (339 stars)
Source  code and model GALD net (BMVC-2019) and Dual-Seg Net (BMVC-2019) (Python, 2019-02-27, 2024-01-30)
173. [Illumina/strelka](https://github.com/Illumina/strelka) (338 stars)
Strelka2 germline and somatic small variant caller (C++, 2016-10-17, 2024-02-06)
174. [pastas/pastas](https://github.com/pastas/pastas) (336 stars)
:spaghetti: Pastas is an open-source Python framework for the analysis of groundwater time series. (Python, 2016-04-15, 2024-01-21)
175. [haddocking/pdb-tools](https://github.com/haddocking/pdb-tools) (334 stars)
A dependency-free cross-platform swiss army knife for PDB files. (Python, 2014-11-27, 2024-02-06)
176. [researchmm/AOT-GAN-for-Inpainting](https://github.com/researchmm/AOT-GAN-for-Inpainting) (332 stars)
[TVCG'2023] AOT-GAN for High-Resolution Image Inpainting (codebase for image inpainting) (Python, 2021-03-28, 2024-02-07)
177. [JialeCao001/SipMask](https://github.com/JialeCao001/SipMask) (331 stars)
SipMask: Spatial Information Preservation for Fast Image and Video Instance Segmentation (ECCV2020) (Jupyter Notebook, 2020-07-27, 2024-01-08)
178. [gpertea/stringtie](https://github.com/gpertea/stringtie) (330 stars)
Transcript assembly and quantification for RNA-Seq (C++, 2014-10-13, 2024-02-06)
179. [MolecularAI/Reinvent](https://github.com/MolecularAI/Reinvent) (329 stars)
None (Python, 2020-03-17, 2024-01-28)
180. [dingkeyan93/DISTS](https://github.com/dingkeyan93/DISTS) (329 stars)
IQA: Deep Image Structure and Texture Similarity Metric (Python, 2020-04-15, 2024-01-26)
181. [clovaai/wsolevaluation](https://github.com/clovaai/wsolevaluation) (326 stars)
Evaluating Weakly Supervised Object Localization Methods Right (CVPR 2020) (Python, 2020-01-21, 2024-02-05)
182. [gpertea/gffread](https://github.com/gpertea/gffread) (323 stars)
GFF/GTF utility providing format conversions, region filtering, FASTA sequence extraction and more (C++, 2015-12-17, 2024-01-30)
183. [carrierlxk/COSNet](https://github.com/carrierlxk/COSNet) (319 stars)
See More, Know More: Unsupervised Video Object Segmentation with Co-Attention Siamese Networks (CVPR19) (Python, 2019-03-28, 2024-01-04)
184. [ChenyangLEI/deep-video-prior](https://github.com/ChenyangLEI/deep-video-prior) (316 stars)
[NeurIPS 2020] Blind Video Temporal Consistency via Deep Video Prior (Python, 2020-09-26, 2024-01-15)
185. [dstahlke/gnuplot-iostream](https://github.com/dstahlke/gnuplot-iostream) (316 stars)
C++ interface to gnuplot (C++, 2015-04-01, 2024-01-24)
186. [beringresearch/ivis](https://github.com/beringresearch/ivis) (314 stars)
Dimensionality reduction in very large datasets using Siamese Networks (Python, 2018-08-13, 2024-01-26)
187. [gangweiX/Fast-ACVNet](https://github.com/gangweiX/Fast-ACVNet) (313 stars)
[TPAMI 2023] Fast-ACV: Fast Attention Concatenation Volume for Accurate and Real-time Stereo Matching (Python, 2022-05-03, 2024-02-07)
188. [jtamames/SqueezeMeta](https://github.com/jtamames/SqueezeMeta) (313 stars)
A complete pipeline for metagenomic analysis (C, 2018-03-15, 2024-02-01)
189. [taozh2017/RGBD-SODsurvey](https://github.com/taozh2017/RGBD-SODsurvey) (307 stars)
RGB-D Salient Object Detection: A Survey (MATLAB, 2020-06-15, 2024-02-05)
190. [cheerss/CrossFormer](https://github.com/cheerss/CrossFormer) (304 stars)
The official code for the paper: https://openreview.net/forum?id=_PHymLIxuI (Python, 2021-07-27, 2024-02-01)
191. [stamatak/standard-RAxML](https://github.com/stamatak/standard-RAxML) (303 stars)
None (C, 2011-06-28, 2023-12-14)
192. [mcahny/vps](https://github.com/mcahny/vps) (301 stars)
Official pytorch implementation for "Video Panoptic Segmentation" (CVPR 2020 Oral) (Python, 2020-04-21, 2024-01-04)
193. [ChenJoya/sampling-free](https://github.com/ChenJoya/sampling-free) (295 stars)
IEEE TIP: Is heuristic sampling necessary in training deep object detectors? Try sampling-free object detectors! (Python, 2019-09-11, 2024-01-17)
194. [arq5x/lumpy-sv](https://github.com/arq5x/lumpy-sv) (295 stars)
lumpy: a general probabilistic framework for structural variant discovery (C, 2012-10-08, 2024-01-17)
195. [oushujun/EDTA](https://github.com/oushujun/EDTA) (294 stars)
Extensive de-novo TE Annotator (Perl, 2019-05-23, 2024-01-23)
196. [jonathanking/sidechainnet](https://github.com/jonathanking/sidechainnet) (291 stars)
An all-atom protein structure dataset for machine learning. (Python, 2020-03-12, 2024-02-04)
197. [whatshap/whatshap](https://github.com/whatshap/whatshap) (291 stars)
Read-based phasing of genomic variants, also called haplotype assembly (C++, 2020-07-02, 2024-01-26)
198. [IDKiro/DehazeFormer](https://github.com/IDKiro/DehazeFormer) (289 stars)
[IEEE TIP] Vision Transformers for Single Image Dehazing (Python, 2022-03-28, 2024-02-06)
199. [BBillot/SynthSeg](https://github.com/BBillot/SynthSeg) (288 stars)
Contrast-agnostic segmentation of MRI scans (Python, 2020-04-21, 2024-02-06)
200. [bcgsc/abyss](https://github.com/bcgsc/abyss) (288 stars)
:microscope: Assemble large genomes using short reads (C++, 2012-03-19, 2024-01-22)
201. [saketkc/pysradb](https://github.com/saketkc/pysradb) (287 stars)
Package for fetching metadata and downloading data from SRA/ENA/GEO (Python, 2018-11-29, 2023-12-20)
202. [lh3/miniasm](https://github.com/lh3/miniasm) (285 stars)
Ultrafast de novo assembly for long noisy reads (though having no consensus step) (TeX, 2015-11-05, 2024-02-02)
203. [TissueImageAnalytics/tiatoolbox](https://github.com/TissueImageAnalytics/tiatoolbox) (283 stars)
Computational Pathology Toolbox developed by TIA Centre, University of Warwick. (Python, 2020-05-28, 2024-02-07)
204. [imagej/imglib](https://github.com/imagej/imglib) (283 stars)
A generic next-generation Java library for image processing (Java, 2011-12-16, 2024-02-02)
205. [zhunzhong07/CamStyle](https://github.com/zhunzhong07/CamStyle) (283 stars)
Camera Style Adaptation for Person Re-identification CVPR 2018 (Python, 2018-03-17, 2024-01-04)
206. [mengyao/Complete-Striped-Smith-Waterman-Library](https://github.com/mengyao/Complete-Striped-Smith-Waterman-Library) (281 stars)
None (C, 2011-10-19, 2024-01-25)
207. [philres/ngmlr](https://github.com/philres/ngmlr) (281 stars)
NGMLR is a long-read mapper designed to align PacBio or Oxford Nanopore (standard and ultra-long) to a reference genome with a focus on reads that span structural variations (C++, 2016-09-01, 2024-01-21)
208. [williamgilpin/pypdb](https://github.com/williamgilpin/pypdb) (281 stars)
A Python API for the RCSB Protein Data Bank (PDB) (Python, 2015-05-13, 2024-01-29)
209. [MarioniLab/miloR](https://github.com/MarioniLab/miloR) (278 stars)
R package implementation of Milo for testing for differential abundance in KNN graphs (HTML, 2020-07-30, 2024-01-27)
210. [yikaiw/CEN](https://github.com/yikaiw/CEN) (270 stars)
[TPAMI 2022, NeurIPS 2020] Code release for "Deep Multimodal Fusion by Channel Exchanging" (Python, 2020-09-29, 2024-02-01)
211. [PhanstielLab/plotgardener](https://github.com/PhanstielLab/plotgardener) (266 stars)
None (R, 2019-07-23, 2024-02-02)
212. [uncbiag/registration](https://github.com/uncbiag/registration) (265 stars)
Image Registration (TeX, 2019-04-04, 2024-02-05)
213. [ncbi/pgap](https://github.com/ncbi/pgap) (264 stars)
NCBI Prokaryotic Genome Annotation Pipeline (Common Workflow Language, 2018-09-05, 2024-01-25)
214. [usnistgov/jarvis](https://github.com/usnistgov/jarvis) (263 stars)
JARVIS-Tools: an open-source software package for data-driven atomistic materials design. Publications: https://scholar.google.com/citations?user=3w6ej94AAAAJ (Python, 2017-06-22, 2024-01-16)
215. [hafen/trelliscopejs](https://github.com/hafen/trelliscopejs) (262 stars)
TrelliscopeJS R Package (R, 2016-10-04, 2024-01-20)
216. [twhui/LiteFlowNet2](https://github.com/twhui/LiteFlowNet2) (256 stars)
A Lightweight Optical Flow CNN - Revisiting Data Fidelity and Regularization, TPAMI 2020 (Python, 2019-02-05, 2024-01-25)
217. [kqwang/phase-recovery](https://github.com/kqwang/phase-recovery) (255 stars)
Resources for phase recovery (also called phase imaging, phase retrieval, or phase reconstruction) (None, 2023-05-31, 2024-02-07)
218. [layumi/person-reid-3d](https://github.com/layumi/person-reid-3d) (255 stars)
TNNLS'22 :statue_of_liberty: Parameter-Efficient Person Re-identification in the 3D Space :statue_of_liberty: (Python, 2020-06-06, 2024-02-06)
219. [eeyhsong/EEG-Conformer](https://github.com/eeyhsong/EEG-Conformer) (250 stars)
EEG Transformer 2.0. i. Convolutional Transformer for EEG Decoding. ii. Novel visualization - Class Activation Topography. (Python, 2022-07-26, 2024-02-03)
220. [marbl/MashMap](https://github.com/marbl/MashMap) (248 stars)
A fast approximate aligner for long DNA sequences (C++, 2017-01-13, 2024-01-31)
221. [marianattestad/ribbon](https://github.com/marianattestad/ribbon) (248 stars)
A genome browser that shows long reads and complex variants better (JavaScript, 2016-06-30, 2024-02-05)
222. [spglib/spglib](https://github.com/spglib/spglib) (248 stars)
C library for finding and handling crystal symmetries (C, 2013-06-17, 2024-02-01)
223. [zjhthu/OANet](https://github.com/zjhthu/OANet) (246 stars)
Implementation of ICCV19 Paper "Learning Two-View Correspondences and Geometry Using Order-Aware Network" (Python, 2019-08-13, 2024-01-24)
224. [schneebergerlab/plotsr](https://github.com/schneebergerlab/plotsr) (245 stars)
Tool to plot synteny and structural rearrangements between genomes (Python, 2021-12-22, 2024-02-02)
225. [tSchlegl/f-AnoGAN](https://github.com/tSchlegl/f-AnoGAN) (242 stars)
Code for reproducing f-AnoGAN training and anomaly scoring (Python, 2019-02-06, 2024-01-28)
226. [brentp/somalier](https://github.com/brentp/somalier) (241 stars)
fast sample-swap and relatedness checks on BAMs/CRAMs/VCFs/GVCFs... "like damn that is one smart wine guy" (Nim, 2018-08-07, 2024-02-06)
227. [dmcable/RCTD](https://github.com/dmcable/RCTD) (241 stars)
Spatial-eXpression-R: Cell type identification (including cell type mixtures) and cell type-specific differential expression for spatial transcriptomics (HTML, 2020-02-06, 2024-02-05)
228. [dmcable/spacexr](https://github.com/dmcable/spacexr) (241 stars)
Spatial-eXpression-R: Cell type identification (including cell type mixtures) and cell type-specific differential expression for spatial transcriptomics (HTML, 2020-02-06, 2024-02-05)
229. [yangji9181/HNE](https://github.com/yangji9181/HNE) (241 stars)
Heterogeneous Network Embedding: Survey, Benchmark, Evaluation, and Beyond (Python, 2020-02-16, 2024-01-11)
230. [mahmoodlab/PathomicFusion](https://github.com/mahmoodlab/PathomicFusion) (240 stars)
Fusing Histology and Genomics via Deep Learning - IEEE TMI (Jupyter Notebook, 2019-12-14, 2024-02-07)
231. [theislab/scgen](https://github.com/theislab/scgen) (237 stars)
Single cell perturbation prediction (Python, 2018-11-28, 2024-01-30)
232. [sigven/pcgr](https://github.com/sigven/pcgr) (236 stars)
Personal Cancer Genome Reporter (PCGR) (R, 2017-03-20, 2023-12-06)
233. [maickrau/GraphAligner](https://github.com/maickrau/GraphAligner) (235 stars)
None (C++, 2017-05-03, 2024-01-21)
234. [opencobra/cobratoolbox](https://github.com/opencobra/cobratoolbox) (233 stars)
The COnstraint-Based Reconstruction and Analysis Toolbox.              Documentation: (MATLAB, 2012-11-02, 2024-02-02)
235. [JiayuYANG/CVP-MVSNet](https://github.com/JiayuYANG/CVP-MVSNet) (232 stars)
Cost Volume Pyramid Based Depth Inference for Multi-View Stereo (CVPR 2020 Oral) (Python, 2020-01-28, 2024-02-02)
236. [jtextor/dagitty](https://github.com/jtextor/dagitty) (231 stars)
Graphical analysis of structural causal models / graphical causal models. (JavaScript, 2015-11-25, 2024-02-06)
237. [Kinggerm/GetOrganelle](https://github.com/Kinggerm/GetOrganelle) (230 stars)
Organelle Genome Assembly Toolkit (Chloroplast/Mitocondrial/ITS) (Python, 2016-04-20, 2024-02-02)
238. [brentp/slivar](https://github.com/brentp/slivar) (230 stars)
genetic variant expressions, annotation, and filtering for great good. (Nim, 2018-12-17, 2024-02-05)
239. [DirtyHarryLYL/Transferable-Interactiveness-Network](https://github.com/DirtyHarryLYL/Transferable-Interactiveness-Network) (228 stars)
Code for Transferable Interactiveness Knowledge for Human-Object Interaction Detection. (CVPR'19, TPAMI'21) (Python, 2019-03-23, 2024-01-04)
240. [TaoHuang2018/Neighbor2Neighbor](https://github.com/TaoHuang2018/Neighbor2Neighbor) (228 stars)
Neighbor2Neighbor: Self-Supervised Denoising from Single Noisy Images (Python, 2021-03-09, 2024-02-05)
241. [hongfz16/DS-Net](https://github.com/hongfz16/DS-Net) (228 stars)
[CVPR 2021/TPAMI 2023] Rank 1st in the public leaderboard of SemanticKITTI Panoptic Segmentation (2020-11-16) (Python, 2020-11-24, 2024-01-25)
242. [changlin31/DNA](https://github.com/changlin31/DNA) (226 stars)
(CVPR 2020) Block-wisely Supervised Neural Architecture Search with Knowledge Distillation (Python, 2019-11-29, 2024-01-19)
243. [chunbolang/BAM](https://github.com/chunbolang/BAM) (226 stars)
Official PyTorch Implementation of Learning What Not to Segment: A New Perspective on Few-Shot Segmentation (CVPR'22 Oral & TPAMI'23). (Python, 2022-03-03, 2024-01-31)
244. [HUANGLIZI/LViT](https://github.com/HUANGLIZI/LViT) (225 stars)
[IEEE Transactions on Medical Imaging/TMI] This repo is the official implementation of "LViT: Language meets Vision Transformer in Medical Image Segmentation" (Python, 2022-03-10, 2024-01-31)
245. [biocommons/hgvs](https://github.com/biocommons/hgvs) (225 stars)
Python library to parse, format, validate, normalize, and map sequence variants. `pip install hgvs` (Python, 2017-03-09, 2024-01-20)
246. [dasmoth/dalliance](https://github.com/dasmoth/dalliance) (225 stars)
Interactive web-based genome browser. (JavaScript, 2010-07-01, 2024-02-04)
247. [jeffdaily/parasail](https://github.com/jeffdaily/parasail) (223 stars)
Pairwise Sequence Alignment Library (C, 2015-05-21, 2024-02-03)
248. [lmdu/pyfastx](https://github.com/lmdu/pyfastx) (223 stars)
a python package for fast random access to sequences from plain and gzipped FASTA/Q files (C, 2019-03-19, 2024-01-31)
249. [Annbless/DUTCode](https://github.com/Annbless/DUTCode) (222 stars)
Pytorch implementation of DUT: Learning Video Stabilization by Simply Watching Unstable Videos (Python, 2020-11-23, 2024-02-04)
250. [ssadedin/bpipe](https://github.com/ssadedin/bpipe) (222 stars)
Bpipe - a tool for running and managing bioinformatics pipelines (Groovy, 2014-10-20, 2023-11-29)
251. [adamewing/bamsurgeon](https://github.com/adamewing/bamsurgeon) (221 stars)
tools for adding mutations to existing .bam files, used for testing mutation callers (Python, 2012-05-10, 2024-02-02)
252. [lukasturcani/stk](https://github.com/lukasturcani/stk) (220 stars)
A Python library which allows construction and manipulation of complex molecules, as well as automatic molecular design and the creation of molecular databases. (Python, 2018-03-18, 2024-01-31)
253. [RasmussenLab/vamb](https://github.com/RasmussenLab/vamb) (219 stars)
Variational autoencoder for metagenomic binning (Python, 2018-05-08, 2024-01-30)
254. [alekseyzimin/masurca](https://github.com/alekseyzimin/masurca) (218 stars)
None (M4, 2018-01-13, 2024-01-28)
255. [changlin31/DS-Net](https://github.com/changlin31/DS-Net) (218 stars)
(CVPR 2021, Oral) Dynamic Slimmable Network (Python, 2021-03-23, 2024-02-01)
256. [gtonkinhill/panaroo](https://github.com/gtonkinhill/panaroo) (216 stars)
An updated pipeline for pangenome investigation (Python, 2018-12-18, 2024-02-02)
257. [smirarab/ASTRAL](https://github.com/smirarab/ASTRAL) (216 stars)
Accurate Species TRee ALgorithm (Java, 2014-06-04, 2024-02-07)
258. [xiangyue9607/BioNEV](https://github.com/xiangyue9607/BioNEV) (215 stars)
Graph Embedding Evaluation / Code and Datasets for  "Graph Embedding on Biomedical Networks: Methods, Applications, and Evaluations"  (Bioinformatics 2020) (Python, 2019-04-01, 2024-01-04)
259. [yangxuntu/SGAE](https://github.com/yangxuntu/SGAE) (215 stars)
None (OpenEdge ABL, 2019-03-29, 2024-01-04)
260. [ryanlayer/giggle](https://github.com/ryanlayer/giggle) (214 stars)
Interval data structure (C, 2015-12-04, 2024-01-04)
261. [shiyinzhang/Inside-Outside-Guidance](https://github.com/shiyinzhang/Inside-Outside-Guidance) (214 stars)
Interactive Object Segmentation with Inside-Outside Guidance (Python, 2020-03-16, 2024-01-04)
262. [tjiangHIT/cuteSV](https://github.com/tjiangHIT/cuteSV) (214 stars)
Long read based human genomic structural variation detection with cuteSV (Python, 2018-09-11, 2024-02-05)
263. [qiicr/dcmqi](https://github.com/qiicr/dcmqi) (212 stars)
dcmqi (DICOM for Quantitative Imaging) is a free, open source C++ library for conversion between imaging research formats and the standard DICOM representation for image analysis results (C++, 2016-01-29, 2024-01-28)
264. [schatzlab/genomescope](https://github.com/schatzlab/genomescope) (212 stars)
Fast genome analysis from unassembled short reads (JavaScript, 2016-02-23, 2024-01-10)
265. [marbl/winnowmap](https://github.com/marbl/winnowmap) (211 stars)
Long read / genome alignment software (C, 2020-02-11, 2024-01-12)
266. [EpistasisLab/Aliro](https://github.com/EpistasisLab/Aliro) (210 stars)
Aliro: AI-Driven Data Science (JavaScript, 2017-05-09, 2024-02-06)
267. [zju3dv/disprcnn](https://github.com/zju3dv/disprcnn) (210 stars)
Code release for Stereo 3D Object Detection via Shape Prior Guided Instance Disparity Estimation (CVPR 2020, TPAMI 2021) (Jupyter Notebook, 2020-03-16, 2024-01-04)
268. [wenguanwang/AGS](https://github.com/wenguanwang/AGS) (209 stars)
Learning Unsupervised Video Object Segmentation through Visual Attention (CVPR19, PAMI20) (Python, 2019-03-29, 2024-01-04)
269. [choderalab/pymbar](https://github.com/choderalab/pymbar) (208 stars)
Python implementation of the multistate Bennett acceptance ratio (MBAR) (Python, 2013-05-10, 2024-02-02)
270. [debbiemarkslab/evcouplings](https://github.com/debbiemarkslab/evcouplings) (208 stars)
Evolutionary couplings from protein and RNA sequence alignments (Jupyter Notebook, 2016-07-20, 2024-01-31)
271. [GregoryFaust/samblaster](https://github.com/GregoryFaust/samblaster) (206 stars)
samblaster: a tool to mark duplicates and extract discordant and split reads from sam files. (C++, 2014-02-11, 2024-02-06)
272. [PacificBiosciences/FALCON](https://github.com/PacificBiosciences/FALCON) (205 stars)
FALCON: experimental PacBio diploid assembler -- Out-of-date -- Please use a binary release: https://github.com/PacificBiosciences/FALCON_unzip/wiki/Binaries (Python, 2014-01-07, 2024-02-01)
273. [carrierlxk/AGNN](https://github.com/carrierlxk/AGNN) (205 stars)
Zero-shot Video Object Segmentation via Attentive Graph Neural Networks (ICCV2019 Oral) (Python, 2019-08-12, 2024-01-04)
274. [RGLab/MAST](https://github.com/RGLab/MAST) (204 stars)
Tools and methods for analysis of single cell assay data in R (R, 2012-09-11, 2024-02-05)
275. [brentp/goleft](https://github.com/brentp/goleft) (204 stars)
goleft is a collection of bioinformatics tools distributed under MIT license in a single static binary (Go, 2016-09-19, 2024-02-05)
276. [cambridgeltl/MTL-Bioinformatics-2016](https://github.com/cambridgeltl/MTL-Bioinformatics-2016) (204 stars)
None (Python, 2016-09-22, 2024-01-21)
277. [fbreitwieser/krakenuniq](https://github.com/fbreitwieser/krakenuniq) (204 stars)
ðŸ™ KrakenUniq: Metagenomics classifier with unique k-mer counting for more specific results (C++, 2017-11-11, 2024-02-01)
278. [HajkD/biomartr](https://github.com/HajkD/biomartr) (202 stars)
Genomic Data Retrieval with R (R, 2014-08-05, 2023-12-25)
279. [KamilSJaron/smudgeplot](https://github.com/KamilSJaron/smudgeplot) (202 stars)
Inference of ploidy and heterozygosity structure using whole genome sequencing data (R, 2018-08-07, 2024-02-02)
280. [hkmztrk/DeepDTA](https://github.com/hkmztrk/DeepDTA) (201 stars)
None (Python, 2018-01-28, 2024-01-09)
281. [chaofengc/Face-SPARNet](https://github.com/chaofengc/Face-SPARNet) (200 stars)
Pytorch codes for "Learning Spatial Attention for Face Super-Resolution",  TIP 2020. (Python, 2019-12-26, 2024-01-24)
282. [bcgsc/NanoSim](https://github.com/bcgsc/NanoSim) (199 stars)
Nanopore sequence read simulator (Python, 2015-11-05, 2024-01-30)
283. [jeetsukumaran/DendroPy](https://github.com/jeetsukumaran/DendroPy) (199 stars)
A Python library for phylogenetic scripting, simulation, data processing and manipulation. (Python, 2009-10-06, 2024-02-06)
284. [niuchuangnn/SPICE](https://github.com/niuchuangnn/SPICE) (199 stars)
None (Python, 2021-03-16, 2024-01-28)
285. [griffithlab/GenVisR](https://github.com/griffithlab/GenVisR) (198 stars)
Genome data visualizations (R, 2015-01-26, 2024-02-03)
286. [ucscGenomeBrowser/kent](https://github.com/ucscGenomeBrowser/kent) (198 stars)
UCSC Genome Browser source tree. Stable branch: "beta". (C, 2015-04-09, 2024-01-26)
287. [ZJUFanLab/scCATCH](https://github.com/ZJUFanLab/scCATCH) (197 stars)
Automatic Annotation on Cell Types of Clusters from Single-Cell RNA Sequencing Data (R, 2019-08-06, 2024-01-13)
288. [taowenmicro/EasyMicrobiomeR](https://github.com/taowenmicro/EasyMicrobiomeR) (196 stars)
Using R language in microbiome analysis (HTML, 2019-07-06, 2024-01-23)
289. [rdk/p2rank](https://github.com/rdk/p2rank) (195 stars)
P2Rank: Protein-ligand binding site prediction tool based on machine learning. Stand-alone command line program / Java library for predicting ligand binding pockets from protein structure. (Groovy, 2016-09-25, 2024-01-26)
290. [shervinea/enzynet](https://github.com/shervinea/enzynet) (195 stars)
EnzyNet: enzyme classification using 3D convolutional neural networks on spatial representation (Python, 2017-07-19, 2024-01-29)
291. [labgem/PPanGGOLiN](https://github.com/labgem/PPanGGOLiN) (193 stars)
Build a partitioned pangenome graph from microbial genomes (Python, 2017-10-20, 2024-01-30)
292. [yangyanli/DO-Conv](https://github.com/yangyanli/DO-Conv) (193 stars)
Depthwise Over-parameterized Convolutional Layer (Python, 2020-06-22, 2024-01-23)
293. [iqtree/iqtree2](https://github.com/iqtree/iqtree2) (192 stars)
NEW location of IQ-TREE software for efficient phylogenomic software by maximum likelihood http://www.iqtree.org (C++, 2020-06-17, 2024-02-05)
294. [choderalab/espaloma](https://github.com/choderalab/espaloma) (191 stars)
Extensible Surrogate Potential of Ab initio Learned and Optimized by Message-passing Algorithm  ðŸ¹https://arxiv.org/abs/2010.01196 (Python, 2020-02-10, 2024-02-01)
295. [dariober/ASCIIGenome](https://github.com/dariober/ASCIIGenome) (191 stars)
Text Only Genome Viewer! (Java, 2016-05-16, 2024-02-06)
296. [kormilitzin/med7](https://github.com/kormilitzin/med7) (191 stars)
None (None, 2020-02-26, 2024-01-22)
297. [tfzhou/MATNet](https://github.com/tfzhou/MATNet) (191 stars)
Motion-Attentive Transition for Zero-Shot Video Object Segmentation (AAAI2020&TIP2021) (Python, 2019-11-17, 2024-01-19)
298. [INCATools/ontology-development-kit](https://github.com/INCATools/ontology-development-kit) (190 stars)
Bootstrap an OBO Library ontology (Jinja, 2015-12-15, 2024-02-04)
299. [atks/vt](https://github.com/atks/vt) (190 stars)
A tool set for short variant discovery in genetic sequence data. (C, 2013-10-03, 2024-01-24)
300. [TGAC/KAT](https://github.com/TGAC/KAT) (189 stars)
The K-mer Analysis Toolkit (KAT) contains a number of tools that analyse and compare K-mer spectra. (C++, 2013-10-03, 2024-02-02)
301. [showteeth/ggcoverage](https://github.com/showteeth/ggcoverage) (189 stars)
Visualize and annotate genomic coverage with ggplot2 (R, 2022-07-18, 2024-02-01)
302. [thinng/GraphDTA](https://github.com/thinng/GraphDTA) (189 stars)
GraphDTA: Predicting drug-target binding affinity with graph neural networks (Python, 2019-06-11, 2024-02-02)
303. [vibansal/HapCUT2](https://github.com/vibansal/HapCUT2) (189 stars)
software tools for haplotype assembly from sequence data (C, 2016-08-02, 2024-01-29)
304. [pmelsted/bifrost](https://github.com/pmelsted/bifrost) (185 stars)
Bifrost: Highly parallel construction and indexing of colored and compacted de Bruijn graphs (C++, 2013-05-16, 2024-01-31)
305. [yumingj/C2-Matching](https://github.com/yumingj/C2-Matching) (185 stars)
Code for C2-Matching (CVPR2021). Paper: Robust Reference-based Super-Resolution via C2-Matching. (Python, 2021-03-20, 2024-01-07)
306. [basiralab/GNNs-in-Network-Neuroscience](https://github.com/basiralab/GNNs-in-Network-Neuroscience) (184 stars)
A review of papers proposing novel GNN methods with application to brain connectivity published in 2017-2020. (None, 2021-06-07, 2024-01-31)
307. [cerr/CERR](https://github.com/cerr/CERR) (184 stars)
Matlab/Octave based platform for Radiological Research. (MATLAB, 2011-09-23, 2024-01-16)
308. [mirnylab/cooler](https://github.com/mirnylab/cooler) (184 stars)
A cool place to store your Hi-C (Python, 2016-01-13, 2024-02-01)
309. [craftGBD/craftGBD](https://github.com/craftGBD/craftGBD) (182 stars)
code for CUImage and CUVideo (Python, 2016-10-05, 2024-01-05)
310. [AstrobioMike/GToTree](https://github.com/AstrobioMike/GToTree) (181 stars)
A user-friendly workflow for phylogenomics (Shell, 2018-12-09, 2024-01-17)
311. [SajadMo/SleepEEGNet](https://github.com/SajadMo/SleepEEGNet) (181 stars)
SleepEEGNet: Automated Sleep Stage Scoring with Sequence to Sequence Deep Learning Approach (Python, 2019-02-24, 2024-01-31)
312. [CapsuleEndoscope/EndoSLAM](https://github.com/CapsuleEndoscope/EndoSLAM) (180 stars)
EndoSLAM Dataset and an Unsupervised Monocular Visual Odometry and Depth Estimation Approach for Endoscopic Videos: Endo-SfMLearner (Python, 2020-09-14, 2024-01-31)
313. [danmcduff/iphys-toolbox](https://github.com/danmcduff/iphys-toolbox) (180 stars)
A MATLAB toolbox for iPPG analysis.  The toolbox includes implementations of commonly used methods. (MATLAB, 2018-04-30, 2024-02-07)
314. [rdkit/mmpdb](https://github.com/rdkit/mmpdb) (180 stars)
A package to identify matched molecular pairs and use them to predict property changes. (Python, 2017-08-12, 2024-01-29)
315. [tensorboy/PIDOptimizer](https://github.com/tensorboy/PIDOptimizer) (180 stars)
Code for this CVPR 2018 paper: "A PID Controller Approach for Stochastic Optimization of Deep Networks",  Wangpeng An, Haoqian Wang, Qingyun Sun, Jun Xu, Qionghai Dai, Lei Zhang. (Python, 2018-03-09, 2024-02-02)
316. [debbiemarkslab/DeepSequence](https://github.com/debbiemarkslab/DeepSequence) (179 stars)
A generative latent variable model for biological sequence families. (Python, 2018-04-20, 2024-01-06)
317. [noriakis/ggkegg](https://github.com/noriakis/ggkegg) (179 stars)
Analyzing and visualizing KEGG information using the grammar of graphics (R, 2023-04-04, 2024-02-04)
318. [openvax/mhcflurry](https://github.com/openvax/mhcflurry) (178 stars)
Peptide-MHC I binding affinity prediction (Python, 2015-06-29, 2024-02-07)
319. [isovic/graphmap](https://github.com/isovic/graphmap) (177 stars)
GraphMap - A highly sensitive and accurate mapper for long, error-prone reads http://www.nature.com/ncomms/2016/160415/ncomms11307/full/ncomms11307.html Note: This was the original repository which will no longer be officially maintained. Please use the new official repository here: (C++, 2014-12-11, 2024-01-08)
320. [LiWentomng/boxlevelset](https://github.com/LiWentomng/boxlevelset) (176 stars)
The code for "Box-supervised Instance Segmentation with Level Set Evolution(ECCV2022)" (Python, 2021-12-05, 2024-01-31)
321. [theislab/paga](https://github.com/theislab/paga) (175 stars)
Mapping out the coarse-grained connectivity structures of complex manifolds. (Jupyter Notebook, 2018-06-08, 2024-02-06)
322. [anjany/verse](https://github.com/anjany/verse) (174 stars)
Everything about the 'Large Scale Vertebrae Segmentation Challenge' @ MICCAI 2019-2020 (Jupyter Notebook, 2020-12-16, 2024-02-04)
323. [Ha0Tang/GestureGAN](https://github.com/Ha0Tang/GestureGAN) (173 stars)
[ACM MM 2018 Oral] GestureGAN for Hand Gesture-to-Gesture Translation in the Wild (Python, 2018-08-08, 2024-01-27)
324. [aquaskyline/Clairvoyante](https://github.com/aquaskyline/Clairvoyante) (172 stars)
Clairvoyante: a multi-task convolutional deep neural network for variant calling in Single Molecule Sequencing (Python, 2017-07-21, 2024-01-22)
325. [cccorn/AP-loss](https://github.com/cccorn/AP-loss) (172 stars)
The implementation of "Towards accurate one-stage object detection with AP-loss". (Python, 2020-01-21, 2024-01-04)
326. [pangenome/odgi](https://github.com/pangenome/odgi) (172 stars)
Optimized Dynamic Genome/Graph Implementation: understanding pangenome graphs (C++, 2018-12-11, 2024-02-06)
327. [tfzhou/VS-Survey](https://github.com/tfzhou/VS-Survey) (172 stars)
None (None, 2021-12-21, 2024-01-26)
328. [JingZhang617/UCNet](https://github.com/JingZhang617/UCNet) (171 stars)
UC-Net: Uncertainty Inspired RGB-D Saliency Detectionvia Conditional Variational Autoencoders, CVPR2020 (Python, 2020-03-03, 2024-01-11)
329. [Nextomics/NextPolish](https://github.com/Nextomics/NextPolish) (171 stars)
 Fast and accurately polish the genome generated by long reads. (None, 2019-05-08, 2024-02-02)
330. [ekimb/rust-mdbg](https://github.com/ekimb/rust-mdbg) (171 stars)
minimizer-space de Bruijn graphs (mdBG) for whole genome assembly (Rust, 2020-11-06, 2024-01-29)
331. [wdecoster/nanopack](https://github.com/wdecoster/nanopack) (171 stars)
An overview of all nanopack tools (Python, 2017-10-02, 2024-01-23)
332. [Duankaiwen/PyCenterNet](https://github.com/Duankaiwen/PyCenterNet) (170 stars)
None (Python, 2022-04-18, 2024-01-30)
333. [Genetalks/gtz](https://github.com/Genetalks/gtz) (170 stars)
A high performance and compression ratio compressor for genomic data, powered by GTXLab of Genetalks. (None, 2016-11-11, 2024-01-18)
334. [privefl/bigsnpr](https://github.com/privefl/bigsnpr) (170 stars)
R package for the analysis of massive SNP arrays. (R, 2016-07-05, 2024-02-06)
335. [MoonBlvd/Detection-of-Traffic-Anomaly](https://github.com/MoonBlvd/Detection-of-Traffic-Anomaly) (169 stars)
This is the repo for our Detection of Traffic Anomaly (DoTA) dataset. (Python, 2020-04-06, 2024-02-06)
336. [machinegun/SALSA](https://github.com/machinegun/SALSA) (169 stars)
SALSA: A tool to scaffold long read assemblies with Hi-C data (Python, 2016-06-06, 2024-01-23)
337. [gpertea/gffcompare](https://github.com/gpertea/gffcompare) (168 stars)
classify, merge, tracking and annotation of GFF files by comparing to a reference annotation GFF (C++, 2015-12-17, 2024-02-02)
338. [mczhuge/ICON](https://github.com/mczhuge/ICON) (168 stars)
(TPAMI2022) Salient Object Detection via Integrity Learning. (Python, 2020-12-25, 2024-01-12)
339. [jbkinney/logomaker](https://github.com/jbkinney/logomaker) (167 stars)
Software for the visualization of sequence-function relationships (Jupyter Notebook, 2017-10-05, 2024-02-01)
340. [shenwei356/kmcp](https://github.com/shenwei356/kmcp) (167 stars)
Accurate metagenomic profiling && Fast large-scale sequence/genome searching (Go, 2020-10-30, 2024-01-22)
341. [choderalab/perses](https://github.com/choderalab/perses) (166 stars)
Experiments with expanded ensembles to explore chemical space (Python, 2014-11-24, 2024-01-22)
342. [mortazavilab/PyWGCNA](https://github.com/mortazavilab/PyWGCNA) (166 stars)
PyWGCNA is a Python package designed to do Weighted Gene Correlation Network analysis (WGCNA) (Jupyter Notebook, 2022-01-28, 2024-02-06)
343. [JialeCao001/PedSurvey](https://github.com/JialeCao001/PedSurvey) (165 stars)
From Handcrafted to Deep Features for Pedestrian Detection: A Survey (TPAMI 2021) (None, 2020-08-13, 2024-01-09)
344. [Bobholamovic/CDLab](https://github.com/Bobholamovic/CDLab) (164 stars)
Yet another repository for developing and benchmarking deep learning-based change detection methods. (Python, 2020-12-16, 2024-02-02)
345. [JaminFong/FNA](https://github.com/JaminFong/FNA) (164 stars)
Fast Neural Network Adaptation via Parameter Remapping and Architecture Search (ICLR2020 & TPAMI) (Python, 2019-12-25, 2024-01-31)
346. [KristapsE/DP4-AI](https://github.com/KristapsE/DP4-AI) (164 stars)
Python workflow for DP5 and DP4 analysis of organic molecules (Python, 2015-08-26, 2024-01-04)
347. [malonge/RaGOO](https://github.com/malonge/RaGOO) (164 stars)
RaGOO is no longer supported. Please use RagTag instead: https://github.com/malonge/RagTag (Python, 2018-02-01, 2024-01-16)
348. [baoe/AlignGraph](https://github.com/baoe/AlignGraph) (163 stars)
Algorithm for secondary de novo genome assembly guided by closely related references (C++, 2014-01-10, 2023-10-23)
349. [kexinhuang12345/moltrans](https://github.com/kexinhuang12345/moltrans) (163 stars)
MolTrans: Molecular Interaction Transformer for Drug Target Interaction Prediction (Bioinformatics) (Jupyter Notebook, 2020-07-16, 2024-02-06)
350. [molgenis/systemsgenetics](https://github.com/molgenis/systemsgenetics) (163 stars)
Generic Java genotype reader / writer, QTL mapping software, Strand alignment tool (Java, 2013-07-11, 2024-01-12)
351. [ConesaLab/SQANTI3](https://github.com/ConesaLab/SQANTI3) (161 stars)
Tool for the Quality Control of Long-Read Defined Transcriptomes (HTML, 2020-05-11, 2024-02-04)
352. [Illumina/ExpansionHunter](https://github.com/Illumina/ExpansionHunter) (161 stars)
A tool for estimating repeat sizes (C++, 2016-11-10, 2024-02-06)
353. [vgteam/sequenceTubeMap](https://github.com/vgteam/sequenceTubeMap) (161 stars)
displays multiple genomic sequences in the form of a tube map (JavaScript, 2016-06-21, 2024-02-02)
354. [wanggrun/SYSU-30k](https://github.com/wanggrun/SYSU-30k) (161 stars)
SYSU-30k Dataset of "Weakly Supervised Person Re-ID: Differentiable Graphical Learning and A New Benchmark" https://arxiv.org/abs/1904.03845 (Python, 2019-04-12, 2024-01-29)
355. [ndierckx/NOVOPlasty](https://github.com/ndierckx/NOVOPlasty) (160 stars)
NOVOPlasty - The organelle assembler and heteroplasmy caller (Perl, 2015-12-07, 2024-02-06)
356. [soedinglab/metaeuk](https://github.com/soedinglab/metaeuk) (160 stars)
MetaEuk - sensitive, high-throughput gene discovery and annotation for large-scale eukaryotic metagenomics (C, 2018-03-07, 2024-02-01)
357. [stephaneguindon/phyml](https://github.com/stephaneguindon/phyml) (160 stars)
PhyML -- Phylogenetic estimation using (Maximum) Likelihood (C, 2014-09-15, 2024-01-16)
358. [HadrienG/InSilicoSeq](https://github.com/HadrienG/InSilicoSeq) (159 stars)
:rocket: A sequencing simulator (Python, 2016-11-16, 2024-01-23)
359. [alexanderkroner/saliency](https://github.com/alexanderkroner/saliency) (159 stars)
Contextual Encoder-Decoder Network for Visual Saliency Prediction [Neural Networks 2020] (Python, 2019-02-18, 2024-01-11)
360. [chrisamiller/fishplot](https://github.com/chrisamiller/fishplot) (159 stars)
Create timecourse "fish plots" that show changes in the clonal architecture of tumors (R, 2016-05-06, 2024-01-30)
361. [franciscozorrilla/metaGEM](https://github.com/franciscozorrilla/metaGEM) (159 stars)
:gem: An easy-to-use workflow for generating context specific genome-scale metabolic models and predicting metabolic interactions within microbial communities directly from metagenomic data (Python, 2018-06-14, 2024-01-29)
362. [Superzchen/iFeature](https://github.com/Superzchen/iFeature) (158 stars)
iFeature is a comprehensive Python-based toolkit for generating various numerical feature representation schemes from protein or peptide sequences. iFeature is capable of calculating and extracting a wide spectrum of 18 major sequence encoding schemes that encompass 53 different types of feature descriptors. Furthermore, iFeature also integrates five kinds of frequently used feature clustering algorithms, four feature selection algorithms and three dimensionality reduction algorithms. (Python, 2017-11-14, 2024-01-13)
363. [alanwilter/acpype](https://github.com/alanwilter/acpype) (158 stars)
OFFICIAL: AnteChamber PYthon Parser interfacE (Python, 2017-10-20, 2024-02-04)
364. [fnzhan/Illumination-Estimation](https://github.com/fnzhan/Illumination-Estimation) (158 stars)
[AAAI 2021] EMLight: Lighting Estimation via Spherical Distribution Approximation, [TIP] GMLight: Lighting Estimation via Geometric Distribution Approximation (Python, 2021-02-15, 2024-01-29)
365. [galaxyproject/tools-iuc](https://github.com/galaxyproject/tools-iuc) (156 stars)
Tool Shed repositories maintained by the Intergalactic Utilities Commission (HTML, 2014-09-13, 2024-01-14)
366. [dnbaker/dashing](https://github.com/dnbaker/dashing) (155 stars)
Fast and accurate genomic distances using HyperLogLog (C++, 2018-04-03, 2024-01-28)
367. [sorgerlab/indra](https://github.com/sorgerlab/indra) (155 stars)
INDRA (Integrated Network and Dynamical Reasoning Assembler) is an automated model assembly system interfacing with NLP systems and databases to collect knowledge, and through a process of assembly, produce causal graphs and dynamical models. (Python, 2014-08-11, 2024-01-24)
368. [taoyang1122/MutualNet](https://github.com/taoyang1122/MutualNet) (155 stars)
[ECCV'20 Oral] MutualNet: Adaptive ConvNet via Mutual Learning from Network Width and Resolution (Python, 2020-03-22, 2024-01-23)
369. [yiluheihei/microbiomeMarker](https://github.com/yiluheihei/microbiomeMarker) (155 stars)
R package for microbiome biomarker discovery (R, 2019-10-17, 2024-01-28)
370. [BaselAbujamous/clust](https://github.com/BaselAbujamous/clust) (154 stars)
Automatic and optimised consensus clustering of one or more heterogeneous datasets (Python, 2017-03-07, 2023-12-01)
371. [ChiLiubio/microeco](https://github.com/ChiLiubio/microeco) (154 stars)
An R package for data analysis in microbial community ecology (R, 2020-04-16, 2024-02-06)
372. [UCLCheminformatics/ScaffoldGraph](https://github.com/UCLCheminformatics/ScaffoldGraph) (154 stars)
ScaffoldGraph is an open-source cheminformatics library, built using RDKit and NetworkX, for the generation and analysis of scaffold networks and scaffold trees. (Python, 2019-12-10, 2024-02-04)
373. [liaohaofu/adn](https://github.com/liaohaofu/adn) (154 stars)
ADN: Artifact Disentanglement Network for Unsupervised Metal Artifact Reduction (Python, 2019-06-27, 2024-01-30)
374. [nygenome/lancet](https://github.com/nygenome/lancet) (154 stars)
Microassembly based somatic variant caller for NGS data (C, 2016-02-03, 2023-11-29)
375. [brentp/hts-nim](https://github.com/brentp/hts-nim) (153 stars)
nim wrapper for htslib for parsing genomics data files (Nim, 2017-08-15, 2023-10-19)
376. [mfumagalli/ngsTools](https://github.com/mfumagalli/ngsTools) (153 stars)
Programs to analyse NGS data for population genetics purposes (R, 2014-01-06, 2024-01-11)
377. [vsbuffalo/scidataflow](https://github.com/vsbuffalo/scidataflow) (152 stars)
Command line scientific data management tool (Rust, 2023-08-02, 2024-02-06)
378. [3dpose/3D-Multi-Person-Pose](https://github.com/3dpose/3D-Multi-Person-Pose) (151 stars)
3D Multi-Person Pose Estimation by Integrating Top-Down and Bottom-Up Networks (Python, 2021-03-01, 2024-02-01)
379. [AnantharamanLab/METABOLIC](https://github.com/AnantharamanLab/METABOLIC) (151 stars)
A scalable high-throughput metabolic and biogeochemical functional trait profiler (Perl, 2019-08-28, 2024-02-01)
380. [bjing2016/EigenFold](https://github.com/bjing2016/EigenFold) (151 stars)
EigenFold: Generative Protein Structure Prediction with Diffusion Models (Jupyter Notebook, 2023-04-02, 2024-01-26)
381. [divelab/MoleculeX](https://github.com/divelab/MoleculeX) (151 stars)
None (Python, 2020-08-15, 2024-01-30)
382. [herrmannlab/highdicom](https://github.com/herrmannlab/highdicom) (151 stars)
High-level DICOM abstractions for the Python programming language (Python, 2019-08-01, 2024-02-01)
383. [mkirsche/Jasmine](https://github.com/mkirsche/Jasmine) (151 stars)
Jasmine: SV Merging Across Samples (Java, 2019-11-15, 2024-01-17)
384. [pymzML/pymzML](https://github.com/pymzML/pymzML) (151 stars)
pymzML - an interface between Python and mzML Mass spectrometry Files (Python, 2012-01-11, 2024-01-15)
385. [dsehnal/LiteMol](https://github.com/dsehnal/LiteMol) (150 stars)
A library/plugin for handling 3D structural molecular data (not only) in the browser. (JavaScript, 2016-08-23, 2024-01-27)
386. [XinhaoLi74/SmilesPE](https://github.com/XinhaoLi74/SmilesPE) (149 stars)
SMILES Pair Encoding: A data-driven substructure representation of chemicals (Jupyter Notebook, 2020-03-16, 2024-01-30)
387. [duartegroup/autodE](https://github.com/duartegroup/autodE) (149 stars)
automated reaction profile generation (Python, 2019-07-09, 2024-01-28)
388. [DianboWork/SPN4RE](https://github.com/DianboWork/SPN4RE) (147 stars)
None (Python, 2020-10-19, 2024-01-04)
389. [KorfLab/StochHMM](https://github.com/KorfLab/StochHMM) (147 stars)
A Hidden Markov Model Application and C++ Library for Rapid and Flexible Development of HMMs (C++, 2012-10-05, 2024-02-01)
390. [abyzovlab/CNVpytor](https://github.com/abyzovlab/CNVpytor) (147 stars)
a python extension of CNVnator -- a tool for CNV analysis from depth-of-coverage by mapped reads (Python, 2019-09-11, 2024-01-23)
391. [crsl4/PhyloNetworks](https://github.com/crsl4/PhyloNetworks) (146 stars)
A Julia package for statistical inference, data manipulation and visualization of phylogenetic networks (Julia, 2014-08-26, 2024-01-20)
392. [venyao/shinyCircos](https://github.com/venyao/shinyCircos) (146 stars)
an R/shiny application for creation of Circos plot interactively (HTML, 2017-07-21, 2024-01-22)
393. [Ha0Tang/LGGAN](https://github.com/Ha0Tang/LGGAN) (145 stars)
[CVPR 2020] Local Class-Specific and Global Image-Level Generative Adversarial Networks for Semantic-Guided Scene Generation (Python, 2019-07-23, 2024-01-27)
394. [marcelauliano/MitoHiFi](https://github.com/marcelauliano/MitoHiFi) (145 stars)
Find, circularise and annotate mitogenome from PacBio assemblies (Python, 2020-08-18, 2024-02-04)
395. [BenevolentAI/DeeplyTough](https://github.com/BenevolentAI/DeeplyTough) (144 stars)
DeeplyTough: Learning Structural Comparison of Protein Binding Sites (Python, 2019-04-03, 2024-01-04)
396. [ZJUDataIntelligence/TEMPO](https://github.com/ZJUDataIntelligence/TEMPO) (144 stars)
None (Python, 2022-07-28, 2023-11-15)
397. [divonlan/genozip](https://github.com/divonlan/genozip) (144 stars)
A modern compressor for genomic files (FASTQ, SAM/BAM/CRAM, VCF, FASTA, GFF/GTF/GVF, 23andMe...), up to 5x better than gzip and faster too (C, 2019-12-15, 2024-01-18)
398. [statgen/locuszoom](https://github.com/statgen/locuszoom) (144 stars)
A Javascript/d3 embeddable plugin for interactively visualizing statistical genetic data from customizable sources. (JavaScript, 2015-08-24, 2024-02-06)
399. [tsudalab/ChemTS](https://github.com/tsudalab/ChemTS) (144 stars)
Molecule Design using Monte Carlo Tree Search with Neural Rollout (Python, 2017-08-27, 2024-02-06)
400. [ComparativeGenomicsToolkit/hal](https://github.com/ComparativeGenomicsToolkit/hal) (143 stars)
Hierarchical Alignment Format (C++, 2012-03-20, 2024-02-01)
401. [OpenGene/MutScan](https://github.com/OpenGene/MutScan) (143 stars)
Detect and visualize target mutations by scanning FastQ files directly (C, 2016-07-23, 2024-02-05)
402. [clovaai/lffont](https://github.com/clovaai/lffont) (143 stars)
Official PyTorch implementation of LF-Font (Few-shot Font Generation with Localized Style Representations and Factorization) AAAI 2021 (Python, 2020-09-23, 2024-01-29)
403. [glennhickey/hal](https://github.com/glennhickey/hal) (143 stars)
Hierarchical Alignment Format (C++, 2012-03-20, 2024-02-01)
404. [ekg/seqwish](https://github.com/ekg/seqwish) (141 stars)
alignment to variation graph inducer (C++, 2018-06-11, 2023-12-07)
405. [mschubert/clustermq](https://github.com/mschubert/clustermq) (141 stars)
R package to send function calls as jobs on LSF, SGE, Slurm, PBS/Torque, or each via SSH (R, 2016-06-18, 2024-02-02)
406. [rajeevyasarla/Syn2Real](https://github.com/rajeevyasarla/Syn2Real) (140 stars)
Syn2Real Transfer Learning for Image Deraining using Gaussian Processes (Python, 2020-05-14, 2024-01-20)
407. [CellProfiler/CellProfiler-Analyst](https://github.com/CellProfiler/CellProfiler-Analyst) (139 stars)
Open-source software for exploring and analyzing large, high-dimensional image-derived data. (Python, 2011-11-10, 2024-01-19)
408. [LinearFold/LinearFold](https://github.com/LinearFold/LinearFold) (139 stars)
The new source code of the LinearFold, linear-time prediction for RNA secondary structures (C++, 2018-11-27, 2024-02-06)
409. [saezlab/decoupleR](https://github.com/saezlab/decoupleR) (139 stars)
R package to infer biological activities from omics data using a collection of methods. (R, 2019-11-21, 2024-01-24)
410. [eldariont/svim](https://github.com/eldariont/svim) (138 stars)
Structural Variant Identification Method using Long Reads (Python, 2018-02-01, 2024-01-24)
411. [fenderglass/Ragout](https://github.com/fenderglass/Ragout) (138 stars)
Chromosome-level scaffolding using multiple references (Python, 2013-08-03, 2024-02-01)
412. [neurodata/neuroparc](https://github.com/neurodata/neuroparc) (138 stars)
None (Jupyter Notebook, 2019-01-14, 2024-01-31)
413. [sequana/sequana](https://github.com/sequana/sequana) (138 stars)
Sequana: a set of Snakemake NGS pipelines (Jupyter Notebook, 2016-03-07, 2024-01-29)
414. [wanglimin/MRCNN-Scene-Recognition](https://github.com/wanglimin/MRCNN-Scene-Recognition) (138 stars)
MR-CNNs for Large-Scale Scene Recognition (Matlab, 2016-10-04, 2024-01-05)
415. [emadeldeen24/AttnSleep](https://github.com/emadeldeen24/AttnSleep) (137 stars)
[TNSRE 2021] "An Attention-based Deep Learning Approach for Sleep Stage Classification with Single-Channel EEG" (Python, 2020-10-21, 2024-01-22)
416. [huangnengCSU/compleasm](https://github.com/huangnengCSU/compleasm) (137 stars)
A genome completeness evaluation tool based on miniprot (Python, 2023-04-04, 2024-02-06)
417. [huangnengCSU/minibusco](https://github.com/huangnengCSU/minibusco) (137 stars)
A genome completeness evaluation tool based on miniprot (Python, 2023-04-04, 2024-02-06)
418. [millanek/Dsuite](https://github.com/millanek/Dsuite) (137 stars)
Fast calculation of Patterson's D (ABBA-BABA) and the f4-ratio statistics across many populations/species (C++, 2019-04-11, 2024-02-01)
419. [bjornwallner/DockQ](https://github.com/bjornwallner/DockQ) (136 stars)
DockQ is a single continuous quality measure for protein docked models based on the CAPRI evaluation protocol (Python, 2015-12-12, 2024-02-02)
420. [AdmiralenOla/Scoary](https://github.com/AdmiralenOla/Scoary) (135 stars)
Pan-genome wide association studies (Python, 2016-02-03, 2024-02-01)
421. [mlyg/unified-focal-loss](https://github.com/mlyg/unified-focal-loss) (135 stars)
None (Python, 2021-02-03, 2024-01-18)
422. [ncbi/icn3d](https://github.com/ncbi/icn3d) (135 stars)
web-based protein structure viewer and analysis tool interactively or in batch mode (JavaScript, 2016-01-20, 2024-01-18)
423. [smarco/WFA](https://github.com/smarco/WFA) (135 stars)
WFA-lib: Wavefront alignment algorithm library v2 (C, 2021-06-13, 2024-01-31)
424. [theislab/kBET](https://github.com/theislab/kBET) (135 stars)
An R package to test for batch effects in high-dimensional single-cell RNA sequencing data. (HTML, 2016-12-06, 2024-01-30)
425. [Ensembl/ensembl-rest](https://github.com/Ensembl/ensembl-rest) (134 stars)
Language agnostic RESTful data access to Ensembl data over HTTP (Perl, 2013-12-13, 2024-01-22)
426. [getzlab/rnaseqc](https://github.com/getzlab/rnaseqc) (134 stars)
Fast, efficient RNA-Seq metrics for quality control and process optimization (C++, 2017-08-10, 2024-01-29)
427. [krishnanlab/PecanPy](https://github.com/krishnanlab/PecanPy) (134 stars)
A fast, parallelized, memory efficient, and cache-optimized Python implementation of node2vec (Python, 2020-03-16, 2024-02-06)
428. [lsyads/fisheye-ORB-SLAM](https://github.com/lsyads/fisheye-ORB-SLAM) (134 stars)
A real-time robust monocular visual SLAM system based on ORB-SLAM for fisheye cameras, without rectifying or cropping the input images (C++, 2019-07-09, 2024-01-04)
429. [masqm/Faster-Mean-Shift](https://github.com/masqm/Faster-Mean-Shift) (134 stars)
GPU accelerated Faster Mean-shift for cosine embedding clustering (Python, 2020-07-11, 2024-02-02)
430. [xlliu7/TadTR](https://github.com/xlliu7/TadTR) (134 stars)
[TIP 2022] End-to-end Temporal Action Detection with Transformer (Python, 2021-05-06, 2024-01-28)
431. [jotech/gapseq](https://github.com/jotech/gapseq) (133 stars)
Informed prediction and analysis of bacterial metabolic pathways and genome-scale networks (R, 2017-11-01, 2024-02-06)
432. [molML/MoleculeACE](https://github.com/molML/MoleculeACE) (133 stars)
A tool for evaluating the predictive performance on activity cliff compounds of machine learning models (Python, 2022-03-26, 2024-01-26)
433. [ryanlayer/gqt](https://github.com/ryanlayer/gqt) (133 stars)
A genotype query interface. (C, 2014-07-07, 2023-12-08)
434. [wenguanwang/DHF1K](https://github.com/wenguanwang/DHF1K) (133 stars)
Revisiting Video Saliency: A Large-scale Benchmark and a New Model (CVPR18, PAMI19) (MATLAB, 2018-03-15, 2024-01-08)
435. [OpenBioLink/OpenBioLink](https://github.com/OpenBioLink/OpenBioLink) (132 stars)
OpenBioLink is a resource and evaluation framework for evaluating link prediction models on heterogeneous biomedical graph data. (Python, 2019-07-30, 2024-01-30)
436. [royfrancis/pophelper](https://github.com/royfrancis/pophelper) (132 stars)
An R package to analyse and visualise admixture proportions from STRUCTURE, fastSTRUCTURE, TESS, ADMIXTURE etc. (R, 2014-02-16, 2024-01-09)
437. [stemangiola/tidyseurat](https://github.com/stemangiola/tidyseurat) (132 stars)
Seurat meets tidyverse. The best of both worlds. (R, 2020-08-23, 2024-02-07)
438. [walaj/SeqLib](https://github.com/walaj/SeqLib) (132 stars)
C++ htslib/bwa-mem/fermi interface for interrogating sequence data (C++, 2015-05-07, 2023-12-04)
439. [CBICA/GaNDLF](https://github.com/CBICA/GaNDLF) (131 stars)
A generalizable application framework for segmentation, regression, and classification using PyTorch (Python, 2021-02-26, 2024-02-06)
440. [HRanWang/Spatial-Re-Scaling](https://github.com/HRanWang/Spatial-Re-Scaling) (131 stars)
None (Python, 2018-12-02, 2024-01-04)
441. [SheffieldML/GPmat](https://github.com/SheffieldML/GPmat) (131 stars)
Matlab implementations of Gaussian processes and other machine learning tools. (Matlab, 2013-02-26, 2024-01-28)
442. [liugang1234567/Gang-neuron](https://github.com/liugang1234567/Gang-neuron) (131 stars)
It may be time to improve the neuron of artificial neural network (IEEE preprints-IEEE preprints ranking ï¼ˆTechRxivï¼‰: Top 1 in yearly popularity-You should cite it.): https://doi.org/10.36227/techrxiv.12477266. Dendrite Net: A White-Box Module for Classification, Regression, and System Identification: https://arxiv.org/abs/2004.03955. (Python, 2020-06-15, 2024-01-14)
443. [aertslab/cistopic](https://github.com/aertslab/cistopic) (130 stars)
cisTopic: Probabilistic modelling of cis-regulatory topics from single cell epigenomics data (R, 2018-05-23, 2024-01-23)
444. [benjjneb/decontam](https://github.com/benjjneb/decontam) (130 stars)
Simple statistical identification and removal of contaminants in marker-gene and metagenomics sequencing data (R, 2017-04-04, 2024-02-01)
445. [deepmodeling/DMFF](https://github.com/deepmodeling/DMFF) (130 stars)
DMFF (Differentiable Molecular Force Field) is a Jax-based python package that provides a full differentiable implementation of molecular force field models. (Python, 2022-02-14, 2024-01-27)
446. [iCGY96/ARPL](https://github.com/iCGY96/ARPL) (130 stars)
[TPAMI 2021] Adversarial Reciprocal Points Learning for Open Set Recognition (Python, 2020-11-08, 2024-01-27)
447. [ncclabsustech/EEGdenoiseNet](https://github.com/ncclabsustech/EEGdenoiseNet) (130 stars)
EEGdenoiseNet, a benchmark dataset, that is suited for training and testing deep learning-based EEG denoising models, as well as for comparing the performance across different models. (Jupyter Notebook, 2020-08-19, 2024-02-03)
448. [tjubiit/TJU-DHD](https://github.com/tjubiit/TJU-DHD) (130 stars)
A newly built high-resolution dataset for object detection and pedestrian detection (IEEE TIP 2020) (None, 2019-09-03, 2024-02-02)
449. [Teichlab/bbknn](https://github.com/Teichlab/bbknn) (129 stars)
Batch balanced KNN (Python, 2018-06-18, 2024-02-02)
450. [galaxyproject/SARS-CoV-2](https://github.com/galaxyproject/SARS-CoV-2) (128 stars)
Ongoing analysis of COVID-19 using Galaxy, BioConda and public research infrastructures (Jupyter Notebook, 2020-02-13, 2024-01-04)
451. [lanl001/halc](https://github.com/lanl001/halc) (128 stars)
High Throughput Algorithm for Long Read Error Correction (C++, 2016-02-18, 2023-02-24)
452. [DengPingFan/D3NetBenchmark](https://github.com/DengPingFan/D3NetBenchmark) (126 stars)
Rethinking RGB-D Salient Object Detection: Models, Datasets, and Large-Scale Benchmarks, IEEE TNNLS 2020 (Python, 2019-07-16, 2024-02-04)
453. [Leotju/MGAN](https://github.com/Leotju/MGAN) (126 stars)
Mask-Guided Attention Network for Occluded Pedestrian Detection. (ICCV'19) (Python, 2019-08-27, 2024-01-04)
454. [brentp/echtvar](https://github.com/brentp/echtvar) (126 stars)
using all the bits for echt rapid variant annotation and filtering (Rust, 2021-11-22, 2024-01-19)
455. [churchmanlab/genewalk](https://github.com/churchmanlab/genewalk) (126 stars)
GeneWalk identifies relevant gene functions for a biological context using network representation learning (Python, 2019-01-22, 2024-02-06)
456. [marianattestad/assemblytics](https://github.com/marianattestad/assemblytics) (126 stars)
Assemblytics is a bioinformatics tool to detect and analyze structural variants from a genome assembly by comparing it to a reference genome. (JavaScript, 2016-03-15, 2024-02-06)
457. [nadeemlab/DeepLIIF](https://github.com/nadeemlab/DeepLIIF) (126 stars)
Deep Learning Inferred Multiplex ImmunoFluorescence for IHC Image Quantification (https://deepliif.org) [Nature Machine Intelligence'22, CVPR'22, MICCAI'23, Histopathology'23] (Python, 2021-04-15, 2024-02-05)
458. [nadeemlab/deepliif](https://github.com/nadeemlab/deepliif) (126 stars)
Deep Learning Inferred Multiplex ImmunoFluorescence for IHC Image Quantification (https://deepliif.org) [Nature Machine Intelligence'22, CVPR'22, MICCAI'23, Histopathology'23] (Python, 2021-04-15, 2024-02-05)
459. [CCChenhao997/DualGCN-ABSA](https://github.com/CCChenhao997/DualGCN-ABSA) (125 stars)
Dual Graph Convolutional Networks for Aspect-based Sentiment Analysis (Python, 2021-05-07, 2024-01-28)
460. [gabraham/flashpca](https://github.com/gabraham/flashpca) (125 stars)
Fast Principal Component Analysis of Large-Scale Genome-Wide Data (C++, 2014-01-23, 2024-01-17)
461. [theislab/scCODA](https://github.com/theislab/scCODA) (125 stars)
A Bayesian model for compositional single-cell data analysis (Jupyter Notebook, 2019-01-21, 2024-02-06)
462. [vals/umis](https://github.com/vals/umis) (125 stars)
Tools for processing UMI RNA-tag data (Python, 2015-06-18, 2024-01-04)
463. [ATOMconsortium/AMPL](https://github.com/ATOMconsortium/AMPL) (124 stars)
The ATOM Modeling PipeLine (AMPL) is an open-source, modular, extensible software pipeline for building and sharing models to advance in silico drug discovery. (Jupyter Notebook, 2019-11-07, 2023-12-14)
464. [antigenomics/vdjdb-db](https://github.com/antigenomics/vdjdb-db) (124 stars)
ðŸ—‚ï¸ [vdjdb.cdr3.net is up and running] Git-based TCR database storage & management. Submissions welcome! (Groovy, 2016-01-06, 2024-01-30)
465. [hakyimlab/MetaXcan](https://github.com/hakyimlab/MetaXcan) (124 stars)
MetaXcan software and manuscript (Python, 2016-02-10, 2024-01-26)
466. [zhanxw/rvtests](https://github.com/zhanxw/rvtests) (124 stars)
Rare variant test software for next generation sequencing data (C++, 2013-01-09, 2024-01-23)
467. [LayneH/self-adaptive-training](https://github.com/LayneH/self-adaptive-training) (123 stars)
[TPAMI2022 & NeurIPS2020] Official implementation of Self-Adaptive Training (Python, 2020-02-22, 2024-01-04)
468. [hewm2008/NGenomeSyn](https://github.com/hewm2008/NGenomeSyn) (123 stars)
Any Way to Show  Multi genomic Synteny (Perl, 2022-03-31, 2024-02-02)
469. [mikessh/vdjtools](https://github.com/mikessh/vdjtools) (123 stars)
Post-analysis of immune repertoire sequencing data (Groovy, 2014-06-06, 2024-02-04)
470. [CapsuleEndoscope/VirtualCapsuleEndoscopy](https://github.com/CapsuleEndoscope/VirtualCapsuleEndoscopy) (122 stars)
VR-Caps: A Virtual Environment for Active Capsule Endoscopy (C#, 2020-08-06, 2024-01-20)
471. [imgag/ngs-bits](https://github.com/imgag/ngs-bits) (122 stars)
Short-read sequencing tools (C++, 2015-06-25, 2024-01-26)
472. [rrwick/Deepbinner](https://github.com/rrwick/Deepbinner) (122 stars)
a signal-level demultiplexer for Oxford Nanopore reads (Python, 2018-02-19, 2024-02-02)
473. [speedinghzl/AlignSeg](https://github.com/speedinghzl/AlignSeg) (122 stars)
AlignSeg: Feature-Aligned Segmentation Networks (TPAMI 2021) (Python, 2020-02-23, 2024-01-04)
474. [xlucpu/MOVICS](https://github.com/xlucpu/MOVICS) (122 stars)
Multi-Omics integration and VIsualization in Cancer Subtyping (R, 2020-09-01, 2024-01-31)
475. [akshaydudhane16/BIPNet](https://github.com/akshaydudhane16/BIPNet) (121 stars)
[CVPR 2022--Oral, Best paper Finalist] Burst Image Restoration and Enhancement. SOTA for Burst Super-resolution, Low-light Burst Image Enhancement, Burst Image De-noising (Python, 2022-04-03, 2024-01-16)
476. [bigdatagenomics/mango](https://github.com/bigdatagenomics/mango) (121 stars)
A scalable genome browser. Apache 2 licensed. (Scala, 2015-03-04, 2024-01-12)
477. [gbouras13/pharokka](https://github.com/gbouras13/pharokka) (121 stars)
fast phage annotation program (Python, 2021-12-08, 2024-02-05)
478. [healthylaife/MIMIC-IV-Data-Pipeline](https://github.com/healthylaife/MIMIC-IV-Data-Pipeline) (121 stars)
A customizable pipeline for data extraction from MIMIC-IV. (Jupyter Notebook, 2021-08-06, 2024-02-07)
479. [lh3/bedtk](https://github.com/lh3/bedtk) (121 stars)
A simple toolset for BED files (warning: CLI may change before bedtk becomes stable) (C, 2019-10-02, 2024-02-06)
480. [ncbi/TPMCalculator](https://github.com/ncbi/TPMCalculator) (121 stars)
TPMCalculator quantifies mRNA abundance directly from the alignments by parsing BAM files (C++, 2018-03-09, 2023-12-01)
481. [stefpeschel/NetCoMi](https://github.com/stefpeschel/NetCoMi) (121 stars)
Network construction, analysis, and comparison for microbial compositional data (R, 2020-04-29, 2024-01-22)
482. [vztu/VIDEVAL](https://github.com/vztu/VIDEVAL) (121 stars)
[IEEE TIP'2021] "UGC-VQA: Benchmarking Blind Video Quality Assessment for User Generated Content", Zhengzhong Tu, Yilin Wang, Neil Birkbeck, Balu Adsumilli, Alan C. Bovik (MATLAB, 2020-05-18, 2024-01-29)
483. [SGDDNB/ShinyCell](https://github.com/SGDDNB/ShinyCell) (120 stars)
Shiny Interactive Web Apps for Single-Cell Data (R, 2020-10-08, 2024-01-31)
484. [muhammedtalo/COVID-19](https://github.com/muhammedtalo/COVID-19) (120 stars)
Automated Detection of COVID-19 Cases Using Deep Neural Networks with X-Ray Images (Jupyter Notebook, 2020-04-20, 2024-02-05)
485. [songc001/remilo](https://github.com/songc001/remilo) (120 stars)
Reference Assisted Misassembly Detection Algorithm  Using Short and Long Reads (C++, 2017-01-17, 2023-02-26)
486. [BBillot/SynthSR](https://github.com/BBillot/SynthSR) (119 stars)
A framework for joint super-resolution and image synthesis, without requiring real training data (Python, 2020-12-09, 2024-01-18)
487. [echen102/us-pres-elections-2020](https://github.com/echen102/us-pres-elections-2020) (119 stars)
The repository contains a collection of tweets IDs associated with the 2020 U.S. Presidential Elections through 6 months post-inauguration. (Python, 2020-09-28, 2024-01-10)
488. [xulabs/aitom](https://github.com/xulabs/aitom) (119 stars)
AI for tomography (Python, 2019-06-13, 2024-01-22)
489. [Illumina/canvas](https://github.com/Illumina/canvas) (118 stars)
Canvas - Copy number variant (CNV) calling from DNA sequencing data (C#, 2015-07-20, 2023-12-19)
490. [baoe/flas](https://github.com/baoe/flas) (118 stars)
Fast and High Throughput Algorithm for PacBio Long Read Self-Correction (C++, 2018-11-02, 2023-06-15)
491. [hasindu2008/f5c](https://github.com/hasindu2008/f5c) (118 stars)
Ultra-fast methylation calling and event alignment tool for nanopore sequencing data (supports CUDA acceleration) (C, 2018-07-20, 2024-01-22)
492. [spetti/SMURF](https://github.com/spetti/SMURF) (118 stars)
None (Python, 2021-08-19, 2024-02-05)
493. [theislab/sc-pert](https://github.com/theislab/sc-pert) (118 stars)
Models and datasets for perturbational single-cell omics (Jupyter Notebook, 2020-12-20, 2024-01-16)
494. [BlueBrain/NeuroMorphoVis](https://github.com/BlueBrain/NeuroMorphoVis) (117 stars)
A lightweight, interactive, extensible and cross-platform framework for building, visualizing and analyzing digital reconstructions of neuronal morphology skeletons extracted from microscopy stacks. (Python, 2018-03-21, 2024-02-06)
495. [EnyaHermite/Picasso](https://github.com/EnyaHermite/Picasso) (117 stars)
Geometric deep learning for surface parsing from 3-D triangular meshes (Python, 2020-11-17, 2024-01-10)
496. [Gabaldonlab/redundans](https://github.com/Gabaldonlab/redundans) (117 stars)
Redundans is a pipeline that assists an assembly of heterozygous/polymorphic genomes. (C++, 2015-04-07, 2024-02-05)
497. [saezlab/decoupler-py](https://github.com/saezlab/decoupler-py) (117 stars)
Python package to perform enrichment analysis from omics data. (Python, 2021-11-16, 2024-02-05)
498. [jdidion/atropos](https://github.com/jdidion/atropos) (116 stars)
An NGS read trimming tool that is specific, sensitive, and speedy. (production) (Python, 2016-06-17, 2023-11-24)
499. [parklab/NGSCheckMate](https://github.com/parklab/NGSCheckMate) (116 stars)
Software program for checking sample matching for NGS data (Python, 2016-01-08, 2024-01-05)
500. [zkkli/PSAQ-ViT](https://github.com/zkkli/PSAQ-ViT) (116 stars)
[ECCV 2022] Patch Similarity Aware Data-Free Quantization for Vision Transformers (Python, 2022-07-17, 2024-01-21)
501. [Merck/BioPhi](https://github.com/Merck/BioPhi) (115 stars)
BioPhi is an open-source antibody design platform. It features methods for automated antibody humanization (Sapiens), humanness evaluation (OASis) and an interface for computer-assisted antibody sequence design. (Python, 2021-07-29, 2024-01-23)
502. [OpenGene/repaq](https://github.com/OpenGene/repaq) (115 stars)
A fast lossless FASTQ compressor with ultra-high compression ratio (C, 2018-07-20, 2024-02-03)
503. [cruizperez/MicrobeAnnotator](https://github.com/cruizperez/MicrobeAnnotator) (115 stars)
Pipeline for metabolic annotation of microbial genomes (Python, 2020-03-04, 2024-01-26)
504. [DiltheyLab/HLA-LA](https://github.com/DiltheyLab/HLA-LA) (114 stars)
Fast HLA type inference from whole-genome data (C++, 2017-03-14, 2024-02-07)
505. [ksahlin/strobealign](https://github.com/ksahlin/strobealign) (114 stars)
Aligns short reads using dynamic seed size with strobemers (C++, 2021-05-23, 2024-01-08)
506. [lppllppl920/EndoscopyDepthEstimation-Pytorch](https://github.com/lppllppl920/EndoscopyDepthEstimation-Pytorch) (114 stars)
Official Repo for the paper "Dense Depth Estimation in Monocular Endoscopy with Self-supervised Learning Methods" (TMI) (Python, 2018-12-01, 2024-01-27)
507. [yatisht/usher](https://github.com/yatisht/usher) (114 stars)
Ultrafast Sample Placement on Existing Trees (C++, 2020-09-16, 2024-02-02)
508. [XiaoleiLiuBio/G2P](https://github.com/XiaoleiLiuBio/G2P) (113 stars)
G2P: A Genome-Wide-Association-Study Simulation Tool for Genotype Simulation, Phenotype Simulation, and Power Evaluation (None, 2018-07-22, 2024-01-19)
509. [hammerlab/cytokit](https://github.com/hammerlab/cytokit) (113 stars)
Microscopy Image Cytometry Toolkit (Jupyter Notebook, 2018-04-17, 2024-01-25)
510. [oxpig/DeLinker](https://github.com/oxpig/DeLinker) (113 stars)
None (Python, 2019-11-04, 2024-02-04)
511. [spacegraphcats/spacegraphcats](https://github.com/spacegraphcats/spacegraphcats) (113 stars)
Indexing & querying large assembly graphs -- in space, no one can hear you miao! (Standard ML, 2016-05-06, 2024-01-24)
512. [wanderine/BROCCOLI](https://github.com/wanderine/BROCCOLI) (113 stars)
BROCCOLI: Software for Fast fMRI Analysis on Many-Core CPUs and GPUs (C++, 2013-04-30, 2024-01-31)
513. [wangzy22/P2P](https://github.com/wangzy22/P2P) (113 stars)
[NeurIPS 2022 Spotlight] P2P: Tuning Pre-trained Image Models for Point Cloud Analysis with Point-to-Pixel Prompting (Python, 2022-08-03, 2024-01-31)
514. [AndersenLab/VCF-kit](https://github.com/AndersenLab/VCF-kit) (112 stars)
VCF-kit: Assorted utilities for the variant call format (Python, 2014-07-16, 2024-02-01)
515. [ChaissonLab/LRA](https://github.com/ChaissonLab/LRA) (112 stars)
Long read aligner (C++, 2018-06-15, 2024-01-18)
516. [HuiZeng/Grid-Anchor-based-Image-Cropping-Pytorch](https://github.com/HuiZeng/Grid-Anchor-based-Image-Cropping-Pytorch) (112 stars)
PyTorch implementation of "Grid anchor based image cropping" (Python, 2019-09-18, 2024-02-01)
517. [OpenGene/GeneFuse](https://github.com/OpenGene/GeneFuse) (112 stars)
Gene fusion detection and visualization (C, 2017-01-20, 2024-02-02)
518. [biojava/biojava-tutorial](https://github.com/biojava/biojava-tutorial) (112 stars)
:book::books:An introductory tutorial for BioJava (Python, 2013-09-18, 2024-02-02)
519. [torognes/swarm](https://github.com/torognes/swarm) (112 stars)
A robust and fast clustering method for amplicon-based studies (C++, 2012-11-10, 2024-02-06)
520. [neuropoly/axondeepseg](https://github.com/neuropoly/axondeepseg) (111 stars)
Axon/Myelin segmentation using Deep Learning (Python, 2016-09-26, 2024-01-19)
521. [tudelft/cuSNN](https://github.com/tudelft/cuSNN) (111 stars)
Spiking Neural Networks in C++ with strong GPU acceleration through CUDA (Cuda, 2018-10-16, 2024-01-04)
522. [MRCIEU/PHESANT](https://github.com/MRCIEU/PHESANT) (110 stars)
PHESANT - PHEnome Scan ANalysis Tool (pheWAS, Mendelian randomisation (MR)-pheWAS etc.) in UK Biobank (R, 2016-10-03, 2024-01-19)
523. [lingluodlut/Att-ChemdNER](https://github.com/lingluodlut/Att-ChemdNER) (110 stars)
Att-ChemdNER (Python, 2017-07-18, 2024-01-16)
524. [JiekaiLab/scDIOR](https://github.com/JiekaiLab/scDIOR) (109 stars)
scDIOR: Single cell data IO softwaRe (HTML, 2021-01-16, 2024-02-05)
525. [jameshadfield/phandango](https://github.com/jameshadfield/phandango) (109 stars)
an interactive viewer for populations of bacterial genomes linked by a phylogeny (JavaScript, 2015-02-04, 2024-01-26)
526. [mcveanlab/mccortex](https://github.com/mcveanlab/mccortex) (109 stars)
De novo genome assembly and multisample variant calling (C, 2014-05-07, 2023-12-22)
527. [biocore/mmvec](https://github.com/biocore/mmvec) (108 stars)
Neural networks for microbe-metabolite interaction analysis (Python, 2018-05-27, 2024-01-30)
528. [dzhang32/ggtranscript](https://github.com/dzhang32/ggtranscript) (108 stars)
Visualizing transcript structure and annotation using ggplot2 (R, 2021-12-23, 2024-01-11)
529. [guigolab/ggsashimi](https://github.com/guigolab/ggsashimi) (108 stars)
Command-line tool for the visualization of splicing events across multiple samples (Python, 2017-06-14, 2024-01-22)
530. [lh3/fermikit](https://github.com/lh3/fermikit) (108 stars)
De novo assembly based variant calling pipeline for Illumina short reads (TeX, 2015-04-11, 2023-10-30)
531. [timolassmann/kalign](https://github.com/timolassmann/kalign) (108 stars)
A fast multiple sequence alignment program. (C, 2019-04-18, 2024-01-23)
532. [Xinglab/DARTS](https://github.com/Xinglab/DARTS) (107 stars)
Deep-learning Augmented RNA-seq analysis of Transcript Splicing (Python, 2018-02-10, 2024-01-02)
533. [juglab/EmbedSeg](https://github.com/juglab/EmbedSeg) (107 stars)
Code Implementation for EmbedSeg, an Instance Segmentation Method for Microscopy Images (Python, 2021-01-13, 2024-01-19)
534. [lykaust15/DeepSimulator](https://github.com/lykaust15/DeepSimulator) (107 stars)
The first deep learning based Nanopore simulator which can simulate the process of Nanopore sequencing. (C, 2017-12-07, 2023-09-15)
535. [mikevoets/jama16-retina-replication](https://github.com/mikevoets/jama16-retina-replication) (107 stars)
JAMA 2016; 316(22) Replication Study (Python, 2018-02-19, 2024-01-11)
536. [mnakanishi/TRCA-SSVEP](https://github.com/mnakanishi/TRCA-SSVEP) (107 stars)
Task-related component analysis (TRCA)-based algorithm for detecting steady-state visual evoked potentials (SSVEPs) toward a high-speed brain-computer interface (BCI). (MATLAB, 2017-12-23, 2024-01-29)
537. [molgenis/molgenis](https://github.com/molgenis/molgenis) (107 stars)
MOLGENIS - for scientific data: management, exploration, integration and analysis. (Java, 2013-02-04, 2024-02-04)
538. [sydney0zq/covid-19-detection](https://github.com/sydney0zq/covid-19-detection) (107 stars)
The implementation of "A Weakly-supervised Framework for COVID-19 Classification and Lesion Localization from Chest CT" (Python, 2020-03-07, 2024-01-04)
539. [ChristofferFlensburg/SuperFreq](https://github.com/ChristofferFlensburg/SuperFreq) (106 stars)
Analysis pipeline for cancer sequencing data (R, 2016-02-17, 2024-01-31)
540. [OpenGene/fastv](https://github.com/OpenGene/fastv) (106 stars)
An ultra-fast tool for identification of SARS-CoV-2 and other microbes from sequencing data. This tool can be used to detect viral infectious diseases, like COVID-19. (C++, 2020-03-26, 2023-12-15)
541. [OpenGene/gencore](https://github.com/OpenGene/gencore) (106 stars)
Generate duplex/single consensus reads to reduce sequencing noises and remove duplications (C++, 2018-05-03, 2024-01-03)
542. [OpenTreeOfLife/opentree](https://github.com/OpenTreeOfLife/opentree) (106 stars)
Opentree browsing and curation web site.  For overarching or cross-repo concerns, please see the 'germinator' repo. (JavaScript, 2013-02-18, 2023-10-13)
543. [QiaoLiuHit/LSOTB-TIR](https://github.com/QiaoLiuHit/LSOTB-TIR) (106 stars)
LSOTB-TIR: A Large-Scale High-Diversity Thermal Infrared Object Tracking Benchmark (ACM MM2020) (C, 2020-08-01, 2024-01-27)
544. [igvteam/igv-webapp](https://github.com/igvteam/igv-webapp) (106 stars)
IGV Web App (JavaScript, 2018-02-19, 2024-02-02)
545. [jiaweihe1996/GMTracker](https://github.com/jiaweihe1996/GMTracker) (106 stars)
Official PyTorch implementation of "Learnable Graph Matching: Incorporating Graph Partitioning with Deep Feature Learning for Multiple Object Tracking" (CVPR 2021). (Python, 2021-03-02, 2024-01-21)
546. [linzhi2013/MitoZ](https://github.com/linzhi2013/MitoZ) (106 stars)
MitoZ: A toolkit for assembly, annotation, and visualization of animal mitochondrial genomes (Shell, 2018-11-18, 2024-02-01)
547. [vanessajurtz/lasagne4bio](https://github.com/vanessajurtz/lasagne4bio) (106 stars)
None (Jupyter Notebook, 2016-09-20, 2024-01-04)
548. [xiaochuanle/MECAT](https://github.com/xiaochuanle/MECAT) (106 stars)
MECAT: an ultra-fast mapping, error correction and de novo assembly tool for single-molecule sequencing reads (C++, 2016-12-16, 2023-10-23)
549. [YaqiangCao/cLoops](https://github.com/YaqiangCao/cLoops) (105 stars)
Accurate and flexible loops calling tool for 3D genomic data. (Python, 2017-06-26, 2023-12-28)
550. [cansyl/DEEPscreen](https://github.com/cansyl/DEEPscreen) (105 stars)
DEEPScreen: Virtual Screening with Deep Convolutional Neural Networks Using Compound Images (Python, 2019-01-09, 2024-02-02)
551. [evolbioinfo/gotree](https://github.com/evolbioinfo/gotree) (105 stars)
Gotree is a set of command line tools and an API to manipulate phylogenetic trees. It is implemented in Go language. (Go, 2016-07-27, 2023-07-17)
552. [lh3/minimap](https://github.com/lh3/minimap) (105 stars)
This repo is DEPRECATED. Please use minimap2, the successor of minimap. (C, 2015-09-06, 2024-01-11)
553. [parklab/bamsnap](https://github.com/parklab/bamsnap) (105 stars)
None (HTML, 2018-06-04, 2023-12-14)
554. [zju3dv/CoLi-BA](https://github.com/zju3dv/CoLi-BA) (105 stars)
None (C++, 2022-07-29, 2024-02-01)
555. [JosephKJ/iOD](https://github.com/JosephKJ/iOD) (104 stars)
(TPAMI 2021) iOD: Incremental Object Detection via Meta-Learning (Python, 2021-10-19, 2024-01-17)
556. [PheWAS/PheWAS](https://github.com/PheWAS/PheWAS) (104 stars)
The PheWAS R package (R, 2014-07-14, 2024-02-06)
557. [morrislab/phylowgs](https://github.com/morrislab/phylowgs) (104 stars)
Application for inferring subclonal composition and evolution from whole-genome sequencing data. (Python, 2015-01-22, 2023-08-22)
558. [thiviyanT/torch-rgcn](https://github.com/thiviyanT/torch-rgcn) (104 stars)
A PyTorch implementation of the Relational Graph Convolutional Network (RGCN). (Python, 2020-05-19, 2024-01-30)
559. [yihongxu/transcenter](https://github.com/yihongxu/transcenter) (104 stars)
This is the official implementation of TransCenter (TPAMI). The code  and pretrained models are now available here: https://gitlab.inria.fr/yixu/TransCenter_official. (None, 2021-04-06, 2024-01-31)
560. [BaderLab/saber](https://github.com/BaderLab/saber) (103 stars)
Saber is a deep-learning based tool for information extraction in the biomedical domain. Pull requests are welcome! Note: this is a work in progress. Many things are broken, and the codebase is not stable. (Python, 2018-02-15, 2023-12-16)
561. [DeepRank/DeepRank-GNN](https://github.com/DeepRank/DeepRank-GNN) (103 stars)
Graph Network for protein-protein interface (Python, 2018-07-13, 2024-01-26)
562. [MASILab/SLANTbrainSeg](https://github.com/MASILab/SLANTbrainSeg) (103 stars)
Deep Whole Brain Segmentation Using SLANT Method (Python, 2018-05-23, 2024-01-07)
563. [bioinfomaticsCSU/deepsignal](https://github.com/bioinfomaticsCSU/deepsignal) (103 stars)
Detecting methylation using signal-level features from Nanopore sequencing reads (Python, 2018-12-11, 2024-01-29)
564. [broadinstitute/ml4h](https://github.com/broadinstitute/ml4h) (103 stars)
None (Jupyter Notebook, 2019-04-10, 2024-02-07)
565. [carmonalab/UCell](https://github.com/carmonalab/UCell) (103 stars)
An R package for single-cell gene signature scoring (R, 2021-04-06, 2024-02-05)
566. [hellloxiaotian/ADNet](https://github.com/hellloxiaotian/ADNet) (103 stars)
Attention-guided CNN for image denoising(Neural Networks,2020) (Python, 2020-01-10, 2024-02-04)
567. [thomas0809/MolScribe](https://github.com/thomas0809/MolScribe) (103 stars)
Robust Molecular Structure Recognition with Image-to-Graph Generation (Python, 2022-11-10, 2024-01-29)
568. [yanwu2014/swne](https://github.com/yanwu2014/swne) (103 stars)
Similarity Weighted Nonnegative Embedding (SWNE), a method for visualizing high dimensional datasets (R, 2018-01-03, 2024-01-07)
569. [RL4M/PCRLv2](https://github.com/RL4M/PCRLv2) (102 stars)
An official implementation of PCRLv2 (pre-training and fine-tuning code are included). (Python, 2022-12-14, 2023-12-09)
570. [RabadanLab/arcasHLA](https://github.com/RabadanLab/arcasHLA) (102 stars)
Fast and accurate in silico inference of HLA genotypes from RNA-seq (Python, 2018-10-29, 2023-11-15)
571. [csxmli2016/DMDNet](https://github.com/csxmli2016/DMDNet) (102 stars)
Learning Dual Memory Dictionaries for Blind Face Restoration (Python, 2021-08-11, 2024-01-29)
572. [csyben/PYRO-NN](https://github.com/csyben/PYRO-NN) (102 stars)
Python Reconstruction Operators in Neural Networks. High level python API for PYRO-NN-Layers (Python, 2019-03-12, 2024-01-04)
573. [pavancm/CONTRIQUE](https://github.com/pavancm/CONTRIQUE) (102 stars)
Official implementation for "Image Quality Assessment using Contrastive Learning" (Python, 2021-10-25, 2024-01-18)
574. [bvieth/powsimR](https://github.com/bvieth/powsimR) (101 stars)
Power analysis is essential to optimize the design of RNA-seq experiments and to assess and compare the power to detect differentially expressed genes. PowsimR is a flexible tool to simulate and evaluate differential expression from bulk and especially single-cell RNA-seq data making it suitable for a priori and posterior power analyses. (HTML, 2017-04-16, 2023-10-10)
575. [chrishah/MITObim](https://github.com/chrishah/MITObim) (101 stars)
MITObim - mitochondrial baiting and iterative mapping (Perl, 2012-10-09, 2023-09-28)
576. [cozygene/FEAST](https://github.com/cozygene/FEAST) (101 stars)
Fast expectation maximization for microbial source tracking (R, 2018-02-04, 2024-01-25)
577. [louiejtaylor/grabseqs](https://github.com/louiejtaylor/grabseqs) (101 stars)
A utility for easy downloading of reads from next-gen sequencing repositories like NCBI SRA (Python, 2018-10-23, 2024-01-29)
578. [zhxiaokang/RASflow](https://github.com/zhxiaokang/RASflow) (101 stars)
RNA-Seq analysis workflow (HTML, 2017-04-04, 2023-12-27)
579. [FePhyFoFum/phyx](https://github.com/FePhyFoFum/phyx) (100 stars)
phylogenetics tools for linux (and other mostly posix compliant) computers (C++, 2013-02-14, 2023-09-07)
580. [althonos/pyhmmer](https://github.com/althonos/pyhmmer) (100 stars)
Cython bindings and Python interface to HMMER3. (Cython, 2020-10-30, 2024-01-28)
581. [mabessa/F3DAS](https://github.com/mabessa/F3DAS) (100 stars)
Framework for Data-Driven Design & Analysis of Structures & Materials (F3DASM) (Python, 2019-09-16, 2024-02-01)
582. [oxpig/AbLang](https://github.com/oxpig/AbLang) (100 stars)
 AbLang: A language model for antibodies (Python, 2021-12-31, 2024-02-01)
583. [ratschlab/spladder](https://github.com/ratschlab/spladder) (100 stars)
Tool for the detection and quantification of alternative splicing events from RNA-Seq data. (Python, 2014-07-22, 2023-12-27)
584. [shubhamchandak94/SPRING](https://github.com/shubhamchandak94/SPRING) (100 stars)
FASTQ compression (C, 2018-04-03, 2024-02-06)
585. [szpiech/selscan](https://github.com/szpiech/selscan) (100 stars)
Haplotype based scans for selection (C, 2013-12-24, 2024-01-03)
586. [yangao07/abPOA](https://github.com/yangao07/abPOA) (100 stars)
abPOA: an SIMD-based C library for fast partial order alignment using adaptive band (C, 2018-02-11, 2024-02-05)
587. [CostaLab/reg-gen](https://github.com/CostaLab/reg-gen) (99 stars)
Regulatory Genomics Toolbox: Python library and set of tools for the integrative analysis of high throughput regulatory genomics data. (Python, 2015-06-17, 2024-01-22)
588. [SBRG/ssbio](https://github.com/SBRG/ssbio) (99 stars)
A Python framework for structural systems biology (Python, 2016-01-25, 2024-01-07)
589. [biopragmatics/bioregistry](https://github.com/biopragmatics/bioregistry) (99 stars)
ðŸ“® An integrative registry of biological databases, ontologies, and nomenclatures. (Python, 2020-12-08, 2024-02-01)
590. [hakyimlab/PrediXcan](https://github.com/hakyimlab/PrediXcan) (99 stars)
PrediXcan Project (Python, 2015-04-23, 2024-01-23)
591. [ncbi/SKESA](https://github.com/ncbi/SKESA) (99 stars)
SKESA assembler (C++, 2018-07-11, 2024-01-05)
592. [rrwick/perfect-bacterial-genome-tutorial](https://github.com/rrwick/perfect-bacterial-genome-tutorial) (99 stars)
None (Python, 2022-09-08, 2024-01-28)
593. [3DGenomes/tadbit](https://github.com/3DGenomes/tadbit) (98 stars)
TADbit is a complete Python library to deal with all steps to analyze, model and explore 3C-based data. With TADbit the user can map FASTQ files to obtain raw interaction binned matrices (Hi-C like matrices), normalize and correct interaction matrices, identify and compare the so-called Topologically Associating Domains (TADs), build 3D models from the interaction matrices, and finally, extract structural properties from the models. TADbit is complemented by TADkit for visualizing 3D models (Python, 2013-08-29, 2024-01-09)
594. [BioinfoMachineLearning/bio-diffusion](https://github.com/BioinfoMachineLearning/bio-diffusion) (98 stars)
A PyTorch hub of denoising diffusion probabilistic models designed to generate novel biological data (Python, 2023-02-09, 2024-01-30)
595. [SunPengChuan/wgdi](https://github.com/SunPengChuan/wgdi) (98 stars)
WGDI: A user-friendly toolkit for evolutionary analyses of whole-genome duplications and ancestral karyotypes (Python, 2018-08-30, 2024-01-29)
596. [brentp/duphold](https://github.com/brentp/duphold) (98 stars)
don't get DUP'ed or DEL'ed by your putative SVs. (Nim, 2018-09-07, 2024-01-22)
597. [verilylifesciences/genomewarp](https://github.com/verilylifesciences/genomewarp) (98 stars)
GenomeWarp translates genetic variants from one genome assembly version to another. (Java, 2016-08-23, 2023-12-29)
598. [whitead/molbloom](https://github.com/whitead/molbloom) (98 stars)
Molecular bloom filter tool (Python, 2022-10-27, 2024-01-19)
599. [zeeev/wham](https://github.com/zeeev/wham) (98 stars)
Structural variant detection and association testing (C++, 2013-05-06, 2024-01-18)
600. [zhjohnchan/M3AE](https://github.com/zhjohnchan/M3AE) (98 stars)
[MICCAI-2022] This is the official implementation of Multi-Modal Masked Autoencoders for Medical Vision-and-Language Pre-Training. (Python, 2022-07-12, 2024-01-30)
601. [HDCVLab/EDFace-Celeb-1M](https://github.com/HDCVLab/EDFace-Celeb-1M) (97 stars)
None (None, 2021-10-10, 2024-01-04)
602. [ZJULearning/SSG](https://github.com/ZJULearning/SSG) (97 stars)
code for satellite system graphs (C++, 2019-07-10, 2024-01-23)
603. [adamewing/methylartist](https://github.com/adamewing/methylartist) (97 stars)
Tools for plotting methylation data in various ways (Python, 2020-12-21, 2024-02-06)
604. [labsyspharm/ashlar](https://github.com/labsyspharm/ashlar) (97 stars)
ASHLAR: Alignment by Simultaneous Harmonization of Layer/Adjacency Registration (Python, 2017-05-15, 2024-02-02)
605. [openPupil/Open-PupilEXT](https://github.com/openPupil/Open-PupilEXT) (97 stars)
Official implementation of a free pupillometry platform called PupilEXT proposed in the article "PupilEXT: flexible open-source platform for high resolution pupillometry in vision research". (C++, 2020-11-11, 2024-01-29)
606. [rnabioco/clustifyr](https://github.com/rnabioco/clustifyr) (97 stars)
Infer cell types in scRNA-seq data using bulk RNA-seq or gene sets (R, 2018-06-21, 2024-01-08)
607. [vaquerizaslab/fanc](https://github.com/vaquerizaslab/fanc) (97 stars)
FAN-C: Framework for the ANalysis of C-like data (Python, 2020-02-03, 2024-01-30)
608. [DIAGNijmegen/StreamingCNN](https://github.com/DIAGNijmegen/StreamingCNN) (96 stars)
To train deep convolutional neural networks, the input data and the activations need to be kept in memory. Given the limited memory available in current GPUs, this limits the maximum dimensions of the input data. Here we demonstrate a method to train convolutional neural networks while holding only parts of the image in memory. (Jupyter Notebook, 2018-04-10, 2024-01-09)
609. [arseha/peakonly](https://github.com/arseha/peakonly) (96 stars)
None (Python, 2019-09-09, 2024-01-03)
610. [evoldoers/biomake](https://github.com/evoldoers/biomake) (96 stars)
GNU-Make-like utility for managing builds and complex workflows (Prolog, 2011-09-10, 2024-01-02)
611. [florisvb/PyNumDiff](https://github.com/florisvb/PyNumDiff) (96 stars)
Methods for numerical differentiation of noisy data in python (Python, 2018-11-29, 2024-02-05)
612. [gurkirt/road-dataset](https://github.com/gurkirt/road-dataset) (96 stars)
Download and pre-processing of ROAD dataset (Python, 2021-02-18, 2024-01-11)
613. [microsoft/frame-flow](https://github.com/microsoft/frame-flow) (96 stars)
Fast protein backbone generation with SE(3) flow matching. (Python, 2023-08-16, 2024-02-01)
614. [ycwu1997/MC-Net](https://github.com/ycwu1997/MC-Net) (96 stars)
Official Code for our MedIA paper "Mutual Consistency Learning for Semi-supervised Medical Image Segmentation" (Python, 2022-04-11, 2024-01-09)
615. [GATB/bcalm](https://github.com/GATB/bcalm) (95 stars)
compacted de Bruijn graph construction in low memory (Python, 2014-12-25, 2024-01-08)
616. [MathOnco/NeoPredPipe](https://github.com/MathOnco/NeoPredPipe) (95 stars)
Neoantigens prediction pipeline for multi- or single-region vcf files using ANNOVAR and netMHCpan. (Python, 2018-12-08, 2023-11-03)
617. [j3xugit/RaptorX-3DModeling](https://github.com/j3xugit/RaptorX-3DModeling) (95 stars)
Note that current version does not include search of very large metagenome data. For some proteins, metagenome data is important. We will update this as soon as possible. (C, 2020-09-12, 2024-02-01)
618. [jlianglab/Fixed-Point-GAN](https://github.com/jlianglab/Fixed-Point-GAN) (95 stars)
Official implementation of Fixed-Point GAN - ICCV 2019 (Python, 2019-10-22, 2024-01-31)
619. [abhi4ssj/few-shot-segmentation](https://github.com/abhi4ssj/few-shot-segmentation) (94 stars)
PyTorch implementation of 'Squeeze and Excite' Guided Few Shot Segmentation of Volumetric Scans (Python, 2018-10-31, 2024-01-17)
620. [grimmlab/MicrobiomeBestPracticeReview](https://github.com/grimmlab/MicrobiomeBestPracticeReview) (94 stars)
Current Challenges and Best Practice Protocols for Microbiome Analysis using Amplicon and Metagenomic Sequencing (Shell, 2018-11-16, 2024-01-19)
621. [hugheslab/snpgenie](https://github.com/hugheslab/snpgenie) (94 stars)
Program for estimating Ï€N/Ï€S, dN/dS, and other diversity measures from next-generation sequencing data (Perl, 2015-05-24, 2023-12-02)
622. [makism/dyconnmap](https://github.com/makism/dyconnmap) (94 stars)
A dynamic connectome mapping module in python. (Python, 2017-07-28, 2024-01-23)
623. [roblanf/sangeranalyseR](https://github.com/roblanf/sangeranalyseR) (94 stars)
functions to analyse sanger sequencing reads in R (R, 2015-12-21, 2024-02-03)
624. [bbci/wyrm](https://github.com/bbci/wyrm) (93 stars)
Python toolbox for Brain-Computer Interfacing (BCI) (Python, 2015-01-22, 2023-10-15)
625. [bytedance/TWIST](https://github.com/bytedance/TWIST) (93 stars)
Official codes: Self-Supervised Learning by Estimating Twin Class Distribution (Python, 2021-10-22, 2024-01-04)
626. [dd1github/DeepSMOTE](https://github.com/dd1github/DeepSMOTE) (93 stars)
Pytorch implementation of "DeepSMOTE: Fusing Deep Learning and SMOTE for Imbalanced Data". (Python, 2021-03-05, 2024-01-31)
627. [lh3/bgt](https://github.com/lh3/bgt) (93 stars)
Flexible genotype query among 30,000+ samples whole-genome (C, 2015-05-02, 2023-11-06)
628. [treangen/MetAMOS](https://github.com/treangen/MetAMOS) (93 stars)
A metagenomic and isolate assembly and analysis pipeline built with AMOS (Roff, 2011-07-16, 2023-10-25)
629. [BEETL/BEETL](https://github.com/BEETL/BEETL) (92 stars)
BEETL (C++, 2012-03-08, 2023-07-20)
630. [ZJUFanLab/scDeepSort](https://github.com/ZJUFanLab/scDeepSort) (92 stars)
Cell-type Annotation for Single-cell Transcriptomics using Deep Learning with a Weighted Graph Neural Network (Python, 2020-04-18, 2023-12-15)
631. [lppllppl920/SAGE-SLAM](https://github.com/lppllppl920/SAGE-SLAM) (92 stars)
Official repo for the paper "SAGE: SLAM with Appearance and Geometry Prior for Endoscopy" (ICRA 2022) (C++, 2022-02-06, 2024-01-23)
632. [nmquijada/tormes](https://github.com/nmquijada/tormes) (92 stars)
Making whole bacterial genome sequencing data analysis easy (Shell, 2018-11-27, 2023-12-31)
633. [pulimeng/DeepDrug3D](https://github.com/pulimeng/DeepDrug3D) (92 stars)
A convolutional neural network for classifying binding pockets based on spatial and chemical information extracted from the pockets. (Python, 2018-04-25, 2023-11-24)
634. [ziplab/SPViT](https://github.com/ziplab/SPViT) (92 stars)
[TPAMI 2024] This is the official repository for our paper: ''Pruning Self-attentions into Convolutional Layers in Single Path''. (Python, 2021-11-23, 2024-01-22)
635. [zstephens/neat-genreads](https://github.com/zstephens/neat-genreads) (92 stars)
NEAT read simulation tools (Python, 2016-02-01, 2024-01-22)
636. [AlexandrovLab/SigProfilerMatrixGenerator](https://github.com/AlexandrovLab/SigProfilerMatrixGenerator) (91 stars)
SigProfilerMatrixGenerator creates mutational matrices for all types of somatic mutations. It allows downsizing the generated mutations only to parts for the genome (e.g., exome or a custom BED file). The tool seamlessly integrates with other SigProfiler tools. (Python, 2018-08-27, 2023-12-09)
637. [BigDataBiology/SemiBin](https://github.com/BigDataBiology/SemiBin) (91 stars)
SemiBin: metagenomics binning with self-supervised deep learning (Python, 2020-11-18, 2024-02-05)
638. [Illumina/Pisces](https://github.com/Illumina/Pisces) (91 stars)
Somatic and germline variant caller for amplicon data.  Recommended caller for tumor-only workflows. (C#, 2015-12-17, 2023-12-15)
639. [MolecularAI/DockStream](https://github.com/MolecularAI/DockStream) (91 stars)
DockStream: A Docking Wrapper to Enhance De Novo Molecular Design (Python, 2021-08-02, 2024-01-06)
640. [RagnarGrootKoerkamp/astar-pairwise-aligner](https://github.com/RagnarGrootKoerkamp/astar-pairwise-aligner) (91 stars)
A pairwise sequence aligner written in Rust (Rust, 2021-12-05, 2024-02-01)
641. [ZJUFanLab/bulk2space](https://github.com/ZJUFanLab/bulk2space) (91 stars)
a spatial deconvolution method based on deep learning frameworks, which converts bulk transcriptomes into spatially resolved single-cell expression profiles (Jupyter Notebook, 2021-12-31, 2024-01-18)
642. [aistairc/DeepEventMine](https://github.com/aistairc/DeepEventMine) (91 stars)
DeepEventMine: End-to-end Neural Nested Event Extraction from Biomedical Texts (Python, 2019-11-26, 2024-01-04)
643. [hahnlab/CAFE5](https://github.com/hahnlab/CAFE5) (91 stars)
Version 5 of the CAFE phylogenetics software (C++, 2018-09-06, 2023-12-03)
644. [immunogenomics/symphony](https://github.com/immunogenomics/symphony) (91 stars)
Efficient and precise single-cell reference atlas mapping with Symphony (Jupyter Notebook, 2020-10-06, 2024-01-21)
645. [ropensci/nasapower](https://github.com/ropensci/nasapower) (91 stars)
API Client for NASA POWER Global Meteorology, Surface Solar Energy and Climatology in R (R, 2017-11-02, 2024-02-06)
646. [GXYM/TextPMs](https://github.com/GXYM/TextPMs) (90 stars)
Arbitrary Shape Text Detection via Segmentation with Probability Maps; accepted by TPAMI2022 (C++, 2020-06-01, 2024-01-27)
647. [caitiecollins/treeWAS](https://github.com/caitiecollins/treeWAS) (90 stars)
treeWAS: A Phylogenetic Tree-Based Tool for Genome-Wide Association Studies in Microbes (R, 2015-03-23, 2023-12-10)
648. [cytoscape/cyjShiny](https://github.com/cytoscape/cyjShiny) (90 stars)
An R/shiny widget for cytoscape.js (JavaScript, 2014-08-05, 2024-02-06)
649. [gaocong13/Projective-Spatial-Transformers](https://github.com/gaocong13/Projective-Spatial-Transformers) (90 stars)
Generalizing Spatial Transformers to Projective Geometry with Applications to 2D/3D Registration (Python, 2020-03-23, 2024-01-17)
650. [qubekit/QUBEKit](https://github.com/qubekit/QUBEKit) (90 stars)
  Quantum Mechanical Bespoke Force Field Derivation Toolkit (Python, 2019-04-10, 2024-01-27)
651. [PangzeCheung/Dual-task-Pose-Transformer-Network](https://github.com/PangzeCheung/Dual-task-Pose-Transformer-Network) (89 stars)
[CVPR 2022] Exploring Dual-task Correlation for Pose Guided Person Image Generation (Python, 2022-03-02, 2024-01-04)
652. [SysBioChalmers/RAVEN](https://github.com/SysBioChalmers/RAVEN) (89 stars)
The RAVEN Toolbox for genome scale model reconstruction, curation and analysis. (MATLAB, 2016-04-05, 2024-01-24)
653. [bcgsc/ARCS](https://github.com/bcgsc/ARCS) (89 stars)
ðŸŒˆScaffold genome sequence assemblies using linked or long read sequencing data (C++, 2016-06-06, 2024-01-26)
654. [jbohnslav/deepethogram](https://github.com/jbohnslav/deepethogram) (89 stars)
None (Python, 2020-08-18, 2024-01-22)
655. [mgalardini/pyseer](https://github.com/mgalardini/pyseer) (89 stars)
SEER, reimplemented in python ðŸðŸ”® (Python, 2017-09-22, 2024-01-27)
656. [microsoft/genomicsnotebook](https://github.com/microsoft/genomicsnotebook) (89 stars)
Jupyter Notebooks on Azure for Genomics Data Analysis (Jupyter Notebook, 2020-07-10, 2024-01-29)
657. [mr7495/COVID-CTset](https://github.com/mr7495/COVID-CTset) (89 stars)
Large Covid-19 CT scans dataset from the paper: https://doi.org/10.1016/j.bspc.2021.102588 (Python, 2020-06-01, 2023-11-30)
658. [nawrockie/vadr](https://github.com/nawrockie/vadr) (89 stars)
Viral Annotation DefineR: classification and annotation of viral sequences based on RefSeq annotation (Perl, 2015-08-10, 2023-12-05)
659. [ncbi/vadr](https://github.com/ncbi/vadr) (89 stars)
Viral Annotation DefineR: classification and annotation of viral sequences based on RefSeq annotation (Perl, 2015-08-10, 2023-12-05)
660. [theosanderson/taxonium](https://github.com/theosanderson/taxonium) (89 stars)
A tool for exploring very large trees in the browser (JavaScript, 2021-06-24, 2024-01-23)
661. [bcgsc/ntHash](https://github.com/bcgsc/ntHash) (88 stars)
Fast hash function for DNA/RNA sequences (C++, 2015-05-15, 2024-01-10)
662. [genomicsITER/NanoCLUST](https://github.com/genomicsITER/NanoCLUST) (88 stars)
NanoCLUST is an analysis pipeline for UMAP-based classification of amplicon-based full-length 16S rRNA nanopore reads (Nextflow, 2020-04-02, 2023-12-18)
663. [labsquare/cutevariant](https://github.com/labsquare/cutevariant) (88 stars)
A standalone and free application to explore genetics variations from VCF file (Python, 2018-11-02, 2024-02-01)
664. [mossmatters/HybPiper](https://github.com/mossmatters/HybPiper) (88 stars)
Recovering genes from targeted sequence capture data (Python, 2014-02-20, 2024-02-02)
665. [giesselmann/nanopype](https://github.com/giesselmann/nanopype) (87 stars)
Snakemake pipelines for nanopore sequencing data archiving and processing (Python, 2018-11-12, 2023-06-30)
666. [mozack/abra2](https://github.com/mozack/abra2) (87 stars)
ABRA2 (Java, 2016-08-25, 2024-01-17)
667. [CDDLeiden/DrugEx](https://github.com/CDDLeiden/DrugEx) (86 stars)
De Novo Drug Design with RNNs and Transformers (Jupyter Notebook, 2022-04-04, 2024-01-28)
668. [blab/nextflu](https://github.com/blab/nextflu) (86 stars)
Real-time tracking of influenza evolution (JavaScript, 2015-01-26, 2024-02-02)
669. [budach/pysster](https://github.com/budach/pysster) (86 stars)
pysster: Learning Sequence And Structure Motifs In Biological Sequences Using Convolutional Neural Networks (Python, 2017-12-02, 2024-02-01)
670. [jminnier/STARTapp](https://github.com/jminnier/STARTapp) (86 stars)
The START App: R Shiny Transcriptome Analysis Resource Tool (HTML, 2016-03-21, 2024-01-25)
671. [orbkit/orbkit](https://github.com/orbkit/orbkit) (86 stars)
A Toolbox for Post-Processing Quantum Chemical Wavefunction Data (Python, 2015-11-10, 2023-09-02)
672. [CogStack/SemEHR](https://github.com/CogStack/SemEHR) (85 stars)
Surfacing Semantic Data from Clinical Notes in Electronic Health Records for Tailored Care, Trial Recruitment and Clinical Research (JavaScript, 2016-10-14, 2023-11-20)
673. [Hzzone/ProPos](https://github.com/Hzzone/ProPos) (85 stars)
Self-Supervised Learning for Deep Clustering (TPAMI 2022) (Python, 2021-11-23, 2024-02-02)
674. [JLiangLab/DiRA](https://github.com/JLiangLab/DiRA) (85 stars)
Official PyTorch Implementation for DiRA: Discriminative, Restorative, and Adversarial Learning for Self-supervised Medical Image Analysis - CVPR 2022 (Python, 2022-04-24, 2024-02-02)
675. [biocore/deblur](https://github.com/biocore/deblur) (85 stars)
Deblur is a greedy deconvolution algorithm based on known read error profiles. (Python, 2015-01-20, 2023-12-27)
676. [mogvision/FFD](https://github.com/mogvision/FFD) (85 stars)
Fast-Feature-Detector (Python, 2020-09-11, 2024-01-31)
677. [mrzResearchArena/PyFeat](https://github.com/mrzResearchArena/PyFeat) (85 stars)
A Python-based Effective Feature Generation Tool from DNA, RNA, and Protein Sequences (Python, 2018-05-02, 2023-11-30)
678. [romanhaa/Cerebro](https://github.com/romanhaa/Cerebro) (85 stars)
Visualization of scRNA-seq data. (HTML, 2019-01-08, 2024-01-15)
679. [BUStools/bustools](https://github.com/BUStools/bustools) (84 stars)
Tools for working with BUS files (C, 2018-11-12, 2023-12-13)
680. [ChristopherWilks/megadepth](https://github.com/ChristopherWilks/megadepth) (84 stars)
BigWig and BAM utilities (C++, 2018-12-13, 2023-11-21)
681. [FastTrackOrg/FastTrack](https://github.com/FastTrackOrg/FastTrack) (84 stars)
FastTrack is a cross-platform application designed to track multiple objects in video recording. (C++, 2020-05-20, 2024-01-16)
682. [KatherLab/HIA](https://github.com/KatherLab/HIA) (84 stars)
Histopathology Image Analysis (Python, 2021-07-12, 2024-02-01)
683. [cpockrandt/genmap](https://github.com/cpockrandt/genmap) (84 stars)
GenMap - Fast and Exact Computation of Genome Mappability (C++, 2019-02-11, 2024-01-02)
684. [jodyphelan/TBProfiler](https://github.com/jodyphelan/TBProfiler) (84 stars)
Profiling tool for Mycobacterium tuberculosis to detect ressistance and strain type from WGS data (Python, 2016-12-27, 2024-02-03)
685. [llnl/fast](https://github.com/llnl/fast) (84 stars)
Fusion models for Atomic and molecular STructures (FAST) (Python, 2020-05-05, 2024-01-10)
686. [meren/illumina-utils](https://github.com/meren/illumina-utils) (84 stars)
A library and collection of scripts to work with Illumina paired-end data (for CASAVA 1.7+ pipeline). (Python, 2011-08-25, 2024-01-16)
687. [ncbi/MedCPT](https://github.com/ncbi/MedCPT) (84 stars)
Code for MedCPT, a model for zero-shot biomedical information retrieval. (Python, 2023-05-04, 2024-02-03)
688. [ToniCreswell/InvertingGAN](https://github.com/ToniCreswell/InvertingGAN) (83 stars)
Invert a pre-trained GAN model (includes code for training a GAN on celebA) (Jupyter Notebook, 2018-01-11, 2024-01-04)
689. [bergmanlab/mcclintock](https://github.com/bergmanlab/mcclintock) (83 stars)
Meta-pipeline to identify transposable element insertions using next generation sequencing data (Python, 2014-02-10, 2023-11-05)
690. [bigchem/transformer-cnn](https://github.com/bigchem/transformer-cnn) (83 stars)
Transformer CNN for QSAR/QSPR modelling (Python, 2019-08-20, 2024-02-03)
691. [ccipd/MRQy](https://github.com/ccipd/MRQy) (83 stars)
MRQy is a quality assurance and checking tool for quantitative assessment of magnetic resonance imaging (MRI) data. (JavaScript, 2019-10-08, 2024-01-27)
692. [cdcgov/microbetrace](https://github.com/cdcgov/microbetrace) (83 stars)
The Visualization Multitool for Molecular Epidemiology and Bioinformatics (TypeScript, 2018-09-21, 2024-02-03)
693. [htseq/htseq](https://github.com/htseq/htseq) (83 stars)
HTSeq is a Python library to facilitate processing and analysis of data from  high-throughput sequencing (HTS) experiments. (Python, 2016-11-11, 2024-02-05)
694. [parashardhapola/scarf](https://github.com/parashardhapola/scarf) (83 stars)
Toolkit for highly memory efficient analysis of single-cell RNA-Seq, scATAC-Seq and CITE-Seq data. Analyze atlas scale datasets with millions of cells on laptop. (Python, 2020-05-07, 2024-01-21)
695. [uwrit/leaf](https://github.com/uwrit/leaf) (83 stars)
Leaf Clinical Data Explorer (TypeScript, 2019-03-28, 2024-02-06)
696. [MannLabs/alphapeptdeep](https://github.com/MannLabs/alphapeptdeep) (82 stars)
Deep learning framework for proteomics (Jupyter Notebook, 2021-10-01, 2024-01-15)
697. [Vini2/GraphBin](https://github.com/Vini2/GraphBin) (82 stars)
âœ¨ðŸ§¬ Refined binning of metagenomic contigs using assembly graphs (Python, 2019-02-18, 2024-01-24)
698. [bcgsc/RNA-Bloom](https://github.com/bcgsc/RNA-Bloom) (82 stars)
:hibiscus: reference-free transcriptome assembly for short and long reads (Java, 2018-06-30, 2024-01-26)
699. [catavallejos/BASiCS](https://github.com/catavallejos/BASiCS) (82 stars)
BASiCS: Bayesian Analysis of Single-Cell Sequencing Data. This is an unstable experimental version. Please see http://bioconductor.org/packages/BASiCS/ for the official release version (R, 2015-05-19, 2024-01-31)
700. [lanagarmire/DeepImpute](https://github.com/lanagarmire/DeepImpute) (82 stars)
An accurate and efficient deep learning method for single-cell RNA-seq data imputation (Python, 2018-05-25, 2023-12-04)
701. [openbiox/UCSCXenaShiny](https://github.com/openbiox/UCSCXenaShiny) (82 stars)
ðŸ“Š An R package for interactively exploring UCSC Xena https://xenabrowser.net/datapages/ (R, 2019-03-17, 2024-01-26)
702. [shervinmin/DeepCovid](https://github.com/shervinmin/DeepCovid) (82 stars)
None (Python, 2020-04-20, 2024-01-04)
703. [Lixy1997/Group-WSSS](https://github.com/Lixy1997/Group-WSSS) (81 stars)
Group-Wise Semantic Mining for Weakly Supervised Semantic Segmentation (Python, 2020-12-02, 2023-12-11)
704. [ShuweiShao/AF-SfMLearner](https://github.com/ShuweiShao/AF-SfMLearner) (81 stars)
[MedIA2022 & ICRA2021] Self-Supervised Monocular Depth and Ego-Motion Estimation in Endoscopy: Appearance Flow to the Rescue (Python, 2021-10-07, 2024-02-01)
705. [modeloriented/survex](https://github.com/modeloriented/survex) (81 stars)
Explainable Machine Learning in Survival Analysis (R, 2022-08-31, 2024-02-03)
706. [nunofonseca/irap](https://github.com/nunofonseca/irap) (81 stars)
integrated RNA-seq Analysis Pipeline (R, 2015-03-19, 2024-01-22)
707. [parklab/xTea](https://github.com/parklab/xTea) (81 stars)
Comprehensive TE insertion identification with WGS/WES data from multiple sequencing technics (Python, 2018-07-12, 2024-01-23)
708. [splatlab/squeakr](https://github.com/splatlab/squeakr) (81 stars)
Squeakr: An Exact and Approximate k -mer Counting System (C++, 2017-02-04, 2024-02-06)
709. [vanallenlab/comut](https://github.com/vanallenlab/comut) (81 stars)
CoMut is a Python library for visualizing genomic and phenotypic information via comutation plots (Python, 2020-03-11, 2024-02-05)
710. [0liliulei/Mem3D](https://github.com/0liliulei/Mem3D) (80 stars)
None (Python, 2021-06-18, 2024-01-15)
711. [CenterForMedicalGeneticsGhent/WisecondorX](https://github.com/CenterForMedicalGeneticsGhent/WisecondorX) (80 stars)
WisecondorX â€” An evolved WISECONDOR (Python, 2018-05-15, 2024-02-05)
712. [Superzchen/iLearnPlus](https://github.com/Superzchen/iLearnPlus) (80 stars)
iLearnPlus is the first machine-learning platform with both graphical- and web-based user interface that enables the construction of automated machine-learning pipelines for computational analysis and predictions using nucleic acid and protein sequences. (Python, 2020-10-22, 2024-01-11)
713. [balbasty/nitorch](https://github.com/balbasty/nitorch) (80 stars)
Neuroimaging in PyTorch (Python, 2020-04-17, 2024-01-04)
714. [carmonalab/scGate](https://github.com/carmonalab/scGate) (80 stars)
marker-based purification of cell types from single-cell RNA-seq datasets (R, 2021-08-11, 2024-02-05)
715. [fwhelan/coinfinder](https://github.com/fwhelan/coinfinder) (80 stars)
A tool for the identification of coincident (associating and dissociating) genes in pangenomes. (C++, 2019-08-27, 2024-01-25)
716. [gamcil/cblaster](https://github.com/gamcil/cblaster) (80 stars)
Find clustered hits from a BLAST search (HTML, 2019-07-18, 2024-01-07)
717. [gui11aume/starcode](https://github.com/gui11aume/starcode) (80 stars)
All pairs search and sequence clustering (C, 2013-06-11, 2023-10-23)
718. [ramiromagno/gwasrapidd](https://github.com/ramiromagno/gwasrapidd) (80 stars)
gwasrapidd: an R package to query, download and wrangle GWAS Catalog data (R, 2018-10-31, 2024-01-26)
719. [vanderschaarlab/AutoPrognosis](https://github.com/vanderschaarlab/AutoPrognosis) (80 stars)
A system for automating the design of predictive modeling pipelines tailored for clinical prognosis. (Python, 2022-01-03, 2024-02-01)
720. [BioJulia/BioStructures](https://github.com/BioJulia/BioStructures) (79 stars)
A Julia package to read, write and manipulate macromolecular structures (particularly proteins) (Julia, 2017-08-16, 2024-02-02)
721. [citiususc/veryfasttree](https://github.com/citiususc/veryfasttree) (79 stars)
Efficient phylogenetic tree inference for massive taxonomic datasets (C++, 2019-12-02, 2024-01-31)
722. [nstrodt/UDSMProt](https://github.com/nstrodt/UDSMProt) (79 stars)
Protein sequence classification with self-supervised pretraining (Python, 2019-09-02, 2024-01-04)
723. [smirarab/pasta](https://github.com/smirarab/pasta) (79 stars)
PASTA (Practical Alignment using SATe and Transitivity) (Python, 2013-10-14, 2023-12-20)
724. [splatlab/mantis](https://github.com/splatlab/mantis) (79 stars)
Mantis: A Fast, Small, and Exact Large-Scale Sequence-Search Index (C++, 2017-11-10, 2024-01-02)
725. [xie186/ViewBS](https://github.com/xie186/ViewBS) (79 stars)
ViewBS - a powerful toolkit for visualization of high-throughput bisulfite sequencing data (Perl, 2016-07-07, 2024-01-29)
726. [CoSMoMVPA/CoSMoMVPA](https://github.com/CoSMoMVPA/CoSMoMVPA) (78 stars)
A lightweight multivariate pattern analysis (MVPA) toolbox in Matlab / Octave (MATLAB, 2013-08-27, 2023-10-19)
727. [DlutMedimgGroup/AnatomySketch-Software](https://github.com/DlutMedimgGroup/AnatomySketch-Software) (78 stars)
A medical image segmentation software supporting user interventions and user-defined funtion extention (C++, 2019-08-06, 2024-02-01)
728. [bcgsc/ntjoin](https://github.com/bcgsc/ntjoin) (78 stars)
ðŸ”—Genome assembly scaffolder using minimizer graphs (Python, 2019-06-04, 2024-02-02)
729. [glennhickey/progressiveCactus](https://github.com/glennhickey/progressiveCactus) (78 stars)
Distribution package for the Prgressive Cactus multiple genome aligner.  Dependencies are linked as submodules (Python, 2012-09-12, 2024-02-01)
730. [lseventeen/FR-UNet](https://github.com/lseventeen/FR-UNet) (78 stars)
[JBHI2021] Full-Resolution Network and Dual-ThresholdIteration for Retinal Vessel and CoronaryAngiograph Segmentation (Python, 2021-08-22, 2024-01-23)
731. [pquochuy/SeqSleepNet](https://github.com/pquochuy/SeqSleepNet) (78 stars)
SeqSleepNet: End-to-End Hierarchical Recurrent Neural Network for Sequence-to-Sequence Automatic Sleep Staging (Python, 2018-09-28, 2024-02-01)
732. [wdecoster/methplotlib](https://github.com/wdecoster/methplotlib) (78 stars)
Plotting tools for nanopore methylation data (Python, 2018-10-31, 2023-11-13)
733. [EtienneCmb/tensorpac](https://github.com/EtienneCmb/tensorpac) (77 stars)
Phase-Amplitude Coupling under Python (Python, 2017-06-04, 2024-02-06)
734. [KeenThera/SECSE](https://github.com/KeenThera/SECSE) (77 stars)
Systemic Evolutionary Chemical Space Exploration for Drug Discovery (Python, 2021-12-05, 2024-02-01)
735. [MICCoM/SSAGES-public](https://github.com/MICCoM/SSAGES-public) (77 stars)
Software Suite for Advanced General Ensemble Simulations (C++, 2016-09-30, 2023-12-12)
736. [MeHelmy/princess](https://github.com/MeHelmy/princess) (77 stars)
None (Python, 2019-04-07, 2024-01-25)
737. [PayneLab/cptac](https://github.com/PayneLab/cptac) (77 stars)
Python packaging for CPTAC data (Jupyter Notebook, 2018-09-18, 2024-01-29)
738. [Snowfallingplum/SSAT](https://github.com/Snowfallingplum/SSAT) (77 stars)
[AAAI 2022] This is the official  pytorch code for SSAT: A Symmetric Semantic-Aware Transformer Network for Makeup Transfer and Removal (Python, 2021-12-07, 2024-02-06)
739. [Vaa3D/vaa3d](https://github.com/Vaa3D/vaa3d) (77 stars)
www.vaa3d.org (None, 2015-03-30, 2024-01-19)
740. [genome/gms](https://github.com/genome/gms) (77 stars)
The Genome Modeling System installer (Perl, 2013-08-28, 2023-08-23)
741. [jermp/sshash](https://github.com/jermp/sshash) (77 stars)
A compressed, associative, exact, and weighted dictionary for k-mers. (C++, 2022-01-13, 2024-01-04)
742. [mdsufz/MuDoGeR](https://github.com/mdsufz/MuDoGeR) (77 stars)
MuDoGeR makes the recovery of genomes from prokaryotes, viruses, and eukaryotes from metagenomes easy. (Shell, 2020-09-01, 2024-01-22)
743. [moiexpositoalonsolab/grenepipe](https://github.com/moiexpositoalonsolab/grenepipe) (77 stars)
A flexible, scalable, and reproducible pipeline to automate variant calling from sequence reads. (Python, 2020-05-06, 2024-01-23)
744. [stephens999/ashr](https://github.com/stephens999/ashr) (77 stars)
An R package for adaptive shrinkage (R, 2015-02-27, 2024-02-06)
745. [yjx1217/simuG](https://github.com/yjx1217/simuG) (77 stars)
simuG: a general-purpose genome simulator (Perl, 2018-12-04, 2024-01-22)
746. [arzwa/wgd](https://github.com/arzwa/wgd) (76 stars)
Python package and CLI for whole-genome duplication related analyses. This package is deprecated in favor of https://github.com/heche-psb/wgd. (Python, 2018-01-17, 2024-02-05)
747. [calico/solo](https://github.com/calico/solo) (76 stars)
software to detect doublets (Python, 2019-02-28, 2023-11-28)
748. [crazyhottommy/scclusteval](https://github.com/crazyhottommy/scclusteval) (76 stars)
Single Cell Cluster Evaluation (R, 2018-11-30, 2023-12-04)
749. [futianfan/clinical-trial-outcome-prediction](https://github.com/futianfan/clinical-trial-outcome-prediction) (76 stars)
benchmark dataset and Deep learning method (Hierarchical Interaction Network, HINT) for clinical trial approval probability prediction, published in Cell Patterns 2022. (Python, 2021-02-18, 2024-01-25)
750. [s312569/clj-biosequence](https://github.com/s312569/clj-biosequence) (76 stars)
A Clojure library designed to make the manipulation of biological sequence data easier. (Clojure, 2013-07-10, 2023-09-25)
751. [scvae/scvae](https://github.com/scvae/scvae) (76 stars)
Deep learning for single-cell transcript counts (Python, 2017-02-13, 2024-01-23)
752. [seqan/flexbar](https://github.com/seqan/flexbar) (76 stars)
flexible barcode and adapter removal (C++, 2015-09-16, 2023-09-28)
753. [sjspielman/pyvolve](https://github.com/sjspielman/pyvolve) (76 stars)
Python library to simulate evolutionary sequence data (Python, 2014-03-09, 2023-11-09)
754. [zzstefan/BrainNetClass](https://github.com/zzstefan/BrainNetClass) (76 stars)
None (MATLAB, 2019-05-01, 2024-01-14)
755. [bcgsc/ntCard](https://github.com/bcgsc/ntCard) (75 stars)
Estimating k-mer coverage histogram of genomics data (C++, 2016-05-30, 2024-01-23)
756. [oxpig/ImmuneBuilder](https://github.com/oxpig/ImmuneBuilder) (75 stars)
Predict the structure of immune receptor proteins (Jupyter Notebook, 2022-11-05, 2024-01-23)
757. [smirarab/sepp](https://github.com/smirarab/sepp) (75 stars)
Ensemble of HMM methods (SEPP, TIPP, UPP) (Python, 2012-07-05, 2023-11-07)
758. [JLiangLab/SemanticGenesis](https://github.com/JLiangLab/SemanticGenesis) (74 stars)
Official Keras & PyTorch Implementation and Pre-trained Models for Semantic Genesis - MICCAI 2020 (Python, 2020-07-10, 2024-01-17)
759. [arthurpessa/ordpy](https://github.com/arthurpessa/ordpy) (74 stars)
A Python package for data analysis with permutation entropy and ordinal network methods. (Python, 2020-08-06, 2024-01-12)
760. [asad/ReactionDecoder](https://github.com/asad/ReactionDecoder) (74 stars)
Reaction Decoder Tool (RDT) - Atom Atom Mapping Tool (Java, 2015-11-02, 2024-01-28)
761. [lh3/fermi](https://github.com/lh3/fermi) (74 stars)
A WGS de novo assembler based on the FMD-index for large genomes (C, 2012-01-06, 2023-11-09)
762. [tfzhou/MG-HumanParsing](https://github.com/tfzhou/MG-HumanParsing) (74 stars)
Differentiable Multi-Granularity Human Representation Learning for Instance-Aware Human Semantic Parsing (Python, 2021-03-04, 2024-01-04)
763. [NJUyued/MutexMatch4SSL](https://github.com/NJUyued/MutexMatch4SSL) (73 stars)
"MutexMatch: Semi-Supervised Learning with Mutex-Based Consistency Regularization" by Yue Duan (TNNLS) (Python, 2022-03-22, 2024-01-15)
764. [Oshlack/JAFFA](https://github.com/Oshlack/JAFFA) (73 stars)
JAFFA is a multi-step pipeline that takes either raw RNA-Seq reads, or pre-assembled transcripts, then searches for gene fusions (C++, 2015-04-17, 2023-12-21)
765. [cecret3350/DEA-Net](https://github.com/cecret3350/DEA-Net) (73 stars)
[IEEE TIP 2024] DEA-Net: Single image dehazing based on detail-enhanced convolution and content-guided attention (Python, 2023-01-10, 2024-02-04)
766. [eldariont/svim-asm](https://github.com/eldariont/svim-asm) (73 stars)
Structural Variant Identification Method using Genome Assemblies (Python, 2019-08-07, 2024-01-19)
767. [gevaertlab/MultimodalPrognosis](https://github.com/gevaertlab/MultimodalPrognosis) (73 stars)
Deep Learning with Multimodal Representation for Pancancer Prognosis Prediction (Python, 2018-11-19, 2024-01-19)
768. [grailbio/bio](https://github.com/grailbio/bio) (73 stars)
Bioinformatic infrastructure libraries (Go, 2018-03-26, 2024-01-23)
769. [kamwoh/DeepIPR](https://github.com/kamwoh/DeepIPR) (73 stars)
[NeurIPS 2019] This is the code repo of our novel passport-based DNN ownership verification schemes, i.e. we embed passport layer into various deep learning architectures (e.g. AlexNet, ResNet) for Intellectual Property Right (IPR) protection. (Python, 2019-09-16, 2024-01-12)
770. [Cassie94/LSTD](https://github.com/Cassie94/LSTD) (72 stars)
An object detection architecture combines the advantage of SSD and Faster-RCNN. (C++, 2017-09-06, 2023-12-28)
771. [HRGV/phyloFlash](https://github.com/HRGV/phyloFlash) (72 stars)
phyloFlash - A pipeline to rapidly reconstruct the SSU rRNAs and explore phylogenetic composition of an illumina (meta)genomic dataset. (Perl, 2015-02-17, 2024-02-06)
772. [diazlab/CONICS](https://github.com/diazlab/CONICS) (72 stars)
CONICS: COpy-Number analysis In single-Cell RNA-Sequencing (R, 2017-02-26, 2023-11-01)
773. [jokergoo/rGREAT](https://github.com/jokergoo/rGREAT) (72 stars)
GREAT Analysis - Functional Enrichment on Genomic Regions (R, 2014-12-24, 2024-01-08)
774. [jwalabroad/VariantBam](https://github.com/jwalabroad/VariantBam) (72 stars)
Filtering and profiling of next-generational sequencing data using region-specific rules (Makefile, 2015-05-21, 2023-11-15)
775. [netZoo/netZooPy](https://github.com/netZoo/netZooPy) (72 stars)
netZooPy is a network biology package implemented in Python. (Jupyter Notebook, 2019-06-13, 2024-02-06)
776. [rezacsedu/XAI-for-bioinformatics](https://github.com/rezacsedu/XAI-for-bioinformatics) (72 stars)
Explainable AI for Bioinformatics (Jupyter Notebook, 2022-11-08, 2024-01-12)
777. [SaskiaFreytag/schex](https://github.com/SaskiaFreytag/schex) (71 stars)
None (R, 2019-06-25, 2024-01-30)
778. [SsGood/MMGL](https://github.com/SsGood/MMGL) (71 stars)
Multi-modal Graph learning for Disease Prediction (IEEE Trans. on Medical imaging, TMI2022) (Jupyter Notebook, 2022-01-01, 2024-01-17)
779. [TinkerTools/tinker-hp](https://github.com/TinkerTools/tinker-hp) (71 stars)
Tinker-HP: High-Performance Massively Parallel Evolution of Tinker on CPUs & GPUs (C, 2018-06-12, 2024-01-24)
780. [changwn/scFEA](https://github.com/changwn/scFEA) (71 stars)
single cell Flux Estimation Analysis (scFEA) Try the below web server! (Jupyter Notebook, 2020-06-14, 2024-01-11)
781. [compgenome365/TCGA-Assembler-2](https://github.com/compgenome365/TCGA-Assembler-2) (71 stars)
TCGA-Assembler 2 (R, 2017-12-05, 2023-12-18)
782. [gurkirt/3D-RetinaNet](https://github.com/gurkirt/3D-RetinaNet) (71 stars)
3D-RetinaNet a baseline models on ROAD dataset (Python, 2020-05-10, 2024-01-04)
783. [twolinin/LongPhase](https://github.com/twolinin/LongPhase) (71 stars)
None (C, 2021-09-08, 2024-02-02)
784. [xavierdidelot/BactDating](https://github.com/xavierdidelot/BactDating) (71 stars)
Bayesian inference of ancestral dates on bacterial phylogenetic trees (R, 2017-09-22, 2024-01-08)
785. [ytZhang99/CF-Net](https://github.com/ytZhang99/CF-Net) (71 stars)
Official repository of "Deep Coupled Feedback Network for Joint Exposure Fusion and Image Super-Resolution" (Python, 2021-01-07, 2023-12-20)
786. [zengqunzhao/MA-Net](https://github.com/zengqunzhao/MA-Net) (71 stars)
[TIP'21] Learning Deep Global Multi-scale and Local Attention Features for Facial Expression Recognition in the Wild (Python, 2021-01-17, 2024-01-17)
787. [BioEasy/EasyCodeML](https://github.com/BioEasy/EasyCodeML) (70 stars)
 EasyCodeML: A visual tool for analysis of selection using CodeML (C, 2016-08-06, 2023-11-27)
788. [FangShancheng/ABINet-PP](https://github.com/FangShancheng/ABINet-PP) (70 stars)
ABINet++: Autonomous, Bidirectional and Iterative Language Modeling for Scene Text Spotting (Python, 2022-01-18, 2024-01-16)
789. [FangpingWan/NeoDTI](https://github.com/FangpingWan/NeoDTI) (70 stars)
NeoDTI: Neural integration of neighbor information from a heterogeneous network for discovering new drug-target interactions (Python, 2018-03-26, 2024-01-16)
790. [PapenfussLab/bionix](https://github.com/PapenfussLab/bionix) (70 stars)
Functional highly reproducible bioinformatics pipelines (Nix, 2018-08-31, 2024-01-31)
791. [bigbio/proteomics-metadata-standard](https://github.com/bigbio/proteomics-metadata-standard) (70 stars)
The Proteomics Experimental Design file format: Standard for experimental design annotation (Python, 2018-06-12, 2024-01-31)
792. [chentao1999/MedicalRelationExtraction](https://github.com/chentao1999/MedicalRelationExtraction) (70 stars)
The depository support training and testing BERT-CNN model on three medical relation extraction corpora: BioCreative V CDR task corpus, traditional Chinese medicine  literature corpus, and i2b2 temporal relation corpus. (Python, 2019-03-10, 2024-01-28)
793. [cosconatilab/PyRMD](https://github.com/cosconatilab/PyRMD) (70 stars)
AI-powered Virtual Screening (Python, 2021-03-25, 2024-02-07)
794. [ddarriba/modeltest](https://github.com/ddarriba/modeltest) (70 stars)
Best-fit model selection (C++, 2015-08-13, 2023-12-04)
795. [frankvogt/vcf2gwas](https://github.com/frankvogt/vcf2gwas) (70 stars)
Python API for comprehensive GWAS analysis using GEMMA (Python, 2021-05-10, 2024-02-07)
796. [jltsiren/gcsa2](https://github.com/jltsiren/gcsa2) (70 stars)
BWT-based index for graphs (C++, 2015-03-12, 2024-01-02)
797. [lh3/dna-nn](https://github.com/lh3/dna-nn) (70 stars)
Model and predict short DNA sequence features with neural networks (C, 2018-12-16, 2023-11-15)
798. [lmmpf/PyAutoFEP](https://github.com/lmmpf/PyAutoFEP) (70 stars)
PyAutoFEP: an automated FEP workflow for GROMACS integrating enhanced sampling methods (Python, 2021-02-12, 2024-02-06)
799. [mateidavid/nanocall](https://github.com/mateidavid/nanocall) (70 stars)
An Oxford Nanopore Basecaller (C++, 2016-03-28, 2023-03-22)
800. [montilab/hypeR](https://github.com/montilab/hypeR) (70 stars)
An R Package for Geneset Enrichment Workflows (R, 2018-12-17, 2024-01-30)
801. [xmengli999/CANet](https://github.com/xmengli999/CANet) (70 stars)
TMI 19: CANet: Cross-disease Attention Network for Joint Diabetic Retinopathy and Diabetic Macular Edema Grading (Python, 2019-11-03, 2024-01-11)
802. [zyjwuyan/BBS-Net](https://github.com/zyjwuyan/BBS-Net) (70 stars)
PyTorch implementation of the paper 'BBS-Net: RGB-D Salient Object Detection with a Bifurcated Backbone Strategy Network, ECCV2020' (Python, 2020-06-03, 2024-01-18)
803. [GfellerLab/EPIC](https://github.com/GfellerLab/EPIC) (69 stars)
Repository for the R package EPIC, to Estimate the Proportion of Immune and Cancer cells from bulk gene expression data. (R, 2017-01-18, 2023-07-17)
804. [MannLabs/alphatims](https://github.com/MannLabs/alphatims) (69 stars)
An open-source Python package for efficient accession and visualization of Bruker TimsTOF raw data from the Mann Labs at the Max Planck Institute of Biochemistry. (Python, 2020-11-07, 2024-01-09)
805. [RemiAllio/MitoFinder](https://github.com/RemiAllio/MitoFinder) (69 stars)
MitoFinder: efficient automated large-scale extraction of mitogenomic data from high throughput sequencing data (Python, 2019-07-05, 2024-01-30)
806. [Xianhang/EDSC-pytorch](https://github.com/Xianhang/EDSC-pytorch) (69 stars)
Code for Multiple Video Frame Interpolation via Enhanced Deformable Separable Convolution (Python, 2020-06-14, 2024-01-04)
807. [boreshkinai/delta-interpolator](https://github.com/boreshkinai/delta-interpolator) (69 stars)
None (Python, 2022-01-18, 2024-01-26)
808. [chainsawriot/rang](https://github.com/chainsawriot/rang) (69 stars)
ðŸ¶ (Re)constructing R computational environments (R, 2023-01-14, 2024-01-05)
809. [mozack/abra](https://github.com/mozack/abra) (69 stars)
Assembly Based ReAligner (Java, 2012-11-07, 2024-01-12)
810. [tderrien/FEELnc](https://github.com/tderrien/FEELnc) (69 stars)
FEELnc : FlExible Extraction of LncRNA (Perl, 2015-01-30, 2023-12-05)
811. [wulalago/DAF3D](https://github.com/wulalago/DAF3D) (69 stars)
Deep Attentive Features for Prostate Segmentation in 3D Transrectal Ultrasound (Python, 2019-04-21, 2024-01-12)
812. [yukiteruono/pbsim2](https://github.com/yukiteruono/pbsim2) (69 stars)
PBSIM2: a simulator for long read sequencers with a novel generative model of quality scores (C++, 2020-06-20, 2024-01-02)
813. [zuoxinian/CCS](https://github.com/zuoxinian/CCS) (69 stars)
Connectome Computation System (MATLAB, 2014-10-28, 2024-01-31)
814. [Liuxg16/GeoPPI](https://github.com/Liuxg16/GeoPPI) (68 stars)
None (Python, 2021-05-09, 2024-01-23)
815. [MaterialsDiscovery/PyChemia](https://github.com/MaterialsDiscovery/PyChemia) (68 stars)
Python Materials Discovery Framework (Python, 2014-05-14, 2023-12-31)
816. [Phlya/coolpuppy](https://github.com/Phlya/coolpuppy) (68 stars)
A versatile tool to perform pile-up analysis on Hi-C data in .cool format. (Python, 2018-09-03, 2024-01-29)
817. [ShellyCoder/cellcall](https://github.com/ShellyCoder/cellcall) (68 stars)
inferring cell-cell communication from scRNA-seq of ligand-receptor (R, 2020-12-11, 2024-02-03)
818. [canlab/MediationToolbox](https://github.com/canlab/MediationToolbox) (68 stars)
Single-level and multi-level mediation analyses for any kind of data, with bootstrap-based significance testing. Neuroimaging-oriented functions allow for mediation effect parametric mapping (mapping of mediation effects across the brain) and multivariate mediation. (HTML, 2015-02-24, 2024-01-31)
819. [citiususc/SparkBWA](https://github.com/citiususc/SparkBWA) (68 stars)
SparkBWA is a new tool that exploits the capabilities of a Big Data technology as Apache Spark to boost the performance of one of the most widely adopted sequence aligner, the Burrows-Wheeler Aligner (BWA). (C, 2016-03-29, 2023-07-10)
820. [eturro/mmseq](https://github.com/eturro/mmseq) (68 stars)
Haplotype, isoform and gene level expression analysis using multi-mapping RNA-seq reads (C++, 2013-05-11, 2023-10-11)
821. [lh3/bfc](https://github.com/lh3/bfc) (68 stars)
High-performance error correction for Illumina resequencing data (TeX, 2014-12-30, 2023-12-15)
822. [lh3/ropebwt2](https://github.com/lh3/ropebwt2) (68 stars)
Incremental construction of FM-index for DNA sequences (TeX, 2013-08-02, 2023-11-22)
823. [natir/yacrd](https://github.com/natir/yacrd) (68 stars)
Yet Another Chimeric Read Detector (Rust, 2018-03-28, 2024-01-24)
824. [ncbi/fcs](https://github.com/ncbi/fcs) (68 stars)
Foreign Contamination Screening caller scripts and documentation (None, 2022-06-15, 2024-02-06)
825. [wengzejia1/Open-VCLIP](https://github.com/wengzejia1/Open-VCLIP) (68 stars)
None (Python, 2023-05-31, 2024-02-07)
826. [Genentech/pviz](https://github.com/Genentech/pviz) (67 stars)
Pviz (JavaScript, 2013-08-20, 2023-10-26)
827. [LiLabAtVT/DeepTE](https://github.com/LiLabAtVT/DeepTE) (67 stars)
Neural network classification of TE (Python, 2019-09-24, 2024-01-26)
828. [MannLabs/alphamap](https://github.com/MannLabs/alphamap) (67 stars)
An open-source Python package for the visual annotation of proteomics data with sequence specific knowledge. (Jupyter Notebook, 2020-10-27, 2024-01-26)
829. [MartinPernus/MaskFaceGAN](https://github.com/MartinPernus/MaskFaceGAN) (67 stars)
High resolution fine-grained face editing (Python, 2021-05-27, 2023-12-28)
830. [csjunxu/Noisy-As-Clean](https://github.com/csjunxu/Noisy-As-Clean) (67 stars)
PyTorch Code for "Noisy-As-Clean: Learning Self-supervised Denoising from Corrupted Image", TIP 2020. (Python, 2020-05-09, 2024-01-04)
831. [hdaoud/ORBSLAMM](https://github.com/hdaoud/ORBSLAMM) (67 stars)
None (C++, 2018-04-05, 2024-01-03)
832. [instadeepai/manyfold](https://github.com/instadeepai/manyfold) (67 stars)
ðŸ§¬ ManyFold: An efficient and flexible library for training and validating protein folding models (Python, 2022-08-31, 2024-02-06)
833. [sebwouters/chemps2](https://github.com/sebwouters/chemps2) (67 stars)
CheMPS2: a spin-adapted implementation of DMRG for ab initio quantum chemistry (C++, 2013-11-18, 2023-11-01)
834. [simonvh/fluff](https://github.com/simonvh/fluff) (67 stars)
Fluff is a Python package that contains several scripts to produce pretty, publication-quality figures for next-generation sequencing experiments. (Python, 2012-05-16, 2024-02-03)
835. [DerKevinRiehl/TransposonUltimate](https://github.com/DerKevinRiehl/TransposonUltimate) (66 stars)
TransposonUltimate - a holistic set of tools for transposon identification (None, 2021-03-05, 2024-02-02)
836. [Pbdas/epa-ng](https://github.com/Pbdas/epa-ng) (66 stars)
Massively parallel phylogenetic placement of genetic sequences (C++, 2015-11-03, 2023-12-10)
837. [TheWebMonks/meshmonk](https://github.com/TheWebMonks/meshmonk) (66 stars)
Open mesh-to-mesh registration framework (C++, 2017-02-03, 2023-12-25)
838. [brinkmanlab/GenomeD3Plot](https://github.com/brinkmanlab/GenomeD3Plot) (66 stars)
SVG based genome viewer written in javascript using D3 (JavaScript, 2015-04-21, 2023-09-13)
839. [comprna/METEORE](https://github.com/comprna/METEORE) (66 stars)
Automatic DNA methylation detection from nanopore tools and their consensus model (R, 2020-07-16, 2024-01-14)
840. [epiviz/epiviz](https://github.com/epiviz/epiviz) (66 stars)
EpiViz is a scientific information visualization tool for genetic and epigenetic data, used to aid in the exploration and understanding of correlations between various genome features. (JavaScript, 2013-05-08, 2023-11-15)
841. [fkong7/MeshDeformNet](https://github.com/fkong7/MeshDeformNet) (66 stars)
A deep-learning approach for direct whole-heart mesh reconstruction (Python, 2021-06-21, 2023-12-11)
842. [illumina/REViewer](https://github.com/illumina/REViewer) (66 stars)
A tool for visualizing alignments of reads in regions containing tandem repeats (C++, 2020-12-10, 2023-11-24)
843. [xiezhq/ISEScan](https://github.com/xiezhq/ISEScan) (66 stars)
A python pipeline to identify IS (Insertion Sequence) elements in genome and metagenome (Python, 2016-10-18, 2024-01-25)
844. [zoecastillo/microbiomeExplorer](https://github.com/zoecastillo/microbiomeExplorer) (66 stars)
The Microbiome Explorer provides methods to analyze and visualize microbial community sequencing data either from the R command line or through a Shiny application. Written by Janina Reeder and Joseph N. Paulson. (R, 2020-07-27, 2024-01-09)
845. [AllonKleinLab/SPRING](https://github.com/AllonKleinLab/SPRING) (65 stars)
None (OpenEdge ABL, 2016-08-12, 2024-01-23)
846. [Cwuwhu/FCD-GAN-pytorch](https://github.com/Cwuwhu/FCD-GAN-pytorch) (65 stars)
A pytorch project of the algorithm FCD-GAN. Fully Convolutional Change Detection Framework with Generative Adversarial Network (FCD-GAN) is a newly proposed framework for change detection in multi-temporal remote sensing images. (Python, 2022-03-21, 2024-01-12)
847. [VoiceBeer/MS-MDA](https://github.com/VoiceBeer/MS-MDA) (65 stars)
Code of paper MS-MDA: Multisource Marginal Distribution Adaptation for Cross-subject and Cross-session EEG Emotion Recognition (Jupyter Notebook, 2021-03-22, 2024-01-30)
848. [chenyilun95/DSGN2](https://github.com/chenyilun95/DSGN2) (65 stars)
DSGN++: Exploiting Visual-Spatial Relation for Stereo-based 3D Detectors (T-PAMI 2022) (Python, 2022-03-28, 2024-01-18)
849. [dereneaton/ipyrad](https://github.com/dereneaton/ipyrad) (65 stars)
Interactive assembly and analysis of RAD-seq data sets (Jupyter Notebook, 2015-10-07, 2024-01-19)
850. [gnina/models](https://github.com/gnina/models) (65 stars)
Trained caffe models (Python, 2016-11-04, 2024-01-25)
851. [jwwangchn/BONAI](https://github.com/jwwangchn/BONAI) (65 stars)
Official code for "Learning to Extract Building Footprints from Off-Nadir Aerial Images" (Python, 2021-05-04, 2024-02-02)
852. [marekborowiec/AMAS](https://github.com/marekborowiec/AMAS) (65 stars)
Calculate summary statistics and manipulate multiple sequence alignments (Python, 2015-04-24, 2024-01-26)
853. [scwatts/FastSpar](https://github.com/scwatts/FastSpar) (65 stars)
Rapid and scalable correlation estimation for compositional data (C++, 2016-07-01, 2023-10-24)
854. [GanjinZero/CODER](https://github.com/GanjinZero/CODER) (64 stars)
CODER: Knowledge infused cross-lingual medical term embedding for term normalization. [JBI, ACL-BioNLP 2022] (Python, 2020-08-12, 2024-01-19)
855. [Scarecrow0/SGTR](https://github.com/Scarecrow0/SGTR) (64 stars)
None (Python, 2022-03-29, 2024-01-31)
856. [WatsonLab/MAGpy](https://github.com/WatsonLab/MAGpy) (64 stars)
Snakemake pipeline for downstream analysis of metagenome-assembled genomes (MAGs) (pronounced mag-pie) (Python, 2017-11-27, 2023-12-19)
857. [aatapa/RLScore](https://github.com/aatapa/RLScore) (64 stars)
None (Python, 2012-09-17, 2023-11-23)
858. [amcrisan/Adjutant](https://github.com/amcrisan/Adjutant) (64 stars)
Runs a pubmed query, returns results and allows user to explore high-level structure of returned documents (R, 2017-12-12, 2024-01-29)
859. [broadinstitute/oncotator](https://github.com/broadinstitute/oncotator) (64 stars)
None (Python, 2013-05-20, 2024-01-25)
860. [czodrowskilab/Machine-learning-meets-pKa](https://github.com/czodrowskilab/Machine-learning-meets-pKa) (64 stars)
None (Jupyter Notebook, 2020-01-31, 2023-09-29)
861. [dphansti/Sushi](https://github.com/dphansti/Sushi) (64 stars)
Tools for visualizing genomics data (R, 2014-02-19, 2024-01-19)
862. [hrlblab/CircleNet](https://github.com/hrlblab/CircleNet) (64 stars)
The official implementation of CircleNet, MICCAI 2020 (Python, 2020-09-14, 2024-02-01)
863. [lidaof/eg-react](https://github.com/lidaof/eg-react) (64 stars)
WashU Epigenome Browser (JavaScript, 2017-07-26, 2024-02-01)
864. [tlemane/kmtricks](https://github.com/tlemane/kmtricks) (64 stars)
modular k-mer count matrix and Bloom filter construction for large read collections (C++, 2020-07-08, 2024-02-02)
865. [vertgenlab/gonomics](https://github.com/vertgenlab/gonomics) (64 stars)
A collection of genomics software tools written in Go (golang). (Go, 2018-07-16, 2024-01-19)
866. [FactoDeepLearning/VerticalAttentionOCR](https://github.com/FactoDeepLearning/VerticalAttentionOCR) (63 stars)
None (Python, 2020-10-21, 2024-01-28)
867. [OpenWaterAnalytics/epanet-dev](https://github.com/OpenWaterAnalytics/epanet-dev) (63 stars)
Development Repository for the object-oriented implementation of EPANET (C++, 2016-07-08, 2024-02-02)
868. [lanagarmire/DeepProg](https://github.com/lanagarmire/DeepProg) (63 stars)
Deep-Learning framework for multi-omic and survival data integration (Python, 2020-06-08, 2024-01-28)
869. [pymodproject/pymod](https://github.com/pymodproject/pymod) (63 stars)
PyMod 3 - sequence similarity searches, multiple sequence/structure alignments, and homology modeling within PyMOL. (Python, 2016-04-04, 2024-01-19)
870. [Vandermode/QRNN3D](https://github.com/Vandermode/QRNN3D) (62 stars)
3D Quasi-Recurrent Neural Network for Hyperspectral Image Denoising (TNNLS 2020) (Python, 2019-06-07, 2024-01-26)
871. [ZhifangDeng/MISSFormer](https://github.com/ZhifangDeng/MISSFormer) (62 stars)
None (Python, 2021-12-19, 2024-01-25)
872. [chaoszhang/ASTER](https://github.com/chaoszhang/ASTER) (62 stars)
Accurate Species Tree EstimatoR series: a family of optimation algorithms for species tree inference implemented in C++ (including ASTRAL-Pro & Weighted ASTRAL) (C++, 2020-05-18, 2024-01-27)
873. [fanglab/nanodisco](https://github.com/fanglab/nanodisco) (62 stars)
nanodisco: a toolbox for discovering and exploiting multiple types of DNA methylation from individual bacteria and microbiomes using nanopore sequencing. (R, 2020-02-14, 2024-02-05)
874. [genesofeve/projectR](https://github.com/genesofeve/projectR) (62 stars)
None (HTML, 2016-09-03, 2024-01-23)
875. [ggonnella/gfaviz](https://github.com/ggonnella/gfaviz) (62 stars)
Graphical interactive tool for the visualization of sequence graphs in GFA format. (C++, 2018-02-07, 2024-01-17)
876. [histocartography/hact-net](https://github.com/histocartography/hact-net) (62 stars)
None (Python, 2021-05-19, 2024-01-25)
877. [jwintersinger/kablammo](https://github.com/jwintersinger/kablammo) (62 stars)
Interactive, web-based BLAST results visualization tool. Accessible at https://kablammo.wasmuthlab.org. (JavaScript, 2013-07-09, 2023-12-05)
878. [meringlab/FlashWeave](https://github.com/meringlab/FlashWeave) (62 stars)
Inference of microbial interaction networks from large-scale heterogeneous abundance data (Julia, 2017-03-06, 2024-01-18)
879. [microsoft/IGT-Intermolecular-Graph-Transformer](https://github.com/microsoft/IGT-Intermolecular-Graph-Transformer) (62 stars)
None (None, 2022-04-13, 2024-01-26)
880. [tk2/RetroSeq](https://github.com/tk2/RetroSeq) (62 stars)
RetroSeq is a bioinformatics tool that searches for mobile element insertions from aligned reads in a BAM file and a library of reference transposable elements. Please read the wiki page (link below) for usage instructions. Also, there is a page on the wiki describing how the 1000 genomes CEU trio was carried out with the files and parameters used for the various steps. (Perl, 2011-01-13, 2023-11-05)
881. [ParkerLab/ataqv](https://github.com/ParkerLab/ataqv) (61 stars)
A toolkit for QC and visualization of ATAC-seq results. (C++, 2015-11-10, 2024-01-23)
882. [RitchieLab/eQTpLot](https://github.com/RitchieLab/eQTpLot) (61 stars)
 Visualization of Colocalization Between eQTL and GWAS Data (R, 2020-08-12, 2024-01-13)
883. [WenkeHuang/FCCL](https://github.com/WenkeHuang/FCCL) (61 stars)
TPAMI2023 & CVPR2022 - Generalizable Heterogeneous Federated Cross-Correlation and Instance Similarity Learning & Learn From Others and Be Yourself in Heterogeneous Federated Learning (Python, 2022-03-09, 2024-01-26)
884. [XinshaoAmosWang/Ranked-List-Loss-for-DML](https://github.com/XinshaoAmosWang/Ranked-List-Loss-for-DML) (61 stars)
CVPR 2019: Ranked List Loss for Deep Metric Learning, with extension for TPAMI submission (Shell, 2019-06-29, 2024-01-18)
885. [cafferychen777/ggpicrust2](https://github.com/cafferychen777/ggpicrust2) (61 stars)
Make Picrust2 Output Analysis and Visualization Easier (R, 2022-12-17, 2024-01-31)
886. [deprekate/PHANOTATE](https://github.com/deprekate/PHANOTATE) (61 stars)
PHANOTATE: a tool to annotate phage genomes. (Python, 2017-05-16, 2023-11-27)
887. [hellloxiaotian/ESRGCNN](https://github.com/hellloxiaotian/ESRGCNN) (61 stars)
Image Super-resolution with An Enhanced Group Convolutional Neural Network (Neural Networks, 2022) (Python, 2021-01-24, 2024-01-04)
888. [huangzhii/SALMON](https://github.com/huangzhii/SALMON) (61 stars)
SALMON: Survival Analysis Learning with Multi-Omics Neural Networks (Python, 2019-01-21, 2024-01-17)
889. [kweimann/ecg-transfer-learning](https://github.com/kweimann/ecg-transfer-learning) (61 stars)
None (Python, 2020-08-07, 2024-01-04)
890. [nadeemlab/CEP](https://github.com/nadeemlab/CEP) (61 stars)
Computational Endoscopy Platform (advanced deep learning toolset for analyzing endoscopy videos) [MICCAI'22, MICCAI'21, ISBI'21, CVPR'20] (Python, 2021-04-22, 2024-02-05)
891. [ncbi/BAMscale](https://github.com/ncbi/BAMscale) (61 stars)
BAMscale is a one-step tool for either 1) quantifying and normalizing the coverage of peaks or 2) generated scaled BigWig files for easy visualization of commonly used DNA-seq capture based methods. (C, 2019-03-14, 2024-01-21)
892. [niemasd/TreeCluster](https://github.com/niemasd/TreeCluster) (61 stars)
Efficient phylogenetic clustering of viral sequences (Python, 2017-10-06, 2024-01-24)
893. [nikhilroxtomar/TGANet](https://github.com/nikhilroxtomar/TGANet) (61 stars)
TGANet: Text-guided attention for improved polyp segmentation [Early Accepted & Student Travel Award at MICCAI 2022] (Python, 2022-05-09, 2024-01-26)
894. [ocxtal/libgaba](https://github.com/ocxtal/libgaba) (61 stars)
Adaptive semi-global banded alignment on string graphs (C, 2016-04-13, 2024-01-29)
895. [superlouis/GATGNN](https://github.com/superlouis/GATGNN) (61 stars)
Pytorch Repository for our work: Graph convolutional neural networks with global attention for improved materials property prediction (Python, 2020-06-21, 2023-10-04)
896. [tripal/tripal](https://github.com/tripal/tripal) (61 stars)
The Tripal package is  a suite of Drupal modules for creating biological (genomic, genetic, breeding) websites. Visit the Tripal homepage at http://tripal.info for documentation, support, and other information.  The Drupal project page is at http://drupal.org/project/tripal. (PHP, 2015-09-17, 2024-01-23)
897. [FactoDeepLearning/DAN](https://github.com/FactoDeepLearning/DAN) (60 stars)
None (Python, 2022-03-04, 2024-01-30)
898. [danielhuson/megan-ce](https://github.com/danielhuson/megan-ce) (60 stars)
MEGAN Community Edition (Java, 2016-05-09, 2023-10-06)
899. [freeseek/score](https://github.com/freeseek/score) (60 stars)
Tools to work with GWAS-VCF summary statistics files (C, 2022-12-21, 2024-02-07)
900. [griffithlab/pVAC-Seq](https://github.com/griffithlab/pVAC-Seq) (60 stars)
DEPRECATED. This tool has been superseded by https://github.com/griffithlab/pVACtools (Python, 2015-08-13, 2024-01-04)
901. [husonlab/megan-ce](https://github.com/husonlab/megan-ce) (60 stars)
MEGAN Community Edition (Java, 2016-05-09, 2023-10-06)
902. [kimmo1019/DeepCDR](https://github.com/kimmo1019/DeepCDR) (60 stars)
Cancer Drug Response Prediction via a Hybrid Graph Convolutional Network (Python, 2019-11-25, 2024-01-21)
903. [pleaseconnectwifi/PNAL](https://github.com/pleaseconnectwifi/PNAL) (60 stars)
Learning with Noisy Labels for Robust Point Cloud Segmentation (ICCV2021 Oral) (Python, 2021-07-29, 2024-01-15)
904. [qiyunzhu/woltka](https://github.com/qiyunzhu/woltka) (60 stars)
Woltka: a versatile meta'omic data classifier (Python, 2020-01-20, 2024-02-01)
905. [simonhmartin/twisst](https://github.com/simonhmartin/twisst) (60 stars)
Topology weighting by iterative sampling of sub-trees (Python, 2016-05-26, 2023-10-22)
906. [tshizys/CoupleNet](https://github.com/tshizys/CoupleNet) (60 stars)
Code for training 'CoupleNet: Coupling Global Structure with Local Parts for Object Detection' in ICCV 2017 (Python, 2017-08-03, 2024-01-05)
907. [198808xc/OrganSegRSTN](https://github.com/198808xc/OrganSegRSTN) (59 stars)
An end-to-end coarse-to-fine framework for organ segmentation (Python, 2018-04-05, 2024-02-03)
908. [KCCG/ClinSV](https://github.com/KCCG/ClinSV) (59 stars)
Robust detection of clinically relevant structural and copy number variation from whole genome sequencing data (Perl, 2017-05-12, 2024-01-06)
909. [Oshlack/STRetch](https://github.com/Oshlack/STRetch) (59 stars)
Method for detecting STR expansions from short-read sequencing data (Python, 2017-01-03, 2023-11-04)
910. [VCCRI/CIDR](https://github.com/VCCRI/CIDR) (59 stars)
Clustering through Imputation and Dimensionality Reduction (R, 2016-08-09, 2024-01-09)
911. [dewyman/TranscriptClean](https://github.com/dewyman/TranscriptClean) (59 stars)
Correct mismatches, microindels, and noncanonical splice junctions in long reads that have been mapped to the genome (Python, 2017-08-08, 2023-12-12)
912. [powellgenomicslab/scPred](https://github.com/powellgenomicslab/scPred) (59 stars)
scPred package for cell type prediction from scRNA-seq data (R, 2019-09-06, 2024-01-30)
913. [ymcui/expmrc](https://github.com/ymcui/expmrc) (59 stars)
ExpMRC: Explainability Evaluation for Machine Reading Comprehension (Python, 2020-10-28, 2024-01-04)
914. [HALFpipe/HALFpipe](https://github.com/HALFpipe/HALFpipe) (58 stars)
ENIGMA HALFpipe is a user-friendly software that facilitates reproducible analysis of fMRI data (Python, 2020-06-30, 2024-01-31)
915. [JLiangLab/TransVW](https://github.com/JLiangLab/TransVW) (58 stars)
Official Keras & PyTorch Implementation and Pre-trained Models for TransVW (Python, 2020-08-12, 2024-01-04)
916. [KingJamesSong/DifferentiableSVD](https://github.com/KingJamesSong/DifferentiableSVD) (58 stars)
A collection of differentiable SVD methods and ICCV21 "Why Approximate Matrix Square Root Outperforms Accurate SVD in Global Covariance Pooling?" (Python, 2021-07-23, 2024-01-23)
917. [NBISweden/EMBLmyGFF3](https://github.com/NBISweden/EMBLmyGFF3) (58 stars)
An efficient way to convert gff3 annotation files into EMBL format ready to submit. (Python, 2016-08-04, 2023-12-22)
918. [RVanDamme/MUFFIN](https://github.com/RVanDamme/MUFFIN) (58 stars)
hybrid assembly and differential binning workflow for metagenomics, transcriptomics and pathway analysis (Nextflow, 2019-06-27, 2024-01-16)
919. [bcgsc/ntedit](https://github.com/bcgsc/ntedit) (58 stars)
âœï¸ultra fast, scalable genome assembly polishing and snv detection (C++, 2018-12-03, 2023-12-12)
920. [brettc/partitionfinder](https://github.com/brettc/partitionfinder) (58 stars)
PartitionFinder discovers optimal partitioning schemes for DNA sequences. (Python, 2011-07-02, 2023-12-18)
921. [dtamayo/spock](https://github.com/dtamayo/spock) (58 stars)
ðŸ–– A package to determine whether planetary orbital configurations will live long and prosper (Python, 2019-10-30, 2024-01-04)
922. [eliberis/parapred](https://github.com/eliberis/parapred) (58 stars)
Paratope Prediction using Deep Learning (Python, 2017-01-05, 2024-01-24)
923. [jamesjiadazhan/dietaryindex](https://github.com/jamesjiadazhan/dietaryindex) (58 stars)
R Package for Calculating Healthy Eating Index-2020 (HEI2020), Alternative Healthy Eating Index (AHEI),  Dietary Approaches to Stop Hypertension (DASH), Mediterranean Diet (MED), Dietary Inflammation Index (DII), and Planetary Health Diet Index from the EAT-Lancet Commission (PHDI) for the NHANES, ASA24, DHQ, and other dietary assessments (R, 2022-06-30, 2024-02-06)
924. [lrkrol/SEREEGA](https://github.com/lrkrol/SEREEGA) (58 stars)
SEREEGA: Simulating Event-Related EEG Activity (MATLAB, 2017-06-14, 2024-01-19)
925. [mourisl/Rcorrector](https://github.com/mourisl/Rcorrector) (58 stars)
Error correction for Illumina RNA-seq reads (C++, 2014-08-21, 2023-12-30)
926. [robotoD/GenoVi](https://github.com/robotoD/GenoVi) (58 stars)
GenoVi, an automated customizable circular genome visualizer for bacteria and archaea (Python, 2020-12-02, 2024-02-05)
927. [samsledje/D-SCRIPT](https://github.com/samsledje/D-SCRIPT) (58 stars)
 A structure-aware interpretable deep learning model for sequence-based prediction of protein-protein interactions (Jupyter Notebook, 2020-10-29, 2024-01-05)
928. [spyflying/CMPC-Refseg](https://github.com/spyflying/CMPC-Refseg) (58 stars)
Code for Referring Image Segmentation via Cross-Modal Progressive Comprehension, CVPR2020. (Python, 2020-03-08, 2024-01-04)
929. [tzt101/CLADE](https://github.com/tzt101/CLADE) (58 stars)
Efficient Semantic Image Synthesis via Class-Adaptive Normalization (TPAMI 2021) (Python, 2020-12-04, 2024-01-04)
930. [zqgao22/HIGH-PPI](https://github.com/zqgao22/HIGH-PPI) (58 stars)
None (Python, 2022-10-16, 2024-01-15)
931. [KailiWang1/DeepDTAF](https://github.com/KailiWang1/DeepDTAF) (57 stars)
 a deep learning architecture for protein-ligand binding affinity prediction (Jupyter Notebook, 2020-08-18, 2024-02-03)
932. [OpenGene/CfdnaPattern](https://github.com/OpenGene/CfdnaPattern) (57 stars)
Pattern Recognition for Cell-free DNA (Python, 2016-07-28, 2024-02-03)
933. [QuKunLab/ATAC-pipe](https://github.com/QuKunLab/ATAC-pipe) (57 stars)
Analysis pipeline for ATAC-seq Data (Python, 2017-12-18, 2024-01-30)
934. [SAMtoBAM/MUMandCo](https://github.com/SAMtoBAM/MUMandCo) (57 stars)
MUM&Co is a simple bash script that uses Whole Genome Alignment information provided by MUMmer (only v4) to detect Structural Variation (Shell, 2019-05-17, 2024-01-18)
935. [SRKH/S4NN](https://github.com/SRKH/S4NN) (57 stars)
Temporal backpropagation for spiking neural networks with one spike per neuron, by S. R. Kheradpisheh and T. Masquelier, International Journal of Neural Systems (2020), doi: 10.1142/S0129065720500276 (Jupyter Notebook, 2020-01-16, 2024-01-30)
936. [ShuGuoJ/GiGCN](https://github.com/ShuGuoJ/GiGCN) (57 stars)
None (Python, 2021-09-06, 2023-12-29)
937. [ablab/viralVerify](https://github.com/ablab/viralVerify) (57 stars)
viralVerify: viral contig verification tool (Python, 2019-10-02, 2024-01-29)
938. [dantaki/SV2](https://github.com/dantaki/SV2) (57 stars)
Support Vector Structural Variation Genotyper (Python, 2017-01-26, 2023-02-15)
939. [ggonnella/gfapy](https://github.com/ggonnella/gfapy) (57 stars)
Gfapy: a flexible and extensible software library for handling sequence graphs in Python (Python, 2017-03-17, 2024-01-15)
940. [harbourlab/UPhyloplot2](https://github.com/harbourlab/UPhyloplot2) (57 stars)
Draw phylogenetic trees of tumor evolution (Python, 2019-10-16, 2024-01-30)
941. [iprada/Circle-Map](https://github.com/iprada/Circle-Map) (57 stars)
A method for circular DNA detection based on probabilistic mapping of ultrashort reads (Python, 2017-03-22, 2024-02-05)
942. [ksahlin/BESST](https://github.com/ksahlin/BESST) (57 stars)
BESST - scaffolder for genomic assemblies (Python, 2013-01-22, 2024-01-11)
943. [ksahlin/ultra](https://github.com/ksahlin/ultra) (57 stars)
Long-read splice alignment with high accuracy (Python, 2019-08-20, 2024-02-06)
944. [neurogenomics/rworkflows](https://github.com/neurogenomics/rworkflows) (57 stars)
Continuous integration for R packages. ðŸ”€ Automates testing âœ…, documentation website building ðŸ“¦, & containerised deployment ðŸ³. (HTML, 2021-10-28, 2024-02-04)
945. [osmanberke/Deep-SSVEP-BCI](https://github.com/osmanberke/Deep-SSVEP-BCI) (57 stars)
Official Repository of 'A Deep Neural Network for SSVEP-Based Brain-Computer Interfaces' (MATLAB, 2020-10-18, 2024-01-26)
946. [soeaver/AirNet-PyTorch](https://github.com/soeaver/AirNet-PyTorch) (57 stars)
Implementation of the paper ''Attention Inspiring Receptive-fields Network'' (Python, 2018-03-14, 2023-05-30)
947. [telatin/BamToCov](https://github.com/telatin/BamToCov) (57 stars)
ðŸ”  coverage extraction from BAM/CRAM files, supporting targets ðŸ“Š (Nim, 2021-02-01, 2023-12-21)
948. [CGPxy/AAU-net](https://github.com/CGPxy/AAU-net) (56 stars)
AAU-net: An Adaptive Attention U-net for Breast Lesions Segmentation in Ultrasound Images (Python, 2022-09-30, 2024-02-06)
949. [JhuangLab/BioInstaller](https://github.com/JhuangLab/BioInstaller) (56 stars)
A comprehensive R package to construct interactive and reproducible biological data analysis applications based on the R platform (R, 2017-03-15, 2024-01-06)
950. [Vini2/phables](https://github.com/Vini2/phables) (56 stars)
ðŸ«§ðŸ§¬ From fragmented assemblies to high-quality bacteriophage genomes (Python, 2022-07-21, 2024-01-25)
951. [bene51/3Dscript](https://github.com/bene51/3Dscript) (56 stars)
Create 3D/4D animations using a natural language-based syntax (Java, 2018-10-04, 2024-01-19)
952. [cyang03/CHECKED](https://github.com/cyang03/CHECKED) (56 stars)
None (Jupyter Notebook, 2020-10-14, 2024-02-04)
953. [ddofer/ProFET](https://github.com/ddofer/ProFET) (56 stars)
ProFET: Protein Feature Engineering Toolkit for Machine Learning (Python, 2015-01-16, 2024-01-02)
954. [jltsiren/gbwt](https://github.com/jltsiren/gbwt) (56 stars)
Substring index for paths in a graph (C++, 2017-04-11, 2023-12-07)
955. [materialsintelligence/matscholar](https://github.com/materialsintelligence/matscholar) (56 stars)
Code to access the Matscholar public API. (Python, 2018-10-03, 2024-01-31)
956. [rnacentral/R2DT](https://github.com/rnacentral/R2DT) (56 stars)
Visualise RNA secondary structure in consistent, reproducible and recognisable layouts (PostScript, 2018-08-14, 2024-02-01)
957. [rundherum/pymia](https://github.com/rundherum/pymia) (56 stars)
pymia: A Python package for data handling and evaluation in deep learning-based medical image analysis (Python, 2018-08-03, 2023-11-13)
958. [volkamerlab/KinFragLib](https://github.com/volkamerlab/KinFragLib) (56 stars)
Kinase-focused fragment library (Jupyter Notebook, 2020-02-21, 2024-01-18)
959. [winycg/HSAKD](https://github.com/winycg/HSAKD) (56 stars)
[IJCAI-2021&&TNNLS-2022] Official implementation of Hierarchical Self-supervised Augmented Knowledge Distillation (Python, 2021-05-10, 2023-12-07)
960. [xavierdidelot/TransPhylo](https://github.com/xavierdidelot/TransPhylo) (56 stars)
Reconstruction of transmission trees using genomic data (R, 2015-06-11, 2023-12-30)
961. [BTheDragonMaster/pikachu](https://github.com/BTheDragonMaster/pikachu) (55 stars)
Python-based Informatics Kit for Analysing Chemical Units (Python, 2022-01-04, 2023-12-17)
962. [Data2Dynamics/d2d](https://github.com/Data2Dynamics/d2d) (55 stars)
a modeling environment tailored to parameter estimation in dynamical systems (MATLAB, 2016-06-14, 2023-12-22)
963. [Genotek/ClassifyCNV](https://github.com/Genotek/ClassifyCNV) (55 stars)
ClassifyCNV: a tool for clinical annotation of copy-number variants (Python, 2020-07-15, 2023-10-29)
964. [IRMVLab/EfficientLO-Net](https://github.com/IRMVLab/EfficientLO-Net) (55 stars)
EfficientLO-Net: Efficient 3D Deep LiDAR Odometry (PAMI 2022) (Python, 2022-07-23, 2024-01-23)
965. [JialeCao001/HSD](https://github.com/JialeCao001/HSD) (55 stars)
Hierarchical Shot Detector (ICCV2019) (Python, 2019-09-26, 2024-01-04)
966. [M3DV/RibSeg](https://github.com/M3DV/RibSeg) (55 stars)
[MICCAI'21 & TMI'23] RibSeg Dataset and Point Cloud Baselines for Rib Segmentation from CT Scans (Python, 2021-06-22, 2024-01-29)
967. [OpenNFT/OpenNFT](https://github.com/OpenNFT/OpenNFT) (55 stars)
OpenNFT is an integrated software package designed for neurofeedback training and automatic real-time fMRI quality assessment (rtQA). (MATLAB, 2017-03-14, 2024-01-20)
968. [PapenfussLab/bioshake](https://github.com/PapenfussLab/bioshake) (55 stars)
Bioinformatics pipelines with Haskell and Shake (Haskell, 2017-02-28, 2023-03-31)
969. [RuleWorld/bionetgen](https://github.com/RuleWorld/bionetgen) (55 stars)
Rule-based modeling framework (Perl, 2015-03-16, 2024-01-18)
970. [abehr/pong](https://github.com/abehr/pong) (55 stars)
Fast analysis and visualization of latent clusters in population genetic data (Python, 2016-03-23, 2024-01-24)
971. [bede/hostile](https://github.com/bede/hostile) (55 stars)
Accurate host read removal (Python, 2023-05-31, 2024-01-24)
972. [ethanchen1301/SSDraw](https://github.com/ethanchen1301/SSDraw) (55 stars)
None (Jupyter Notebook, 2023-07-26, 2024-01-15)
973. [giannimonaco/ABIS](https://github.com/giannimonaco/ABIS) (55 stars)
None (R, 2018-08-14, 2023-10-15)
974. [lczech/genesis](https://github.com/lczech/genesis) (55 stars)
A library for working with phylogenetic and population genetic data. (C++, 2014-10-24, 2023-11-27)
975. [muellan/metacache](https://github.com/muellan/metacache) (55 stars)
memory efficient, fast & precise taxnomomic classification system for metagenomic read mapping (C++, 2017-02-07, 2024-01-24)
976. [ncbi/SSDraw](https://github.com/ncbi/SSDraw) (55 stars)
None (Jupyter Notebook, 2023-07-26, 2024-01-15)
977. [pratas/cryfa](https://github.com/pratas/cryfa) (55 stars)
A secure encryption tool for genomic data (C++, 2016-12-12, 2024-01-11)
978. [twestbrookunh/paladin](https://github.com/twestbrookunh/paladin) (55 stars)
Protein Alignment and Detection Interface (C, 2015-05-22, 2023-12-14)
979. [vembrane/vembrane](https://github.com/vembrane/vembrane) (55 stars)
vembrane filters VCF records using python expressions (Python, 2020-07-01, 2024-02-05)
980. [xiaofeng94/RefineDNet-for-dehazing](https://github.com/xiaofeng94/RefineDNet-for-dehazing) (55 stars)
The Pytorch implementation for "RefineDNet: A Weakly Supervised Refinement Framework for Single Image Dehazing" (TIP 2021) (Python, 2020-06-24, 2024-01-25)
981. [BenoitMorel/GeneRax](https://github.com/BenoitMorel/GeneRax) (54 stars)
None (C++, 2018-09-24, 2023-12-20)
982. [CherBass/ICAM](https://github.com/CherBass/ICAM) (54 stars)
ICAM: Interpretable Classification via Disentangled Representations and Feature Attribution Mapping (Python, 2020-05-17, 2024-01-10)
983. [DengPingFan/SODBenchmark](https://github.com/DengPingFan/SODBenchmark) (54 stars)
Salient objects in clutter, TPAMI, 2022 (None, 2022-04-04, 2024-02-01)
984. [Proteomicslab57357/UniprotR](https://github.com/Proteomicslab57357/UniprotR) (54 stars)
Retrieving Information of Proteins from Uniprot (R, 2019-08-19, 2023-11-28)
985. [RyanCook94/inphared](https://github.com/RyanCook94/inphared) (54 stars)
Providing up-to-date phage genome databases, metrics and useful input files for a number of bioinformatic pipelines. (Perl, 2021-01-14, 2024-01-13)
986. [YifanDengWHU/DDIMDL](https://github.com/YifanDengWHU/DDIMDL) (54 stars)
None (Python, 2019-11-30, 2024-01-08)
987. [carmonalab/STACAS](https://github.com/carmonalab/STACAS) (54 stars)
R package for semi-supervised single-cell data integration (R, 2020-06-12, 2024-02-06)
988. [eskederks/eMAGMA-tutorial](https://github.com/eskederks/eMAGMA-tutorial) (54 stars)
A step by step guide on how to use eMAGMA, an approach to conducting eQTL informed gene-based tests. (None, 2019-08-26, 2023-12-11)
989. [kiharalab/DOVE](https://github.com/kiharalab/DOVE) (54 stars)
A Deep-learning based dOcking decoy eValuation mEthod (Python, 2019-06-27, 2023-06-24)
990. [lcnhappe/happe](https://github.com/lcnhappe/happe) (54 stars)
Harvard Automated Pre-Processing Pipeline for EEG (Matlab, 2017-09-30, 2023-09-23)
991. [lczech/gappa](https://github.com/lczech/gappa) (54 stars)
A toolkit for analyzing and visualizing phylogenetic (placement) data (C++, 2017-10-20, 2024-02-01)
992. [mlin/spVCF](https://github.com/mlin/spVCF) (54 stars)
Sparse Project VCF: evolution of VCF to encode population genotype matrices efficiently (C++, 2018-06-26, 2023-07-02)
993. [paiardin/DockingPie](https://github.com/paiardin/DockingPie) (54 stars)
A Consensus Docking Plugin for PyMOL (Python, 2022-02-14, 2024-01-12)
994. [sequencing/NxTrim](https://github.com/sequencing/NxTrim) (54 stars)
Adapter trimming and virtual library creation for Illumina Nextera Mate Pair libraries. (C++, 2014-08-05, 2022-04-21)
995. [smfleming/HMeta-d](https://github.com/smfleming/HMeta-d) (54 stars)
Hierarchical meta-d' model (MATLAB, 2014-07-17, 2023-12-20)
996. [tanghaibao/quota-alignment](https://github.com/tanghaibao/quota-alignment) (54 stars)
Guided synteny alignment between duplicated genomes (within specified quota constraint) (Python, 2010-02-08, 2024-01-19)
997. [ushaham/BatchEffectRemoval](https://github.com/ushaham/BatchEffectRemoval) (54 stars)
None (Python, 2016-11-04, 2023-09-29)
998. [ygidtu/trackplot](https://github.com/ygidtu/trackplot) (54 stars)
trackplot is a tool for visualizing various next-generation sequencing (NGS) data, including DNA-seq, RNA-seq, single-cell RNA-seq and full-length sequencing datasets. (Python, 2022-04-27, 2024-02-04)
999. [10XGenomics/scHLAcount](https://github.com/10XGenomics/scHLAcount) (53 stars)
Count HLA alleles in single-cell RNA-seq data (TeX, 2019-07-16, 2023-10-21)
1000. [MannLabs/ProteomicsVisualization](https://github.com/MannLabs/ProteomicsVisualization) (53 stars)
This is a collection of minimal code to visualize various aspects of proteomics data. (Jupyter Notebook, 2021-06-11, 2024-01-25)

