# PubMed-GitHub

A collection of GitHub repositories that in PubMed abstract.

## Top 20 Authors

| Rank | Author | Count | Stars | Score |
| ---: | -----: | ----: | ----: | ----: |
| 1 | [brentp](https://github.com/brentp) | 11 | 2464 | 81.44 |
| 2 | [lh3](https://github.com/lh3) | 10 | 2611 | 70.26 |
| 3 | [theislab](https://github.com/theislab) | 9 | 3081 | 67.43 |
| 4 | [ncbi](https://github.com/ncbi) | 9 | 2131 | 65.00 |
| 5 | [bcgsc](https://github.com/bcgsc) | 10 | 1052 | 64.43 |
| 6 | [OpenGene](https://github.com/OpenGene) | 6 | 2281 | 45.02 |
| 7 | [Illumina](https://github.com/Illumina) | 5 | 1089 | 37.69 |
| 8 | [MolecularAI](https://github.com/MolecularAI) | 5 | 1032 | 35.15 |
| 9 | [broadinstitute](https://github.com/broadinstitute) | 4 | 2276 | 32.40 |
| 10 | [rrwick](https://github.com/rrwick) | 4 | 1278 | 31.66 |
| 11 | [oxpig](https://github.com/oxpig) | 5 | 379 | 30.67 |
| 12 | [EpistasisLab](https://github.com/EpistasisLab) | 3 | 10401 | 30.52 |
| 13 | [marbl](https://github.com/marbl) | 4 | 874 | 29.83 |
| 14 | [MannLabs](https://github.com/MannLabs) | 5 | 315 | 29.69 |
| 15 | [choderalab](https://github.com/choderalab) | 4 | 617 | 28.31 |
| 16 | [nadeemlab](https://github.com/nadeemlab) | 4 | 358 | 25.37 |
| 17 | [tanghaibao](https://github.com/tanghaibao) | 3 | 1419 | 24.58 |
| 18 | [microsoft](https://github.com/microsoft) | 4 | 298 | 24.57 |
| 19 | [galaxyproject](https://github.com/galaxyproject) | 3 | 1506 | 24.54 |
| 20 | [pachterlab](https://github.com/pachterlab) | 3 | 1391 | 24.33 |


***Note: only count repositories with stars not less than top 1000 threshold (40).***

## Top 1000 Repositories

***Format: description, language, created date, updated date.***

1. [CSSEGISandData/COVID-19](https://github.com/CSSEGISandData/COVID-19) (‚≠ê29205 ¬∑ None) Novel Coronavirus (COVID-19) Cases, provided by JHU CSSE ([32834919](https://pubmed.ncbi.nlm.nih.gov/32834919); [33121426](https://pubmed.ncbi.nlm.nih.gov/33121426); [34248208](https://pubmed.ncbi.nlm.nih.gov/34248208); [34790453](https://pubmed.ncbi.nlm.nih.gov/34790453))
2. [kubernetes/charts](https://github.com/kubernetes/charts) (‚≠ê15534 ¬∑ Go) ‚ö†Ô∏è(OBSOLETE) Curated applications for Kubernetes ([30101309](https://pubmed.ncbi.nlm.nih.gov/30101309))
3. [flairNLP/flair](https://github.com/flairNLP/flair) (‚≠ê13411 ¬∑ Python) A very simple framework for state-of-the-art Natural Language Processing (NLP) ([33508086](https://pubmed.ncbi.nlm.nih.gov/33508086))
4. [NVlabs/stylegan2](https://github.com/NVlabs/stylegan2) (‚≠ê10721 ¬∑ Python) StyleGAN2 - Official TensorFlow Implementation ([35871688](https://pubmed.ncbi.nlm.nih.gov/35871688))
5. [EpistasisLab/tpot](https://github.com/EpistasisLab/tpot) (‚≠ê9418 ¬∑ Python) A Python Automated Machine Learning tool that optimizes machine learning pipelines using genetic programming. ([31165141](https://pubmed.ncbi.nlm.nih.gov/31165141); [32998684](https://pubmed.ncbi.nlm.nih.gov/32998684))
6. [pachyderm/pachyderm](https://github.com/pachyderm/pachyderm) (‚≠ê6048 ¬∑ Go) Data-Centric Pipelines and Data Versioning ([30101309](https://pubmed.ncbi.nlm.nih.gov/30101309))
7. [owid/covid-19-data](https://github.com/owid/covid-19-data) (‚≠ê5622 ¬∑ Python) Data on COVID-19 (coronavirus) cases, deaths, hospitalizations, tests ‚Ä¢ All countries ‚Ä¢ Updated daily by Our World in Data ([34610042](https://pubmed.ncbi.nlm.nih.gov/34610042))
8. [tyiannak/pyAudioAnalysis](https://github.com/tyiannak/pyAudioAnalysis) (‚≠ê5586 ¬∑ Python) Python Audio Analysis Library: Feature Extraction, Classification, Segmentation and Applications ([26656189](https://pubmed.ncbi.nlm.nih.gov/26656189))
9. [mdbloice/Augmentor](https://github.com/mdbloice/Augmentor) (‚≠ê5006 ¬∑ Python) Image augmentation library in Python for machine learning. ([30989173](https://pubmed.ncbi.nlm.nih.gov/30989173))
10. [deepchem/deepchem](https://github.com/deepchem/deepchem) (‚≠ê4926 ¬∑ Python) Democratizing Deep-Learning for Drug Discovery, Quantum Chemistry, Materials Science and Biology ([28470045](https://pubmed.ncbi.nlm.nih.gov/28470045))
11. [google/deepvariant](https://github.com/google/deepvariant) (‚≠ê3021 ¬∑ Python) DeepVariant is an analysis pipeline that uses a deep neural network to call genetic variants from next-generation DNA sequencing data. ([33399819](https://pubmed.ncbi.nlm.nih.gov/33399819))
12. [OpenGene/fastp](https://github.com/OpenGene/fastp) (‚≠ê1698 ¬∑ C++) An ultra-fast all-in-one FASTQ preprocessor (QC/adapters/trimming/filtering/splitting/merging...) ([30423086](https://pubmed.ncbi.nlm.nih.gov/30423086))
13. [theislab/Scanpy](https://github.com/theislab/Scanpy) (‚≠ê1694 ¬∑ Python) Single-cell analysis in Python. Scales to >1M cells. ([29409532](https://pubmed.ncbi.nlm.nih.gov/29409532))
14. [lh3/minimap2](https://github.com/lh3/minimap2) (‚≠ê1619 ¬∑ C) A versatile pairwise aligner for genomic and spliced nucleotide sequences ([29750242](https://pubmed.ncbi.nlm.nih.gov/29750242); [34623391](https://pubmed.ncbi.nlm.nih.gov/34623391))
15. [sokrypton/ColabFold](https://github.com/sokrypton/ColabFold) (‚≠ê1597 ¬∑ Jupyter Notebook) Making Protein folding accessible to all! ([35637307](https://pubmed.ncbi.nlm.nih.gov/35637307))
16. [broadinstitute/gatk](https://github.com/broadinstitute/gatk) (‚≠ê1561 ¬∑ Java) Official code repository for GATK versions 4 and up ([29982281](https://pubmed.ncbi.nlm.nih.gov/29982281); [31830260](https://pubmed.ncbi.nlm.nih.gov/31830260))
17. [samtools/samtools](https://github.com/samtools/samtools) (‚≠ê1508 ¬∑ C) Tools (written in C using htslib) for manipulating next-generation sequencing data ([24974206](https://pubmed.ncbi.nlm.nih.gov/24974206))
18. [SmartDataAnalytics/PyKEEN](https://github.com/SmartDataAnalytics/PyKEEN) (‚≠ê1452 ¬∑ Python) ü§ñ A Python library for learning and evaluating knowledge graph embeddings ([30768158](https://pubmed.ncbi.nlm.nih.gov/30768158))
19. [galaxyproject/galaxy](https://github.com/galaxyproject/galaxy) (‚≠ê1222 ¬∑ Python) Data intensive science for everyone. ([31197310](https://pubmed.ncbi.nlm.nih.gov/31197310); [35669346](https://pubmed.ncbi.nlm.nih.gov/35669346))
20. [glample/tagger](https://github.com/glample/tagger) (‚≠ê1149 ¬∑ Python) Named Entity Recognition Tool ([28881963](https://pubmed.ncbi.nlm.nih.gov/28881963))
21. [shenwei356/seqkit](https://github.com/shenwei356/seqkit) (‚≠ê1145 ¬∑ Go) A cross-platform and ultrafast toolkit for FASTA/Q file manipulation ([27706213](https://pubmed.ncbi.nlm.nih.gov/27706213))
22. [ewels/MultiQC](https://github.com/ewels/MultiQC) (‚≠ê1137 ¬∑ JavaScript) Aggregate results from bioinformatics analyses across many samples into a single report. ([35799367](https://pubmed.ncbi.nlm.nih.gov/35799367))
23. [YosefLab/scVI](https://github.com/YosefLab/scVI) (‚≠ê1073 ¬∑ Python) Deep probabilistic analysis of single-cell omics data ([30504886](https://pubmed.ncbi.nlm.nih.gov/30504886); [32176273](https://pubmed.ncbi.nlm.nih.gov/32176273))
24. [fahadshamshad/awesome-transformers-in-medical-imaging](https://github.com/fahadshamshad/awesome-transformers-in-medical-imaging) (‚≠ê1041 ¬∑ None) A collection of resources on applications of Transformers in Medical Imaging. ([37315483](https://pubmed.ncbi.nlm.nih.gov/37315483))
25. [vgteam/vg](https://github.com/vgteam/vg) (‚≠ê1033 ¬∑ C++) tools for working with genome variation graphs ([31406990](https://pubmed.ncbi.nlm.nih.gov/31406990); [32657356](https://pubmed.ncbi.nlm.nih.gov/32657356))
26. [ncbi/sra-tools](https://github.com/ncbi/sra-tools) (‚≠ê1023 ¬∑ C) SRA Tools ([36698146](https://pubmed.ncbi.nlm.nih.gov/36698146))
27. [vlawhern/arl-eegmodels](https://github.com/vlawhern/arl-eegmodels) (‚≠ê1019 ¬∑ Python) This is the Army Research Laboratory (ARL) EEGModels Project: A Collection of Convolutional Neural Network (CNN) models for EEG signal classification, using Keras and Tensorflow ([29932424](https://pubmed.ncbi.nlm.nih.gov/29932424))
28. [tensorflow/gan](https://github.com/tensorflow/gan) (‚≠ê895 ¬∑ Jupyter Notebook) Tooling for GANs in TensorFlow ([33381813](https://pubmed.ncbi.nlm.nih.gov/33381813))
29. [kexinhuang12345/DeepPurpose](https://github.com/kexinhuang12345/DeepPurpose) (‚≠ê867 ¬∑ Jupyter Notebook) A Deep Learning Toolkit for DTI, Drug Property, PPI, DDI, Protein Function Prediction (Bioinformatics) ([33275143](https://pubmed.ncbi.nlm.nih.gov/33275143))
30. [pachterlab/gget](https://github.com/pachterlab/gget) (‚≠ê831 ¬∑ Python) üß¨ gget enables efficient querying of genomic reference databases ([36610989](https://pubmed.ncbi.nlm.nih.gov/36610989))
31. [jdblischak/workflowr](https://github.com/jdblischak/workflowr) (‚≠ê794 ¬∑ R) Organize your project into a research website ([31723427](https://pubmed.ncbi.nlm.nih.gov/31723427))
32. [InsightSoftwareConsortium/SimpleITK-Notebooks](https://github.com/InsightSoftwareConsortium/SimpleITK-Notebooks) (‚≠ê786 ¬∑ Jupyter Notebook) Jupyter notebooks for learning how to use SimpleITK ([29181613](https://pubmed.ncbi.nlm.nih.gov/29181613))
33. [RainCloudPlots/RainCloudPlots](https://github.com/RainCloudPlots/RainCloudPlots) (‚≠ê777 ¬∑ HTML) Code and tutorials to visualise your data that is both beautiful *and* statistically valid ([31069261](https://pubmed.ncbi.nlm.nih.gov/31069261))
34. [EpistasisLab/pmlb](https://github.com/EpistasisLab/pmlb) (‚≠ê770 ¬∑ Python) PMLB: A large, curated repository of benchmark datasets for evaluating supervised machine learning algorithms. ([34677586](https://pubmed.ncbi.nlm.nih.gov/34677586))
35. [arose/nglview](https://github.com/arose/nglview) (‚≠ê765 ¬∑ Jupyter Notebook) Jupyter widget to interactively view molecular structures and trajectories ([29236954](https://pubmed.ncbi.nlm.nih.gov/29236954))
36. [molecularsets/moses](https://github.com/molecularsets/moses) (‚≠ê762 ¬∑ Python) Molecular Sets (MOSES): A Benchmarking Platform for Molecular Generation Models ([33390943](https://pubmed.ncbi.nlm.nih.gov/33390943))
37. [MrGiovanni/ModelsGenesis](https://github.com/MrGiovanni/ModelsGenesis) (‚≠ê709 ¬∑ Jupyter Notebook) [MICCAI 2019] [MEDIA 2020] Models Genesis ([32766570](https://pubmed.ncbi.nlm.nih.gov/32766570); [33188996](https://pubmed.ncbi.nlm.nih.gov/33188996))
38. [tanghaibao/goatools](https://github.com/tanghaibao/goatools) (‚≠ê694 ¬∑ Python) Python library to handle Gene Ontology (GO) terms ([30022098](https://pubmed.ncbi.nlm.nih.gov/30022098))
39. [deeptools/pyGenomeTracks](https://github.com/deeptools/pyGenomeTracks) (‚≠ê692 ¬∑ Python) python module to plot beautiful and highly customizable genome browser tracks ([32745185](https://pubmed.ncbi.nlm.nih.gov/32745185))
40. [taoliu/MACS](https://github.com/taoliu/MACS) (‚≠ê674 ¬∑ Cython) MACS -- Model-based Analysis of ChIP-Seq ([24743991](https://pubmed.ncbi.nlm.nih.gov/24743991))
41. [tanghaibao/jcvi](https://github.com/tanghaibao/jcvi) (‚≠ê671 ¬∑ Python) Python library to facilitate genome assembly, annotation, and comparative genomics ([25583564](https://pubmed.ncbi.nlm.nih.gov/25583564))
42. [awslabs/dgl-lifesci](https://github.com/awslabs/dgl-lifesci) (‚≠ê662 ¬∑ Python) Python package for graph neural networks in chemistry and biology ([34693143](https://pubmed.ncbi.nlm.nih.gov/34693143))
43. [beoutbreakprepared/nCoV2019](https://github.com/beoutbreakprepared/nCoV2019) (‚≠ê658 ¬∑ HTML) Location for summaries and analysis of data related to n-CoV 2019, first reported in Wuhan, China ([33051749](https://pubmed.ncbi.nlm.nih.gov/33051749))
44. [ablab/spades](https://github.com/ablab/spades) (‚≠ê642 ¬∑ C++) SPAdes Genome Assembler ([31510642](https://pubmed.ncbi.nlm.nih.gov/31510642); [32413137](https://pubmed.ncbi.nlm.nih.gov/32413137))
45. [volkamerlab/TeachOpenCADD](https://github.com/volkamerlab/TeachOpenCADD) (‚≠ê637 ¬∑ Jupyter Notebook) TeachOpenCADD: a teaching platform for computer-aided drug design (CADD) using open source packages and data ([30963287](https://pubmed.ncbi.nlm.nih.gov/30963287))
46. [arose/ngl](https://github.com/arose/ngl) (‚≠ê630 ¬∑ TypeScript) WebGL protein viewer ([29850778](https://pubmed.ncbi.nlm.nih.gov/29850778))
47. [brentp/mosdepth](https://github.com/brentp/mosdepth) (‚≠ê630 ¬∑ Nim) fast BAM/CRAM depth calculation for WGS, exome, or targeted sequencing ([29096012](https://pubmed.ncbi.nlm.nih.gov/29096012))
48. [naver/biobert-pretrained](https://github.com/naver/biobert-pretrained) (‚≠ê625 ¬∑ None) BioBERT: a pre-trained biomedical language representation model for biomedical text mining ([31501885](https://pubmed.ncbi.nlm.nih.gov/31501885))
49. [torognes/vsearch](https://github.com/torognes/vsearch) (‚≠ê614 ¬∑ C++) Versatile open-source tool for microbiome analysis ([27781170](https://pubmed.ncbi.nlm.nih.gov/27781170))
50. [davidemms/OrthoFinder](https://github.com/davidemms/OrthoFinder) (‚≠ê611 ¬∑ Python) Phylogenetic orthology inference for comparative genomics ([31727128](https://pubmed.ncbi.nlm.nih.gov/31727128))
51. [samtools/BCFtools](https://github.com/samtools/BCFtools) (‚≠ê598 ¬∑ C) This is the official development repository for BCFtools. See installation instructions and other documentation here http://samtools.github.io/bcftools/howtos/install.html ([26826718](https://pubmed.ncbi.nlm.nih.gov/26826718))
52. [igvteam/igv](https://github.com/igvteam/igv) (‚≠ê597 ¬∑ Java) Integrative Genomics Viewer. Fast, efficient, scalable visualization tool for genomics data and annotations ([36562559](https://pubmed.ncbi.nlm.nih.gov/36562559))
53. [KlugerLab/FIt-SNE](https://github.com/KlugerLab/FIt-SNE) (‚≠ê575 ¬∑ C++) Fast Fourier Transform-accelerated Interpolation-based  t-SNE (FIt-SNE) ([30742040](https://pubmed.ncbi.nlm.nih.gov/30742040))
54. [NeuroTechX/moabb](https://github.com/NeuroTechX/moabb) (‚≠ê571 ¬∑ Python) Mother of All BCI Benchmarks ([30177583](https://pubmed.ncbi.nlm.nih.gov/30177583))
55. [ialbert/biostar-central](https://github.com/ialbert/biostar-central) (‚≠ê571 ¬∑ Python) Biostar Q&A ([32640986](https://pubmed.ncbi.nlm.nih.gov/32640986))
56. [biojava/biojava](https://github.com/biojava/biojava) (‚≠ê566 ¬∑ Java) :book::microscope::coffee: BioJava is an open-source project dedicated to providing a Java library for processing biological data. ([25505094](https://pubmed.ncbi.nlm.nih.gov/25505094); [28334105](https://pubmed.ncbi.nlm.nih.gov/28334105); [30735498](https://pubmed.ncbi.nlm.nih.gov/30735498))
57. [molstar/molstar](https://github.com/molstar/molstar) (‚≠ê552 ¬∑ TypeScript) A comprehensive macromolecular library ([34864908](https://pubmed.ncbi.nlm.nih.gov/34864908))
58. [rrwick/Bandage](https://github.com/rrwick/Bandage) (‚≠ê550 ¬∑ C++) a Bioinformatics Application for Navigating De novo Assembly Graphs Easily ([26099265](https://pubmed.ncbi.nlm.nih.gov/26099265))
59. [broadinstitute/keras-rcnn](https://github.com/broadinstitute/keras-rcnn) (‚≠ê548 ¬∑ Python) Keras package for region-based convolutional neural networks (RCNNs) ([32652926](https://pubmed.ncbi.nlm.nih.gov/32652926))
60. [ThomasYeoLab/CBIG](https://github.com/ThomasYeoLab/CBIG) (‚≠ê534 ¬∑ MATLAB) None ([28981612](https://pubmed.ncbi.nlm.nih.gov/28981612); [29770530](https://pubmed.ncbi.nlm.nih.gov/29770530); [30974241](https://pubmed.ncbi.nlm.nih.gov/30974241); [31229658](https://pubmed.ncbi.nlm.nih.gov/31229658); [33942058](https://pubmed.ncbi.nlm.nih.gov/33942058); [36918136](https://pubmed.ncbi.nlm.nih.gov/36918136))
61. [gnina/gnina](https://github.com/gnina/gnina) (‚≠ê522 ¬∑ C++) A deep learning framework for molecular docking ([34108002](https://pubmed.ncbi.nlm.nih.gov/34108002))
62. [jerryji1993/DNABERT](https://github.com/jerryji1993/DNABERT) (‚≠ê518 ¬∑ Python) DNABERT: pre-trained Bidirectional Encoder Representations from Transformers model for DNA-language in genome ([33538820](https://pubmed.ncbi.nlm.nih.gov/33538820))
63. [ZhaoZhibin/DL-based-Intelligent-Diagnosis-Benchmark](https://github.com/ZhaoZhibin/DL-based-Intelligent-Diagnosis-Benchmark) (‚≠ê516 ¬∑ Python) Source codes for the paper "Deep Learning Algorithms for Rotating Machinery Intelligent Diagnosis: An Open Source Benchmark Study" ([32854956](https://pubmed.ncbi.nlm.nih.gov/32854956))
64. [MolecularAI/aizynthfinder](https://github.com/MolecularAI/aizynthfinder) (‚≠ê512 ¬∑ Python) A tool for retrosynthetic planning ([34342428](https://pubmed.ncbi.nlm.nih.gov/34342428))
65. [skovaka/UNCALLED](https://github.com/skovaka/UNCALLED) (‚≠ê511 ¬∑ C++) Raw nanopore signal mapper that enables real-time targeted sequencing ([33257863](https://pubmed.ncbi.nlm.nih.gov/33257863))
66. [pachterlab/ffq](https://github.com/pachterlab/ffq) (‚≠ê510 ¬∑ Python) A tool to find sequencing data and metadata from public databases. ([36610997](https://pubmed.ncbi.nlm.nih.gov/36610997))
67. [rrwick/Unicycler](https://github.com/rrwick/Unicycler) (‚≠ê506 ¬∑ C++) hybrid assembly pipeline for bacterial genomes ([28594827](https://pubmed.ncbi.nlm.nih.gov/28594827))
68. [fritzsedlazeck/Sniffles](https://github.com/fritzsedlazeck/Sniffles) (‚≠ê489 ¬∑ Python) Structural variation caller using third generation sequencing ([28456410](https://pubmed.ncbi.nlm.nih.gov/28456410); [29713083](https://pubmed.ncbi.nlm.nih.gov/29713083))
69. [gamcil/clinker](https://github.com/gamcil/clinker) (‚≠ê488 ¬∑ Python) Gene cluster comparison figure generator ([33459763](https://pubmed.ncbi.nlm.nih.gov/33459763))
70. [ruanjue/wtdbg2](https://github.com/ruanjue/wtdbg2) (‚≠ê486 ¬∑ C) Redbean: A fuzzy Bruijn graph approach to long noisy reads assembly ([31819265](https://pubmed.ncbi.nlm.nih.gov/31819265))
71. [theislab/anndata](https://github.com/theislab/anndata) (‚≠ê481 ¬∑ Python) Annotated data. ([29409532](https://pubmed.ncbi.nlm.nih.gov/29409532))
72. [biojs/biojs](https://github.com/biojs/biojs) (‚≠ê480 ¬∑ None) üî¨A library of JavaScript components to represent biological data ([24627804](https://pubmed.ncbi.nlm.nih.gov/24627804); [24715980](https://pubmed.ncbi.nlm.nih.gov/24715980); [24741440](https://pubmed.ncbi.nlm.nih.gov/24741440); [24860644](https://pubmed.ncbi.nlm.nih.gov/24860644); [25075287](https://pubmed.ncbi.nlm.nih.gov/25075287); [25075288](https://pubmed.ncbi.nlm.nih.gov/25075288); [25075290](https://pubmed.ncbi.nlm.nih.gov/25075290))
73. [owkin/PyDESeq2](https://github.com/owkin/PyDESeq2) (‚≠ê480 ¬∑ Python) A Python implementation of the DESeq2 pipeline for bulk RNA-seq DEA. ([37669147](https://pubmed.ncbi.nlm.nih.gov/37669147))
74. [zqfang/GSEApy](https://github.com/zqfang/GSEApy) (‚≠ê480 ¬∑ Python) Gene Set Enrichment Analysis in Python ([36426870](https://pubmed.ncbi.nlm.nih.gov/36426870))
75. [CERN/TIGRE](https://github.com/CERN/TIGRE) (‚≠ê476 ¬∑ MATLAB) TIGRE: Tomographic Iterative GPU-based Reconstruction Toolbox ([36088800](https://pubmed.ncbi.nlm.nih.gov/36088800))
76. [Martinsos/edlib](https://github.com/Martinsos/edlib) (‚≠ê476 ¬∑ C++) Lightweight, super fast C/C++ (& Python) library for sequence alignment using edit (Levenshtein) distance. ([28453688](https://pubmed.ncbi.nlm.nih.gov/28453688))
77. [soedinglab/hh-suite](https://github.com/soedinglab/hh-suite) (‚≠ê476 ¬∑ C) Remote protein homology detection suite. ([31521110](https://pubmed.ncbi.nlm.nih.gov/31521110))
78. [ncoudray/DeepPATH](https://github.com/ncoudray/DeepPATH) (‚≠ê473 ¬∑ Python) Classification of Lung cancer slide images using deep-learning ([30224757](https://pubmed.ncbi.nlm.nih.gov/30224757))
79. [immunogenomics/harmony](https://github.com/immunogenomics/harmony) (‚≠ê468 ¬∑ R) Fast, sensitive and accurate integration of single-cell data with Harmony ([31740819](https://pubmed.ncbi.nlm.nih.gov/31740819))
80. [Ildaron/ironbci](https://github.com/Ildaron/ironbci) (‚≠ê467 ¬∑ C) Open-Source Brain-Computer Interface, ADS1299 and STM32 ([34586477](https://pubmed.ncbi.nlm.nih.gov/34586477))
81. [ryanlayer/samplot](https://github.com/ryanlayer/samplot) (‚≠ê467 ¬∑ Python) Plot structural variant signals from many BAMs and CRAMs ([34034781](https://pubmed.ncbi.nlm.nih.gov/34034781))
82. [BMIRDS/deepslide](https://github.com/BMIRDS/deepslide) (‚≠ê466 ¬∑ Python) Code for the Nature Scientific Reports paper "Pathologist-level classification of histologic patterns on resected lung adenocarcinoma slides with deep neural networks." A sliding window framework for classification of high resolution whole-slide images, often microscopy or histopathology images. ([30833650](https://pubmed.ncbi.nlm.nih.gov/30833650))
83. [etal/cnvkit](https://github.com/etal/cnvkit) (‚≠ê460 ¬∑ Python) Copy number variant detection from targeted DNA sequencing ([27100738](https://pubmed.ncbi.nlm.nih.gov/27100738))
84. [lindenb/jvarkit](https://github.com/lindenb/jvarkit) (‚≠ê455 ¬∑ Java) Java utilities for Bioinformatics ([29186339](https://pubmed.ncbi.nlm.nih.gov/29186339))
85. [seqan/seqan](https://github.com/seqan/seqan) (‚≠ê446 ¬∑ C++) SeqAn's official repository. ([29232861](https://pubmed.ncbi.nlm.nih.gov/29232861))
86. [OpenMS/OpenMS](https://github.com/OpenMS/OpenMS) (‚≠ê441 ¬∑ C++) The codebase of the OpenMS project ([25927999](https://pubmed.ncbi.nlm.nih.gov/25927999))
87. [HazyResearch/hyena-dna](https://github.com/HazyResearch/hyena-dna) (‚≠ê436 ¬∑ Assembly) Official implementation for HyenaDNA, a long-range genomic foundation model built with Hyena ([37426456](https://pubmed.ncbi.nlm.nih.gov/37426456))
88. [benjjneb/dada2](https://github.com/benjjneb/dada2) (‚≠ê436 ¬∑ R) Accurate sample inference from amplicon data with single nucleotide resolution ([27214047](https://pubmed.ncbi.nlm.nih.gov/27214047))
89. [aiqm/torchani](https://github.com/aiqm/torchani) (‚≠ê428 ¬∑ Python) Accurate Neural Network Potential on PyTorch ([32568524](https://pubmed.ncbi.nlm.nih.gov/32568524))
90. [saeyslab/nichenetr](https://github.com/saeyslab/nichenetr) (‚≠ê415 ¬∑ R) NicheNet: predict active ligand-target links between interacting cells ([31819264](https://pubmed.ncbi.nlm.nih.gov/31819264))
91. [pezmaster31/bamtools](https://github.com/pezmaster31/bamtools) (‚≠ê405 ¬∑ C++) C++ API & command-line toolkit for working with BAM data ([23620359](https://pubmed.ncbi.nlm.nih.gov/23620359))
92. [ecogenomics/gtdbtk](https://github.com/ecogenomics/gtdbtk) (‚≠ê400 ¬∑ Python) GTDB-Tk: a toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes. ([31730192](https://pubmed.ncbi.nlm.nih.gov/31730192); [36218463](https://pubmed.ncbi.nlm.nih.gov/36218463))
93. [aristoteleo/dynamo-release](https://github.com/aristoteleo/dynamo-release) (‚≠ê386 ¬∑ Python) Inclusive model of expression dynamics with conventional or metabolic labeling based scRNA-seq / multiomics, vector field reconstruction and differential geometry analyses ([35108499](https://pubmed.ncbi.nlm.nih.gov/35108499))
94. [Illumina/manta](https://github.com/Illumina/manta) (‚≠ê381 ¬∑ C++) Structural variant and indel caller for mapped sequencing data ([26647377](https://pubmed.ncbi.nlm.nih.gov/26647377))
95. [oddt/oddt](https://github.com/oddt/oddt) (‚≠ê380 ¬∑ Python) Open Drug Discovery Toolkit ([26101548](https://pubmed.ncbi.nlm.nih.gov/26101548); [30202917](https://pubmed.ncbi.nlm.nih.gov/30202917))
96. [agshumate/Liftoff](https://github.com/agshumate/Liftoff) (‚≠ê376 ¬∑ Python) An accurate GFF3/GTF lift over pipeline ([33320174](https://pubmed.ncbi.nlm.nih.gov/33320174))
97. [prody/ProDy](https://github.com/prody/ProDy) (‚≠ê374 ¬∑ Python) A Python Package for Protein Dynamics Analysis ([33822884](https://pubmed.ncbi.nlm.nih.gov/33822884); [34240100](https://pubmed.ncbi.nlm.nih.gov/34240100))
98. [oschwengers/bakta](https://github.com/oschwengers/bakta) (‚≠ê372 ¬∑ Python) Rapid & standardized annotation of bacterial genomes, MAGs & plasmids ([34739369](https://pubmed.ncbi.nlm.nih.gov/34739369))
99. [cabani/MaskedFace-Net](https://github.com/cabani/MaskedFace-Net) (‚≠ê371 ¬∑ None) MaskedFace-Net is a dataset of human faces with a correctly and incorrectly worn mask based on the dataset Flickr-Faces-HQ (FFHQ). ([33521223](https://pubmed.ncbi.nlm.nih.gov/33521223))
100. [digitalmethodsinitiative/dmi-tcat](https://github.com/digitalmethodsinitiative/dmi-tcat) (‚≠ê365 ¬∑ PHP) Digital Methods Initiative - Twitter Capture and Analysis Toolset ([33665150](https://pubmed.ncbi.nlm.nih.gov/33665150))
101. [marbl/mash](https://github.com/marbl/mash) (‚≠ê365 ¬∑ C++) Fast genome and metagenome distance estimation using MinHash ([27323842](https://pubmed.ncbi.nlm.nih.gov/27323842))
102. [nservant/HiC-Pro](https://github.com/nservant/HiC-Pro) (‚≠ê354 ¬∑ Python) HiC-Pro: An optimized and flexible pipeline for Hi-C data processing ([26619908](https://pubmed.ncbi.nlm.nih.gov/26619908))
103. [bxlab/metaWRAP](https://github.com/bxlab/metaWRAP) (‚≠ê352 ¬∑ Shell) MetaWRAP - a flexible pipeline for genome-resolved metagenomic data analysis ([30219103](https://pubmed.ncbi.nlm.nih.gov/30219103))
104. [lishen/end2end-all-conv](https://github.com/lishen/end2end-all-conv) (‚≠ê349 ¬∑ Jupyter Notebook) Deep Learning to Improve Breast Cancer Detection on Screening Mammography ([31467326](https://pubmed.ncbi.nlm.nih.gov/31467326))
105. [mlpen/Nystromformer](https://github.com/mlpen/Nystromformer) (‚≠ê348 ¬∑ Python) None ([34745767](https://pubmed.ncbi.nlm.nih.gov/34745767))
106. [wmayner/pyphi](https://github.com/wmayner/pyphi) (‚≠ê348 ¬∑ Python) A toolbox for integrated information theory. ([30048445](https://pubmed.ncbi.nlm.nih.gov/30048445))
107. [amkozlov/raxml-ng](https://github.com/amkozlov/raxml-ng) (‚≠ê346 ¬∑ C++) RAxML Next Generation: faster, easier-to-use and more flexible ([31070718](https://pubmed.ncbi.nlm.nih.gov/31070718); [31755898](https://pubmed.ncbi.nlm.nih.gov/31755898))
108. [brentp/cyvcf2](https://github.com/brentp/cyvcf2) (‚≠ê346 ¬∑ Cython) cython + htslib == fast VCF and BCF processing ([28165109](https://pubmed.ncbi.nlm.nih.gov/28165109))
109. [brentp/vcfanno](https://github.com/brentp/vcfanno) (‚≠ê342 ¬∑ Go) annotate a VCF with other VCFs/BEDs/tabixed files ([27250555](https://pubmed.ncbi.nlm.nih.gov/27250555))
110. [fbreitwieser/pavian](https://github.com/fbreitwieser/pavian) (‚≠ê339 ¬∑ R) üåà Interactive analysis of metagenomics data ([31553437](https://pubmed.ncbi.nlm.nih.gov/31553437))
111. [Illumina/strelka](https://github.com/Illumina/strelka) (‚≠ê338 ¬∑ C++) Strelka2 germline and somatic small variant caller ([30013048](https://pubmed.ncbi.nlm.nih.gov/30013048))
112. [pastas/pastas](https://github.com/pastas/pastas) (‚≠ê336 ¬∑ Python) :spaghetti: Pastas is an open-source Python framework for the analysis of groundwater time series. ([31347164](https://pubmed.ncbi.nlm.nih.gov/31347164))
113. [haddocking/pdb-tools](https://github.com/haddocking/pdb-tools) (‚≠ê335 ¬∑ Python) A dependency-free cross-platform swiss army knife for PDB files. ([30705752](https://pubmed.ncbi.nlm.nih.gov/30705752))
114. [gpertea/stringtie](https://github.com/gpertea/stringtie) (‚≠ê333 ¬∑ C++) Transcript assembly and quantification for RNA-Seq ([35648784](https://pubmed.ncbi.nlm.nih.gov/35648784))
115. [MolecularAI/Reinvent](https://github.com/MolecularAI/Reinvent) (‚≠ê329 ¬∑ Python) None ([33118816](https://pubmed.ncbi.nlm.nih.gov/33118816))
116. [gpertea/gffread](https://github.com/gpertea/gffread) (‚≠ê323 ¬∑ C++) GFF/GTF utility providing format conversions, region filtering, FASTA sequence extraction and more ([32489650](https://pubmed.ncbi.nlm.nih.gov/32489650))
117. [dstahlke/gnuplot-iostream](https://github.com/dstahlke/gnuplot-iostream) (‚≠ê316 ¬∑ C++) C++ interface to gnuplot ([27402903](https://pubmed.ncbi.nlm.nih.gov/27402903))
118. [jtamames/SqueezeMeta](https://github.com/jtamames/SqueezeMeta) (‚≠ê315 ¬∑ C) A complete pipeline for metagenomic analysis ([30733714](https://pubmed.ncbi.nlm.nih.gov/30733714))
119. [beringresearch/ivis](https://github.com/beringresearch/ivis) (‚≠ê314 ¬∑ Python) Dimensionality reduction in very large datasets using Siamese Networks ([31222035](https://pubmed.ncbi.nlm.nih.gov/31222035))
120. [taozh2017/RGBD-SODsurvey](https://github.com/taozh2017/RGBD-SODsurvey) (‚≠ê308 ¬∑ MATLAB) RGB-D Salient Object Detection: A Survey ([33432275](https://pubmed.ncbi.nlm.nih.gov/33432275))
121. [stamatak/standard-RAxML](https://github.com/stamatak/standard-RAxML) (‚≠ê303 ¬∑ C) None ([34410861](https://pubmed.ncbi.nlm.nih.gov/34410861); [37337442](https://pubmed.ncbi.nlm.nih.gov/37337442))
122. [arq5x/lumpy-sv](https://github.com/arq5x/lumpy-sv) (‚≠ê295 ¬∑ C) lumpy: a general probabilistic framework for structural variant discovery ([24970577](https://pubmed.ncbi.nlm.nih.gov/24970577))
123. [oushujun/EDTA](https://github.com/oushujun/EDTA) (‚≠ê294 ¬∑ Perl) Extensive de-novo TE Annotator ([31843001](https://pubmed.ncbi.nlm.nih.gov/31843001); [33900591](https://pubmed.ncbi.nlm.nih.gov/33900591))
124. [whatshap/whatshap](https://github.com/whatshap/whatshap) (‚≠ê293 ¬∑ C++) Read-based phasing of genomic variants, also called haplotype assembly ([29949989](https://pubmed.ncbi.nlm.nih.gov/29949989); [37928341](https://pubmed.ncbi.nlm.nih.gov/37928341))
125. [jonathanking/sidechainnet](https://github.com/jonathanking/sidechainnet) (‚≠ê292 ¬∑ Python) An all-atom protein structure dataset for machine learning. ([34213059](https://pubmed.ncbi.nlm.nih.gov/34213059))
126. [bcgsc/abyss](https://github.com/bcgsc/abyss) (‚≠ê291 ¬∑ C++) :microscope: Assemble large genomes using short reads ([26209068](https://pubmed.ncbi.nlm.nih.gov/26209068); [35729491](https://pubmed.ncbi.nlm.nih.gov/35729491))
127. [BBillot/SynthSeg](https://github.com/BBillot/SynthSeg) (‚≠ê289 ¬∑ Python) Contrast-agnostic segmentation of MRI scans ([34367472](https://pubmed.ncbi.nlm.nih.gov/34367472))
128. [saketkc/pysradb](https://github.com/saketkc/pysradb) (‚≠ê287 ¬∑ Python) Package for fetching metadata and downloading data from SRA/ENA/GEO ([31114675](https://pubmed.ncbi.nlm.nih.gov/31114675))
129. [TissueImageAnalytics/tiatoolbox](https://github.com/TissueImageAnalytics/tiatoolbox) (‚≠ê286 ¬∑ Python) Computational Pathology Toolbox developed by TIA Centre, University of Warwick. ([35640384](https://pubmed.ncbi.nlm.nih.gov/35640384))
130. [lh3/miniasm](https://github.com/lh3/miniasm) (‚≠ê285 ¬∑ TeX) Ultrafast de novo assembly for long noisy reads (though having no consensus step) ([27153593](https://pubmed.ncbi.nlm.nih.gov/27153593))
131. [imagej/imglib](https://github.com/imagej/imglib) (‚≠ê283 ¬∑ Java) A generic next-generation Java library for image processing ([22962343](https://pubmed.ncbi.nlm.nih.gov/22962343))
132. [ncbi/GeneGPT](https://github.com/ncbi/GeneGPT) (‚≠ê283 ¬∑ Python) Code and data for GeneGPT. ([38341654](https://pubmed.ncbi.nlm.nih.gov/38341654))
133. [MarioniLab/miloR](https://github.com/MarioniLab/miloR) (‚≠ê282 ¬∑ HTML) R package implementation of Milo for testing for differential abundance in KNN graphs ([34594043](https://pubmed.ncbi.nlm.nih.gov/34594043))
134. [mengyao/Complete-Striped-Smith-Waterman-Library](https://github.com/mengyao/Complete-Striped-Smith-Waterman-Library) (‚≠ê282 ¬∑ C) None ([24324759](https://pubmed.ncbi.nlm.nih.gov/24324759))
135. [williamgilpin/pypdb](https://github.com/williamgilpin/pypdb) (‚≠ê282 ¬∑ Python) A Python API for the RCSB Protein Data Bank (PDB) ([26369703](https://pubmed.ncbi.nlm.nih.gov/26369703))
136. [philres/ngmlr](https://github.com/philres/ngmlr) (‚≠ê281 ¬∑ C++) NGMLR is a long-read mapper designed to align PacBio or Oxford Nanopore (standard and ultra-long) to a reference genome with a focus on reads that span structural variations ([29713083](https://pubmed.ncbi.nlm.nih.gov/29713083))
137. [kqwang/phase-recovery](https://github.com/kqwang/phase-recovery) (‚≠ê267 ¬∑ None) Resources for phase recovery (also called phase imaging, phase retrieval, or phase reconstruction) ([38161203](https://pubmed.ncbi.nlm.nih.gov/38161203))
138. [PhanstielLab/plotgardener](https://github.com/PhanstielLab/plotgardener) (‚≠ê266 ¬∑ R) None ([35134826](https://pubmed.ncbi.nlm.nih.gov/35134826))
139. [ncbi/pgap](https://github.com/ncbi/pgap) (‚≠ê266 ¬∑ Common Workflow Language) NCBI Prokaryotic Genome Annotation Pipeline ([37962425](https://pubmed.ncbi.nlm.nih.gov/37962425))
140. [uncbiag/registration](https://github.com/uncbiag/registration) (‚≠ê264 ¬∑ TeX) Image Registration ([32523327](https://pubmed.ncbi.nlm.nih.gov/32523327); [36081637](https://pubmed.ncbi.nlm.nih.gov/36081637))
141. [usnistgov/jarvis](https://github.com/usnistgov/jarvis) (‚≠ê263 ¬∑ Python) JARVIS-Tools: an open-source software package for data-driven atomistic materials design. Publications: https://scholar.google.com/citations?user=3w6ej94AAAAJ ([30117438](https://pubmed.ncbi.nlm.nih.gov/30117438); [32165788](https://pubmed.ncbi.nlm.nih.gov/32165788); [32165790](https://pubmed.ncbi.nlm.nih.gov/32165790); [33850146](https://pubmed.ncbi.nlm.nih.gov/33850146))
142. [hafen/trelliscopejs](https://github.com/hafen/trelliscopejs) (‚≠ê261 ¬∑ R) TrelliscopeJS R Package ([36622218](https://pubmed.ncbi.nlm.nih.gov/36622218))
143. [spglib/spglib](https://github.com/spglib/spglib) (‚≠ê250 ¬∑ C) C library for finding and handling crystal symmetries ([37668050](https://pubmed.ncbi.nlm.nih.gov/37668050))
144. [marbl/MashMap](https://github.com/marbl/MashMap) (‚≠ê248 ¬∑ C++) A fast approximate aligner for long DNA sequences ([30423094](https://pubmed.ncbi.nlm.nih.gov/30423094); [37603771](https://pubmed.ncbi.nlm.nih.gov/37603771))
145. [marianattestad/ribbon](https://github.com/marianattestad/ribbon) (‚≠ê248 ¬∑ JavaScript) A genome browser that shows long reads and complex variants better ([32766814](https://pubmed.ncbi.nlm.nih.gov/32766814))
146. [schneebergerlab/plotsr](https://github.com/schneebergerlab/plotsr) (‚≠ê246 ¬∑ Python) Tool to plot synteny and structural rearrangements between genomes ([35561173](https://pubmed.ncbi.nlm.nih.gov/35561173))
147. [dmcable/RCTD](https://github.com/dmcable/RCTD) (‚≠ê243 ¬∑ HTML) Spatial-eXpression-R: Cell type identification (including cell type mixtures) and cell type-specific differential expression for spatial transcriptomics ([33603203](https://pubmed.ncbi.nlm.nih.gov/33603203))
148. [dmcable/spacexr](https://github.com/dmcable/spacexr) (‚≠ê243 ¬∑ HTML) Spatial-eXpression-R: Cell type identification (including cell type mixtures) and cell type-specific differential expression for spatial transcriptomics ([36050488](https://pubmed.ncbi.nlm.nih.gov/36050488))
149. [tSchlegl/f-AnoGAN](https://github.com/tSchlegl/f-AnoGAN) (‚≠ê242 ¬∑ Python) Code for reproducing f-AnoGAN training and anomaly scoring ([30831356](https://pubmed.ncbi.nlm.nih.gov/30831356))
150. [brentp/somalier](https://github.com/brentp/somalier) (‚≠ê241 ¬∑ Nim) fast sample-swap and relatedness checks on BAMs/CRAMs/VCFs/GVCFs... "like damn that is one smart wine guy" ([32664994](https://pubmed.ncbi.nlm.nih.gov/32664994))
151. [sigven/pcgr](https://github.com/sigven/pcgr) (‚≠ê238 ¬∑ R) Personal Cancer Genome Reporter (PCGR) ([29272339](https://pubmed.ncbi.nlm.nih.gov/29272339))
152. [theislab/scgen](https://github.com/theislab/scgen) (‚≠ê238 ¬∑ Python) Single cell perturbation prediction ([31363220](https://pubmed.ncbi.nlm.nih.gov/31363220))
153. [maickrau/GraphAligner](https://github.com/maickrau/GraphAligner) (‚≠ê237 ¬∑ C++) None ([30851095](https://pubmed.ncbi.nlm.nih.gov/30851095); [32972461](https://pubmed.ncbi.nlm.nih.gov/32972461))
154. [opencobra/cobratoolbox](https://github.com/opencobra/cobratoolbox) (‚≠ê234 ¬∑ MATLAB) The COnstraint-Based Reconstruction and Analysis Toolbox.              Documentation: ([27993782](https://pubmed.ncbi.nlm.nih.gov/27993782); [28158334](https://pubmed.ncbi.nlm.nih.gov/28158334); [30052768](https://pubmed.ncbi.nlm.nih.gov/30052768); [37697651](https://pubmed.ncbi.nlm.nih.gov/37697651))
155. [brentp/slivar](https://github.com/brentp/slivar) (‚≠ê232 ¬∑ Nim) genetic variant expressions, annotation, and filtering for great good. ([34267211](https://pubmed.ncbi.nlm.nih.gov/34267211))
156. [jtextor/dagitty](https://github.com/jtextor/dagitty) (‚≠ê231 ¬∑ JavaScript) Graphical analysis of structural causal models / graphical causal models. ([28089956](https://pubmed.ncbi.nlm.nih.gov/28089956))
157. [Kinggerm/GetOrganelle](https://github.com/Kinggerm/GetOrganelle) (‚≠ê230 ¬∑ Python) Organelle Genome Assembly Toolkit (Chloroplast/Mitocondrial/ITS) ([32912315](https://pubmed.ncbi.nlm.nih.gov/32912315))
158. [jeffdaily/parasail](https://github.com/jeffdaily/parasail) (‚≠ê226 ¬∑ C) Pairwise Sequence Alignment Library ([26864881](https://pubmed.ncbi.nlm.nih.gov/26864881))
159. [biocommons/hgvs](https://github.com/biocommons/hgvs) (‚≠ê225 ¬∑ Python) Python library to parse, format, validate, normalize, and map sequence variants. `pip install hgvs` ([28967166](https://pubmed.ncbi.nlm.nih.gov/28967166))
160. [dasmoth/dalliance](https://github.com/dasmoth/dalliance) (‚≠ê225 ¬∑ JavaScript) Interactive web-based genome browser. ([26315906](https://pubmed.ncbi.nlm.nih.gov/26315906))
161. [lmdu/pyfastx](https://github.com/lmdu/pyfastx) (‚≠ê224 ¬∑ C) a python package for fast random access to sequences from plain and gzipped FASTA/Q files ([33341884](https://pubmed.ncbi.nlm.nih.gov/33341884))
162. [lukasturcani/stk](https://github.com/lukasturcani/stk) (‚≠ê223 ¬∑ Python) A Python library which allows construction and manipulation of complex molecules, as well as automatic molecular design and the creation of molecular databases. ([34240979](https://pubmed.ncbi.nlm.nih.gov/34240979))
163. [adamewing/bamsurgeon](https://github.com/adamewing/bamsurgeon) (‚≠ê222 ¬∑ Python) tools for adding mutations to existing .bam files, used for testing mutation callers ([25984700](https://pubmed.ncbi.nlm.nih.gov/25984700); [30400818](https://pubmed.ncbi.nlm.nih.gov/30400818))
164. [ssadedin/bpipe](https://github.com/ssadedin/bpipe) (‚≠ê222 ¬∑ Groovy) Bpipe - a tool for running and managing bioinformatics pipelines ([26460497](https://pubmed.ncbi.nlm.nih.gov/26460497))
165. [RasmussenLab/vamb](https://github.com/RasmussenLab/vamb) (‚≠ê221 ¬∑ Python) Variational autoencoder for metagenomic binning ([33398153](https://pubmed.ncbi.nlm.nih.gov/33398153))
166. [alekseyzimin/masurca](https://github.com/alekseyzimin/masurca) (‚≠ê220 ¬∑ M4) None ([35120119](https://pubmed.ncbi.nlm.nih.gov/35120119))
167. [gtonkinhill/panaroo](https://github.com/gtonkinhill/panaroo) (‚≠ê218 ¬∑ Python) An updated pipeline for pangenome investigation ([32698896](https://pubmed.ncbi.nlm.nih.gov/32698896))
168. [xiangyue9607/BioNEV](https://github.com/xiangyue9607/BioNEV) (‚≠ê217 ¬∑ Python) Graph Embedding Evaluation / Code and Datasets for  "Graph Embedding on Biomedical Networks: Methods, Applications, and Evaluations"  (Bioinformatics 2020) ([31584634](https://pubmed.ncbi.nlm.nih.gov/31584634))
169. [schatzlab/genomescope](https://github.com/schatzlab/genomescope) (‚≠ê216 ¬∑ JavaScript) Fast genome analysis from unassembled short reads ([28369201](https://pubmed.ncbi.nlm.nih.gov/28369201))
170. [smirarab/ASTRAL](https://github.com/smirarab/ASTRAL) (‚≠ê216 ¬∑ Java) Accurate Species TRee ALgorithm ([25161245](https://pubmed.ncbi.nlm.nih.gov/25161245); [26072508](https://pubmed.ncbi.nlm.nih.gov/26072508); [30903685](https://pubmed.ncbi.nlm.nih.gov/30903685))
171. [ryanlayer/giggle](https://github.com/ryanlayer/giggle) (‚≠ê215 ¬∑ C) Interval data structure ([29309061](https://pubmed.ncbi.nlm.nih.gov/29309061))
172. [tjiangHIT/cuteSV](https://github.com/tjiangHIT/cuteSV) (‚≠ê215 ¬∑ Python) Long read based human genomic structural variation detection with cuteSV ([32746918](https://pubmed.ncbi.nlm.nih.gov/32746918); [35751813](https://pubmed.ncbi.nlm.nih.gov/35751813))
173. [EpistasisLab/Aliro](https://github.com/EpistasisLab/Aliro) (‚≠ê213 ¬∑ JavaScript) Aliro: AI-Driven Data Science ([37796839](https://pubmed.ncbi.nlm.nih.gov/37796839))
174. [qiicr/dcmqi](https://github.com/qiicr/dcmqi) (‚≠ê213 ¬∑ C++) dcmqi (DICOM for Quantitative Imaging) is a free, open source C++ library for conversion between imaging research formats and the standard DICOM representation for image analysis results ([29092948](https://pubmed.ncbi.nlm.nih.gov/29092948))
175. [marbl/winnowmap](https://github.com/marbl/winnowmap) (‚≠ê211 ¬∑ C) Long read / genome alignment software ([32657365](https://pubmed.ncbi.nlm.nih.gov/32657365))
176. [debbiemarkslab/evcouplings](https://github.com/debbiemarkslab/evcouplings) (‚≠ê210 ¬∑ Jupyter Notebook) Evolutionary couplings from protein and RNA sequence alignments ([30304492](https://pubmed.ncbi.nlm.nih.gov/30304492))
177. [choderalab/pymbar](https://github.com/choderalab/pymbar) (‚≠ê209 ¬∑ Python) Python implementation of the multistate Bennett acceptance ratio (MBAR) ([25808134](https://pubmed.ncbi.nlm.nih.gov/25808134))
178. [RGLab/MAST](https://github.com/RGLab/MAST) (‚≠ê208 ¬∑ R) Tools and methods for analysis of single cell assay data in R ([26653891](https://pubmed.ncbi.nlm.nih.gov/26653891))
179. [GregoryFaust/samblaster](https://github.com/GregoryFaust/samblaster) (‚≠ê207 ¬∑ C++) samblaster: a tool to mark duplicates and extract discordant and split reads from sam files. ([24812344](https://pubmed.ncbi.nlm.nih.gov/24812344))
180. [brentp/goleft](https://github.com/brentp/goleft) (‚≠ê206 ¬∑ Go) goleft is a collection of bioinformatics tools distributed under MIT license in a single static binary ([29048539](https://pubmed.ncbi.nlm.nih.gov/29048539))
181. [fbreitwieser/krakenuniq](https://github.com/fbreitwieser/krakenuniq) (‚≠ê206 ¬∑ C++) üêô KrakenUniq: Metagenomics classifier with unique k-mer counting for more specific results ([30445993](https://pubmed.ncbi.nlm.nih.gov/30445993))
182. [PacificBiosciences/FALCON](https://github.com/PacificBiosciences/FALCON) (‚≠ê204 ¬∑ Python) FALCON: experimental PacBio diploid assembler -- Out-of-date -- Please use a binary release: https://github.com/PacificBiosciences/FALCON_unzip/wiki/Binaries ([27749838](https://pubmed.ncbi.nlm.nih.gov/27749838))
183. [cambridgeltl/MTL-Bioinformatics-2016](https://github.com/cambridgeltl/MTL-Bioinformatics-2016) (‚≠ê204 ¬∑ Python) None ([32415963](https://pubmed.ncbi.nlm.nih.gov/32415963))
184. [HajkD/biomartr](https://github.com/HajkD/biomartr) (‚≠ê203 ¬∑ R) Genomic Data Retrieval with R ([28110292](https://pubmed.ncbi.nlm.nih.gov/28110292))
185. [bcgsc/NanoSim](https://github.com/bcgsc/NanoSim) (‚≠ê203 ¬∑ Python) Nanopore sequence read simulator ([32520350](https://pubmed.ncbi.nlm.nih.gov/32520350); [36939007](https://pubmed.ncbi.nlm.nih.gov/36939007))
186. [KamilSJaron/smudgeplot](https://github.com/KamilSJaron/smudgeplot) (‚≠ê202 ¬∑ R) Inference of ploidy and heterozygosity structure using whole genome sequencing data ([32188846](https://pubmed.ncbi.nlm.nih.gov/32188846))
187. [hkmztrk/DeepDTA](https://github.com/hkmztrk/DeepDTA) (‚≠ê201 ¬∑ Python) None ([30423097](https://pubmed.ncbi.nlm.nih.gov/30423097))
188. [jeetsukumaran/DendroPy](https://github.com/jeetsukumaran/DendroPy) (‚≠ê200 ¬∑ Python) A Python library for phylogenetic scripting, simulation, data processing and manipulation. ([20421198](https://pubmed.ncbi.nlm.nih.gov/20421198))
189. [griffithlab/GenVisR](https://github.com/griffithlab/GenVisR) (‚≠ê199 ¬∑ R) Genome data visualizations ([27288499](https://pubmed.ncbi.nlm.nih.gov/27288499))
190. [taowenmicro/EasyMicrobiomeR](https://github.com/taowenmicro/EasyMicrobiomeR) (‚≠ê199 ¬∑ HTML) Using R language in microbiome analysis ([37128855](https://pubmed.ncbi.nlm.nih.gov/37128855))
191. [ucscGenomeBrowser/kent](https://github.com/ucscGenomeBrowser/kent) (‚≠ê199 ¬∑ C) UCSC Genome Browser source tree. Stable branch: "beta". ([28108446](https://pubmed.ncbi.nlm.nih.gov/28108446))
192. [ZJUFanLab/scCATCH](https://github.com/ZJUFanLab/scCATCH) (‚≠ê198 ¬∑ R) Automatic Annotation on Cell Types of Clusters from Single-Cell RNA Sequencing Data ([32062421](https://pubmed.ncbi.nlm.nih.gov/32062421))
193. [rdk/p2rank](https://github.com/rdk/p2rank) (‚≠ê198 ¬∑ Groovy) P2Rank: Protein-ligand binding site prediction tool based on machine learning. Stand-alone command line program / Java library for predicting ligand binding pockets from protein structure. ([31114880](https://pubmed.ncbi.nlm.nih.gov/31114880))
194. [labgem/PPanGGOLiN](https://github.com/labgem/PPanGGOLiN) (‚≠ê197 ¬∑ Python) Build a partitioned pangenome graph from microbial genomes ([32191703](https://pubmed.ncbi.nlm.nih.gov/32191703); [33381850](https://pubmed.ncbi.nlm.nih.gov/33381850))
195. [iqtree/iqtree2](https://github.com/iqtree/iqtree2) (‚≠ê196 ¬∑ C++) NEW location of IQ-TREE software for efficient phylogenomic software by maximum likelihood http://www.iqtree.org ([36383168](https://pubmed.ncbi.nlm.nih.gov/36383168); [37656933](https://pubmed.ncbi.nlm.nih.gov/37656933))
196. [shervinea/enzynet](https://github.com/shervinea/enzynet) (‚≠ê195 ¬∑ Python) EnzyNet: enzyme classification using 3D convolutional neural networks on spatial representation ([29740518](https://pubmed.ncbi.nlm.nih.gov/29740518))
197. [INCATools/ontology-development-kit](https://github.com/INCATools/ontology-development-kit) (‚≠ê192 ¬∑ Jinja) Bootstrap an OBO Library ontology ([36208225](https://pubmed.ncbi.nlm.nih.gov/36208225))
198. [choderalab/espaloma](https://github.com/choderalab/espaloma) (‚≠ê192 ¬∑ Python) Extensible Surrogate Potential of Ab initio Learned and Optimized by Message-passing Algorithm  üçπhttps://arxiv.org/abs/2010.01196 ([36349096](https://pubmed.ncbi.nlm.nih.gov/36349096))
199. [dariober/ASCIIGenome](https://github.com/dariober/ASCIIGenome) (‚≠ê192 ¬∑ Java) Text Only Genome Viewer! ([28119307](https://pubmed.ncbi.nlm.nih.gov/28119307))
200. [kormilitzin/med7](https://github.com/kormilitzin/med7) (‚≠ê192 ¬∑ None) None ([34412834](https://pubmed.ncbi.nlm.nih.gov/34412834))
201. [TGAC/KAT](https://github.com/TGAC/KAT) (‚≠ê190 ¬∑ C++) The K-mer Analysis Toolkit (KAT) contains a number of tools that analyse and compare K-mer spectra. ([27797770](https://pubmed.ncbi.nlm.nih.gov/27797770))
202. [atks/vt](https://github.com/atks/vt) (‚≠ê190 ¬∑ C) A tool set for short variant discovery in genetic sequence data. ([25701572](https://pubmed.ncbi.nlm.nih.gov/25701572))
203. [showteeth/ggcoverage](https://github.com/showteeth/ggcoverage) (‚≠ê190 ¬∑ R) Visualize and annotate genomic coverage with ggplot2 ([37559015](https://pubmed.ncbi.nlm.nih.gov/37559015))
204. [thinng/GraphDTA](https://github.com/thinng/GraphDTA) (‚≠ê189 ¬∑ Python) GraphDTA: Predicting drug-target binding affinity with graph neural networks ([33119053](https://pubmed.ncbi.nlm.nih.gov/33119053))
205. [vibansal/HapCUT2](https://github.com/vibansal/HapCUT2) (‚≠ê189 ¬∑ C) software tools for haplotype assembly from sequence data ([36335497](https://pubmed.ncbi.nlm.nih.gov/36335497))
206. [cerr/CERR](https://github.com/cerr/CERR) (‚≠ê185 ¬∑ MATLAB) Matlab/Octave based platform for Radiological Research. ([34874302](https://pubmed.ncbi.nlm.nih.gov/34874302))
207. [mirnylab/cooler](https://github.com/mirnylab/cooler) (‚≠ê185 ¬∑ Python) A cool place to store your Hi-C ([31290943](https://pubmed.ncbi.nlm.nih.gov/31290943); [33135074](https://pubmed.ncbi.nlm.nih.gov/33135074))
208. [pmelsted/bifrost](https://github.com/pmelsted/bifrost) (‚≠ê185 ¬∑ C++) Bifrost: Highly parallel construction and indexing of colored and compacted de Bruijn graphs ([32943081](https://pubmed.ncbi.nlm.nih.gov/32943081))
209. [AstrobioMike/GToTree](https://github.com/AstrobioMike/GToTree) (‚≠ê182 ¬∑ Shell) A user-friendly workflow for phylogenomics ([30865266](https://pubmed.ncbi.nlm.nih.gov/30865266))
210. [SajadMo/SleepEEGNet](https://github.com/SajadMo/SleepEEGNet) (‚≠ê182 ¬∑ Python) SleepEEGNet: Automated Sleep Stage Scoring with Sequence to Sequence Deep Learning Approach ([31063501](https://pubmed.ncbi.nlm.nih.gov/31063501))
211. [theislab/paga](https://github.com/theislab/paga) (‚≠ê182 ¬∑ Jupyter Notebook) Mapping out the coarse-grained connectivity structures of complex manifolds. ([30890159](https://pubmed.ncbi.nlm.nih.gov/30890159))
212. [CapsuleEndoscope/EndoSLAM](https://github.com/CapsuleEndoscope/EndoSLAM) (‚≠ê181 ¬∑ Python) EndoSLAM Dataset and an Unsupervised Monocular Visual Odometry and Depth Estimation Approach for Endoscopic Videos: Endo-SfMLearner ([33930829](https://pubmed.ncbi.nlm.nih.gov/33930829))
213. [rdkit/mmpdb](https://github.com/rdkit/mmpdb) (‚≠ê181 ¬∑ Python) A package to identify matched molecular pairs and use them to predict property changes. ([29770697](https://pubmed.ncbi.nlm.nih.gov/29770697))
214. [danmcduff/iphys-toolbox](https://github.com/danmcduff/iphys-toolbox) (‚≠ê180 ¬∑ MATLAB) A MATLAB toolbox for iPPG analysis.  The toolbox includes implementations of commonly used methods. ([31947335](https://pubmed.ncbi.nlm.nih.gov/31947335))
215. [debbiemarkslab/DeepSequence](https://github.com/debbiemarkslab/DeepSequence) (‚≠ê180 ¬∑ Python) A generative latent variable model for biological sequence families. ([30250057](https://pubmed.ncbi.nlm.nih.gov/30250057))
216. [noriakis/ggkegg](https://github.com/noriakis/ggkegg) (‚≠ê180 ¬∑ R) Analyzing and visualizing KEGG information using the grammar of graphics ([37846038](https://pubmed.ncbi.nlm.nih.gov/37846038))
217. [openvax/mhcflurry](https://github.com/openvax/mhcflurry) (‚≠ê179 ¬∑ Python) Peptide-MHC I binding affinity prediction ([32124315](https://pubmed.ncbi.nlm.nih.gov/32124315); [32711842](https://pubmed.ncbi.nlm.nih.gov/32711842))
218. [isovic/graphmap](https://github.com/isovic/graphmap) (‚≠ê177 ¬∑ C++) GraphMap - A highly sensitive and accurate mapper for long, error-prone reads http://www.nature.com/ncomms/2016/160415/ncomms11307/full/ncomms11307.html Note: This was the original repository which will no longer be officially maintained. Please use the new official repository here: ([27079541](https://pubmed.ncbi.nlm.nih.gov/27079541))
219. [anjany/verse](https://github.com/anjany/verse) (‚≠ê176 ¬∑ Jupyter Notebook) Everything about the 'Large Scale Vertebrae Segmentation Challenge' @ MICCAI 2019-2020 ([34340104](https://pubmed.ncbi.nlm.nih.gov/34340104))
220. [Nextomics/NextPolish](https://github.com/Nextomics/NextPolish) (‚≠ê172 ¬∑ None)  Fast and accurately polish the genome generated by long reads. ([31778144](https://pubmed.ncbi.nlm.nih.gov/31778144))
221. [aquaskyline/Clairvoyante](https://github.com/aquaskyline/Clairvoyante) (‚≠ê172 ¬∑ Python) Clairvoyante: a multi-task convolutional deep neural network for variant calling in Single Molecule Sequencing ([30824707](https://pubmed.ncbi.nlm.nih.gov/30824707))
222. [pangenome/odgi](https://github.com/pangenome/odgi) (‚≠ê172 ¬∑ C++) Optimized Dynamic Genome/Graph Implementation: understanding pangenome graphs ([35552372](https://pubmed.ncbi.nlm.nih.gov/35552372))
223. [ekimb/rust-mdbg](https://github.com/ekimb/rust-mdbg) (‚≠ê171 ¬∑ Rust) minimizer-space de Bruijn graphs (mdBG) for whole genome assembly ([34525345](https://pubmed.ncbi.nlm.nih.gov/34525345))
224. [wdecoster/nanopack](https://github.com/wdecoster/nanopack) (‚≠ê171 ¬∑ Python) An overview of all nanopack tools ([29547981](https://pubmed.ncbi.nlm.nih.gov/29547981); [37171891](https://pubmed.ncbi.nlm.nih.gov/37171891))
225. [Genetalks/gtz](https://github.com/Genetalks/gtz) (‚≠ê170 ¬∑ None) A high performance and compression ratio compressor for genomic data, powered by GTXLab of Genetalks. ([29297296](https://pubmed.ncbi.nlm.nih.gov/29297296))
226. [privefl/bigsnpr](https://github.com/privefl/bigsnpr) (‚≠ê170 ¬∑ R) R package for the analysis of massive SNP arrays. ([32415959](https://pubmed.ncbi.nlm.nih.gov/32415959))
227. [gpertea/gffcompare](https://github.com/gpertea/gffcompare) (‚≠ê169 ¬∑ C++) classify, merge, tracking and annotation of GFF files by comparing to a reference annotation GFF ([32489650](https://pubmed.ncbi.nlm.nih.gov/32489650))
228. [machinegun/SALSA](https://github.com/machinegun/SALSA) (‚≠ê169 ¬∑ Python) SALSA: A tool to scaffold long read assemblies with Hi-C data ([31433799](https://pubmed.ncbi.nlm.nih.gov/31433799))
229. [jbkinney/logomaker](https://github.com/jbkinney/logomaker) (‚≠ê167 ¬∑ Jupyter Notebook) Software for the visualization of sequence-function relationships ([31821414](https://pubmed.ncbi.nlm.nih.gov/31821414))
230. [mortazavilab/PyWGCNA](https://github.com/mortazavilab/PyWGCNA) (‚≠ê167 ¬∑ Jupyter Notebook) PyWGCNA is a Python package designed to do Weighted Gene Correlation Network analysis (WGCNA) ([37399090](https://pubmed.ncbi.nlm.nih.gov/37399090))
231. [shenwei356/kmcp](https://github.com/shenwei356/kmcp) (‚≠ê167 ¬∑ Go) Accurate metagenomic profiling && Fast large-scale sequence/genome searching ([36579886](https://pubmed.ncbi.nlm.nih.gov/36579886))
232. [choderalab/perses](https://github.com/choderalab/perses) (‚≠ê166 ¬∑ Python) Experiments with expanded ensembles to explore chemical space ([37450482](https://pubmed.ncbi.nlm.nih.gov/37450482))
233. [KristapsE/DP4-AI](https://github.com/KristapsE/DP4-AI) (‚≠ê164 ¬∑ Python) Python workflow for DP5 and DP4 analysis of organic molecules ([34122893](https://pubmed.ncbi.nlm.nih.gov/34122893))
234. [kexinhuang12345/moltrans](https://github.com/kexinhuang12345/moltrans) (‚≠ê164 ¬∑ Jupyter Notebook) MolTrans: Molecular Interaction Transformer for Drug Target Interaction Prediction (Bioinformatics) ([33070179](https://pubmed.ncbi.nlm.nih.gov/33070179))
235. [malonge/RaGOO](https://github.com/malonge/RaGOO) (‚≠ê164 ¬∑ Python) RaGOO is no longer supported. Please use RagTag instead: https://github.com/malonge/RagTag ([31661016](https://pubmed.ncbi.nlm.nih.gov/31661016))
236. [stephaneguindon/phyml](https://github.com/stephaneguindon/phyml) (‚≠ê164 ¬∑ C) PhyML -- Phylogenetic estimation using (Maximum) Likelihood ([30977781](https://pubmed.ncbi.nlm.nih.gov/30977781))
237. [baoe/AlignGraph](https://github.com/baoe/AlignGraph) (‚≠ê163 ¬∑ C++) Algorithm for secondary de novo genome assembly guided by closely related references ([24932000](https://pubmed.ncbi.nlm.nih.gov/24932000))
238. [molgenis/systemsgenetics](https://github.com/molgenis/systemsgenetics) (‚≠ê163 ¬∑ Java) Generic Java genotype reader / writer, QTL mapping software, Strand alignment tool ([32532224](https://pubmed.ncbi.nlm.nih.gov/32532224); [34792549](https://pubmed.ncbi.nlm.nih.gov/34792549))
239. [vgteam/sequenceTubeMap](https://github.com/vgteam/sequenceTubeMap) (‚≠ê162 ¬∑ JavaScript) displays multiple genomic sequences in the form of a tube map ([31368484](https://pubmed.ncbi.nlm.nih.gov/31368484))
240. [HadrienG/InSilicoSeq](https://github.com/HadrienG/InSilicoSeq) (‚≠ê161 ¬∑ Python) :rocket: A sequencing simulator ([30016412](https://pubmed.ncbi.nlm.nih.gov/30016412))
241. [Illumina/ExpansionHunter](https://github.com/Illumina/ExpansionHunter) (‚≠ê161 ¬∑ C++) A tool for estimating repeat sizes ([31134279](https://pubmed.ncbi.nlm.nih.gov/31134279))
242. [ndierckx/NOVOPlasty](https://github.com/ndierckx/NOVOPlasty) (‚≠ê161 ¬∑ Perl) NOVOPlasty - The organelle assembler and heteroplasmy caller ([28204566](https://pubmed.ncbi.nlm.nih.gov/28204566); [33575563](https://pubmed.ncbi.nlm.nih.gov/33575563))
243. [soedinglab/metaeuk](https://github.com/soedinglab/metaeuk) (‚≠ê161 ¬∑ C) MetaEuk - sensitive, high-throughput gene discovery and annotation for large-scale eukaryotic metagenomics ([32245390](https://pubmed.ncbi.nlm.nih.gov/32245390))
244. [Superzchen/iFeature](https://github.com/Superzchen/iFeature) (‚≠ê160 ¬∑ Python) iFeature is a comprehensive Python-based toolkit for generating various numerical feature representation schemes from protein or peptide sequences. iFeature is capable of calculating and extracting a wide spectrum of 18 major sequence encoding schemes that encompass 53 different types of feature descriptors. Furthermore, iFeature also integrates five kinds of frequently used feature clustering algorithms, four feature selection algorithms and three dimensionality reduction algorithms. ([29528364](https://pubmed.ncbi.nlm.nih.gov/29528364))
245. [alexanderkroner/saliency](https://github.com/alexanderkroner/saliency) (‚≠ê159 ¬∑ Python) Contextual Encoder-Decoder Network for Visual Saliency Prediction [Neural Networks 2020] ([32563023](https://pubmed.ncbi.nlm.nih.gov/32563023))
246. [chrisamiller/fishplot](https://github.com/chrisamiller/fishplot) (‚≠ê159 ¬∑ R) Create timecourse "fish plots" that show changes in the clonal architecture of tumors ([27821060](https://pubmed.ncbi.nlm.nih.gov/27821060))
247. [franciscozorrilla/metaGEM](https://github.com/franciscozorrilla/metaGEM) (‚≠ê159 ¬∑ Python) :gem: An easy-to-use workflow for generating context specific genome-scale metabolic models and predicting metabolic interactions within microbial communities directly from metagenomic data ([34614189](https://pubmed.ncbi.nlm.nih.gov/34614189))
248. [alanwilter/acpype](https://github.com/alanwilter/acpype) (‚≠ê158 ¬∑ Python) OFFICIAL: AnteChamber PYthon Parser interfacE ([37252824](https://pubmed.ncbi.nlm.nih.gov/37252824))
249. [galaxyproject/tools-iuc](https://github.com/galaxyproject/tools-iuc) (‚≠ê156 ¬∑ HTML) Tool Shed repositories maintained by the Intergalactic Utilities Commission ([34096994](https://pubmed.ncbi.nlm.nih.gov/34096994))
250. [vsbuffalo/scidataflow](https://github.com/vsbuffalo/scidataflow) (‚≠ê156 ¬∑ Rust) Command line scientific data management tool ([38180848](https://pubmed.ncbi.nlm.nih.gov/38180848))
251. [yiluheihei/microbiomeMarker](https://github.com/yiluheihei/microbiomeMarker) (‚≠ê156 ¬∑ R) R package for microbiome biomarker discovery ([35771644](https://pubmed.ncbi.nlm.nih.gov/35771644))
252. [ChiLiubio/microeco](https://github.com/ChiLiubio/microeco) (‚≠ê155 ¬∑ R) An R package for data analysis in microbial community ecology ([33332530](https://pubmed.ncbi.nlm.nih.gov/33332530))
253. [UCLCheminformatics/ScaffoldGraph](https://github.com/UCLCheminformatics/ScaffoldGraph) (‚≠ê155 ¬∑ Python) ScaffoldGraph is an open-source cheminformatics library, built using RDKit and NetworkX, for the generation and analysis of scaffold networks and scaffold trees. ([32232438](https://pubmed.ncbi.nlm.nih.gov/32232438))
254. [dnbaker/dashing](https://github.com/dnbaker/dashing) (‚≠ê155 ¬∑ C++) Fast and accurate genomic distances using HyperLogLog ([31801633](https://pubmed.ncbi.nlm.nih.gov/31801633))
255. [sorgerlab/indra](https://github.com/sorgerlab/indra) (‚≠ê155 ¬∑ Python) INDRA (Integrated Network and Dynamical Reasoning Assembler) is an automated model assembly system interfacing with NLP systems and databases to collect knowledge, and through a process of assembly, produce causal graphs and dynamical models. ([31070726](https://pubmed.ncbi.nlm.nih.gov/31070726))
256. [BaselAbujamous/clust](https://github.com/BaselAbujamous/clust) (‚≠ê154 ¬∑ Python) Automatic and optimised consensus clustering of one or more heterogeneous datasets ([30359297](https://pubmed.ncbi.nlm.nih.gov/30359297))
257. [nygenome/lancet](https://github.com/nygenome/lancet) (‚≠ê154 ¬∑ C) Microassembly based somatic variant caller for NGS data ([30271907](https://pubmed.ncbi.nlm.nih.gov/30271907); [33241313](https://pubmed.ncbi.nlm.nih.gov/33241313))
258. [brentp/hts-nim](https://github.com/brentp/hts-nim) (‚≠ê153 ¬∑ Nim) nim wrapper for htslib for parsing genomics data files ([29718142](https://pubmed.ncbi.nlm.nih.gov/29718142))
259. [herrmannlab/highdicom](https://github.com/herrmannlab/highdicom) (‚≠ê153 ¬∑ Python) High-level DICOM abstractions for the Python programming language ([35995898](https://pubmed.ncbi.nlm.nih.gov/35995898))
260. [mfumagalli/ngsTools](https://github.com/mfumagalli/ngsTools) (‚≠ê153 ¬∑ R) Programs to analyse NGS data for population genetics purposes ([24458950](https://pubmed.ncbi.nlm.nih.gov/24458950))
261. [mkirsche/Jasmine](https://github.com/mkirsche/Jasmine) (‚≠ê153 ¬∑ Java) Jasmine: SV Merging Across Samples ([36658279](https://pubmed.ncbi.nlm.nih.gov/36658279))
262. [bjing2016/EigenFold](https://github.com/bjing2016/EigenFold) (‚≠ê152 ¬∑ Jupyter Notebook) EigenFold: Generative Protein Structure Prediction with Diffusion Models ([37064532](https://pubmed.ncbi.nlm.nih.gov/37064532))
263. [AnantharamanLab/METABOLIC](https://github.com/AnantharamanLab/METABOLIC) (‚≠ê151 ¬∑ Perl) A scalable high-throughput metabolic and biogeochemical functional trait profiler ([35172890](https://pubmed.ncbi.nlm.nih.gov/35172890))
264. [XinhaoLi74/SmilesPE](https://github.com/XinhaoLi74/SmilesPE) (‚≠ê151 ¬∑ Jupyter Notebook) SMILES Pair Encoding: A data-driven substructure representation of chemicals ([33715361](https://pubmed.ncbi.nlm.nih.gov/33715361))
265. [divelab/MoleculeX](https://github.com/divelab/MoleculeX) (‚≠ê151 ¬∑ Python) None ([35179547](https://pubmed.ncbi.nlm.nih.gov/35179547))
266. [duartegroup/autodE](https://github.com/duartegroup/autodE) (‚≠ê151 ¬∑ Python) automated reaction profile generation ([33108028](https://pubmed.ncbi.nlm.nih.gov/33108028))
267. [pymzML/pymzML](https://github.com/pymzML/pymzML) (‚≠ê151 ¬∑ Python) pymzML - an interface between Python and mzML Mass spectrometry Files ([29394323](https://pubmed.ncbi.nlm.nih.gov/29394323))
268. [dsehnal/LiteMol](https://github.com/dsehnal/LiteMol) (‚≠ê150 ¬∑ JavaScript) A library/plugin for handling 3D structural molecular data (not only) in the browser. ([30179493](https://pubmed.ncbi.nlm.nih.gov/30179493))
269. [nickabattista/IB2d](https://github.com/nickabattista/IB2d) (‚≠ê150 ¬∑ MATLAB) An easy to use immersed boundary method in 2D, with full implementations in MATLAB and Python that contains over 75 built-in examples, including multiple options for fiber-structure models and advection-diffusion, Boussinesq approximations, and/or artificial forcing. ([28355146](https://pubmed.ncbi.nlm.nih.gov/28355146))
270. [abyzovlab/CNVpytor](https://github.com/abyzovlab/CNVpytor) (‚≠ê149 ¬∑ Python) a python extension of CNVnator -- a tool for CNV analysis from depth-of-coverage by mapped reads ([34817058](https://pubmed.ncbi.nlm.nih.gov/34817058))
271. [KorfLab/StochHMM](https://github.com/KorfLab/StochHMM) (‚≠ê147 ¬∑ C++) A Hidden Markov Model Application and C++ Library for Rapid and Flexible Development of HMMs ([24489371](https://pubmed.ncbi.nlm.nih.gov/24489371))
272. [crsl4/PhyloNetworks](https://github.com/crsl4/PhyloNetworks) (‚≠ê147 ¬∑ Julia) A Julia package for statistical inference, data manipulation and visualization of phylogenetic networks ([28961984](https://pubmed.ncbi.nlm.nih.gov/28961984))
273. [marcelauliano/MitoHiFi](https://github.com/marcelauliano/MitoHiFi) (‚≠ê147 ¬∑ Python) Find, circularise and annotate mitogenome from PacBio assemblies ([37464285](https://pubmed.ncbi.nlm.nih.gov/37464285))
274. [venyao/shinyCircos](https://github.com/venyao/shinyCircos) (‚≠ê146 ¬∑ HTML) an R/shiny application for creation of Circos plot interactively ([29186362](https://pubmed.ncbi.nlm.nih.gov/29186362))
275. [statgen/locuszoom](https://github.com/statgen/locuszoom) (‚≠ê145 ¬∑ JavaScript) A Javascript/d3 embeddable plugin for interactively visualizing statistical genetic data from customizable sources. ([33734315](https://pubmed.ncbi.nlm.nih.gov/33734315))
276. [BenevolentAI/DeeplyTough](https://github.com/BenevolentAI/DeeplyTough) (‚≠ê144 ¬∑ Python) DeeplyTough: Learning Structural Comparison of Protein Binding Sites ([32023053](https://pubmed.ncbi.nlm.nih.gov/32023053))
277. [ComparativeGenomicsToolkit/hal](https://github.com/ComparativeGenomicsToolkit/hal) (‚≠ê144 ¬∑ C++) Hierarchical Alignment Format ([32463100](https://pubmed.ncbi.nlm.nih.gov/32463100))
278. [ZJUDataIntelligence/TEMPO](https://github.com/ZJUDataIntelligence/TEMPO) (‚≠ê144 ¬∑ Python) None ([36535209](https://pubmed.ncbi.nlm.nih.gov/36535209))
279. [divonlan/genozip](https://github.com/divonlan/genozip) (‚≠ê144 ¬∑ C) A modern compressor for genomic files (FASTQ, SAM/BAM/CRAM, VCF, FASTA, GFF/GTF/GVF, 23andMe...), up to 5x better than gzip and faster too ([32407471](https://pubmed.ncbi.nlm.nih.gov/32407471))
280. [glennhickey/hal](https://github.com/glennhickey/hal) (‚≠ê144 ¬∑ C++) Hierarchical Alignment Format ([23505295](https://pubmed.ncbi.nlm.nih.gov/23505295))
281. [tsudalab/ChemTS](https://github.com/tsudalab/ChemTS) (‚≠ê144 ¬∑ Python) Molecule Design using Monte Carlo Tree Search with Neural Rollout ([29435094](https://pubmed.ncbi.nlm.nih.gov/29435094))
282. [OpenGene/MutScan](https://github.com/OpenGene/MutScan) (‚≠ê143 ¬∑ C) Detect and visualize target mutations by scanning FastQ files directly ([29357822](https://pubmed.ncbi.nlm.nih.gov/29357822))
283. [ekg/seqwish](https://github.com/ekg/seqwish) (‚≠ê141 ¬∑ C++) alignment to variation graph inducer ([36448683](https://pubmed.ncbi.nlm.nih.gov/36448683))
284. [mschubert/clustermq](https://github.com/mschubert/clustermq) (‚≠ê141 ¬∑ R) R package to send function calls as jobs on LSF, SGE, Slurm, PBS/Torque, or each via SSH ([31134271](https://pubmed.ncbi.nlm.nih.gov/31134271))
285. [saezlab/decoupleR](https://github.com/saezlab/decoupleR) (‚≠ê141 ¬∑ R) R package to infer biological activities from omics data using a collection of methods. ([36699385](https://pubmed.ncbi.nlm.nih.gov/36699385))
286. [CellProfiler/CellProfiler-Analyst](https://github.com/CellProfiler/CellProfiler-Analyst) (‚≠ê139 ¬∑ Python) Open-source software for exploring and analyzing large, high-dimensional image-derived data. ([27354701](https://pubmed.ncbi.nlm.nih.gov/27354701); [34478488](https://pubmed.ncbi.nlm.nih.gov/34478488))
287. [LinearFold/LinearFold](https://github.com/LinearFold/LinearFold) (‚≠ê139 ¬∑ C++) The new source code of the LinearFold, linear-time prediction for RNA secondary structures ([31510672](https://pubmed.ncbi.nlm.nih.gov/31510672))
288. [eldariont/svim](https://github.com/eldariont/svim) (‚≠ê139 ¬∑ Python) Structural Variant Identification Method using Long Reads ([30668829](https://pubmed.ncbi.nlm.nih.gov/30668829))
289. [fenderglass/Ragout](https://github.com/fenderglass/Ragout) (‚≠ê139 ¬∑ Python) Chromosome-level scaffolding using multiple references ([24931998](https://pubmed.ncbi.nlm.nih.gov/24931998))
290. [huangnengCSU/compleasm](https://github.com/huangnengCSU/compleasm) (‚≠ê139 ¬∑ Python) A genome completeness evaluation tool based on miniprot ([37758247](https://pubmed.ncbi.nlm.nih.gov/37758247))
291. [neurodata/neuroparc](https://github.com/neurodata/neuroparc) (‚≠ê139 ¬∑ Jupyter Notebook) None ([33686079](https://pubmed.ncbi.nlm.nih.gov/33686079))
292. [millanek/Dsuite](https://github.com/millanek/Dsuite) (‚≠ê138 ¬∑ C++) Fast calculation of Patterson's D (ABBA-BABA) and the f4-ratio statistics across many populations/species ([33012121](https://pubmed.ncbi.nlm.nih.gov/33012121))
293. [sequana/sequana](https://github.com/sequana/sequana) (‚≠ê138 ¬∑ Jupyter Notebook) Sequana: a set of Snakemake NGS pipelines ([29361152](https://pubmed.ncbi.nlm.nih.gov/29361152))
294. [krishnanlab/PecanPy](https://github.com/krishnanlab/PecanPy) (‚≠ê137 ¬∑ Python) A fast, parallelized, memory efficient, and cache-optimized Python implementation of node2vec ([33760066](https://pubmed.ncbi.nlm.nih.gov/33760066))
295. [bjornwallner/DockQ](https://github.com/bjornwallner/DockQ) (‚≠ê136 ¬∑ Python) DockQ is a single continuous quality measure for protein docked models based on the CAPRI evaluation protocol ([27560519](https://pubmed.ncbi.nlm.nih.gov/27560519))
296. [ncbi/icn3d](https://github.com/ncbi/icn3d) (‚≠ê136 ¬∑ JavaScript) web-based protein structure viewer and analysis tool interactively or in batch mode ([31218344](https://pubmed.ncbi.nlm.nih.gov/31218344); [35252351](https://pubmed.ncbi.nlm.nih.gov/35252351))
297. [AdmiralenOla/Scoary](https://github.com/AdmiralenOla/Scoary) (‚≠ê135 ¬∑ Python) Pan-genome wide association studies ([27887642](https://pubmed.ncbi.nlm.nih.gov/27887642))
298. [mlyg/unified-focal-loss](https://github.com/mlyg/unified-focal-loss) (‚≠ê135 ¬∑ Python) None ([34953431](https://pubmed.ncbi.nlm.nih.gov/34953431))
299. [smarco/WFA](https://github.com/smarco/WFA) (‚≠ê135 ¬∑ C) WFA-lib: Wavefront alignment algorithm library v2 ([32915952](https://pubmed.ncbi.nlm.nih.gov/32915952))
300. [theislab/kBET](https://github.com/theislab/kBET) (‚≠ê135 ¬∑ HTML) An R package to test for batch effects in high-dimensional single-cell RNA sequencing data. ([30573817](https://pubmed.ncbi.nlm.nih.gov/30573817))
301. [Ensembl/ensembl-rest](https://github.com/Ensembl/ensembl-rest) (‚≠ê134 ¬∑ Perl) Language agnostic RESTful data access to Ensembl data over HTTP ([25236461](https://pubmed.ncbi.nlm.nih.gov/25236461))
302. [getzlab/rnaseqc](https://github.com/getzlab/rnaseqc) (‚≠ê134 ¬∑ C++) Fast, efficient RNA-Seq metrics for quality control and process optimization ([33677499](https://pubmed.ncbi.nlm.nih.gov/33677499))
303. [lsyads/fisheye-ORB-SLAM](https://github.com/lsyads/fisheye-ORB-SLAM) (‚≠ê134 ¬∑ C++) A real-time robust monocular visual SLAM system based on ORB-SLAM for fisheye cameras, without rectifying or cropping the input images ([31623266](https://pubmed.ncbi.nlm.nih.gov/31623266))
304. [masqm/Faster-Mean-Shift](https://github.com/masqm/Faster-Mean-Shift) (‚≠ê134 ¬∑ Python) GPU accelerated Faster Mean-shift for cosine embedding clustering ([33872961](https://pubmed.ncbi.nlm.nih.gov/33872961))
305. [stemangiola/tidyseurat](https://github.com/stemangiola/tidyseurat) (‚≠ê134 ¬∑ R) Seurat meets tidyverse. The best of both worlds. ([34028547](https://pubmed.ncbi.nlm.nih.gov/34028547))
306. [jotech/gapseq](https://github.com/jotech/gapseq) (‚≠ê133 ¬∑ R) Informed prediction and analysis of bacterial metabolic pathways and genome-scale networks ([33691770](https://pubmed.ncbi.nlm.nih.gov/33691770))
307. [molML/MoleculeACE](https://github.com/molML/MoleculeACE) (‚≠ê133 ¬∑ Python) A tool for evaluating the predictive performance on activity cliff compounds of machine learning models ([36456532](https://pubmed.ncbi.nlm.nih.gov/36456532))
308. [ryanlayer/gqt](https://github.com/ryanlayer/gqt) (‚≠ê133 ¬∑ C) A genotype query interface. ([26550772](https://pubmed.ncbi.nlm.nih.gov/26550772))
309. [CBICA/GaNDLF](https://github.com/CBICA/GaNDLF) (‚≠ê132 ¬∑ Python) A generalizable application framework for segmentation, regression, and classification using PyTorch ([35465060](https://pubmed.ncbi.nlm.nih.gov/35465060))
310. [OpenBioLink/OpenBioLink](https://github.com/OpenBioLink/OpenBioLink) (‚≠ê132 ¬∑ Python) OpenBioLink is a resource and evaluation framework for evaluating link prediction models on heterogeneous biomedical graph data. ([32339214](https://pubmed.ncbi.nlm.nih.gov/32339214))
311. [royfrancis/pophelper](https://github.com/royfrancis/pophelper) (‚≠ê132 ¬∑ R) An R package to analyse and visualise admixture proportions from STRUCTURE, fastSTRUCTURE, TESS, ADMIXTURE etc. ([26850166](https://pubmed.ncbi.nlm.nih.gov/26850166))
312. [walaj/SeqLib](https://github.com/walaj/SeqLib) (‚≠ê132 ¬∑ C++) C++ htslib/bwa-mem/fermi interface for interrogating sequence data ([28011768](https://pubmed.ncbi.nlm.nih.gov/28011768))
313. [SheffieldML/GPmat](https://github.com/SheffieldML/GPmat) (‚≠ê131 ¬∑ Matlab) Matlab implementations of Gaussian processes and other machine learning tools. ([24618470](https://pubmed.ncbi.nlm.nih.gov/24618470))
314. [deepmodeling/DMFF](https://github.com/deepmodeling/DMFF) (‚≠ê131 ¬∑ Python) DMFF (Differentiable Molecular Force Field) is a Jax-based python package that provides a full differentiable implementation of molecular force field models. ([37589304](https://pubmed.ncbi.nlm.nih.gov/37589304))
315. [ncclabsustech/EEGdenoiseNet](https://github.com/ncclabsustech/EEGdenoiseNet) (‚≠ê131 ¬∑ Jupyter Notebook) EEGdenoiseNet, a benchmark dataset, that is suited for training and testing deep learning-based EEG denoising models, as well as for comparing the performance across different models. ([34596046](https://pubmed.ncbi.nlm.nih.gov/34596046))
316. [aertslab/cistopic](https://github.com/aertslab/cistopic) (‚≠ê130 ¬∑ R) cisTopic: Probabilistic modelling of cis-regulatory topics from single cell epigenomics data ([30962623](https://pubmed.ncbi.nlm.nih.gov/30962623))
317. [benjjneb/decontam](https://github.com/benjjneb/decontam) (‚≠ê130 ¬∑ R) Simple statistical identification and removal of contaminants in marker-gene and metagenomics sequencing data ([30558668](https://pubmed.ncbi.nlm.nih.gov/30558668))
318. [Teichlab/bbknn](https://github.com/Teichlab/bbknn) (‚≠ê129 ¬∑ Python) Batch balanced KNN ([31400197](https://pubmed.ncbi.nlm.nih.gov/31400197))
319. [galaxyproject/SARS-CoV-2](https://github.com/galaxyproject/SARS-CoV-2) (‚≠ê128 ¬∑ Jupyter Notebook) Ongoing analysis of COVID-19 using Galaxy, BioConda and public research infrastructures ([32790776](https://pubmed.ncbi.nlm.nih.gov/32790776))
320. [lanl001/halc](https://github.com/lanl001/halc) (‚≠ê128 ¬∑ C++) High Throughput Algorithm for Long Read Error Correction ([28381259](https://pubmed.ncbi.nlm.nih.gov/28381259))
321. [theislab/scCODA](https://github.com/theislab/scCODA) (‚≠ê127 ¬∑ Jupyter Notebook) A Bayesian model for compositional single-cell data analysis ([34824236](https://pubmed.ncbi.nlm.nih.gov/34824236))
322. [antigenomics/vdjdb-db](https://github.com/antigenomics/vdjdb-db) (‚≠ê126 ¬∑ Groovy) üóÇÔ∏è [vdjdb.cdr3.net is up and running] Git-based TCR database storage & management. Submissions welcome! ([28977646](https://pubmed.ncbi.nlm.nih.gov/28977646))
323. [brentp/echtvar](https://github.com/brentp/echtvar) (‚≠ê126 ¬∑ Rust) using all the bits for echt rapid variant annotation and filtering ([36300617](https://pubmed.ncbi.nlm.nih.gov/36300617))
324. [churchmanlab/genewalk](https://github.com/churchmanlab/genewalk) (‚≠ê126 ¬∑ Python) GeneWalk identifies relevant gene functions for a biological context using network representation learning ([33526072](https://pubmed.ncbi.nlm.nih.gov/33526072))
325. [marianattestad/assemblytics](https://github.com/marianattestad/assemblytics) (‚≠ê126 ¬∑ JavaScript) Assemblytics is a bioinformatics tool to detect and analyze structural variants from a genome assembly by comparing it to a reference genome. ([27318204](https://pubmed.ncbi.nlm.nih.gov/27318204))
326. [nadeemlab/DeepLIIF](https://github.com/nadeemlab/DeepLIIF) (‚≠ê126 ¬∑ Python) Deep Learning Inferred Multiplex ImmunoFluorescence for IHC Image Quantification (https://deepliif.org) [Nature Machine Intelligence'22, CVPR'22, MICCAI'23, Histopathology'23] ([37292462](https://pubmed.ncbi.nlm.nih.gov/37292462); [37841230](https://pubmed.ncbi.nlm.nih.gov/37841230))
327. [nadeemlab/deepliif](https://github.com/nadeemlab/deepliif) (‚≠ê126 ¬∑ Python) Deep Learning Inferred Multiplex ImmunoFluorescence for IHC Image Quantification (https://deepliif.org) [Nature Machine Intelligence'22, CVPR'22, MICCAI'23, Histopathology'23] ([36118303](https://pubmed.ncbi.nlm.nih.gov/36118303))
328. [vals/umis](https://github.com/vals/umis) (‚≠ê126 ¬∑ Python) Tools for processing UMI RNA-tag data ([28263961](https://pubmed.ncbi.nlm.nih.gov/28263961))
329. [gabraham/flashpca](https://github.com/gabraham/flashpca) (‚≠ê125 ¬∑ C++) Fast Principal Component Analysis of Large-Scale Genome-Wide Data ([28475694](https://pubmed.ncbi.nlm.nih.gov/28475694))
330. [hakyimlab/MetaXcan](https://github.com/hakyimlab/MetaXcan) (‚≠ê125 ¬∑ Python) MetaXcan software and manuscript ([29752479](https://pubmed.ncbi.nlm.nih.gov/29752479))
331. [ATOMconsortium/AMPL](https://github.com/ATOMconsortium/AMPL) (‚≠ê124 ¬∑ Jupyter Notebook) The ATOM Modeling PipeLine (AMPL) is an open-source, modular, extensible software pipeline for building and sharing models to advance in silico drug discovery. ([32243153](https://pubmed.ncbi.nlm.nih.gov/32243153))
332. [CapsuleEndoscope/VirtualCapsuleEndoscopy](https://github.com/CapsuleEndoscope/VirtualCapsuleEndoscopy) (‚≠ê124 ¬∑ C#) VR-Caps: A Virtual Environment for Active Capsule Endoscopy ([33609920](https://pubmed.ncbi.nlm.nih.gov/33609920))
333. [gbouras13/pharokka](https://github.com/gbouras13/pharokka) (‚≠ê124 ¬∑ Python) fast phage annotation program ([36453861](https://pubmed.ncbi.nlm.nih.gov/36453861))
334. [zhanxw/rvtests](https://github.com/zhanxw/rvtests) (‚≠ê124 ¬∑ C++) Rare variant test software for next generation sequencing data ([27153000](https://pubmed.ncbi.nlm.nih.gov/27153000))
335. [healthylaife/MIMIC-IV-Data-Pipeline](https://github.com/healthylaife/MIMIC-IV-Data-Pipeline) (‚≠ê123 ¬∑ Jupyter Notebook) A customizable pipeline for data extraction from MIMIC-IV. ([36686986](https://pubmed.ncbi.nlm.nih.gov/36686986))
336. [hewm2008/NGenomeSyn](https://github.com/hewm2008/NGenomeSyn) (‚≠ê123 ¬∑ Perl) Any Way to Show  Multi genomic Synteny ([36883694](https://pubmed.ncbi.nlm.nih.gov/36883694))
337. [imgag/ngs-bits](https://github.com/imgag/ngs-bits) (‚≠ê123 ¬∑ C++) Short-read sequencing tools ([27161244](https://pubmed.ncbi.nlm.nih.gov/27161244); [28130233](https://pubmed.ncbi.nlm.nih.gov/28130233))
338. [mikessh/vdjtools](https://github.com/mikessh/vdjtools) (‚≠ê123 ¬∑ Groovy) Post-analysis of immune repertoire sequencing data ([26606115](https://pubmed.ncbi.nlm.nih.gov/26606115))
339. [xlucpu/MOVICS](https://github.com/xlucpu/MOVICS) (‚≠ê123 ¬∑ R) Multi-Omics integration and VIsualization in Cancer Subtyping ([33315104](https://pubmed.ncbi.nlm.nih.gov/33315104))
340. [rrwick/Deepbinner](https://github.com/rrwick/Deepbinner) (‚≠ê122 ¬∑ Python) a signal-level demultiplexer for Oxford Nanopore reads ([30458005](https://pubmed.ncbi.nlm.nih.gov/30458005))
341. [stefpeschel/NetCoMi](https://github.com/stefpeschel/NetCoMi) (‚≠ê122 ¬∑ R) Network construction, analysis, and comparison for microbial compositional data ([33264391](https://pubmed.ncbi.nlm.nih.gov/33264391))
342. [SGDDNB/ShinyCell](https://github.com/SGDDNB/ShinyCell) (‚≠ê121 ¬∑ R) Shiny Interactive Web Apps for Single-Cell Data ([33774659](https://pubmed.ncbi.nlm.nih.gov/33774659))
343. [bigdatagenomics/mango](https://github.com/bigdatagenomics/mango) (‚≠ê121 ¬∑ Scala) A scalable genome browser. Apache 2 licensed. ([29308454](https://pubmed.ncbi.nlm.nih.gov/29308454))
344. [lh3/bedtk](https://github.com/lh3/bedtk) (‚≠ê121 ¬∑ C) A simple toolset for BED files (warning: CLI may change before bedtk becomes stable) ([32966548](https://pubmed.ncbi.nlm.nih.gov/32966548))
345. [ncbi/TPMCalculator](https://github.com/ncbi/TPMCalculator) (‚≠ê121 ¬∑ C++) TPMCalculator quantifies mRNA abundance directly from the alignments by parsing BAM files ([30379987](https://pubmed.ncbi.nlm.nih.gov/30379987))
346. [xulabs/aitom](https://github.com/xulabs/aitom) (‚≠ê121 ¬∑ Python) AI for tomography ([32722755](https://pubmed.ncbi.nlm.nih.gov/32722755); [33620460](https://pubmed.ncbi.nlm.nih.gov/33620460); [34897387](https://pubmed.ncbi.nlm.nih.gov/34897387))
347. [BBillot/SynthSR](https://github.com/BBillot/SynthSR) (‚≠ê120 ¬∑ Python) A framework for joint super-resolution and image synthesis, without requiring real training data ([34048902](https://pubmed.ncbi.nlm.nih.gov/34048902))
348. [muhammedtalo/COVID-19](https://github.com/muhammedtalo/COVID-19) (‚≠ê120 ¬∑ Jupyter Notebook) Automated Detection of COVID-19 Cases Using Deep Neural Networks with X-Ray Images ([32568675](https://pubmed.ncbi.nlm.nih.gov/32568675))
349. [songc001/remilo](https://github.com/songc001/remilo) (‚≠ê120 ¬∑ C++) Reference Assisted Misassembly Detection Algorithm  Using Short and Long Reads ([28961789](https://pubmed.ncbi.nlm.nih.gov/28961789))
350. [spetti/SMURF](https://github.com/spetti/SMURF) (‚≠ê119 ¬∑ Python) None ([36355460](https://pubmed.ncbi.nlm.nih.gov/36355460))
351. [theislab/sc-pert](https://github.com/theislab/sc-pert) (‚≠ê119 ¬∑ Jupyter Notebook) Models and datasets for perturbational single-cell omics ([34139164](https://pubmed.ncbi.nlm.nih.gov/34139164))
352. [Illumina/canvas](https://github.com/Illumina/canvas) (‚≠ê118 ¬∑ C#) Canvas - Copy number variant (CNV) calling from DNA sequencing data ([27153601](https://pubmed.ncbi.nlm.nih.gov/27153601); [29028893](https://pubmed.ncbi.nlm.nih.gov/29028893))
353. [baoe/flas](https://github.com/baoe/flas) (‚≠ê118 ¬∑ C++) Fast and High Throughput Algorithm for PacBio Long Read Self-Correction ([30895306](https://pubmed.ncbi.nlm.nih.gov/30895306))
354. [echen102/us-pres-elections-2020](https://github.com/echen102/us-pres-elections-2020) (‚≠ê118 ¬∑ Python) The repository contains a collection of tweets IDs associated with the 2020 U.S. Presidential Elections through 6 months post-inauguration. ([33824934](https://pubmed.ncbi.nlm.nih.gov/33824934))
355. [hasindu2008/f5c](https://github.com/hasindu2008/f5c) (‚≠ê118 ¬∑ C) Ultra-fast methylation calling and event alignment tool for nanopore sequencing data (supports CUDA acceleration) ([32758139](https://pubmed.ncbi.nlm.nih.gov/32758139))
356. [saezlab/decoupler-py](https://github.com/saezlab/decoupler-py) (‚≠ê118 ¬∑ Python) Python package to perform enrichment analysis from omics data. ([36699385](https://pubmed.ncbi.nlm.nih.gov/36699385))
357. [BlueBrain/NeuroMorphoVis](https://github.com/BlueBrain/NeuroMorphoVis) (‚≠ê117 ¬∑ Python) A lightweight, interactive, extensible and cross-platform framework for building, visualizing and analyzing digital reconstructions of neuronal morphology skeletons extracted from microscopy stacks. ([29949998](https://pubmed.ncbi.nlm.nih.gov/29949998))
358. [Gabaldonlab/redundans](https://github.com/Gabaldonlab/redundans) (‚≠ê117 ¬∑ C++) Redundans is a pipeline that assists an assembly of heterozygous/polymorphic genomes. ([27131372](https://pubmed.ncbi.nlm.nih.gov/27131372))
359. [cruizperez/MicrobeAnnotator](https://github.com/cruizperez/MicrobeAnnotator) (‚≠ê117 ¬∑ Python) Pipeline for metabolic annotation of microbial genomes ([33407081](https://pubmed.ncbi.nlm.nih.gov/33407081))
360. [Merck/BioPhi](https://github.com/Merck/BioPhi) (‚≠ê116 ¬∑ Python) BioPhi is an open-source antibody design platform. It features methods for automated antibody humanization (Sapiens), humanness evaluation (OASis) and an interface for computer-assisted antibody sequence design. ([35133949](https://pubmed.ncbi.nlm.nih.gov/35133949))
361. [jdidion/atropos](https://github.com/jdidion/atropos) (‚≠ê116 ¬∑ Python) An NGS read trimming tool that is specific, sensitive, and speedy. (production) ([28875074](https://pubmed.ncbi.nlm.nih.gov/28875074))
362. [parklab/NGSCheckMate](https://github.com/parklab/NGSCheckMate) (‚≠ê116 ¬∑ Python) Software program for checking sample matching for NGS data ([28369524](https://pubmed.ncbi.nlm.nih.gov/28369524))
363. [OpenGene/repaq](https://github.com/OpenGene/repaq) (‚≠ê115 ¬∑ C) A fast lossless FASTQ compressor with ultra-high compression ratio ([37811144](https://pubmed.ncbi.nlm.nih.gov/37811144))
364. [ksahlin/strobealign](https://github.com/ksahlin/strobealign) (‚≠ê115 ¬∑ C++) Aligns short reads using dynamic seed size with strobemers ([36522758](https://pubmed.ncbi.nlm.nih.gov/36522758))
365. [DiltheyLab/HLA-LA](https://github.com/DiltheyLab/HLA-LA) (‚≠ê114 ¬∑ C++) Fast HLA type inference from whole-genome data ([30942877](https://pubmed.ncbi.nlm.nih.gov/30942877))
366. [yatisht/usher](https://github.com/yatisht/usher) (‚≠ê114 ¬∑ C++) Ultrafast Sample Placement on Existing Trees ([34469548](https://pubmed.ncbi.nlm.nih.gov/34469548); [35731204](https://pubmed.ncbi.nlm.nih.gov/35731204))
367. [AndersenLab/VCF-kit](https://github.com/AndersenLab/VCF-kit) (‚≠ê113 ¬∑ Python) VCF-kit: Assorted utilities for the variant call format ([28093408](https://pubmed.ncbi.nlm.nih.gov/28093408))
368. [XiaoleiLiuBio/G2P](https://github.com/XiaoleiLiuBio/G2P) (‚≠ê113 ¬∑ None) G2P: A Genome-Wide-Association-Study Simulation Tool for Genotype Simulation, Phenotype Simulation, and Power Evaluation ([30848784](https://pubmed.ncbi.nlm.nih.gov/30848784))
369. [hammerlab/cytokit](https://github.com/hammerlab/cytokit) (‚≠ê113 ¬∑ Jupyter Notebook) Microscopy Image Cytometry Toolkit ([31477013](https://pubmed.ncbi.nlm.nih.gov/31477013))
370. [oxpig/DeLinker](https://github.com/oxpig/DeLinker) (‚≠ê113 ¬∑ Python) None ([32195587](https://pubmed.ncbi.nlm.nih.gov/32195587))
371. [spacegraphcats/spacegraphcats](https://github.com/spacegraphcats/spacegraphcats) (‚≠ê113 ¬∑ Standard ML) Indexing & querying large assembly graphs -- in space, no one can hear you miao! ([32631445](https://pubmed.ncbi.nlm.nih.gov/32631445))
372. [wanderine/BROCCOLI](https://github.com/wanderine/BROCCOLI) (‚≠ê113 ¬∑ C++) BROCCOLI: Software for Fast fMRI Analysis on Many-Core CPUs and GPUs ([24672471](https://pubmed.ncbi.nlm.nih.gov/24672471))
373. [ChaissonLab/LRA](https://github.com/ChaissonLab/LRA) (‚≠ê112 ¬∑ C++) Long read aligner ([34153026](https://pubmed.ncbi.nlm.nih.gov/34153026))
374. [OpenGene/GeneFuse](https://github.com/OpenGene/GeneFuse) (‚≠ê112 ¬∑ C) Gene fusion detection and visualization ([29989075](https://pubmed.ncbi.nlm.nih.gov/29989075))
375. [biojava/biojava-tutorial](https://github.com/biojava/biojava-tutorial) (‚≠ê112 ¬∑ Python) :book::books:An introductory tutorial for BioJava ([30735498](https://pubmed.ncbi.nlm.nih.gov/30735498))
376. [neuropoly/axondeepseg](https://github.com/neuropoly/axondeepseg) (‚≠ê112 ¬∑ Python) Axon/Myelin segmentation using Deep Learning ([29491478](https://pubmed.ncbi.nlm.nih.gov/29491478))
377. [torognes/swarm](https://github.com/torognes/swarm) (‚≠ê112 ¬∑ C++) A robust and fast clustering method for amplicon-based studies ([34244702](https://pubmed.ncbi.nlm.nih.gov/34244702))
378. [MRCIEU/PHESANT](https://github.com/MRCIEU/PHESANT) (‚≠ê111 ¬∑ R) PHESANT - PHEnome Scan ANalysis Tool (pheWAS, Mendelian randomisation (MR)-pheWAS etc.) in UK Biobank ([29040602](https://pubmed.ncbi.nlm.nih.gov/29040602))
379. [lingluodlut/Att-ChemdNER](https://github.com/lingluodlut/Att-ChemdNER) (‚≠ê110 ¬∑ Python) Att-ChemdNER ([29186323](https://pubmed.ncbi.nlm.nih.gov/29186323))
380. [timolassmann/kalign](https://github.com/timolassmann/kalign) (‚≠ê110 ¬∑ C) A fast multiple sequence alignment program. ([31665271](https://pubmed.ncbi.nlm.nih.gov/31665271))
381. [JiekaiLab/scDIOR](https://github.com/JiekaiLab/scDIOR) (‚≠ê109 ¬∑ HTML) scDIOR: Single cell data IO softwaRe ([34991457](https://pubmed.ncbi.nlm.nih.gov/34991457))
382. [jameshadfield/phandango](https://github.com/jameshadfield/phandango) (‚≠ê109 ¬∑ JavaScript) an interactive viewer for populations of bacterial genomes linked by a phylogeny ([29028899](https://pubmed.ncbi.nlm.nih.gov/29028899))
383. [mcveanlab/mccortex](https://github.com/mcveanlab/mccortex) (‚≠ê109 ¬∑ C) De novo genome assembly and multisample variant calling ([29554215](https://pubmed.ncbi.nlm.nih.gov/29554215))
384. [biocore/mmvec](https://github.com/biocore/mmvec) (‚≠ê108 ¬∑ Python) Neural networks for microbe-metabolite interaction analysis ([31686038](https://pubmed.ncbi.nlm.nih.gov/31686038))
385. [dzhang32/ggtranscript](https://github.com/dzhang32/ggtranscript) (‚≠ê108 ¬∑ R) Visualizing transcript structure and annotation using ggplot2 ([35751589](https://pubmed.ncbi.nlm.nih.gov/35751589))
386. [guigolab/ggsashimi](https://github.com/guigolab/ggsashimi) (‚≠ê108 ¬∑ Python) Command-line tool for the visualization of splicing events across multiple samples ([30118475](https://pubmed.ncbi.nlm.nih.gov/30118475))
387. [lh3/fermikit](https://github.com/lh3/fermikit) (‚≠ê108 ¬∑ TeX) De novo assembly based variant calling pipeline for Illumina short reads ([26220959](https://pubmed.ncbi.nlm.nih.gov/26220959))
388. [molgenis/molgenis](https://github.com/molgenis/molgenis) (‚≠ê108 ¬∑ Java) MOLGENIS - for scientific data: management, exploration, integration and analysis. ([27153686](https://pubmed.ncbi.nlm.nih.gov/27153686); [29036577](https://pubmed.ncbi.nlm.nih.gov/29036577); [29599418](https://pubmed.ncbi.nlm.nih.gov/29599418))
389. [OpenGene/fastv](https://github.com/OpenGene/fastv) (‚≠ê107 ¬∑ C++) An ultra-fast tool for identification of SARS-CoV-2 and other microbes from sequencing data. This tool can be used to detect viral infectious diseases, like COVID-19. ([33003197](https://pubmed.ncbi.nlm.nih.gov/33003197))
390. [Xinglab/DARTS](https://github.com/Xinglab/DARTS) (‚≠ê107 ¬∑ Python) Deep-learning Augmented RNA-seq analysis of Transcript Splicing ([30923373](https://pubmed.ncbi.nlm.nih.gov/30923373))
391. [juglab/EmbedSeg](https://github.com/juglab/EmbedSeg) (‚≠ê107 ¬∑ Python) Code Implementation for EmbedSeg, an Instance Segmentation Method for Microscopy Images ([35926335](https://pubmed.ncbi.nlm.nih.gov/35926335))
392. [linzhi2013/MitoZ](https://github.com/linzhi2013/MitoZ) (‚≠ê107 ¬∑ Shell) MitoZ: A toolkit for assembly, annotation, and visualization of animal mitochondrial genomes ([30864657](https://pubmed.ncbi.nlm.nih.gov/30864657))
393. [lykaust15/DeepSimulator](https://github.com/lykaust15/DeepSimulator) (‚≠ê107 ¬∑ C) The first deep learning based Nanopore simulator which can simulate the process of Nanopore sequencing. ([29659695](https://pubmed.ncbi.nlm.nih.gov/29659695); [31913436](https://pubmed.ncbi.nlm.nih.gov/31913436))
394. [mikevoets/jama16-retina-replication](https://github.com/mikevoets/jama16-retina-replication) (‚≠ê107 ¬∑ Python) JAMA 2016; 316(22) Replication Study ([31170223](https://pubmed.ncbi.nlm.nih.gov/31170223))
395. [ChristofferFlensburg/SuperFreq](https://github.com/ChristofferFlensburg/SuperFreq) (‚≠ê106 ¬∑ R) Analysis pipeline for cancer sequencing data ([34132781](https://pubmed.ncbi.nlm.nih.gov/34132781))
396. [OpenGene/gencore](https://github.com/OpenGene/gencore) (‚≠ê106 ¬∑ C++) Generate duplex/single consensus reads to reduce sequencing noises and remove duplications ([31881822](https://pubmed.ncbi.nlm.nih.gov/31881822))
397. [OpenTreeOfLife/opentree](https://github.com/OpenTreeOfLife/opentree) (‚≠ê106 ¬∑ JavaScript) Opentree browsing and curation web site.  For overarching or cross-repo concerns, please see the 'germinator' repo. ([25940563](https://pubmed.ncbi.nlm.nih.gov/25940563))
398. [igvteam/igv-webapp](https://github.com/igvteam/igv-webapp) (‚≠ê106 ¬∑ JavaScript) IGV Web App ([36562559](https://pubmed.ncbi.nlm.nih.gov/36562559))
399. [vanessajurtz/lasagne4bio](https://github.com/vanessajurtz/lasagne4bio) (‚≠ê106 ¬∑ Jupyter Notebook) None ([28961695](https://pubmed.ncbi.nlm.nih.gov/28961695))
400. [xiaochuanle/MECAT](https://github.com/xiaochuanle/MECAT) (‚≠ê106 ¬∑ C++) MECAT: an ultra-fast mapping, error correction and de novo assembly tool for single-molecule sequencing reads ([28945707](https://pubmed.ncbi.nlm.nih.gov/28945707))
401. [YaqiangCao/cLoops](https://github.com/YaqiangCao/cLoops) (‚≠ê105 ¬∑ Python) Accurate and flexible loops calling tool for 3D genomic data. ([31504161](https://pubmed.ncbi.nlm.nih.gov/31504161))
402. [cansyl/DEEPscreen](https://github.com/cansyl/DEEPscreen) (‚≠ê105 ¬∑ Python) DEEPScreen: Virtual Screening with Deep Convolutional Neural Networks Using Compound Images ([33209251](https://pubmed.ncbi.nlm.nih.gov/33209251))
403. [evolbioinfo/gotree](https://github.com/evolbioinfo/gotree) (‚≠ê105 ¬∑ Go) Gotree is a set of command line tools and an API to manipulate phylogenetic trees. It is implemented in Go language. ([34396097](https://pubmed.ncbi.nlm.nih.gov/34396097))
404. [lh3/minimap](https://github.com/lh3/minimap) (‚≠ê105 ¬∑ C) This repo is DEPRECATED. Please use minimap2, the successor of minimap. ([27153593](https://pubmed.ncbi.nlm.nih.gov/27153593))
405. [parklab/bamsnap](https://github.com/parklab/bamsnap) (‚≠ê105 ¬∑ HTML) None ([33416869](https://pubmed.ncbi.nlm.nih.gov/33416869))
406. [DeepRank/DeepRank-GNN](https://github.com/DeepRank/DeepRank-GNN) (‚≠ê104 ¬∑ Python) Graph Network for protein-protein interface ([36420989](https://pubmed.ncbi.nlm.nih.gov/36420989))
407. [MASILab/SLANTbrainSeg](https://github.com/MASILab/SLANTbrainSeg) (‚≠ê104 ¬∑ Python) Deep Whole Brain Segmentation Using SLANT Method ([30910724](https://pubmed.ncbi.nlm.nih.gov/30910724); [34999162](https://pubmed.ncbi.nlm.nih.gov/34999162))
408. [PheWAS/PheWAS](https://github.com/PheWAS/PheWAS) (‚≠ê104 ¬∑ R) The PheWAS R package ([29912272](https://pubmed.ncbi.nlm.nih.gov/29912272))
409. [carmonalab/UCell](https://github.com/carmonalab/UCell) (‚≠ê104 ¬∑ R) An R package for single-cell gene signature scoring ([34285779](https://pubmed.ncbi.nlm.nih.gov/34285779))
410. [morrislab/phylowgs](https://github.com/morrislab/phylowgs) (‚≠ê104 ¬∑ Python) Application for inferring subclonal composition and evolution from whole-genome sequencing data. ([25786235](https://pubmed.ncbi.nlm.nih.gov/25786235))
411. [thiviyanT/torch-rgcn](https://github.com/thiviyanT/torch-rgcn) (‚≠ê104 ¬∑ Python) A PyTorch implementation of the Relational Graph Convolutional Network (RGCN). ([36426239](https://pubmed.ncbi.nlm.nih.gov/36426239))
412. [BaderLab/saber](https://github.com/BaderLab/saber) (‚≠ê103 ¬∑ Python) Saber is a deep-learning based tool for information extraction in the biomedical domain. Pull requests are welcome! Note: this is a work in progress. Many things are broken, and the codebase is not stable. ([31218364](https://pubmed.ncbi.nlm.nih.gov/31218364))
413. [RabadanLab/arcasHLA](https://github.com/RabadanLab/arcasHLA) (‚≠ê103 ¬∑ Python) Fast and accurate in silico inference of HLA genotypes from RNA-seq ([31173059](https://pubmed.ncbi.nlm.nih.gov/31173059))
414. [bioinfomaticsCSU/deepsignal](https://github.com/bioinfomaticsCSU/deepsignal) (‚≠ê103 ¬∑ Python) Detecting methylation using signal-level features from Nanopore sequencing reads ([30994904](https://pubmed.ncbi.nlm.nih.gov/30994904))
415. [broadinstitute/ml4h](https://github.com/broadinstitute/ml4h) (‚≠ê103 ¬∑ Jupyter Notebook) None ([35157695](https://pubmed.ncbi.nlm.nih.gov/35157695))
416. [csyben/PYRO-NN](https://github.com/csyben/PYRO-NN) (‚≠ê103 ¬∑ Python) Python Reconstruction Operators in Neural Networks. High level python API for PYRO-NN-Layers ([31389023](https://pubmed.ncbi.nlm.nih.gov/31389023))
417. [hellloxiaotian/ADNet](https://github.com/hellloxiaotian/ADNet) (‚≠ê103 ¬∑ Python) Attention-guided CNN for image denoising(Neural Networks,2020) ([31991307](https://pubmed.ncbi.nlm.nih.gov/31991307))
418. [labsyspharm/ashlar](https://github.com/labsyspharm/ashlar) (‚≠ê103 ¬∑ Python) ASHLAR: Alignment by Simultaneous Harmonization of Layer/Adjacency Registration ([35972352](https://pubmed.ncbi.nlm.nih.gov/35972352))
419. [microsoft/frame-flow](https://github.com/microsoft/frame-flow) (‚≠ê103 ¬∑ Python) Fast protein backbone generation with SE(3) flow matching. ([38259348](https://pubmed.ncbi.nlm.nih.gov/38259348))
420. [thomas0809/MolScribe](https://github.com/thomas0809/MolScribe) (‚≠ê103 ¬∑ Python) Robust Molecular Structure Recognition with Image-to-Graph Generation ([36971363](https://pubmed.ncbi.nlm.nih.gov/36971363))
421. [yanwu2014/swne](https://github.com/yanwu2014/swne) (‚≠ê103 ¬∑ R) Similarity Weighted Nonnegative Embedding (SWNE), a method for visualizing high dimensional datasets ([30528274](https://pubmed.ncbi.nlm.nih.gov/30528274))
422. [althonos/pyhmmer](https://github.com/althonos/pyhmmer) (‚≠ê102 ¬∑ Cython) Cython bindings and Python interface to HMMER3. ([37074928](https://pubmed.ncbi.nlm.nih.gov/37074928))
423. [chrishah/MITObim](https://github.com/chrishah/MITObim) (‚≠ê102 ¬∑ Perl) MITObim - mitochondrial baiting and iterative mapping ([23661685](https://pubmed.ncbi.nlm.nih.gov/23661685))
424. [FePhyFoFum/phyx](https://github.com/FePhyFoFum/phyx) (‚≠ê101 ¬∑ C++) phylogenetics tools for linux (and other mostly posix compliant) computers ([28174903](https://pubmed.ncbi.nlm.nih.gov/28174903))
425. [bvieth/powsimR](https://github.com/bvieth/powsimR) (‚≠ê101 ¬∑ HTML) Power analysis is essential to optimize the design of RNA-seq experiments and to assess and compare the power to detect differentially expressed genes. PowsimR is a flexible tool to simulate and evaluate differential expression from bulk and especially single-cell RNA-seq data making it suitable for a priori and posterior power analyses. ([29036287](https://pubmed.ncbi.nlm.nih.gov/29036287))
426. [cozygene/FEAST](https://github.com/cozygene/FEAST) (‚≠ê101 ¬∑ R) Fast expectation maximization for microbial source tracking ([31182859](https://pubmed.ncbi.nlm.nih.gov/31182859))
427. [louiejtaylor/grabseqs](https://github.com/louiejtaylor/grabseqs) (‚≠ê101 ¬∑ Python) A utility for easy downloading of reads from next-gen sequencing repositories like NCBI SRA ([32154830](https://pubmed.ncbi.nlm.nih.gov/32154830))
428. [zhxiaokang/RASflow](https://github.com/zhxiaokang/RASflow) (‚≠ê101 ¬∑ HTML) RNA-Seq analysis workflow ([32183729](https://pubmed.ncbi.nlm.nih.gov/32183729))
429. [CostaLab/reg-gen](https://github.com/CostaLab/reg-gen) (‚≠ê100 ¬∑ Python) Regulatory Genomics Toolbox: Python library and set of tools for the integrative analysis of high throughput regulatory genomics data. ([36879236](https://pubmed.ncbi.nlm.nih.gov/36879236))
430. [mabessa/F3DAS](https://github.com/mabessa/F3DAS) (‚≠ê100 ¬∑ Python) Framework for Data-Driven Design & Analysis of Structures & Materials (F3DASM) ([31608516](https://pubmed.ncbi.nlm.nih.gov/31608516))
431. [ncbi/SKESA](https://github.com/ncbi/SKESA) (‚≠ê100 ¬∑ C++) SKESA assembler ([30286803](https://pubmed.ncbi.nlm.nih.gov/30286803))
432. [openPupil/Open-PupilEXT](https://github.com/openPupil/Open-PupilEXT) (‚≠ê100 ¬∑ C++) Official implementation of a free pupillometry platform called PupilEXT proposed in the article "PupilEXT: flexible open-source platform for high resolution pupillometry in vision research". ([34220432](https://pubmed.ncbi.nlm.nih.gov/34220432))
433. [oxpig/AbLang](https://github.com/oxpig/AbLang) (‚≠ê100 ¬∑ Python)  AbLang: A language model for antibodies ([36699403](https://pubmed.ncbi.nlm.nih.gov/36699403))
434. [ratschlab/spladder](https://github.com/ratschlab/spladder) (‚≠ê100 ¬∑ Python) Tool for the detection and quantification of alternative splicing events from RNA-Seq data. ([26873928](https://pubmed.ncbi.nlm.nih.gov/26873928))
435. [rrwick/perfect-bacterial-genome-tutorial](https://github.com/rrwick/perfect-bacterial-genome-tutorial) (‚≠ê100 ¬∑ Python) None ([36862631](https://pubmed.ncbi.nlm.nih.gov/36862631))
436. [shubhamchandak94/SPRING](https://github.com/shubhamchandak94/SPRING) (‚≠ê100 ¬∑ C) FASTQ compression ([30535063](https://pubmed.ncbi.nlm.nih.gov/30535063))
437. [szpiech/selscan](https://github.com/szpiech/selscan) (‚≠ê100 ¬∑ C) Haplotype based scans for selection ([25015648](https://pubmed.ncbi.nlm.nih.gov/25015648); [38180866](https://pubmed.ncbi.nlm.nih.gov/38180866))
438. [yangao07/abPOA](https://github.com/yangao07/abPOA) (‚≠ê100 ¬∑ C) abPOA: an SIMD-based C library for fast partial order alignment using adaptive band ([33165528](https://pubmed.ncbi.nlm.nih.gov/33165528))
439. [SBRG/ssbio](https://github.com/SBRG/ssbio) (‚≠ê99 ¬∑ Python) A Python framework for structural systems biology ([29444205](https://pubmed.ncbi.nlm.nih.gov/29444205))
440. [SunPengChuan/wgdi](https://github.com/SunPengChuan/wgdi) (‚≠ê99 ¬∑ Python) WGDI: A user-friendly toolkit for evolutionary analyses of whole-genome duplications and ancestral karyotypes ([36307977](https://pubmed.ncbi.nlm.nih.gov/36307977))
441. [biopragmatics/bioregistry](https://github.com/biopragmatics/bioregistry) (‚≠ê99 ¬∑ Python) üìÆ An integrative registry of biological databases, ontologies, and nomenclatures. ([36402838](https://pubmed.ncbi.nlm.nih.gov/36402838))
442. [hakyimlab/PrediXcan](https://github.com/hakyimlab/PrediXcan) (‚≠ê99 ¬∑ Python) PrediXcan Project ([27835642](https://pubmed.ncbi.nlm.nih.gov/27835642))
443. [zeeev/wham](https://github.com/zeeev/wham) (‚≠ê99 ¬∑ C++) Structural variant detection and association testing ([26625158](https://pubmed.ncbi.nlm.nih.gov/26625158))
444. [3DGenomes/tadbit](https://github.com/3DGenomes/tadbit) (‚≠ê98 ¬∑ Python) TADbit is a complete Python library to deal with all steps to analyze, model and explore 3C-based data. With TADbit the user can map FASTQ files to obtain raw interaction binned matrices (Hi-C like matrices), normalize and correct interaction matrices, identify and compare the so-called Topologically Associating Domains (TADs), build 3D models from the interaction matrices, and finally, extract structural properties from the models. TADbit is complemented by TADkit for visualizing 3D models ([28723903](https://pubmed.ncbi.nlm.nih.gov/28723903))
445. [adamewing/methylartist](https://github.com/adamewing/methylartist) (‚≠ê98 ¬∑ Python) Tools for plotting methylation data in various ways ([35482479](https://pubmed.ncbi.nlm.nih.gov/35482479))
446. [brentp/duphold](https://github.com/brentp/duphold) (‚≠ê98 ¬∑ Nim) don't get DUP'ed or DEL'ed by your putative SVs. ([31222198](https://pubmed.ncbi.nlm.nih.gov/31222198))
447. [verilylifesciences/genomewarp](https://github.com/verilylifesciences/genomewarp) (‚≠ê98 ¬∑ Java) GenomeWarp translates genetic variants from one genome assembly version to another. ([30916319](https://pubmed.ncbi.nlm.nih.gov/30916319))
448. [whitead/molbloom](https://github.com/whitead/molbloom) (‚≠ê98 ¬∑ Python) Molecular bloom filter tool ([37828615](https://pubmed.ncbi.nlm.nih.gov/37828615))
449. [zhjohnchan/M3AE](https://github.com/zhjohnchan/M3AE) (‚≠ê98 ¬∑ Python) [MICCAI-2022] This is the official implementation of Multi-Modal Masked Autoencoders for Medical Vision-and-Language Pre-Training. ([37976867](https://pubmed.ncbi.nlm.nih.gov/37976867))
450. [rnabioco/clustifyr](https://github.com/rnabioco/clustifyr) (‚≠ê97 ¬∑ R) Infer cell types in scRNA-seq data using bulk RNA-seq or gene sets ([32765839](https://pubmed.ncbi.nlm.nih.gov/32765839))
451. [vaquerizaslab/fanc](https://github.com/vaquerizaslab/fanc) (‚≠ê97 ¬∑ Python) FAN-C: Framework for the ANalysis of C-like data ([33334380](https://pubmed.ncbi.nlm.nih.gov/33334380))
452. [MathOnco/NeoPredPipe](https://github.com/MathOnco/NeoPredPipe) (‚≠ê96 ¬∑ Python) Neoantigens prediction pipeline for multi- or single-region vcf files using ANNOVAR and netMHCpan. ([31117948](https://pubmed.ncbi.nlm.nih.gov/31117948))
453. [arseha/peakonly](https://github.com/arseha/peakonly) (‚≠ê96 ¬∑ Python) None ([31841624](https://pubmed.ncbi.nlm.nih.gov/31841624))
454. [evoldoers/biomake](https://github.com/evoldoers/biomake) (‚≠ê96 ¬∑ Prolog) GNU-Make-like utility for managing builds and complex workflows ([28486579](https://pubmed.ncbi.nlm.nih.gov/28486579))
455. [ycwu1997/MC-Net](https://github.com/ycwu1997/MC-Net) (‚≠ê96 ¬∑ Python) Official Code for our MedIA paper "Mutual Consistency Learning for Semi-supervised Medical Image Segmentation" ([35839737](https://pubmed.ncbi.nlm.nih.gov/35839737))
456. [GATB/bcalm](https://github.com/GATB/bcalm) (‚≠ê95 ¬∑ Python) compacted de Bruijn graph construction in low memory ([27307618](https://pubmed.ncbi.nlm.nih.gov/27307618))
457. [hugheslab/snpgenie](https://github.com/hugheslab/snpgenie) (‚≠ê95 ¬∑ Perl) Program for estimating œÄN/œÄS, dN/dS, and other diversity measures from next-generation sequencing data ([26227143](https://pubmed.ncbi.nlm.nih.gov/26227143))
458. [j3xugit/RaptorX-3DModeling](https://github.com/j3xugit/RaptorX-3DModeling) (‚≠ê95 ¬∑ C) Note that current version does not include search of very large metagenome data. For some proteins, metagenome data is important. We will update this as soon as possible. ([33961022](https://pubmed.ncbi.nlm.nih.gov/33961022))
459. [jlianglab/Fixed-Point-GAN](https://github.com/jlianglab/Fixed-Point-GAN) (‚≠ê95 ¬∑ Python) Official implementation of Fixed-Point GAN - ICCV 2019 ([32612486](https://pubmed.ncbi.nlm.nih.gov/32612486))
460. [makism/dyconnmap](https://github.com/makism/dyconnmap) (‚≠ê95 ¬∑ Python) A dynamic connectome mapping module in python. ([34250674](https://pubmed.ncbi.nlm.nih.gov/34250674))
461. [abhi4ssj/few-shot-segmentation](https://github.com/abhi4ssj/few-shot-segmentation) (‚≠ê94 ¬∑ Python) PyTorch implementation of 'Squeeze and Excite' Guided Few Shot Segmentation of Volumetric Scans ([31630012](https://pubmed.ncbi.nlm.nih.gov/31630012))
462. [grimmlab/MicrobiomeBestPracticeReview](https://github.com/grimmlab/MicrobiomeBestPracticeReview) (‚≠ê94 ¬∑ Shell) Current Challenges and Best Practice Protocols for Microbiome Analysis using Amplicon and Metagenomic Sequencing ([31848574](https://pubmed.ncbi.nlm.nih.gov/31848574))
463. [roblanf/sangeranalyseR](https://github.com/roblanf/sangeranalyseR) (‚≠ê94 ¬∑ R) functions to analyse sanger sequencing reads in R ([33591316](https://pubmed.ncbi.nlm.nih.gov/33591316))
464. [ZJUFanLab/scDeepSort](https://github.com/ZJUFanLab/scDeepSort) (‚≠ê93 ¬∑ Python) Cell-type Annotation for Single-cell Transcriptomics using Deep Learning with a Weighted Graph Neural Network ([34500471](https://pubmed.ncbi.nlm.nih.gov/34500471))
465. [bbci/wyrm](https://github.com/bbci/wyrm) (‚≠ê93 ¬∑ Python) Python toolbox for Brain-Computer Interfacing (BCI) ([26001643](https://pubmed.ncbi.nlm.nih.gov/26001643))
466. [lh3/bgt](https://github.com/lh3/bgt) (‚≠ê93 ¬∑ C) Flexible genotype query among 30,000+ samples whole-genome ([26500154](https://pubmed.ncbi.nlm.nih.gov/26500154))
467. [nmquijada/tormes](https://github.com/nmquijada/tormes) (‚≠ê93 ¬∑ Shell) Making whole bacterial genome sequencing data analysis easy ([30957837](https://pubmed.ncbi.nlm.nih.gov/30957837))
468. [treangen/MetAMOS](https://github.com/treangen/MetAMOS) (‚≠ê93 ¬∑ Roff) A metagenomic and isolate assembly and analysis pipeline built with AMOS ([23320958](https://pubmed.ncbi.nlm.nih.gov/23320958))
469. [BEETL/BEETL](https://github.com/BEETL/BEETL) (‚≠ê92 ¬∑ C++) BEETL ([22556365](https://pubmed.ncbi.nlm.nih.gov/22556365); [24950811](https://pubmed.ncbi.nlm.nih.gov/24950811))
470. [BigDataBiology/SemiBin](https://github.com/BigDataBiology/SemiBin) (‚≠ê92 ¬∑ Python) SemiBin: metagenomics binning with self-supervised deep learning ([37387171](https://pubmed.ncbi.nlm.nih.gov/37387171))
471. [RagnarGrootKoerkamp/astar-pairwise-aligner](https://github.com/RagnarGrootKoerkamp/astar-pairwise-aligner) (‚≠ê92 ¬∑ Rust) A pairwise sequence aligner written in Rust ([38265119](https://pubmed.ncbi.nlm.nih.gov/38265119))
472. [pulimeng/DeepDrug3D](https://github.com/pulimeng/DeepDrug3D) (‚≠ê92 ¬∑ Python) A convolutional neural network for classifying binding pockets based on spatial and chemical information extracted from the pockets. ([30716081](https://pubmed.ncbi.nlm.nih.gov/30716081))
473. [zstephens/neat-genreads](https://github.com/zstephens/neat-genreads) (‚≠ê92 ¬∑ Python) NEAT read simulation tools ([27893777](https://pubmed.ncbi.nlm.nih.gov/27893777))
474. [AlexandrovLab/SigProfilerMatrixGenerator](https://github.com/AlexandrovLab/SigProfilerMatrixGenerator) (‚≠ê91 ¬∑ Python) SigProfilerMatrixGenerator creates mutational matrices for all types of somatic mutations. It allows downsizing the generated mutations only to parts for the genome (e.g., exome or a custom BED file). The tool seamlessly integrates with other SigProfiler tools. ([31470794](https://pubmed.ncbi.nlm.nih.gov/31470794); [37605126](https://pubmed.ncbi.nlm.nih.gov/37605126))
475. [Illumina/Pisces](https://github.com/Illumina/Pisces) (‚≠ê91 ¬∑ C#) Somatic and germline variant caller for amplicon data.  Recommended caller for tumor-only workflows. ([30304370](https://pubmed.ncbi.nlm.nih.gov/30304370))
476. [MolecularAI/DockStream](https://github.com/MolecularAI/DockStream) (‚≠ê91 ¬∑ Python) DockStream: A Docking Wrapper to Enhance De Novo Molecular Design ([34789335](https://pubmed.ncbi.nlm.nih.gov/34789335))
477. [ZJUFanLab/bulk2space](https://github.com/ZJUFanLab/bulk2space) (‚≠ê91 ¬∑ Jupyter Notebook) a spatial deconvolution method based on deep learning frameworks, which converts bulk transcriptomes into spatially resolved single-cell expression profiles ([36310179](https://pubmed.ncbi.nlm.nih.gov/36310179))
478. [aistairc/DeepEventMine](https://github.com/aistairc/DeepEventMine) (‚≠ê91 ¬∑ Python) DeepEventMine: End-to-end Neural Nested Event Extraction from Biomedical Texts ([33141147](https://pubmed.ncbi.nlm.nih.gov/33141147))
479. [hahnlab/CAFE5](https://github.com/hahnlab/CAFE5) (‚≠ê91 ¬∑ C++) Version 5 of the CAFE phylogenetics software ([33325502](https://pubmed.ncbi.nlm.nih.gov/33325502))
480. [immunogenomics/symphony](https://github.com/immunogenomics/symphony) (‚≠ê91 ¬∑ Jupyter Notebook) Efficient and precise single-cell reference atlas mapping with Symphony ([34620862](https://pubmed.ncbi.nlm.nih.gov/34620862))
481. [mgalardini/pyseer](https://github.com/mgalardini/pyseer) (‚≠ê91 ¬∑ Python) SEER, reimplemented in python üêçüîÆ ([30535304](https://pubmed.ncbi.nlm.nih.gov/30535304))
482. [qubekit/QUBEKit](https://github.com/qubekit/QUBEKit) (‚≠ê91 ¬∑ Python)   Quantum Mechanical Bespoke Force Field Derivation Toolkit ([35792069](https://pubmed.ncbi.nlm.nih.gov/35792069))
483. [ropensci/nasapower](https://github.com/ropensci/nasapower) (‚≠ê91 ¬∑ R) API Client for NASA POWER Global Meteorology, Surface Solar Energy and Climatology in R ([34039412](https://pubmed.ncbi.nlm.nih.gov/34039412))
484. [SysBioChalmers/RAVEN](https://github.com/SysBioChalmers/RAVEN) (‚≠ê90 ¬∑ MATLAB) The RAVEN Toolbox for genome scale model reconstruction, curation and analysis. ([30335785](https://pubmed.ncbi.nlm.nih.gov/30335785))
485. [caitiecollins/treeWAS](https://github.com/caitiecollins/treeWAS) (‚≠ê90 ¬∑ R) treeWAS: A Phylogenetic Tree-Based Tool for Genome-Wide Association Studies in Microbes ([29401456](https://pubmed.ncbi.nlm.nih.gov/29401456))
486. [cytoscape/cyjShiny](https://github.com/cytoscape/cyjShiny) (‚≠ê90 ¬∑ JavaScript) An R/shiny widget for cytoscape.js ([37585474](https://pubmed.ncbi.nlm.nih.gov/37585474))
487. [gaocong13/Projective-Spatial-Transformers](https://github.com/gaocong13/Projective-Spatial-Transformers) (‚≠ê90 ¬∑ Python) Generalizing Spatial Transformers to Projective Geometry with Applications to 2D/3D Registration ([33135014](https://pubmed.ncbi.nlm.nih.gov/33135014))
488. [jbohnslav/deepethogram](https://github.com/jbohnslav/deepethogram) (‚≠ê90 ¬∑ Python) None ([34473051](https://pubmed.ncbi.nlm.nih.gov/34473051))
489. [bcgsc/ARCS](https://github.com/bcgsc/ARCS) (‚≠ê89 ¬∑ C++) üåàScaffold genome sequence assemblies using linked or long read sequencing data ([29069293](https://pubmed.ncbi.nlm.nih.gov/29069293))
490. [microsoft/genomicsnotebook](https://github.com/microsoft/genomicsnotebook) (‚≠ê89 ¬∑ Jupyter Notebook) Jupyter Notebooks on Azure for Genomics Data Analysis ([36726729](https://pubmed.ncbi.nlm.nih.gov/36726729))
491. [mossmatters/HybPiper](https://github.com/mossmatters/HybPiper) (‚≠ê89 ¬∑ Python) Recovering genes from targeted sequence capture data ([27437175](https://pubmed.ncbi.nlm.nih.gov/27437175))
492. [mr7495/COVID-CTset](https://github.com/mr7495/COVID-CTset) (‚≠ê89 ¬∑ Python) Large Covid-19 CT scans dataset from the paper: https://doi.org/10.1016/j.bspc.2021.102588 ([33821166](https://pubmed.ncbi.nlm.nih.gov/33821166))
493. [nawrockie/vadr](https://github.com/nawrockie/vadr) (‚≠ê89 ¬∑ Perl) Viral Annotation DefineR: classification and annotation of viral sequences based on RefSeq annotation ([32448124](https://pubmed.ncbi.nlm.nih.gov/32448124))
494. [theosanderson/taxonium](https://github.com/theosanderson/taxonium) (‚≠ê89 ¬∑ JavaScript) A tool for exploring very large trees in the browser ([36377483](https://pubmed.ncbi.nlm.nih.gov/36377483); [36453872](https://pubmed.ncbi.nlm.nih.gov/36453872))
495. [bcgsc/ntHash](https://github.com/bcgsc/ntHash) (‚≠ê88 ¬∑ C++) Fast hash function for DNA/RNA sequences ([36000872](https://pubmed.ncbi.nlm.nih.gov/36000872))
496. [genomicsITER/NanoCLUST](https://github.com/genomicsITER/NanoCLUST) (‚≠ê88 ¬∑ Nextflow) NanoCLUST is an analysis pipeline for UMAP-based classification of amplicon-based full-length 16S rRNA nanopore reads ([33079990](https://pubmed.ncbi.nlm.nih.gov/33079990))
497. [labsquare/cutevariant](https://github.com/labsquare/cutevariant) (‚≠ê88 ¬∑ Python) A standalone and free application to explore genetics variations from VCF file ([36699349](https://pubmed.ncbi.nlm.nih.gov/36699349))
498. [giesselmann/nanopype](https://github.com/giesselmann/nanopype) (‚≠ê87 ¬∑ Python) Snakemake pipelines for nanopore sequencing data archiving and processing ([31192347](https://pubmed.ncbi.nlm.nih.gov/31192347))
499. [mozack/abra2](https://github.com/mozack/abra2) (‚≠ê87 ¬∑ Java) ABRA2 ([30649250](https://pubmed.ncbi.nlm.nih.gov/30649250))
500. [CDDLeiden/DrugEx](https://github.com/CDDLeiden/DrugEx) (‚≠ê86 ¬∑ Jupyter Notebook) De Novo Drug Design with RNNs and Transformers ([37272707](https://pubmed.ncbi.nlm.nih.gov/37272707))
501. [JLiangLab/DiRA](https://github.com/JLiangLab/DiRA) (‚≠ê86 ¬∑ Python) Official PyTorch Implementation for DiRA: Discriminative, Restorative, and Adversarial Learning for Self-supervised Medical Image Analysis - CVPR 2022 ([36313959](https://pubmed.ncbi.nlm.nih.gov/36313959))
502. [blab/nextflu](https://github.com/blab/nextflu) (‚≠ê86 ¬∑ JavaScript) Real-time tracking of influenza evolution ([26115986](https://pubmed.ncbi.nlm.nih.gov/26115986))
503. [budach/pysster](https://github.com/budach/pysster) (‚≠ê86 ¬∑ Python) pysster: Learning Sequence And Structure Motifs In Biological Sequences Using Convolutional Neural Networks ([29659719](https://pubmed.ncbi.nlm.nih.gov/29659719))
504. [jminnier/STARTapp](https://github.com/jminnier/STARTapp) (‚≠ê86 ¬∑ HTML) The START App: R Shiny Transcriptome Analysis Resource Tool ([28171615](https://pubmed.ncbi.nlm.nih.gov/28171615))
505. [mrzResearchArena/PyFeat](https://github.com/mrzResearchArena/PyFeat) (‚≠ê86 ¬∑ Python) A Python-based Effective Feature Generation Tool from DNA, RNA, and Protein Sequences ([30850831](https://pubmed.ncbi.nlm.nih.gov/30850831))
506. [orbkit/orbkit](https://github.com/orbkit/orbkit) (‚≠ê86 ¬∑ Python) A Toolbox for Post-Processing Quantum Chemical Wavefunction Data ([27043934](https://pubmed.ncbi.nlm.nih.gov/27043934))
507. [CogStack/SemEHR](https://github.com/CogStack/SemEHR) (‚≠ê85 ¬∑ JavaScript) Surfacing Semantic Data from Clinical Notes in Electronic Health Records for Tailored Care, Trial Recruitment and Clinical Research ([29361077](https://pubmed.ncbi.nlm.nih.gov/29361077))
508. [KatherLab/HIA](https://github.com/KatherLab/HIA) (‚≠ê85 ¬∑ Python) Histopathology Image Analysis ([35588568](https://pubmed.ncbi.nlm.nih.gov/35588568))
509. [biocore/deblur](https://github.com/biocore/deblur) (‚≠ê85 ¬∑ Python) Deblur is a greedy deconvolution algorithm based on known read error profiles. ([28289731](https://pubmed.ncbi.nlm.nih.gov/28289731))
510. [meren/illumina-utils](https://github.com/meren/illumina-utils) (‚≠ê85 ¬∑ Python) A library and collection of scripts to work with Illumina paired-end data (for CASAVA 1.7+ pipeline). ([23799126](https://pubmed.ncbi.nlm.nih.gov/23799126))
511. [ncbi/MedCPT](https://github.com/ncbi/MedCPT) (‚≠ê85 ¬∑ Python) Code for MedCPT, a model for zero-shot biomedical information retrieval. ([37930897](https://pubmed.ncbi.nlm.nih.gov/37930897))
512. [romanhaa/Cerebro](https://github.com/romanhaa/Cerebro) (‚≠ê85 ¬∑ HTML) Visualization of scRNA-seq data. ([31764967](https://pubmed.ncbi.nlm.nih.gov/31764967))
513. [BUStools/bustools](https://github.com/BUStools/bustools) (‚≠ê84 ¬∑ C) Tools for working with BUS files ([37129540](https://pubmed.ncbi.nlm.nih.gov/37129540))
514. [ChristopherWilks/megadepth](https://github.com/ChristopherWilks/megadepth) (‚≠ê84 ¬∑ C++) BigWig and BAM utilities ([33693500](https://pubmed.ncbi.nlm.nih.gov/33693500))
515. [FastTrackOrg/FastTrack](https://github.com/FastTrackOrg/FastTrack) (‚≠ê84 ¬∑ C++) FastTrack is a cross-platform application designed to track multiple objects in video recording. ([33571205](https://pubmed.ncbi.nlm.nih.gov/33571205))
516. [cpockrandt/genmap](https://github.com/cpockrandt/genmap) (‚≠ê84 ¬∑ C++) GenMap - Fast and Exact Computation of Genome Mappability ([32246826](https://pubmed.ncbi.nlm.nih.gov/32246826))
517. [jodyphelan/TBProfiler](https://github.com/jodyphelan/TBProfiler) (‚≠ê84 ¬∑ Python) Profiling tool for Mycobacterium tuberculosis to detect ressistance and strain type from WGS data ([33317631](https://pubmed.ncbi.nlm.nih.gov/33317631))
518. [llnl/fast](https://github.com/llnl/fast) (‚≠ê84 ¬∑ Python) Fusion models for Atomic and molecular STructures (FAST) ([33754707](https://pubmed.ncbi.nlm.nih.gov/33754707))
519. [modeloriented/survex](https://github.com/modeloriented/survex) (‚≠ê84 ¬∑ R) Explainable Machine Learning in Survival Analysis ([38039146](https://pubmed.ncbi.nlm.nih.gov/38039146))
520. [parashardhapola/scarf](https://github.com/parashardhapola/scarf) (‚≠ê84 ¬∑ Python) Toolkit for highly memory efficient analysis of single-cell RNA-Seq, scATAC-Seq and CITE-Seq data. Analyze atlas scale datasets with millions of cells on laptop. ([35941103](https://pubmed.ncbi.nlm.nih.gov/35941103))
521. [MannLabs/alphapeptdeep](https://github.com/MannLabs/alphapeptdeep) (‚≠ê83 ¬∑ Jupyter Notebook) Deep learning framework for proteomics ([36433986](https://pubmed.ncbi.nlm.nih.gov/36433986))
522. [bergmanlab/mcclintock](https://github.com/bergmanlab/mcclintock) (‚≠ê83 ¬∑ Python) Meta-pipeline to identify transposable element insertions using next generation sequencing data ([28637810](https://pubmed.ncbi.nlm.nih.gov/28637810); [37452430](https://pubmed.ncbi.nlm.nih.gov/37452430))
523. [bigchem/transformer-cnn](https://github.com/bigchem/transformer-cnn) (‚≠ê83 ¬∑ Python) Transformer CNN for QSAR/QSPR modelling ([33431004](https://pubmed.ncbi.nlm.nih.gov/33431004))
524. [ccipd/MRQy](https://github.com/ccipd/MRQy) (‚≠ê83 ¬∑ JavaScript) MRQy is a quality assurance and checking tool for quantitative assessment of magnetic resonance imaging (MRI) data. ([33176026](https://pubmed.ncbi.nlm.nih.gov/33176026))
525. [cdcgov/microbetrace](https://github.com/cdcgov/microbetrace) (‚≠ê83 ¬∑ TypeScript) The Visualization Multitool for Molecular Epidemiology and Bioinformatics ([34492010](https://pubmed.ncbi.nlm.nih.gov/34492010))
526. [htseq/htseq](https://github.com/htseq/htseq) (‚≠ê83 ¬∑ Python) HTSeq is a Python library to facilitate processing and analysis of data from  high-throughput sequencing (HTS) experiments. ([35561197](https://pubmed.ncbi.nlm.nih.gov/35561197))
527. [uwrit/leaf](https://github.com/uwrit/leaf) (‚≠ê83 ¬∑ TypeScript) Leaf Clinical Data Explorer ([31592524](https://pubmed.ncbi.nlm.nih.gov/31592524))
528. [Vini2/GraphBin](https://github.com/Vini2/GraphBin) (‚≠ê82 ¬∑ Python) ‚ú®üß¨ Refined binning of metagenomic contigs using assembly graphs ([32167528](https://pubmed.ncbi.nlm.nih.gov/32167528))
529. [bcgsc/RNA-Bloom](https://github.com/bcgsc/RNA-Bloom) (‚≠ê82 ¬∑ Java) :hibiscus: reference-free transcriptome assembly for short and long reads ([37217540](https://pubmed.ncbi.nlm.nih.gov/37217540))
530. [catavallejos/BASiCS](https://github.com/catavallejos/BASiCS) (‚≠ê82 ¬∑ R) BASiCS: Bayesian Analysis of Single-Cell Sequencing Data. This is an unstable experimental version. Please see http://bioconductor.org/packages/BASiCS/ for the official release version ([27083558](https://pubmed.ncbi.nlm.nih.gov/27083558))
531. [lanagarmire/DeepImpute](https://github.com/lanagarmire/DeepImpute) (‚≠ê82 ¬∑ Python) An accurate and efficient deep learning method for single-cell RNA-seq data imputation ([31627739](https://pubmed.ncbi.nlm.nih.gov/31627739))
532. [openbiox/UCSCXenaShiny](https://github.com/openbiox/UCSCXenaShiny) (‚≠ê82 ¬∑ R) üìä An R package for interactively exploring UCSC Xena https://xenabrowser.net/datapages/ ([34323947](https://pubmed.ncbi.nlm.nih.gov/34323947))
533. [parklab/xTea](https://github.com/parklab/xTea) (‚≠ê82 ¬∑ Python) Comprehensive TE insertion identification with WGS/WES data from multiple sequencing technics ([34158502](https://pubmed.ncbi.nlm.nih.gov/34158502))
534. [shervinmin/DeepCovid](https://github.com/shervinmin/DeepCovid) (‚≠ê82 ¬∑ Python) None ([32781377](https://pubmed.ncbi.nlm.nih.gov/32781377))
535. [splatlab/squeakr](https://github.com/splatlab/squeakr) (‚≠ê82 ¬∑ C++) Squeakr: An Exact and Approximate k -mer Counting System ([29444235](https://pubmed.ncbi.nlm.nih.gov/29444235))
536. [ShuweiShao/AF-SfMLearner](https://github.com/ShuweiShao/AF-SfMLearner) (‚≠ê81 ¬∑ Python) [MedIA2022 & ICRA2021] Self-Supervised Monocular Depth and Ego-Motion Estimation in Endoscopy: Appearance Flow to the Rescue ([35016079](https://pubmed.ncbi.nlm.nih.gov/35016079))
537. [Superzchen/iLearnPlus](https://github.com/Superzchen/iLearnPlus) (‚≠ê81 ¬∑ Python) iLearnPlus is the first machine-learning platform with both graphical- and web-based user interface that enables the construction of automated machine-learning pipelines for computational analysis and predictions using nucleic acid and protein sequences. ([33660783](https://pubmed.ncbi.nlm.nih.gov/33660783))
538. [nunofonseca/irap](https://github.com/nunofonseca/irap) (‚≠ê81 ¬∑ R) integrated RNA-seq Analysis Pipeline ([28369191](https://pubmed.ncbi.nlm.nih.gov/28369191))
539. [smirarab/pasta](https://github.com/smirarab/pasta) (‚≠ê81 ¬∑ Python) PASTA (Practical Alignment using SATe and Transitivity) ([29931282](https://pubmed.ncbi.nlm.nih.gov/29931282))
540. [vanallenlab/comut](https://github.com/vanallenlab/comut) (‚≠ê81 ¬∑ Python) CoMut is a Python library for visualizing genomic and phenotypic information via comutation plots ([32502231](https://pubmed.ncbi.nlm.nih.gov/32502231))
541. [0liliulei/Mem3D](https://github.com/0liliulei/Mem3D) (‚≠ê80 ¬∑ Python) None ([36327652](https://pubmed.ncbi.nlm.nih.gov/36327652))
542. [CenterForMedicalGeneticsGhent/WisecondorX](https://github.com/CenterForMedicalGeneticsGhent/WisecondorX) (‚≠ê80 ¬∑ Python) WisecondorX ‚Äî An evolved WISECONDOR ([30566647](https://pubmed.ncbi.nlm.nih.gov/30566647))
543. [balbasty/nitorch](https://github.com/balbasty/nitorch) (‚≠ê80 ¬∑ Python) Neuroimaging in PyTorch ([34271531](https://pubmed.ncbi.nlm.nih.gov/34271531))
544. [carmonalab/scGate](https://github.com/carmonalab/scGate) (‚≠ê80 ¬∑ R) marker-based purification of cell types from single-cell RNA-seq datasets ([35258562](https://pubmed.ncbi.nlm.nih.gov/35258562))
545. [citiususc/veryfasttree](https://github.com/citiususc/veryfasttree) (‚≠ê80 ¬∑ C++) Efficient phylogenetic tree inference for massive taxonomic datasets ([32573652](https://pubmed.ncbi.nlm.nih.gov/32573652))
546. [fwhelan/coinfinder](https://github.com/fwhelan/coinfinder) (‚≠ê80 ¬∑ C++) A tool for the identification of coincident (associating and dissociating) genes in pangenomes. ([32100706](https://pubmed.ncbi.nlm.nih.gov/32100706))
547. [gamcil/cblaster](https://github.com/gamcil/cblaster) (‚≠ê80 ¬∑ HTML) Find clustered hits from a BLAST search ([36700093](https://pubmed.ncbi.nlm.nih.gov/36700093))
548. [gui11aume/starcode](https://github.com/gui11aume/starcode) (‚≠ê80 ¬∑ C) All pairs search and sequence clustering ([25638815](https://pubmed.ncbi.nlm.nih.gov/25638815))
549. [ramiromagno/gwasrapidd](https://github.com/ramiromagno/gwasrapidd) (‚≠ê80 ¬∑ R) gwasrapidd: an R package to query, download and wrangle GWAS Catalog data ([31373609](https://pubmed.ncbi.nlm.nih.gov/31373609))
550. [splatlab/mantis](https://github.com/splatlab/mantis) (‚≠ê80 ¬∑ C++) Mantis: A Fast, Small, and Exact Large-Scale Sequence-Search Index ([35325039](https://pubmed.ncbi.nlm.nih.gov/35325039))
551. [vanderschaarlab/AutoPrognosis](https://github.com/vanderschaarlab/AutoPrognosis) (‚≠ê80 ¬∑ Python) A system for automating the design of predictive modeling pipelines tailored for clinical prognosis. ([37347752](https://pubmed.ncbi.nlm.nih.gov/37347752))
552. [BioJulia/BioStructures](https://github.com/BioJulia/BioStructures) (‚≠ê79 ¬∑ Julia) A Julia package to read, write and manipulate macromolecular structures (particularly proteins) ([32407511](https://pubmed.ncbi.nlm.nih.gov/32407511))
553. [PayneLab/cptac](https://github.com/PayneLab/cptac) (‚≠ê79 ¬∑ Jupyter Notebook) Python packaging for CPTAC data ([33560848](https://pubmed.ncbi.nlm.nih.gov/33560848))
554. [nstrodt/UDSMProt](https://github.com/nstrodt/UDSMProt) (‚≠ê79 ¬∑ Python) Protein sequence classification with self-supervised pretraining ([31913448](https://pubmed.ncbi.nlm.nih.gov/31913448))
555. [xie186/ViewBS](https://github.com/xie186/ViewBS) (‚≠ê79 ¬∑ Perl) ViewBS - a powerful toolkit for visualization of high-throughput bisulfite sequencing data ([29087450](https://pubmed.ncbi.nlm.nih.gov/29087450))
556. [CoSMoMVPA/CoSMoMVPA](https://github.com/CoSMoMVPA/CoSMoMVPA) (‚≠ê78 ¬∑ MATLAB) A lightweight multivariate pattern analysis (MVPA) toolbox in Matlab / Octave ([27499741](https://pubmed.ncbi.nlm.nih.gov/27499741))
557. [DlutMedimgGroup/AnatomySketch-Software](https://github.com/DlutMedimgGroup/AnatomySketch-Software) (‚≠ê78 ¬∑ C++) A medical image segmentation software supporting user interventions and user-defined funtion extention ([35768752](https://pubmed.ncbi.nlm.nih.gov/35768752))
558. [Vaa3D/vaa3d](https://github.com/Vaa3D/vaa3d) (‚≠ê78 ¬∑ None) www.vaa3d.org ([37213868](https://pubmed.ncbi.nlm.nih.gov/37213868))
559. [bcgsc/ntjoin](https://github.com/bcgsc/ntjoin) (‚≠ê78 ¬∑ Python) üîóGenome assembly scaffolder using minimizer graphs ([32311025](https://pubmed.ncbi.nlm.nih.gov/32311025))
560. [glennhickey/progressiveCactus](https://github.com/glennhickey/progressiveCactus) (‚≠ê78 ¬∑ Python) Distribution package for the Prgressive Cactus multiple genome aligner.  Dependencies are linked as submodules ([25138168](https://pubmed.ncbi.nlm.nih.gov/25138168))
561. [mdsufz/MuDoGeR](https://github.com/mdsufz/MuDoGeR) (‚≠ê78 ¬∑ Shell) MuDoGeR makes the recovery of genomes from prokaryotes, viruses, and eukaryotes from metagenomes easy. ([37994269](https://pubmed.ncbi.nlm.nih.gov/37994269))
562. [moiexpositoalonsolab/grenepipe](https://github.com/moiexpositoalonsolab/grenepipe) (‚≠ê78 ¬∑ Python) A flexible, scalable, and reproducible pipeline to automate variant calling from sequence reads. ([36053180](https://pubmed.ncbi.nlm.nih.gov/36053180))
563. [wdecoster/methplotlib](https://github.com/wdecoster/methplotlib) (‚≠ê78 ¬∑ Python) Plotting tools for nanopore methylation data ([32053166](https://pubmed.ncbi.nlm.nih.gov/32053166))
564. [EtienneCmb/tensorpac](https://github.com/EtienneCmb/tensorpac) (‚≠ê77 ¬∑ Python) Phase-Amplitude Coupling under Python ([33119593](https://pubmed.ncbi.nlm.nih.gov/33119593))
565. [KeenThera/SECSE](https://github.com/KeenThera/SECSE) (‚≠ê77 ¬∑ Python) Systemic Evolutionary Chemical Space Exploration for Drug Discovery ([35365231](https://pubmed.ncbi.nlm.nih.gov/35365231))
566. [MICCoM/SSAGES-public](https://github.com/MICCoM/SSAGES-public) (‚≠ê77 ¬∑ C++) Software Suite for Advanced General Ensemble Simulations ([29390830](https://pubmed.ncbi.nlm.nih.gov/29390830))
567. [MeHelmy/princess](https://github.com/MeHelmy/princess) (‚≠ê77 ¬∑ Python) None ([34521442](https://pubmed.ncbi.nlm.nih.gov/34521442))
568. [genome/gms](https://github.com/genome/gms) (‚≠ê77 ¬∑ Perl) The Genome Modeling System installer ([26158448](https://pubmed.ncbi.nlm.nih.gov/26158448))
569. [jermp/sshash](https://github.com/jermp/sshash) (‚≠ê77 ¬∑ C++) A compressed, associative, exact, and weighted dictionary for k-mers. ([35758794](https://pubmed.ncbi.nlm.nih.gov/35758794))
570. [scvae/scvae](https://github.com/scvae/scvae) (‚≠ê77 ¬∑ Python) Deep learning for single-cell transcript counts ([32415966](https://pubmed.ncbi.nlm.nih.gov/32415966))
571. [seqan/flexbar](https://github.com/seqan/flexbar) (‚≠ê77 ¬∑ C++) flexible barcode and adapter removal ([28541403](https://pubmed.ncbi.nlm.nih.gov/28541403))
572. [stephens999/ashr](https://github.com/stephens999/ashr) (‚≠ê77 ¬∑ R) An R package for adaptive shrinkage ([27756721](https://pubmed.ncbi.nlm.nih.gov/27756721))
573. [yjx1217/simuG](https://github.com/yjx1217/simuG) (‚≠ê77 ¬∑ Perl) simuG: a general-purpose genome simulator ([31116378](https://pubmed.ncbi.nlm.nih.gov/31116378))
574. [arzwa/wgd](https://github.com/arzwa/wgd) (‚≠ê76 ¬∑ Python) Python package and CLI for whole-genome duplication related analyses. This package is deprecated in favor of https://github.com/heche-psb/wgd. ([30398564](https://pubmed.ncbi.nlm.nih.gov/30398564))
575. [calico/solo](https://github.com/calico/solo) (‚≠ê76 ¬∑ Python) software to detect doublets ([32592658](https://pubmed.ncbi.nlm.nih.gov/32592658))
576. [crazyhottommy/scclusteval](https://github.com/crazyhottommy/scclusteval) (‚≠ê76 ¬∑ R) Single Cell Cluster Evaluation ([33165513](https://pubmed.ncbi.nlm.nih.gov/33165513))
577. [futianfan/clinical-trial-outcome-prediction](https://github.com/futianfan/clinical-trial-outcome-prediction) (‚≠ê76 ¬∑ Python) benchmark dataset and Deep learning method (Hierarchical Interaction Network, HINT) for clinical trial approval probability prediction, published in Cell Patterns 2022. ([35465223](https://pubmed.ncbi.nlm.nih.gov/35465223))
578. [oxpig/ImmuneBuilder](https://github.com/oxpig/ImmuneBuilder) (‚≠ê76 ¬∑ Jupyter Notebook) Predict the structure of immune receptor proteins ([37248282](https://pubmed.ncbi.nlm.nih.gov/37248282))
579. [s312569/clj-biosequence](https://github.com/s312569/clj-biosequence) (‚≠ê76 ¬∑ Clojure) A Clojure library designed to make the manipulation of biological sequence data easier. ([24794932](https://pubmed.ncbi.nlm.nih.gov/24794932))
580. [sjspielman/pyvolve](https://github.com/sjspielman/pyvolve) (‚≠ê76 ¬∑ Python) Python library to simulate evolutionary sequence data ([26397960](https://pubmed.ncbi.nlm.nih.gov/26397960))
581. [zzstefan/BrainNetClass](https://github.com/zzstefan/BrainNetClass) (‚≠ê76 ¬∑ MATLAB) None ([32163221](https://pubmed.ncbi.nlm.nih.gov/32163221))
582. [bcgsc/ntCard](https://github.com/bcgsc/ntCard) (‚≠ê75 ¬∑ C++) Estimating k-mer coverage histogram of genomics data ([28453674](https://pubmed.ncbi.nlm.nih.gov/28453674))
583. [smirarab/sepp](https://github.com/smirarab/sepp) (‚≠ê75 ¬∑ Python) Ensemble of HMM methods (SEPP, TIPP, UPP) ([26076734](https://pubmed.ncbi.nlm.nih.gov/26076734); [28185571](https://pubmed.ncbi.nlm.nih.gov/28185571); [33471121](https://pubmed.ncbi.nlm.nih.gov/33471121); [34791036](https://pubmed.ncbi.nlm.nih.gov/34791036))
584. [JLiangLab/SemanticGenesis](https://github.com/JLiangLab/SemanticGenesis) (‚≠ê74 ¬∑ Python) Official Keras & PyTorch Implementation and Pre-trained Models for Semantic Genesis - MICCAI 2020 ([35695848](https://pubmed.ncbi.nlm.nih.gov/35695848))
585. [arthurpessa/ordpy](https://github.com/arthurpessa/ordpy) (‚≠ê74 ¬∑ Python) A Python package for data analysis with permutation entropy and ordinal network methods. ([34241315](https://pubmed.ncbi.nlm.nih.gov/34241315))
586. [asad/ReactionDecoder](https://github.com/asad/ReactionDecoder) (‚≠ê74 ¬∑ Java) Reaction Decoder Tool (RDT) - Atom Atom Mapping Tool ([27153692](https://pubmed.ncbi.nlm.nih.gov/27153692))
587. [lh3/fermi](https://github.com/lh3/fermi) (‚≠ê74 ¬∑ C) A WGS de novo assembler based on the FMD-index for large genomes ([22569178](https://pubmed.ncbi.nlm.nih.gov/22569178))
588. [Oshlack/JAFFA](https://github.com/Oshlack/JAFFA) (‚≠ê73 ¬∑ C++) JAFFA is a multi-step pipeline that takes either raw RNA-Seq reads, or pre-assembled transcripts, then searches for gene fusions ([26019724](https://pubmed.ncbi.nlm.nih.gov/26019724); [34991664](https://pubmed.ncbi.nlm.nih.gov/34991664))
589. [PapenfussLab/bionix](https://github.com/PapenfussLab/bionix) (‚≠ê73 ¬∑ Nix) Functional highly reproducible bioinformatics pipelines ([33205815](https://pubmed.ncbi.nlm.nih.gov/33205815))
590. [eldariont/svim-asm](https://github.com/eldariont/svim-asm) (‚≠ê73 ¬∑ Python) Structural Variant Identification Method using Genome Assemblies ([33346817](https://pubmed.ncbi.nlm.nih.gov/33346817))
591. [gevaertlab/MultimodalPrognosis](https://github.com/gevaertlab/MultimodalPrognosis) (‚≠ê73 ¬∑ Python) Deep Learning with Multimodal Representation for Pancancer Prognosis Prediction ([31510656](https://pubmed.ncbi.nlm.nih.gov/31510656))
592. [grailbio/bio](https://github.com/grailbio/bio) (‚≠ê73 ¬∑ Go) Bioinformatic infrastructure libraries ([31510681](https://pubmed.ncbi.nlm.nih.gov/31510681))
593. [jokergoo/rGREAT](https://github.com/jokergoo/rGREAT) (‚≠ê73 ¬∑ R) GREAT Analysis - Functional Enrichment on Genomic Regions ([36394265](https://pubmed.ncbi.nlm.nih.gov/36394265))
594. [jwalabroad/VariantBam](https://github.com/jwalabroad/VariantBam) (‚≠ê73 ¬∑ Makefile) Filtering and profiling of next-generational sequencing data using region-specific rules ([27153727](https://pubmed.ncbi.nlm.nih.gov/27153727))
595. [xavierdidelot/BactDating](https://github.com/xavierdidelot/BactDating) (‚≠ê73 ¬∑ R) Bayesian inference of ancestral dates on bacterial phylogenetic trees ([30184106](https://pubmed.ncbi.nlm.nih.gov/30184106))
596. [HRGV/phyloFlash](https://github.com/HRGV/phyloFlash) (‚≠ê72 ¬∑ Perl) phyloFlash - A pipeline to rapidly reconstruct the SSU rRNAs and explore phylogenetic composition of an illumina (meta)genomic dataset. ([33109753](https://pubmed.ncbi.nlm.nih.gov/33109753))
597. [diazlab/CONICS](https://github.com/diazlab/CONICS) (‚≠ê72 ¬∑ R) CONICS: COpy-Number analysis In single-Cell RNA-Sequencing ([29897414](https://pubmed.ncbi.nlm.nih.gov/29897414))
598. [netZoo/netZooPy](https://github.com/netZoo/netZooPy) (‚≠ê72 ¬∑ Jupyter Notebook) netZooPy is a network biology package implemented in Python. ([35156023](https://pubmed.ncbi.nlm.nih.gov/35156023))
599. [rezacsedu/XAI-for-bioinformatics](https://github.com/rezacsedu/XAI-for-bioinformatics) (‚≠ê72 ¬∑ Jupyter Notebook) Explainable AI for Bioinformatics ([37478371](https://pubmed.ncbi.nlm.nih.gov/37478371))
600. [SaskiaFreytag/schex](https://github.com/SaskiaFreytag/schex) (‚≠ê71 ¬∑ R) None ([31794001](https://pubmed.ncbi.nlm.nih.gov/31794001))
601. [TinkerTools/tinker-hp](https://github.com/TinkerTools/tinker-hp) (‚≠ê71 ¬∑ C) Tinker-HP: High-Performance Massively Parallel Evolution of Tinker on CPUs & GPUs ([33173801](https://pubmed.ncbi.nlm.nih.gov/33173801); [33755446](https://pubmed.ncbi.nlm.nih.gov/33755446))
602. [bigbio/proteomics-metadata-standard](https://github.com/bigbio/proteomics-metadata-standard) (‚≠ê71 ¬∑ Python) The Proteomics Experimental Design file format: Standard for experimental design annotation ([32786688](https://pubmed.ncbi.nlm.nih.gov/32786688))
603. [changwn/scFEA](https://github.com/changwn/scFEA) (‚≠ê71 ¬∑ Jupyter Notebook) single cell Flux Estimation Analysis (scFEA) Try the below web server! ([37216602](https://pubmed.ncbi.nlm.nih.gov/37216602))
604. [compgenome365/TCGA-Assembler-2](https://github.com/compgenome365/TCGA-Assembler-2) (‚≠ê71 ¬∑ R) TCGA-Assembler 2 ([29272348](https://pubmed.ncbi.nlm.nih.gov/29272348))
605. [twolinin/LongPhase](https://github.com/twolinin/LongPhase) (‚≠ê71 ¬∑ C) None ([35104333](https://pubmed.ncbi.nlm.nih.gov/35104333))
606. [BioEasy/EasyCodeML](https://github.com/BioEasy/EasyCodeML) (‚≠ê70 ¬∑ C)  EasyCodeML: A visual tool for analysis of selection using CodeML ([31015974](https://pubmed.ncbi.nlm.nih.gov/31015974))
607. [FangpingWan/NeoDTI](https://github.com/FangpingWan/NeoDTI) (‚≠ê70 ¬∑ Python) NeoDTI: Neural integration of neighbor information from a heterogeneous network for discovering new drug-target interactions ([30561548](https://pubmed.ncbi.nlm.nih.gov/30561548))
608. [chentao1999/MedicalRelationExtraction](https://github.com/chentao1999/MedicalRelationExtraction) (‚≠ê70 ¬∑ Python) The depository support training and testing BERT-CNN model on three medical relation extraction corpora: BioCreative V CDR task corpus, traditional Chinese medicine  literature corpus, and i2b2 temporal relation corpus. ([31800044](https://pubmed.ncbi.nlm.nih.gov/31800044))
609. [cosconatilab/PyRMD](https://github.com/cosconatilab/PyRMD) (‚≠ê70 ¬∑ Python) AI-powered Virtual Screening ([34270903](https://pubmed.ncbi.nlm.nih.gov/34270903); [37552222](https://pubmed.ncbi.nlm.nih.gov/37552222))
610. [ddarriba/modeltest](https://github.com/ddarriba/modeltest) (‚≠ê70 ¬∑ C++) Best-fit model selection ([31432070](https://pubmed.ncbi.nlm.nih.gov/31432070))
611. [frankvogt/vcf2gwas](https://github.com/frankvogt/vcf2gwas) (‚≠ê70 ¬∑ Python) Python API for comprehensive GWAS analysis using GEMMA ([34636840](https://pubmed.ncbi.nlm.nih.gov/34636840))
612. [jltsiren/gcsa2](https://github.com/jltsiren/gcsa2) (‚≠ê70 ¬∑ C++) BWT-based index for graphs ([31406990](https://pubmed.ncbi.nlm.nih.gov/31406990))
613. [lh3/dna-nn](https://github.com/lh3/dna-nn) (‚≠ê70 ¬∑ C) Model and predict short DNA sequence features with neural networks ([30989183](https://pubmed.ncbi.nlm.nih.gov/30989183))
614. [lmmpf/PyAutoFEP](https://github.com/lmmpf/PyAutoFEP) (‚≠ê70 ¬∑ Python) PyAutoFEP: an automated FEP workflow for GROMACS integrating enhanced sampling methods ([34142828](https://pubmed.ncbi.nlm.nih.gov/34142828))
615. [mateidavid/nanocall](https://github.com/mateidavid/nanocall) (‚≠ê70 ¬∑ C++) An Oxford Nanopore Basecaller ([27614348](https://pubmed.ncbi.nlm.nih.gov/27614348))
616. [montilab/hypeR](https://github.com/montilab/hypeR) (‚≠ê70 ¬∑ R) An R Package for Geneset Enrichment Workflows ([31498385](https://pubmed.ncbi.nlm.nih.gov/31498385))
617. [GfellerLab/EPIC](https://github.com/GfellerLab/EPIC) (‚≠ê69 ¬∑ R) Repository for the R package EPIC, to Estimate the Proportion of Immune and Cancer cells from bulk gene expression data. ([32124324](https://pubmed.ncbi.nlm.nih.gov/32124324))
618. [Liuxg16/GeoPPI](https://github.com/Liuxg16/GeoPPI) (‚≠ê69 ¬∑ Python) None ([34347784](https://pubmed.ncbi.nlm.nih.gov/34347784))
619. [MannLabs/alphatims](https://github.com/MannLabs/alphatims) (‚≠ê69 ¬∑ Python) An open-source Python package for efficient accession and visualization of Bruker TimsTOF raw data from the Mann Labs at the Max Planck Institute of Biochemistry. ([34543758](https://pubmed.ncbi.nlm.nih.gov/34543758))
620. [RemiAllio/MitoFinder](https://github.com/RemiAllio/MitoFinder) (‚≠ê69 ¬∑ Python) MitoFinder: efficient automated large-scale extraction of mitogenomic data from high throughput sequencing data ([32243090](https://pubmed.ncbi.nlm.nih.gov/32243090))
621. [chainsawriot/rang](https://github.com/chainsawriot/rang) (‚≠ê69 ¬∑ R) üê∂ (Re)constructing R computational environments ([37289797](https://pubmed.ncbi.nlm.nih.gov/37289797))
622. [instadeepai/manyfold](https://github.com/instadeepai/manyfold) (‚≠ê69 ¬∑ Python) üß¨ ManyFold: An efficient and flexible library for training and validating protein folding models ([36495196](https://pubmed.ncbi.nlm.nih.gov/36495196))
623. [mozack/abra](https://github.com/mozack/abra) (‚≠ê69 ¬∑ Java) Assembly Based ReAligner ([24907369](https://pubmed.ncbi.nlm.nih.gov/24907369))
624. [tderrien/FEELnc](https://github.com/tderrien/FEELnc) (‚≠ê69 ¬∑ Perl) FEELnc : FlExible Extraction of LncRNA ([28053114](https://pubmed.ncbi.nlm.nih.gov/28053114))
625. [yukiteruono/pbsim2](https://github.com/yukiteruono/pbsim2) (‚≠ê69 ¬∑ C++) PBSIM2: a simulator for long read sequencers with a novel generative model of quality scores ([32976553](https://pubmed.ncbi.nlm.nih.gov/32976553))
626. [zuoxinian/CCS](https://github.com/zuoxinian/CCS) (‚≠ê69 ¬∑ MATLAB) Connectome Computation System ([36738058](https://pubmed.ncbi.nlm.nih.gov/36738058))
627. [MaterialsDiscovery/PyChemia](https://github.com/MaterialsDiscovery/PyChemia) (‚≠ê68 ¬∑ Python) Python Materials Discovery Framework ([27232694](https://pubmed.ncbi.nlm.nih.gov/27232694))
628. [Phlya/coolpuppy](https://github.com/Phlya/coolpuppy) (‚≠ê68 ¬∑ Python) A versatile tool to perform pile-up analysis on Hi-C data in .cool format. ([32003791](https://pubmed.ncbi.nlm.nih.gov/32003791))
629. [ShellyCoder/cellcall](https://github.com/ShellyCoder/cellcall) (‚≠ê68 ¬∑ R) inferring cell-cell communication from scRNA-seq of ligand-receptor ([34331449](https://pubmed.ncbi.nlm.nih.gov/34331449))
630. [canlab/MediationToolbox](https://github.com/canlab/MediationToolbox) (‚≠ê68 ¬∑ HTML) Single-level and multi-level mediation analyses for any kind of data, with bootstrap-based significance testing. Neuroimaging-oriented functions allow for mediation effect parametric mapping (mapping of mediation effects across the brain) and multivariate mediation. ([36569614](https://pubmed.ncbi.nlm.nih.gov/36569614))
631. [citiususc/SparkBWA](https://github.com/citiususc/SparkBWA) (‚≠ê68 ¬∑ C) SparkBWA is a new tool that exploits the capabilities of a Big Data technology as Apache Spark to boost the performance of one of the most widely adopted sequence aligner, the Burrows-Wheeler Aligner (BWA). ([27182962](https://pubmed.ncbi.nlm.nih.gov/27182962))
632. [eturro/mmseq](https://github.com/eturro/mmseq) (‚≠ê68 ¬∑ C++) Haplotype, isoform and gene level expression analysis using multi-mapping RNA-seq reads ([24281695](https://pubmed.ncbi.nlm.nih.gov/24281695))
633. [lh3/bfc](https://github.com/lh3/bfc) (‚≠ê68 ¬∑ TeX) High-performance error correction for Illumina resequencing data ([25953801](https://pubmed.ncbi.nlm.nih.gov/25953801))
634. [lh3/ropebwt2](https://github.com/lh3/ropebwt2) (‚≠ê68 ¬∑ TeX) Incremental construction of FM-index for DNA sequences ([25107872](https://pubmed.ncbi.nlm.nih.gov/25107872))
635. [natir/yacrd](https://github.com/natir/yacrd) (‚≠ê68 ¬∑ Rust) Yet Another Chimeric Read Detector ([32315402](https://pubmed.ncbi.nlm.nih.gov/32315402))
636. [Genentech/pviz](https://github.com/Genentech/pviz) (‚≠ê67 ¬∑ JavaScript) Pviz ([25147360](https://pubmed.ncbi.nlm.nih.gov/25147360))
637. [LiLabAtVT/DeepTE](https://github.com/LiLabAtVT/DeepTE) (‚≠ê67 ¬∑ Python) Neural network classification of TE ([32415954](https://pubmed.ncbi.nlm.nih.gov/32415954))
638. [MannLabs/alphamap](https://github.com/MannLabs/alphamap) (‚≠ê67 ¬∑ Jupyter Notebook) An open-source Python package for the visual annotation of proteomics data with sequence specific knowledge. ([34586352](https://pubmed.ncbi.nlm.nih.gov/34586352))
639. [hdaoud/ORBSLAMM](https://github.com/hdaoud/ORBSLAMM) (‚≠ê67 ¬∑ C++) None ([29702697](https://pubmed.ncbi.nlm.nih.gov/29702697))
640. [sebwouters/chemps2](https://github.com/sebwouters/chemps2) (‚≠ê67 ¬∑ C++) CheMPS2: a spin-adapted implementation of DMRG for ab initio quantum chemistry ([27497552](https://pubmed.ncbi.nlm.nih.gov/27497552))
641. [simonvh/fluff](https://github.com/simonvh/fluff) (‚≠ê67 ¬∑ Python) Fluff is a Python package that contains several scripts to produce pretty, publication-quality figures for next-generation sequencing experiments. ([27547532](https://pubmed.ncbi.nlm.nih.gov/27547532))
642. [DerKevinRiehl/TransposonUltimate](https://github.com/DerKevinRiehl/TransposonUltimate) (‚≠ê66 ¬∑ None) TransposonUltimate - a holistic set of tools for transposon identification ([35234904](https://pubmed.ncbi.nlm.nih.gov/35234904))
643. [Pbdas/epa-ng](https://github.com/Pbdas/epa-ng) (‚≠ê66 ¬∑ C++) Massively parallel phylogenetic placement of genetic sequences ([30165689](https://pubmed.ncbi.nlm.nih.gov/30165689))
644. [TheWebMonks/meshmonk](https://github.com/TheWebMonks/meshmonk) (‚≠ê66 ¬∑ C++) Open mesh-to-mesh registration framework ([32746977](https://pubmed.ncbi.nlm.nih.gov/32746977))
645. [brinkmanlab/GenomeD3Plot](https://github.com/brinkmanlab/GenomeD3Plot) (‚≠ê66 ¬∑ JavaScript) SVG based genome viewer written in javascript using D3 ([26093150](https://pubmed.ncbi.nlm.nih.gov/26093150))
646. [comprna/METEORE](https://github.com/comprna/METEORE) (‚≠ê66 ¬∑ R) Automatic DNA methylation detection from nanopore tools and their consensus model ([34103501](https://pubmed.ncbi.nlm.nih.gov/34103501))
647. [czodrowskilab/Machine-learning-meets-pKa](https://github.com/czodrowskilab/Machine-learning-meets-pKa) (‚≠ê66 ¬∑ Jupyter Notebook) None ([32226607](https://pubmed.ncbi.nlm.nih.gov/32226607))
648. [dereneaton/ipyrad](https://github.com/dereneaton/ipyrad) (‚≠ê66 ¬∑ Jupyter Notebook) Interactive assembly and analysis of RAD-seq data sets ([31904816](https://pubmed.ncbi.nlm.nih.gov/31904816))
649. [epiviz/epiviz](https://github.com/epiviz/epiviz) (‚≠ê66 ¬∑ JavaScript) EpiViz is a scientific information visualization tool for genetic and epigenetic data, used to aid in the exploration and understanding of correlations between various genome features. ([31782758](https://pubmed.ncbi.nlm.nih.gov/31782758))
650. [fkong7/MeshDeformNet](https://github.com/fkong7/MeshDeformNet) (‚≠ê66 ¬∑ Python) A deep-learning approach for direct whole-heart mesh reconstruction ([34543913](https://pubmed.ncbi.nlm.nih.gov/34543913))
651. [illumina/REViewer](https://github.com/illumina/REViewer) (‚≠ê66 ¬∑ C++) A tool for visualizing alignments of reads in regions containing tandem repeats ([35948990](https://pubmed.ncbi.nlm.nih.gov/35948990))
652. [marekborowiec/AMAS](https://github.com/marekborowiec/AMAS) (‚≠ê66 ¬∑ Python) Calculate summary statistics and manipulate multiple sequence alignments ([26835189](https://pubmed.ncbi.nlm.nih.gov/26835189))
653. [vertgenlab/gonomics](https://github.com/vertgenlab/gonomics) (‚≠ê66 ¬∑ Go) A collection of genomics software tools written in Go (golang). ([37624924](https://pubmed.ncbi.nlm.nih.gov/37624924))
654. [xiezhq/ISEScan](https://github.com/xiezhq/ISEScan) (‚≠ê66 ¬∑ Python) A python pipeline to identify IS (Insertion Sequence) elements in genome and metagenome ([29077810](https://pubmed.ncbi.nlm.nih.gov/29077810))
655. [zoecastillo/microbiomeExplorer](https://github.com/zoecastillo/microbiomeExplorer) (‚≠ê66 ¬∑ R) The Microbiome Explorer provides methods to analyze and visualize microbial community sequencing data either from the R command line or through a Shiny application. Written by Janina Reeder and Joseph N. Paulson. ([32960962](https://pubmed.ncbi.nlm.nih.gov/32960962))
656. [AllonKleinLab/SPRING](https://github.com/AllonKleinLab/SPRING) (‚≠ê65 ¬∑ OpenEdge ABL) None ([29228172](https://pubmed.ncbi.nlm.nih.gov/29228172))
657. [VoiceBeer/MS-MDA](https://github.com/VoiceBeer/MS-MDA) (‚≠ê65 ¬∑ Jupyter Notebook) Code of paper MS-MDA: Multisource Marginal Distribution Adaptation for Cross-subject and Cross-session EEG Emotion Recognition ([34949983](https://pubmed.ncbi.nlm.nih.gov/34949983))
658. [cafferychen777/ggpicrust2](https://github.com/cafferychen777/ggpicrust2) (‚≠ê65 ¬∑ R) Make Picrust2 Output Analysis and Visualization Easier ([37527009](https://pubmed.ncbi.nlm.nih.gov/37527009))
659. [gnina/models](https://github.com/gnina/models) (‚≠ê65 ¬∑ Python) Trained caffe models ([32865404](https://pubmed.ncbi.nlm.nih.gov/32865404))
660. [scwatts/FastSpar](https://github.com/scwatts/FastSpar) (‚≠ê65 ¬∑ C++) Rapid and scalable correlation estimation for compositional data ([30169561](https://pubmed.ncbi.nlm.nih.gov/30169561))
661. [GanjinZero/CODER](https://github.com/GanjinZero/CODER) (‚≠ê64 ¬∑ Python) CODER: Knowledge infused cross-lingual medical term embedding for term normalization. [JBI, ACL-BioNLP 2022] ([34990838](https://pubmed.ncbi.nlm.nih.gov/34990838))
662. [WatsonLab/MAGpy](https://github.com/WatsonLab/MAGpy) (‚≠ê64 ¬∑ Python) Snakemake pipeline for downstream analysis of metagenome-assembled genomes (MAGs) (pronounced mag-pie) ([30418481](https://pubmed.ncbi.nlm.nih.gov/30418481))
663. [aatapa/RLScore](https://github.com/aatapa/RLScore) (‚≠ê64 ¬∑ Python) None ([30329015](https://pubmed.ncbi.nlm.nih.gov/30329015))
664. [amcrisan/Adjutant](https://github.com/amcrisan/Adjutant) (‚≠ê64 ¬∑ R) Runs a pubmed query, returns results and allows user to explore high-level structure of returned documents ([30875428](https://pubmed.ncbi.nlm.nih.gov/30875428))
665. [broadinstitute/oncotator](https://github.com/broadinstitute/oncotator) (‚≠ê64 ¬∑ Python) None ([25703262](https://pubmed.ncbi.nlm.nih.gov/25703262))
666. [dphansti/Sushi](https://github.com/dphansti/Sushi) (‚≠ê64 ¬∑ R) Tools for visualizing genomics data ([24903420](https://pubmed.ncbi.nlm.nih.gov/24903420))
667. [lidaof/eg-react](https://github.com/lidaof/eg-react) (‚≠ê64 ¬∑ JavaScript) WashU Epigenome Browser ([32941613](https://pubmed.ncbi.nlm.nih.gov/32941613))
668. [pymodproject/pymod](https://github.com/pymodproject/pymod) (‚≠ê64 ¬∑ Python) PyMod 3 - sequence similarity searches, multiple sequence/structure alignments, and homology modeling within PyMOL. ([28158668](https://pubmed.ncbi.nlm.nih.gov/28158668); [33010156](https://pubmed.ncbi.nlm.nih.gov/33010156))
669. [tlemane/kmtricks](https://github.com/tlemane/kmtricks) (‚≠ê64 ¬∑ C++) modular k-mer count matrix and Bloom filter construction for large read collections ([36699393](https://pubmed.ncbi.nlm.nih.gov/36699393))
670. [OpenWaterAnalytics/epanet-dev](https://github.com/OpenWaterAnalytics/epanet-dev) (‚≠ê63 ¬∑ C++) Development Repository for the object-oriented implementation of EPANET ([31096428](https://pubmed.ncbi.nlm.nih.gov/31096428))
671. [chaoszhang/ASTER](https://github.com/chaoszhang/ASTER) (‚≠ê63 ¬∑ C++) Accurate Species Tree EstimatoR series: a family of optimation algorithms for species tree inference implemented in C++ (including ASTRAL-Pro & Weighted ASTRAL) ([36094339](https://pubmed.ncbi.nlm.nih.gov/36094339))
672. [fanglab/nanodisco](https://github.com/fanglab/nanodisco) (‚≠ê63 ¬∑ R) nanodisco: a toolbox for discovering and exploiting multiple types of DNA methylation from individual bacteria and microbiomes using nanopore sequencing. ([33820988](https://pubmed.ncbi.nlm.nih.gov/33820988))
673. [freeseek/score](https://github.com/freeseek/score) (‚≠ê63 ¬∑ C) Tools to work with GWAS-VCF summary statistics files ([38261650](https://pubmed.ncbi.nlm.nih.gov/38261650))
674. [histocartography/hact-net](https://github.com/histocartography/hact-net) (‚≠ê63 ¬∑ Python) None ([34781160](https://pubmed.ncbi.nlm.nih.gov/34781160))
675. [jwintersinger/kablammo](https://github.com/jwintersinger/kablammo) (‚≠ê63 ¬∑ JavaScript) Interactive, web-based BLAST results visualization tool. Accessible at https://kablammo.wasmuthlab.org. ([25481007](https://pubmed.ncbi.nlm.nih.gov/25481007))
676. [lanagarmire/DeepProg](https://github.com/lanagarmire/DeepProg) (‚≠ê63 ¬∑ Python) Deep-Learning framework for multi-omic and survival data integration ([34261540](https://pubmed.ncbi.nlm.nih.gov/34261540))
677. [microsoft/IGT-Intermolecular-Graph-Transformer](https://github.com/microsoft/IGT-Intermolecular-Graph-Transformer) (‚≠ê63 ¬∑ None) None ([35514186](https://pubmed.ncbi.nlm.nih.gov/35514186))
678. [deprekate/PHANOTATE](https://github.com/deprekate/PHANOTATE) (‚≠ê62 ¬∑ Python) PHANOTATE: a tool to annotate phage genomes. ([31329826](https://pubmed.ncbi.nlm.nih.gov/31329826))
679. [genesofeve/projectR](https://github.com/genesofeve/projectR) (‚≠ê62 ¬∑ HTML) None ([32167521](https://pubmed.ncbi.nlm.nih.gov/32167521))
680. [ggonnella/gfaviz](https://github.com/ggonnella/gfaviz) (‚≠ê62 ¬∑ C++) Graphical interactive tool for the visualization of sequence graphs in GFA format. ([30596893](https://pubmed.ncbi.nlm.nih.gov/30596893))
681. [huangzhii/SALMON](https://github.com/huangzhii/SALMON) (‚≠ê62 ¬∑ Python) SALMON: Survival Analysis Learning with Multi-Omics Neural Networks ([30906311](https://pubmed.ncbi.nlm.nih.gov/30906311))
682. [kweimann/ecg-transfer-learning](https://github.com/kweimann/ecg-transfer-learning) (‚≠ê62 ¬∑ Python) None ([33664343](https://pubmed.ncbi.nlm.nih.gov/33664343))
683. [meringlab/FlashWeave](https://github.com/meringlab/FlashWeave) (‚≠ê62 ¬∑ Julia) Inference of microbial interaction networks from large-scale heterogeneous abundance data ([31542415](https://pubmed.ncbi.nlm.nih.gov/31542415))
684. [niemasd/TreeCluster](https://github.com/niemasd/TreeCluster) (‚≠ê62 ¬∑ Python) Efficient phylogenetic clustering of viral sequences ([31437182](https://pubmed.ncbi.nlm.nih.gov/31437182))
685. [tk2/RetroSeq](https://github.com/tk2/RetroSeq) (‚≠ê62 ¬∑ Perl) RetroSeq is a bioinformatics tool that searches for mobile element insertions from aligned reads in a BAM file and a library of reference transposable elements. Please read the wiki page (link below) for usage instructions. Also, there is a page on the wiki describing how the 1000 genomes CEU trio was carried out with the files and parameters used for the various steps. ([23233656](https://pubmed.ncbi.nlm.nih.gov/23233656))
686. [tripal/tripal](https://github.com/tripal/tripal) (‚≠ê62 ¬∑ PHP) The Tripal package is  a suite of Drupal modules for creating biological (genomic, genetic, breeding) websites. Visit the Tripal homepage at http://tripal.info for documentation, support, and other information.  The Drupal project page is at http://drupal.org/project/tripal. ([29220446](https://pubmed.ncbi.nlm.nih.gov/29220446))
687. [ParkerLab/ataqv](https://github.com/ParkerLab/ataqv) (‚≠ê61 ¬∑ C++) A toolkit for QC and visualization of ATAC-seq results. ([32213349](https://pubmed.ncbi.nlm.nih.gov/32213349))
688. [RitchieLab/eQTpLot](https://github.com/RitchieLab/eQTpLot) (‚≠ê61 ¬∑ R)  Visualization of Colocalization Between eQTL and GWAS Data ([34273980](https://pubmed.ncbi.nlm.nih.gov/34273980))
689. [hellloxiaotian/ESRGCNN](https://github.com/hellloxiaotian/ESRGCNN) (‚≠ê61 ¬∑ Python) Image Super-resolution with An Enhanced Group Convolutional Neural Network (Neural Networks, 2022) ([35779445](https://pubmed.ncbi.nlm.nih.gov/35779445))
690. [nadeemlab/CEP](https://github.com/nadeemlab/CEP) (‚≠ê61 ¬∑ Python) Computational Endoscopy Platform (advanced deep learning toolset for analyzing endoscopy videos) [MICCAI'22, MICCAI'21, ISBI'21, CVPR'20] ([34642595](https://pubmed.ncbi.nlm.nih.gov/34642595); [35403172](https://pubmed.ncbi.nlm.nih.gov/35403172); [36178456](https://pubmed.ncbi.nlm.nih.gov/36178456))
691. [ncbi/BAMscale](https://github.com/ncbi/BAMscale) (‚≠ê61 ¬∑ C) BAMscale is a one-step tool for either 1) quantifying and normalizing the coverage of peaks or 2) generated scaled BigWig files for easy visualization of commonly used DNA-seq capture based methods. ([32321568](https://pubmed.ncbi.nlm.nih.gov/32321568))
692. [nikhilroxtomar/TGANet](https://github.com/nikhilroxtomar/TGANet) (‚≠ê61 ¬∑ Python) TGANet: Text-guided attention for improved polyp segmentation [Early Accepted & Student Travel Award at MICCAI 2022] ([36780239](https://pubmed.ncbi.nlm.nih.gov/36780239))
693. [ocxtal/libgaba](https://github.com/ocxtal/libgaba) (‚≠ê61 ¬∑ C) Adaptive semi-global banded alignment on string graphs ([29504909](https://pubmed.ncbi.nlm.nih.gov/29504909))
694. [superlouis/GATGNN](https://github.com/superlouis/GATGNN) (‚≠ê61 ¬∑ Python) Pytorch Repository for our work: Graph convolutional neural networks with global attention for improved materials property prediction ([32766627](https://pubmed.ncbi.nlm.nih.gov/32766627))
695. [danielhuson/megan-ce](https://github.com/danielhuson/megan-ce) (‚≠ê60 ¬∑ Java) MEGAN Community Edition ([27327495](https://pubmed.ncbi.nlm.nih.gov/27327495))
696. [griffithlab/pVAC-Seq](https://github.com/griffithlab/pVAC-Seq) (‚≠ê60 ¬∑ Python) DEPRECATED. This tool has been superseded by https://github.com/griffithlab/pVACtools ([26825632](https://pubmed.ncbi.nlm.nih.gov/26825632))
697. [husonlab/megan-ce](https://github.com/husonlab/megan-ce) (‚≠ê60 ¬∑ Java) MEGAN Community Edition ([36825821](https://pubmed.ncbi.nlm.nih.gov/36825821))
698. [kimmo1019/DeepCDR](https://github.com/kimmo1019/DeepCDR) (‚≠ê60 ¬∑ Python) Cancer Drug Response Prediction via a Hybrid Graph Convolutional Network ([33381841](https://pubmed.ncbi.nlm.nih.gov/33381841))
699. [qiyunzhu/woltka](https://github.com/qiyunzhu/woltka) (‚≠ê60 ¬∑ Python) Woltka: a versatile meta'omic data classifier ([36073806](https://pubmed.ncbi.nlm.nih.gov/36073806))
700. [simonhmartin/twisst](https://github.com/simonhmartin/twisst) (‚≠ê60 ¬∑ Python) Topology weighting by iterative sampling of sub-trees ([28341652](https://pubmed.ncbi.nlm.nih.gov/28341652))
701. [HALFpipe/HALFpipe](https://github.com/HALFpipe/HALFpipe) (‚≠ê59 ¬∑ Python) ENIGMA HALFpipe is a user-friendly software that facilitates reproducible analysis of fMRI data ([35305030](https://pubmed.ncbi.nlm.nih.gov/35305030))
702. [KCCG/ClinSV](https://github.com/KCCG/ClinSV) (‚≠ê59 ¬∑ Perl) Robust detection of clinically relevant structural and copy number variation from whole genome sequencing data ([33632298](https://pubmed.ncbi.nlm.nih.gov/33632298))
703. [Oshlack/STRetch](https://github.com/Oshlack/STRetch) (‚≠ê59 ¬∑ Python) Method for detecting STR expansions from short-read sequencing data ([30129428](https://pubmed.ncbi.nlm.nih.gov/30129428))
704. [VCCRI/CIDR](https://github.com/VCCRI/CIDR) (‚≠ê59 ¬∑ R) Clustering through Imputation and Dimensionality Reduction ([28351406](https://pubmed.ncbi.nlm.nih.gov/28351406))
705. [dewyman/TranscriptClean](https://github.com/dewyman/TranscriptClean) (‚≠ê59 ¬∑ Python) Correct mismatches, microindels, and noncanonical splice junctions in long reads that have been mapped to the genome ([29912287](https://pubmed.ncbi.nlm.nih.gov/29912287))
706. [lrkrol/SEREEGA](https://github.com/lrkrol/SEREEGA) (‚≠ê59 ¬∑ MATLAB) SEREEGA: Simulating Event-Related EEG Activity ([30114381](https://pubmed.ncbi.nlm.nih.gov/30114381))
707. [neurogenomics/rworkflows](https://github.com/neurogenomics/rworkflows) (‚≠ê59 ¬∑ HTML) Continuous integration for R packages. üîÄ Automates testing ‚úÖ, documentation website building üì¶, & containerised deployment üê≥. ([38167858](https://pubmed.ncbi.nlm.nih.gov/38167858))
708. [powellgenomicslab/scPred](https://github.com/powellgenomicslab/scPred) (‚≠ê59 ¬∑ R) scPred package for cell type prediction from scRNA-seq data ([31829268](https://pubmed.ncbi.nlm.nih.gov/31829268))
709. [robotoD/GenoVi](https://github.com/robotoD/GenoVi) (‚≠ê59 ¬∑ Python) GenoVi, an automated customizable circular genome visualizer for bacteria and archaea ([37014908](https://pubmed.ncbi.nlm.nih.gov/37014908))
710. [ymcui/expmrc](https://github.com/ymcui/expmrc) (‚≠ê59 ¬∑ Python) ExpMRC: Explainability Evaluation for Machine Reading Comprehension ([35497046](https://pubmed.ncbi.nlm.nih.gov/35497046))
711. [NBISweden/EMBLmyGFF3](https://github.com/NBISweden/EMBLmyGFF3) (‚≠ê58 ¬∑ Python) An efficient way to convert gff3 annotation files into EMBL format ready to submit. ([30103816](https://pubmed.ncbi.nlm.nih.gov/30103816))
712. [RVanDamme/MUFFIN](https://github.com/RVanDamme/MUFFIN) (‚≠ê58 ¬∑ Nextflow) hybrid assembly and differential binning workflow for metagenomics, transcriptomics and pathway analysis ([33561126](https://pubmed.ncbi.nlm.nih.gov/33561126))
713. [bcgsc/ntedit](https://github.com/bcgsc/ntedit) (‚≠ê58 ¬∑ C++) ‚úèÔ∏èultra fast, scalable genome assembly polishing and snv detection ([31095290](https://pubmed.ncbi.nlm.nih.gov/31095290); [35567771](https://pubmed.ncbi.nlm.nih.gov/35567771))
714. [brettc/partitionfinder](https://github.com/brettc/partitionfinder) (‚≠ê58 ¬∑ Python) PartitionFinder discovers optimal partitioning schemes for DNA sequences. ([28013191](https://pubmed.ncbi.nlm.nih.gov/28013191))
715. [carmonalab/STACAS](https://github.com/carmonalab/STACAS) (‚≠ê58 ¬∑ R) R package for semi-supervised single-cell data integration ([32845323](https://pubmed.ncbi.nlm.nih.gov/32845323))
716. [dtamayo/spock](https://github.com/dtamayo/spock) (‚≠ê58 ¬∑ Python) üññ A package to determine whether planetary orbital configurations will live long and prosper ([34599094](https://pubmed.ncbi.nlm.nih.gov/34599094))
717. [iprada/Circle-Map](https://github.com/iprada/Circle-Map) (‚≠ê58 ¬∑ Python) A method for circular DNA detection based on probabilistic mapping of ultrashort reads ([31830908](https://pubmed.ncbi.nlm.nih.gov/31830908))
718. [ksahlin/ultra](https://github.com/ksahlin/ultra) (‚≠ê58 ¬∑ Python) Long-read splice alignment with high accuracy ([34302453](https://pubmed.ncbi.nlm.nih.gov/34302453))
719. [mourisl/Rcorrector](https://github.com/mourisl/Rcorrector) (‚≠ê58 ¬∑ C++) Error correction for Illumina RNA-seq reads ([26500767](https://pubmed.ncbi.nlm.nih.gov/26500767))
720. [samsledje/D-SCRIPT](https://github.com/samsledje/D-SCRIPT) (‚≠ê58 ¬∑ Jupyter Notebook)  A structure-aware interpretable deep learning model for sequence-based prediction of protein-protein interactions ([37897686](https://pubmed.ncbi.nlm.nih.gov/37897686))
721. [zqgao22/HIGH-PPI](https://github.com/zqgao22/HIGH-PPI) (‚≠ê58 ¬∑ Python) None ([36841846](https://pubmed.ncbi.nlm.nih.gov/36841846))
722. [KailiWang1/DeepDTAF](https://github.com/KailiWang1/DeepDTAF) (‚≠ê57 ¬∑ Jupyter Notebook)  a deep learning architecture for protein-ligand binding affinity prediction ([33834190](https://pubmed.ncbi.nlm.nih.gov/33834190))
723. [QuKunLab/ATAC-pipe](https://github.com/QuKunLab/ATAC-pipe) (‚≠ê57 ¬∑ Python) Analysis pipeline for ATAC-seq Data ([29982337](https://pubmed.ncbi.nlm.nih.gov/29982337))
724. [SAMtoBAM/MUMandCo](https://github.com/SAMtoBAM/MUMandCo) (‚≠ê57 ¬∑ Shell) MUM&Co is a simple bash script that uses Whole Genome Alignment information provided by MUMmer (only v4) to detect Structural Variation ([32096823](https://pubmed.ncbi.nlm.nih.gov/32096823))
725. [SRKH/S4NN](https://github.com/SRKH/S4NN) (‚≠ê57 ¬∑ Jupyter Notebook) Temporal backpropagation for spiking neural networks with one spike per neuron, by S. R. Kheradpisheh and T. Masquelier, International Journal of Neural Systems (2020), doi: 10.1142/S0129065720500276 ([32466691](https://pubmed.ncbi.nlm.nih.gov/32466691))
726. [Vini2/phables](https://github.com/Vini2/phables) (‚≠ê57 ¬∑ Python) ü´ßüß¨ From fragmented assemblies to high-quality bacteriophage genomes ([37738590](https://pubmed.ncbi.nlm.nih.gov/37738590))
727. [abehr/pong](https://github.com/abehr/pong) (‚≠ê57 ¬∑ Python) Fast analysis and visualization of latent clusters in population genetic data ([27283948](https://pubmed.ncbi.nlm.nih.gov/27283948))
728. [ablab/viralVerify](https://github.com/ablab/viralVerify) (‚≠ê57 ¬∑ Python) viralVerify: viral contig verification tool ([32413137](https://pubmed.ncbi.nlm.nih.gov/32413137))
729. [dantaki/SV2](https://github.com/dantaki/SV2) (‚≠ê57 ¬∑ Python) Support Vector Structural Variation Genotyper ([29300834](https://pubmed.ncbi.nlm.nih.gov/29300834))
730. [eliberis/parapred](https://github.com/eliberis/parapred) (‚≠ê57 ¬∑ Python) Paratope Prediction using Deep Learning ([29672675](https://pubmed.ncbi.nlm.nih.gov/29672675))
731. [ggonnella/gfapy](https://github.com/ggonnella/gfapy) (‚≠ê57 ¬∑ Python) Gfapy: a flexible and extensible software library for handling sequence graphs in Python ([28645150](https://pubmed.ncbi.nlm.nih.gov/28645150))
732. [harbourlab/UPhyloplot2](https://github.com/harbourlab/UPhyloplot2) (‚≠ê57 ¬∑ Python) Draw phylogenetic trees of tumor evolution ([34090344](https://pubmed.ncbi.nlm.nih.gov/34090344))
733. [ksahlin/BESST](https://github.com/ksahlin/BESST) (‚≠ê57 ¬∑ Python) BESST - scaffolder for genomic assemblies ([27153683](https://pubmed.ncbi.nlm.nih.gov/27153683))
734. [rundherum/pymia](https://github.com/rundherum/pymia) (‚≠ê57 ¬∑ Python) pymia: A Python package for data handling and evaluation in deep learning-based medical image analysis ([33137700](https://pubmed.ncbi.nlm.nih.gov/33137700))
735. [telatin/BamToCov](https://github.com/telatin/BamToCov) (‚≠ê57 ¬∑ Nim) üèî  coverage extraction from BAM/CRAM files, supporting targets üìä ([35199151](https://pubmed.ncbi.nlm.nih.gov/35199151))
736. [JhuangLab/BioInstaller](https://github.com/JhuangLab/BioInstaller) (‚≠ê56 ¬∑ R) A comprehensive R package to construct interactive and reproducible biological data analysis applications based on the R platform ([30402350](https://pubmed.ncbi.nlm.nih.gov/30402350))
737. [PapenfussLab/bioshake](https://github.com/PapenfussLab/bioshake) (‚≠ê56 ¬∑ Haskell) Bioinformatics pipelines with Haskell and Shake ([31328031](https://pubmed.ncbi.nlm.nih.gov/31328031))
738. [bede/hostile](https://github.com/bede/hostile) (‚≠ê56 ¬∑ Python) Accurate host read removal ([38039142](https://pubmed.ncbi.nlm.nih.gov/38039142))
739. [bene51/3Dscript](https://github.com/bene51/3Dscript) (‚≠ê56 ¬∑ Java) Create 3D/4D animations using a natural language-based syntax ([34152405](https://pubmed.ncbi.nlm.nih.gov/34152405))
740. [cyang03/CHECKED](https://github.com/cyang03/CHECKED) (‚≠ê56 ¬∑ Jupyter Notebook) None ([34178179](https://pubmed.ncbi.nlm.nih.gov/34178179))
741. [ddofer/ProFET](https://github.com/ddofer/ProFET) (‚≠ê56 ¬∑ Python) ProFET: Protein Feature Engineering Toolkit for Machine Learning ([26130574](https://pubmed.ncbi.nlm.nih.gov/26130574))
742. [jltsiren/gbwt](https://github.com/jltsiren/gbwt) (‚≠ê56 ¬∑ C++) Substring index for paths in a graph ([31406990](https://pubmed.ncbi.nlm.nih.gov/31406990))
743. [lczech/genesis](https://github.com/lczech/genesis) (‚≠ê56 ¬∑ C++) A library for working with phylogenetic and population genetic data. ([32016344](https://pubmed.ncbi.nlm.nih.gov/32016344))
744. [materialsintelligence/matscholar](https://github.com/materialsintelligence/matscholar) (‚≠ê56 ¬∑ Python) Code to access the Matscholar public API. ([31361962](https://pubmed.ncbi.nlm.nih.gov/31361962))
745. [ncbi/SSDraw](https://github.com/ncbi/SSDraw) (‚≠ê56 ¬∑ Jupyter Notebook) None ([37953705](https://pubmed.ncbi.nlm.nih.gov/37953705))
746. [rnacentral/R2DT](https://github.com/rnacentral/R2DT) (‚≠ê56 ¬∑ PostScript) Visualise RNA secondary structure in consistent, reproducible and recognisable layouts ([34108470](https://pubmed.ncbi.nlm.nih.gov/34108470))
747. [volkamerlab/KinFragLib](https://github.com/volkamerlab/KinFragLib) (‚≠ê56 ¬∑ Jupyter Notebook) Kinase-focused fragment library ([33155465](https://pubmed.ncbi.nlm.nih.gov/33155465))
748. [xavierdidelot/TransPhylo](https://github.com/xavierdidelot/TransPhylo) (‚≠ê56 ¬∑ R) Reconstruction of transmission trees using genomic data ([28100788](https://pubmed.ncbi.nlm.nih.gov/28100788))
749. [ygidtu/trackplot](https://github.com/ygidtu/trackplot) (‚≠ê56 ¬∑ Python) trackplot is a tool for visualizing various next-generation sequencing (NGS) data, including DNA-seq, RNA-seq, single-cell RNA-seq and full-length sequencing datasets. ([37669275](https://pubmed.ncbi.nlm.nih.gov/37669275))
750. [BTheDragonMaster/pikachu](https://github.com/BTheDragonMaster/pikachu) (‚≠ê55 ¬∑ Python) Python-based Informatics Kit for Analysing Chemical Units ([35672769](https://pubmed.ncbi.nlm.nih.gov/35672769))
751. [Data2Dynamics/d2d](https://github.com/Data2Dynamics/d2d) (‚≠ê55 ¬∑ MATLAB) a modeling environment tailored to parameter estimation in dynamical systems ([32176768](https://pubmed.ncbi.nlm.nih.gov/32176768))
752. [Genotek/ClassifyCNV](https://github.com/Genotek/ClassifyCNV) (‚≠ê55 ¬∑ Python) ClassifyCNV: a tool for clinical annotation of copy-number variants ([33230148](https://pubmed.ncbi.nlm.nih.gov/33230148))
753. [MannLabs/ProteomicsVisualization](https://github.com/MannLabs/ProteomicsVisualization) (‚≠ê55 ¬∑ Jupyter Notebook) This is a collection of minimal code to visualize various aspects of proteomics data. ([35107884](https://pubmed.ncbi.nlm.nih.gov/35107884))
754. [OpenNFT/OpenNFT](https://github.com/OpenNFT/OpenNFT) (‚≠ê55 ¬∑ MATLAB) OpenNFT is an integrated software package designed for neurofeedback training and automatic real-time fMRI quality assessment (rtQA). ([28795112](https://pubmed.ncbi.nlm.nih.gov/28795112))
755. [RuleWorld/bionetgen](https://github.com/RuleWorld/bionetgen) (‚≠ê55 ¬∑ Perl) Rule-based modeling framework ([27402907](https://pubmed.ncbi.nlm.nih.gov/27402907))
756. [RyanCook94/inphared](https://github.com/RyanCook94/inphared) (‚≠ê55 ¬∑ Perl) Providing up-to-date phage genome databases, metrics and useful input files for a number of bioinformatic pipelines. ([36159887](https://pubmed.ncbi.nlm.nih.gov/36159887))
757. [YifanDengWHU/DDIMDL](https://github.com/YifanDengWHU/DDIMDL) (‚≠ê55 ¬∑ Python) None ([32407508](https://pubmed.ncbi.nlm.nih.gov/32407508))
758. [giannimonaco/ABIS](https://github.com/giannimonaco/ABIS) (‚≠ê55 ¬∑ R) None ([30726743](https://pubmed.ncbi.nlm.nih.gov/30726743))
759. [lczech/gappa](https://github.com/lczech/gappa) (‚≠ê55 ¬∑ C++) A toolkit for analyzing and visualizing phylogenetic (placement) data ([30169747](https://pubmed.ncbi.nlm.nih.gov/30169747); [32016344](https://pubmed.ncbi.nlm.nih.gov/32016344))
760. [muellan/metacache](https://github.com/muellan/metacache) (‚≠ê55 ¬∑ C++) memory efficient, fast & precise taxnomomic classification system for metagenomic read mapping ([28961782](https://pubmed.ncbi.nlm.nih.gov/28961782))
761. [paiardin/DockingPie](https://github.com/paiardin/DockingPie) (‚≠ê55 ¬∑ Python) A Consensus Docking Plugin for PyMOL ([35792827](https://pubmed.ncbi.nlm.nih.gov/35792827))
762. [pratas/cryfa](https://github.com/pratas/cryfa) (‚≠ê55 ¬∑ C++) A secure encryption tool for genomic data ([30020420](https://pubmed.ncbi.nlm.nih.gov/30020420))
763. [twestbrookunh/paladin](https://github.com/twestbrookunh/paladin) (‚≠ê55 ¬∑ C) Protein Alignment and Detection Interface ([28158639](https://pubmed.ncbi.nlm.nih.gov/28158639))
764. [vembrane/vembrane](https://github.com/vembrane/vembrane) (‚≠ê55 ¬∑ Python) vembrane filters VCF records using python expressions ([36519840](https://pubmed.ncbi.nlm.nih.gov/36519840))
765. [yuhoo0302/Segment-Anything-Model-for-Medical-Images](https://github.com/yuhoo0302/Segment-Anything-Model-for-Medical-Images) (‚≠ê55 ¬∑ Python) Codes and models for Medical Image Analysis (MIA) 2023 paper. Segment Anything Model for Medical Images?. ([38086235](https://pubmed.ncbi.nlm.nih.gov/38086235))
766. [BenoitMorel/GeneRax](https://github.com/BenoitMorel/GeneRax) (‚≠ê54 ¬∑ C++) None ([32502238](https://pubmed.ncbi.nlm.nih.gov/32502238); [35021210](https://pubmed.ncbi.nlm.nih.gov/35021210))
767. [Proteomicslab57357/UniprotR](https://github.com/Proteomicslab57357/UniprotR) (‚≠ê54 ¬∑ R) Retrieving Information of Proteins from Uniprot ([31843688](https://pubmed.ncbi.nlm.nih.gov/31843688))
768. [eskederks/eMAGMA-tutorial](https://github.com/eskederks/eMAGMA-tutorial) (‚≠ê54 ¬∑ None) A step by step guide on how to use eMAGMA, an approach to conducting eQTL informed gene-based tests. ([33624746](https://pubmed.ncbi.nlm.nih.gov/33624746))
769. [kiharalab/DOVE](https://github.com/kiharalab/DOVE) (‚≠ê54 ¬∑ Python) A Deep-learning based dOcking decoy eValuation mEthod ([31746961](https://pubmed.ncbi.nlm.nih.gov/31746961))
770. [lcnhappe/happe](https://github.com/lcnhappe/happe) (‚≠ê54 ¬∑ Matlab) Harvard Automated Pre-Processing Pipeline for EEG ([29535597](https://pubmed.ncbi.nlm.nih.gov/29535597))
771. [mlin/spVCF](https://github.com/mlin/spVCF) (‚≠ê54 ¬∑ C++) Sparse Project VCF: evolution of VCF to encode population genotype matrices efficiently ([33300997](https://pubmed.ncbi.nlm.nih.gov/33300997))
772. [morispi/CONSENT](https://github.com/morispi/CONSENT) (‚≠ê54 ¬∑ C++) Scalable long read self-correction and assembly polishing with multiple sequence alignment ([33436980](https://pubmed.ncbi.nlm.nih.gov/33436980))
773. [sequencing/NxTrim](https://github.com/sequencing/NxTrim) (‚≠ê54 ¬∑ C++) Adapter trimming and virtual library creation for Illumina Nextera Mate Pair libraries. ([25661542](https://pubmed.ncbi.nlm.nih.gov/25661542))
774. [smfleming/HMeta-d](https://github.com/smfleming/HMeta-d) (‚≠ê54 ¬∑ MATLAB) Hierarchical meta-d' model ([29877507](https://pubmed.ncbi.nlm.nih.gov/29877507))
775. [tanghaibao/quota-alignment](https://github.com/tanghaibao/quota-alignment) (‚≠ê54 ¬∑ Python) Guided synteny alignment between duplicated genomes (within specified quota constraint) ([21501495](https://pubmed.ncbi.nlm.nih.gov/21501495))
776. [ushaham/BatchEffectRemoval](https://github.com/ushaham/BatchEffectRemoval) (‚≠ê54 ¬∑ Python) None ([28419223](https://pubmed.ncbi.nlm.nih.gov/28419223))
777. [10XGenomics/scHLAcount](https://github.com/10XGenomics/scHLAcount) (‚≠ê53 ¬∑ TeX) Count HLA alleles in single-cell RNA-seq data ([32330223](https://pubmed.ncbi.nlm.nih.gov/32330223))
778. [RWilton/Arioc](https://github.com/RWilton/Arioc) (‚≠ê53 ¬∑ None) Arioc:  GPU-accelerated DNA short-read alignment ([25780763](https://pubmed.ncbi.nlm.nih.gov/25780763); [29554207](https://pubmed.ncbi.nlm.nih.gov/29554207))
779. [XiaoTaoWang/HiCLift](https://github.com/XiaoTaoWang/HiCLift) (‚≠ê53 ¬∑ Python) A fast and efficient tool for converting chromatin interaction data between genome assemblies ([37335863](https://pubmed.ncbi.nlm.nih.gov/37335863))
780. [biocore/gneiss](https://github.com/biocore/gneiss) (‚≠ê53 ¬∑ Jupyter Notebook) compositional data analysis toolbox ([28144630](https://pubmed.ncbi.nlm.nih.gov/28144630))
781. [insilichem/pychimera](https://github.com/insilichem/pychimera) (‚≠ê53 ¬∑ Python) Use UCSF Chimera Python API in a standard interpreter ([29340616](https://pubmed.ncbi.nlm.nih.gov/29340616))
782. [jag1g13/pycgtool](https://github.com/jag1g13/pycgtool) (‚≠ê53 ¬∑ Python) Generate coarse-grained molecular dynamics models from atomistic trajectories. ([28345910](https://pubmed.ncbi.nlm.nih.gov/28345910))
783. [lasigeBioTM/MER](https://github.com/lasigeBioTM/MER) (‚≠ê53 ¬∑ Shell) Minimal Named-Entity Recognizer (MER) ([30519990](https://pubmed.ncbi.nlm.nih.gov/30519990))
784. [rhysf/Synima](https://github.com/rhysf/Synima) (‚≠ê53 ¬∑ Perl) Synteny Imager ([29162056](https://pubmed.ncbi.nlm.nih.gov/29162056))
785. [theislab/AutoGeneS](https://github.com/theislab/AutoGeneS) (‚≠ê53 ¬∑ Jupyter Notebook) None ([34293324](https://pubmed.ncbi.nlm.nih.gov/34293324))
786. [vibbits/phyd3](https://github.com/vibbits/phyd3) (‚≠ê53 ¬∑ JavaScript) Phylogenetic tree viewer based on D3.js ([28525531](https://pubmed.ncbi.nlm.nih.gov/28525531))
787. [xiaoyeye/GCNG](https://github.com/xiaoyeye/GCNG) (‚≠ê53 ¬∑ Python) using graph convolutional neural network and spaital transcriptomics data to infer cell-cell interactions ([33303016](https://pubmed.ncbi.nlm.nih.gov/33303016))
788. [zhangxaochen/CuFusion](https://github.com/zhangxaochen/CuFusion) (‚≠ê53 ¬∑ C++) None ([28974030](https://pubmed.ncbi.nlm.nih.gov/28974030))
789. [BorgwardtLab/S3M](https://github.com/BorgwardtLab/S3M) (‚≠ê52 ¬∑ C++) A software package for statistically significant shapelet mining ([29949972](https://pubmed.ncbi.nlm.nih.gov/29949972); [33381811](https://pubmed.ncbi.nlm.nih.gov/33381811))
790. [acadTags/Explainable-Automated-Medical-Coding](https://github.com/acadTags/Explainable-Automated-Medical-Coding) (‚≠ê52 ¬∑ Python) Implementation and demo of explainable coding of clinical notes with Hierarchical Label-wise Attention Networks (HLAN) ([33711543](https://pubmed.ncbi.nlm.nih.gov/33711543))
791. [clwgg/nQuire](https://github.com/clwgg/nQuire) (‚≠ê52 ¬∑ C) A statistical framework for ploidy estimation using NGS short-read data ([29618319](https://pubmed.ncbi.nlm.nih.gov/29618319))
792. [cramerlab/warp](https://github.com/cramerlab/warp) (‚≠ê52 ¬∑ C#) None ([31591575](https://pubmed.ncbi.nlm.nih.gov/31591575))
793. [daquang/YAMDA](https://github.com/daquang/YAMDA) (‚≠ê52 ¬∑ Python) Yet Another Motif Discovery Algorithm ([29790915](https://pubmed.ncbi.nlm.nih.gov/29790915))
794. [immunedynamics/spectre](https://github.com/immunedynamics/spectre) (‚≠ê52 ¬∑ HTML) A computational toolkit in R for the integration, exploration, and analysis of high-dimensional single-cell cytometry and imaging data. ([33840138](https://pubmed.ncbi.nlm.nih.gov/33840138))
795. [jpquast/protti](https://github.com/jpquast/protti) (‚≠ê52 ¬∑ R) Picotti lab data analysis package. ([36699412](https://pubmed.ncbi.nlm.nih.gov/36699412))
796. [kundajelab/dfim](https://github.com/kundajelab/dfim) (‚≠ê52 ¬∑ Python) Deep Feature Interaction Maps (DFIM) ([30423062](https://pubmed.ncbi.nlm.nih.gov/30423062))
797. [maickrau/MBG](https://github.com/maickrau/MBG) (‚≠ê52 ¬∑ C++) None ([33475133](https://pubmed.ncbi.nlm.nih.gov/33475133))
798. [minoh0201/DeepMicro](https://github.com/minoh0201/DeepMicro) (‚≠ê52 ¬∑ Python) Deep representation learning for disease prediction based on microbiome data ([32265477](https://pubmed.ncbi.nlm.nih.gov/32265477))
799. [sigven/cpsr](https://github.com/sigven/cpsr) (‚≠ê52 ¬∑ R) Cancer Predisposition Sequencing Reporter (CPSR) ([34310709](https://pubmed.ncbi.nlm.nih.gov/34310709))
800. [theislab/trvae](https://github.com/theislab/trvae) (‚≠ê52 ¬∑ Python) Conditional out-of-distribution prediction ([33381839](https://pubmed.ncbi.nlm.nih.gov/33381839))
801. [BioinfoUNIBA/REDItools](https://github.com/BioinfoUNIBA/REDItools) (‚≠ê51 ¬∑ Python) REDItools are python scripts to investigate RNA editing at genomic scale. ([31996844](https://pubmed.ncbi.nlm.nih.gov/31996844))
802. [MolecularAI/Icolos](https://github.com/MolecularAI/Icolos) (‚≠ê51 ¬∑ Python) Icolos: A workflow manager for structure based post-processing of de novo generated small molecules ([36073898](https://pubmed.ncbi.nlm.nih.gov/36073898))
803. [PedroMTQ/mantis](https://github.com/PedroMTQ/mantis) (‚≠ê51 ¬∑ Python) A package to annotate protein sequences ([34076241](https://pubmed.ncbi.nlm.nih.gov/34076241))
804. [SofieVG/FlowSOM](https://github.com/SofieVG/FlowSOM) (‚≠ê51 ¬∑ R) Using self-organizing maps for visualization and interpretation of cytometry data ([25573116](https://pubmed.ncbi.nlm.nih.gov/25573116))
805. [UCLOrengoGroup/cath-tools](https://github.com/UCLOrengoGroup/cath-tools) (‚≠ê51 ¬∑ C++) Protein structure comparison tools such as SSAP and SNAP ([30295745](https://pubmed.ncbi.nlm.nih.gov/30295745))
806. [charite/jannovar](https://github.com/charite/jannovar) (‚≠ê51 ¬∑ Java) Annotation of VCF variants with functional impact and from databases (executable+library) ([24677618](https://pubmed.ncbi.nlm.nih.gov/24677618))
807. [chopralab/lemon](https://github.com/chopralab/lemon) (‚≠ê51 ¬∑ C++) A framework for rapidly mining structural information from the Protein Data Bank ([30873531](https://pubmed.ncbi.nlm.nih.gov/30873531); [32639154](https://pubmed.ncbi.nlm.nih.gov/32639154))
808. [frankMusacchia/Annocript](https://github.com/frankMusacchia/Annocript) (‚≠ê51 ¬∑ Perl) Annocript is a pipeline for the annotation of de-novo generated  transcriptomes. It executes BLAST analysis with UniProt, NCBI Conserved Domain Database and Nucleotide divisions, Gene Ontology, UniPathways and the Enzyme Commission. It gives information about the longest ORF (using DNA2PEP) and non-coding potential of the sequences (using Portrait). A final heuristic makes Annocript able to identify putative long non-coding RNAs among your transcripts. Like our facebook page to be always updated: https://www.facebook.com/annocript ([25701574](https://pubmed.ncbi.nlm.nih.gov/25701574))
809. [gtonkinhill/fastbaps](https://github.com/gtonkinhill/fastbaps) (‚≠ê51 ¬∑ R) A fast approximation to a Dirichlet Process Mixture model (DPM) for clustering genetic data ([31076776](https://pubmed.ncbi.nlm.nih.gov/31076776))
810. [hillerlab/GenomeAlignmentTools](https://github.com/hillerlab/GenomeAlignmentTools) (‚≠ê51 ¬∑ C) Tools for improving the sensitivity and specificity of genome alignments ([31742600](https://pubmed.ncbi.nlm.nih.gov/31742600))
811. [kassonlab/gmxapi](https://github.com/kassonlab/gmxapi) (‚≠ê51 ¬∑ C++) (outdated) fork of https://gitlab.com/gromacs/gromacs ([29912282](https://pubmed.ncbi.nlm.nih.gov/29912282))
812. [luoyuanlab/Clinical-Longformer](https://github.com/luoyuanlab/Clinical-Longformer) (‚≠ê51 ¬∑ Python) None ([36451266](https://pubmed.ncbi.nlm.nih.gov/36451266))
813. [niemasd/ViralMSA](https://github.com/niemasd/ViralMSA) (‚≠ê51 ¬∑ Python) Reference-guided multiple sequence alignment of viral genomes ([32814953](https://pubmed.ncbi.nlm.nih.gov/32814953))
814. [noriakis/CBNplot](https://github.com/noriakis/CBNplot) (‚≠ê51 ¬∑ R) Bayesian network plot for the enrichment analysis results ([35561164](https://pubmed.ncbi.nlm.nih.gov/35561164))
815. [nygenome/conpair](https://github.com/nygenome/conpair) (‚≠ê51 ¬∑ Python) Concordance and contamination estimator for tumor‚Äìnormal pairs ([27354699](https://pubmed.ncbi.nlm.nih.gov/27354699))
816. [psathyrella/partis](https://github.com/psathyrella/partis) (‚≠ê51 ¬∑ C++) B- and T-cell receptor sequence annotation, simulation, clonal family and germline inference, and affinity prediction ([26751373](https://pubmed.ncbi.nlm.nih.gov/26751373); [31329576](https://pubmed.ncbi.nlm.nih.gov/31329576); [33175831](https://pubmed.ncbi.nlm.nih.gov/33175831))
817. [ETHmodlab/BIMODAL](https://github.com/ETHmodlab/BIMODAL) (‚≠ê50 ¬∑ Python) Supporting code for the paper "Bidirectional Molecule Generation with Recurrent Neural Networks" (J. Chem. Inf. 2020, 60, 3). ([31904964](https://pubmed.ncbi.nlm.nih.gov/31904964))
818. [IBM/fold2seq](https://github.com/IBM/fold2seq) (‚≠ê50 ¬∑ Python) Code for Fold2Seq paper from ICML 2021 ([34423306](https://pubmed.ncbi.nlm.nih.gov/34423306))
819. [RASpicer/MetabolomicsTools](https://github.com/RASpicer/MetabolomicsTools) (‚≠ê50 ¬∑ None) None ([28890673](https://pubmed.ncbi.nlm.nih.gov/28890673))
820. [aldro61/kover](https://github.com/aldro61/kover) (‚≠ê50 ¬∑ Python) Learn interpretable computational phenotyping models from k-merized genomic data ([27671088](https://pubmed.ncbi.nlm.nih.gov/27671088))
821. [choderalab/ensembler](https://github.com/choderalab/ensembler) (‚≠ê50 ¬∑ Python) Automated omics-scale protein modeling and simulation setup. ([27337644](https://pubmed.ncbi.nlm.nih.gov/27337644))
822. [compbiomed/animalcules](https://github.com/compbiomed/animalcules) (‚≠ê50 ¬∑ R) R shiny app for interactive microbiome analysis ([33775256](https://pubmed.ncbi.nlm.nih.gov/33775256))
823. [insilicomedicine/BiAAE](https://github.com/insilicomedicine/BiAAE) (‚≠ê50 ¬∑ Python) Molecular Generation for Desired Transcriptome Changes with Adversarial Autoencoders ([32362822](https://pubmed.ncbi.nlm.nih.gov/32362822))
824. [kamimrcht/REINDEER](https://github.com/kamimrcht/REINDEER) (‚≠ê50 ¬∑ C++) REINDEER  REad Index for abuNDancE quERy ([32657392](https://pubmed.ncbi.nlm.nih.gov/32657392))
825. [marbl/harvest](https://github.com/marbl/harvest) (‚≠ê50 ¬∑ None) None ([25410596](https://pubmed.ncbi.nlm.nih.gov/25410596))
826. [mlbendall/telescope](https://github.com/mlbendall/telescope) (‚≠ê50 ¬∑ Python) Quantification of transposable element expression using RNA-seq ([31568525](https://pubmed.ncbi.nlm.nih.gov/31568525))
827. [oxpig/DEVELOP](https://github.com/oxpig/DEVELOP) (‚≠ê50 ¬∑ Python) None ([34881010](https://pubmed.ncbi.nlm.nih.gov/34881010))
828. [pachterlab/metakallisto](https://github.com/pachterlab/metakallisto) (‚≠ê50 ¬∑ Python) Using kallisto for metagenomic analysis ([28334086](https://pubmed.ncbi.nlm.nih.gov/28334086))
829. [psipred/cgdms](https://github.com/psipred/cgdms) (‚≠ê50 ¬∑ Python) Differentiable molecular simulation of proteins with a coarse-grained potential ([34473813](https://pubmed.ncbi.nlm.nih.gov/34473813))
830. [schmeing/ReSeq](https://github.com/schmeing/ReSeq) (‚≠ê50 ¬∑ C++) More realistic simulator for genomic DNA sequences from Illumina machines that achieves a similar k-mer spectrum as the original ([33608040](https://pubmed.ncbi.nlm.nih.gov/33608040))
831. [IFMlab/ChemFlow](https://github.com/IFMlab/ChemFlow) (‚≠ê49 ¬∑ Python) Computational Chemistry Workflows ([36603846](https://pubmed.ncbi.nlm.nih.gov/36603846); [37620503](https://pubmed.ncbi.nlm.nih.gov/37620503))
832. [KirillKryukov/naf](https://github.com/KirillKryukov/naf) (‚≠ê49 ¬∑ C) Nucleotide Archival Format - Compressed file format for DNA/RNA/protein sequences ([30799504](https://pubmed.ncbi.nlm.nih.gov/30799504))
833. [MolecularAI/Lib-INVENT](https://github.com/MolecularAI/Lib-INVENT) (‚≠ê49 ¬∑ Jupyter Notebook) None ([34460269](https://pubmed.ncbi.nlm.nih.gov/34460269))
834. [TheJacksonLaboratory/SVE](https://github.com/TheJacksonLaboratory/SVE) (‚≠ê49 ¬∑ Python) None ([29559002](https://pubmed.ncbi.nlm.nih.gov/29559002))
835. [haifangong/TRFE-Net-for-thyroid-nodule-segmentation](https://github.com/haifangong/TRFE-Net-for-thyroid-nodule-segmentation) (‚≠ê49 ¬∑ Python) [ISBI'21, CBM'22] TRFE-Net for thyroid nodule segmentation ([36812810](https://pubmed.ncbi.nlm.nih.gov/36812810))
836. [irycisBioinfo/PATO](https://github.com/irycisBioinfo/PATO) (‚≠ê49 ¬∑ R) PATO is a R package designed to analyze pangenomes (set of genomes) intra or inter species. ([34623430](https://pubmed.ncbi.nlm.nih.gov/34623430))
837. [rcs333/VAPiD](https://github.com/rcs333/VAPiD) (‚≠ê49 ¬∑ Shell) VAPiD: Viral Annotation and Identification Pipeline ([30674273](https://pubmed.ncbi.nlm.nih.gov/30674273))
838. [songweizhi/MetaCHIP](https://github.com/songweizhi/MetaCHIP) (‚≠ê49 ¬∑ Python) Horizontal gene transfer (HGT) identification pipeline ([30832740](https://pubmed.ncbi.nlm.nih.gov/30832740))
839. [stephaniehicks/qsmooth](https://github.com/stephaniehicks/qsmooth) (‚≠ê49 ¬∑ R) Smooth quantile normalization (qsmooth) is a generalization of quantile normalization, which is an average of the two types of assumptions about the data generation process: quantile normalization and quantile normalization between groups. ([29036413](https://pubmed.ncbi.nlm.nih.gov/29036413))
840. [stevemussmann/admixturePipeline](https://github.com/stevemussmann/admixturePipeline) (‚≠ê49 ¬∑ Python) A pipeline that accepts a VCF file to run through Admixture ([32727359](https://pubmed.ncbi.nlm.nih.gov/32727359); [38046098](https://pubmed.ncbi.nlm.nih.gov/38046098))
841. [BigDataBiology/macrel](https://github.com/BigDataBiology/macrel) (‚≠ê48 ¬∑ Python) Predict AMPs in (meta)genomes and peptides ([33384902](https://pubmed.ncbi.nlm.nih.gov/33384902))
842. [BojarLab/glycowork](https://github.com/BojarLab/glycowork) (‚≠ê48 ¬∑ Jupyter Notebook) Package for processing and analyzing glycans and their role in biology. ([34192308](https://pubmed.ncbi.nlm.nih.gov/34192308))
843. [Graylab/GeoDock](https://github.com/Graylab/GeoDock) (‚≠ê48 ¬∑ Python) Flexible Protein-Protein Docking with a Multi-Track Iterative Transformer. ([38148272](https://pubmed.ncbi.nlm.nih.gov/38148272))
844. [alfonsosemeraro/pyplutchik](https://github.com/alfonsosemeraro/pyplutchik) (‚≠ê48 ¬∑ Python) Python visualisation for Plutchik annotated corpora. ([34469455](https://pubmed.ncbi.nlm.nih.gov/34469455))
845. [bcgsc/orca](https://github.com/bcgsc/orca) (‚≠ê48 ¬∑ R) :whale: Genomics Research Container Architecture ([31004474](https://pubmed.ncbi.nlm.nih.gov/31004474))
846. [edawson/gfakluge](https://github.com/edawson/gfakluge) (‚≠ê48 ¬∑ C++) A C++ library and utilities for manipulating the Graphical Fragment Assembly format. ([31535074](https://pubmed.ncbi.nlm.nih.gov/31535074))
847. [fgcz/rawrr](https://github.com/fgcz/rawrr) (‚≠ê48 ¬∑ R) Access Orbitrap data in R lang using C# mono assembly - bioconductor package ([33686856](https://pubmed.ncbi.nlm.nih.gov/33686856))
848. [horsepurve/DeepRTplus](https://github.com/horsepurve/DeepRTplus) (‚≠ê48 ¬∑ Python) Deep (Transfer) Learning for Peptide Retention Time Prediction ([30114359](https://pubmed.ncbi.nlm.nih.gov/30114359))
849. [mrazvan22/brain-coloring](https://github.com/mrazvan22/brain-coloring) (‚≠ê48 ¬∑ Python) BrainPainter - Brain Visualisation Software ([33537683](https://pubmed.ncbi.nlm.nih.gov/33537683))
850. [msproteomicstools/msproteomicstools](https://github.com/msproteomicstools/msproteomicstools) (‚≠ê48 ¬∑ Python) Some hopefully useful tools for mass spectrometry applied to proteomics ([25788625](https://pubmed.ncbi.nlm.nih.gov/25788625))
851. [mstsuite/MuST](https://github.com/mstsuite/MuST) (‚≠ê48 ¬∑ Fortran) Multiple Scattering Theory code for first principles calculations ([33344416](https://pubmed.ncbi.nlm.nih.gov/33344416))
852. [sigven/oncoEnrichR](https://github.com/sigven/oncoEnrichR) (‚≠ê48 ¬∑ R) Cancer-dedicated gene set interpretation ([37551617](https://pubmed.ncbi.nlm.nih.gov/37551617))
853. [soedinglab/wish](https://github.com/soedinglab/wish) (‚≠ê48 ¬∑ C++) Predict prokaryotic host for phage metagenomic sequences ([28957499](https://pubmed.ncbi.nlm.nih.gov/28957499))
854. [stephenslab/flashr](https://github.com/stephenslab/flashr) (‚≠ê48 ¬∑ R) R package for Empirical Bayes Factor Analysis. ([37920532](https://pubmed.ncbi.nlm.nih.gov/37920532))
855. [wenqi006/SlideGraph](https://github.com/wenqi006/SlideGraph) (‚≠ê48 ¬∑ Python) None ([35640384](https://pubmed.ncbi.nlm.nih.gov/35640384))
856. [willtownes/nsf-paper](https://github.com/willtownes/nsf-paper) (‚≠ê48 ¬∑ Jupyter Notebook) Nonnegative spatial factorization for multivariate count data ([36587187](https://pubmed.ncbi.nlm.nih.gov/36587187))
857. [yuzhimanhua/lm-lstm-crf](https://github.com/yuzhimanhua/lm-lstm-crf) (‚≠ê48 ¬∑ None) None ([30307536](https://pubmed.ncbi.nlm.nih.gov/30307536))
858. [yvquanli/trimnet](https://github.com/yvquanli/trimnet) (‚≠ê48 ¬∑ Python) Code for paper "TrimNet: learning molecular representation from triplet messages for biomedicine " ([33147620](https://pubmed.ncbi.nlm.nih.gov/33147620))
859. [ADicksonLab/wepy](https://github.com/ADicksonLab/wepy) (‚≠ê47 ¬∑ Python) Weighted Ensemble simulation framework in Python ([33344813](https://pubmed.ncbi.nlm.nih.gov/33344813))
860. [HannesStark/protein-localization](https://github.com/HannesStark/protein-localization) (‚≠ê47 ¬∑ Jupyter Notebook) Using Transformer protein embeddings with a linear attention mechanism to make SOTA de-novo predictions for the subcellular location of proteins :microscope: ([36700108](https://pubmed.ncbi.nlm.nih.gov/36700108))
861. [IARCbioinfo/needlestack](https://github.com/IARCbioinfo/needlestack) (‚≠ê47 ¬∑ R) Multi-sample somatic variant caller ([32363341](https://pubmed.ncbi.nlm.nih.gov/32363341))
862. [KlugerLab/t-SNE-Heatmaps](https://github.com/KlugerLab/t-SNE-Heatmaps) (‚≠ê47 ¬∑ R) Beta version of 1D t-SNE heatmaps to visualize expression patterns of hundreds of genes simultaneously in scRNA-seq ([30742040](https://pubmed.ncbi.nlm.nih.gov/30742040))
863. [LaboratorioBioinformatica/MARVEL](https://github.com/LaboratorioBioinformatica/MARVEL) (‚≠ê47 ¬∑ Python) MARVEL: Metagenomic Analyses and Retrieval of Viral ELements ([30131825](https://pubmed.ncbi.nlm.nih.gov/30131825))
864. [RealityBending/Pyllusion](https://github.com/RealityBending/Pyllusion) (‚≠ê47 ¬∑ Python) A Parametric Framework to Generate Visual Illusions using Python ([34841973](https://pubmed.ncbi.nlm.nih.gov/34841973))
865. [WGLab/bioformer](https://github.com/WGLab/bioformer) (‚≠ê47 ¬∑ None) Bioformer: an efficient BERT model for biomedical text mining ([36945685](https://pubmed.ncbi.nlm.nih.gov/36945685))
866. [brentp/hts-nim-tools](https://github.com/brentp/hts-nim-tools) (‚≠ê47 ¬∑ Nim) useful command-line tools written to showcase hts-nim ([29718142](https://pubmed.ncbi.nlm.nih.gov/29718142))
867. [hkaneko1985/dcekit](https://github.com/hkaneko1985/dcekit) (‚≠ê47 ¬∑ Python) DCEKit (Data Chemical Engineering toolKit) ([35309472](https://pubmed.ncbi.nlm.nih.gov/35309472); [37360490](https://pubmed.ncbi.nlm.nih.gov/37360490))
868. [joshr17/IFM](https://github.com/joshr17/IFM) (‚≠ê47 ¬∑ Python) Code for paper "Can contrastive learning avoid shortcut solutions?" NeurIPS 2021. ([35546903](https://pubmed.ncbi.nlm.nih.gov/35546903))
869. [rothlab/chromozoom](https://github.com/rothlab/chromozoom) (‚≠ê47 ¬∑ JavaScript) ChromoZoom is a fast, fluid web-based genome browser ([23220575](https://pubmed.ncbi.nlm.nih.gov/23220575))
870. [xq141839/DCSAU-Net](https://github.com/xq141839/DCSAU-Net) (‚≠ê47 ¬∑ Python) CIBM-2023: A deeper and more compact split-attention U-Net for medical image segmentation ([36736096](https://pubmed.ncbi.nlm.nih.gov/36736096))
871. [BimberLab/DISCVRSeq](https://github.com/BimberLab/DISCVRSeq) (‚≠ê46 ¬∑ HTML) A collection of command line tools for working with sequencing data ([31309221](https://pubmed.ncbi.nlm.nih.gov/31309221))
872. [JLSteenwyk/PhyKIT](https://github.com/JLSteenwyk/PhyKIT) (‚≠ê46 ¬∑ Python) a UNIX shell toolkit for processing and analyzing multiple sequence alignments and phylogenies ([33560364](https://pubmed.ncbi.nlm.nih.gov/33560364))
873. [LooseLab/bulkvis](https://github.com/LooseLab/bulkvis) (‚≠ê46 ¬∑ Python) None ([30462145](https://pubmed.ncbi.nlm.nih.gov/30462145))
874. [Merck/Matcher](https://github.com/Merck/Matcher) (‚≠ê46 ¬∑ Python) Matcher is a tool for understanding how chemical structure optimization problems have been solved. Matcher enables deep control over searching structure/activity relationships (SAR) derived from large datasets, and takes the form of an accessible web application with simple deployment. Matcher is built around the mmpdb platform. ([36977316](https://pubmed.ncbi.nlm.nih.gov/36977316))
875. [aehrc/cvt2distilgpt2](https://github.com/aehrc/cvt2distilgpt2) (‚≠ê46 ¬∑ Python) Improving Chest X-Ray Report Generation by Leveraging Warm-Starting ([37783533](https://pubmed.ncbi.nlm.nih.gov/37783533))
876. [algbio/themisto](https://github.com/algbio/themisto) (‚≠ê46 ¬∑ C++) Space-efficient pseudoalignment with a colored de Bruijn graph ([37387143](https://pubmed.ncbi.nlm.nih.gov/37387143))
877. [aquaskyline/LRSIM](https://github.com/aquaskyline/LRSIM) (‚≠ê46 ¬∑ C) 10x Genomics Reads Simulator ([29213995](https://pubmed.ncbi.nlm.nih.gov/29213995))
878. [baranwa2/Struct2Graph](https://github.com/baranwa2/Struct2Graph) (‚≠ê46 ¬∑ Python) A PyTorch implementation of GCN with mutual attention for protein-protein interaction prediction ([36088285](https://pubmed.ncbi.nlm.nih.gov/36088285))
879. [blaxterlab/blobology](https://github.com/blaxterlab/blobology) (‚≠ê46 ¬∑ Perl) Tools for making blobplots or Taxon-Annotated-GC-Coverage plots (TAGC plots) to visualise the contents of genome assembly data sets as a QC step (NO LONGER MANTAINED) ([24348509](https://pubmed.ncbi.nlm.nih.gov/24348509))
880. [genomicsITER/NanoDJ](https://github.com/genomicsITER/NanoDJ) (‚≠ê46 ¬∑ Jupyter Notebook) NanoDJ: A Dockerized Jupyter Notebook for Interactive Oxford Nanopore MinION Sequence Manipulation and Genome Assembly ([31072312](https://pubmed.ncbi.nlm.nih.gov/31072312))
881. [hosseinshn/MOLI](https://github.com/hosseinshn/MOLI) (‚≠ê46 ¬∑ Jupyter Notebook) MOLI: Multi-Omics Late Integration with deep neural networks for drug response prediction ([31510700](https://pubmed.ncbi.nlm.nih.gov/31510700))
882. [ignaciorlando/red-lesion-detection](https://github.com/ignaciorlando/red-lesion-detection) (‚≠ê46 ¬∑ MATLAB) This code implements a red lesion detection method based on a combination of hand-crafted features and CNN based descriptors. Our paper is under revision now, so please do not use this repository until we release the paper. ([29157445](https://pubmed.ncbi.nlm.nih.gov/29157445))
883. [jordisr/poreover](https://github.com/jordisr/poreover) (‚≠ê46 ¬∑ Python) Nanopore basecalling and consensus decoding ([33468205](https://pubmed.ncbi.nlm.nih.gov/33468205))
884. [precimed/mixer](https://github.com/precimed/mixer) (‚≠ê46 ¬∑ C) Causal Mixture Model for GWAS summary statistics ([32539089](https://pubmed.ncbi.nlm.nih.gov/32539089))
885. [wgmao/PLIER](https://github.com/wgmao/PLIER) (‚≠ê46 ¬∑ R) Pathway-Level Information Extractor (PLIER): a generative model for gene expression data ([31249421](https://pubmed.ncbi.nlm.nih.gov/31249421))
886. [PayamDiba/SERGIO](https://github.com/PayamDiba/SERGIO) (‚≠ê45 ¬∑ Python) A simulator for single-cell expression data  guided by gene regulatory networks ([32871105](https://pubmed.ncbi.nlm.nih.gov/32871105))
887. [PoonLab/covizu](https://github.com/PoonLab/covizu) (‚≠ê45 ¬∑ JavaScript) Rapid analysis and visualization of coronavirus genome variation ([37124703](https://pubmed.ncbi.nlm.nih.gov/37124703))
888. [SmartDataAnalytics/BioKEEN](https://github.com/SmartDataAnalytics/BioKEEN) (‚≠ê45 ¬∑ Jupyter Notebook) A computational library for learning and evaluating biological knowledge graph embeddings - please see the main PyKEEN repo at https://github.com/pykeen/pykeen/ ([30768158](https://pubmed.ncbi.nlm.nih.gov/30768158))
889. [VCCRI/Sierra](https://github.com/VCCRI/Sierra) (‚≠ê45 ¬∑ R) Discover differential transcript usage from polyA-captured single cell RNA-seq data ([32641141](https://pubmed.ncbi.nlm.nih.gov/32641141))
890. [biopragmatics/biomappings](https://github.com/biopragmatics/biomappings) (‚≠ê45 ¬∑ Jupyter Notebook) üó∫Ô∏è Community curated and predicted equivalences and related mappings between named biological entities that are not available from primary sources. ([36916735](https://pubmed.ncbi.nlm.nih.gov/36916735))
891. [gerberlab/mitre](https://github.com/gerberlab/mitre) (‚≠ê45 ¬∑ Python) The Microbiome Interpretable Temporal Rule Engine ([31477162](https://pubmed.ncbi.nlm.nih.gov/31477162))
892. [jyangfsu/WQChartPy](https://github.com/jyangfsu/WQChartPy) (‚≠ê45 ¬∑ Python) Python Package for Visualization of Water Geochemistry Data ([35157303](https://pubmed.ncbi.nlm.nih.gov/35157303))
893. [kshulgina/codetta](https://github.com/kshulgina/codetta) (‚≠ê45 ¬∑ Python) None ([36511586](https://pubmed.ncbi.nlm.nih.gov/36511586))
894. [nadeemlab/Physics-ArX](https://github.com/nadeemlab/Physics-ArX) (‚≠ê45 ¬∑ Python) Physics-based data augmentation library for quantifying CT and CBCT images in radiotherapy [PMB'23, PMB'21, Medical Physics'21, AAPM'21] ([34245012](https://pubmed.ncbi.nlm.nih.gov/34245012))
895. [oddt/rfscorevs](https://github.com/oddt/rfscorevs) (‚≠ê45 ¬∑ Jupyter Notebook) RF-Score-VS - Random forest based protein-ligand scoring function for Virtual Screening ([28440302](https://pubmed.ncbi.nlm.nih.gov/28440302))
896. [petmri/ROCKETSHIP](https://github.com/petmri/ROCKETSHIP) (‚≠ê45 ¬∑ MATLAB) Dynamic Contrast Enhanced (DCE) MRI Analysis Software ([26076957](https://pubmed.ncbi.nlm.nih.gov/26076957))
897. [qingyuzhao/br-net](https://github.com/qingyuzhao/br-net) (‚≠ê45 ¬∑ Jupyter Notebook) None ([33243992](https://pubmed.ncbi.nlm.nih.gov/33243992))
898. [rinikerlab/Ensembler](https://github.com/rinikerlab/Ensembler) (‚≠ê45 ¬∑ Python) Ensembler is a python package that provides fast and easy access to 1D and 2D model system simulations. It can be used for method development or to deepen understanding of a broad spectrum of modeling methods, from basic sampling techniques to enhanced sampling and free energy calculations. It is easy to install, fast, increases shareability, comparability, and reproducibility of scientific code developments. ([33512157](https://pubmed.ncbi.nlm.nih.gov/33512157))
899. [shilpagarg/WHdenovo](https://github.com/shilpagarg/WHdenovo) (‚≠ê45 ¬∑ Python) None ([31860070](https://pubmed.ncbi.nlm.nih.gov/31860070))
900. [stephenslab/rss](https://github.com/stephenslab/rss) (‚≠ê45 ¬∑ MATLAB) Regression with Summary Statistics. ([29399241](https://pubmed.ncbi.nlm.nih.gov/29399241))
901. [variani/lme4qtl](https://github.com/variani/lme4qtl) (‚≠ê45 ¬∑ R) Mixed models @lme4 + custom covariances + parameter constraints ([29486711](https://pubmed.ncbi.nlm.nih.gov/29486711))
902. [yijingru/ANCIS-Pytorch](https://github.com/yijingru/ANCIS-Pytorch) (‚≠ê45 ¬∑ Python) [Medical Image Analysis 2019] Attentive Neural Cell Instance Segmentation ([31103790](https://pubmed.ncbi.nlm.nih.gov/31103790))
903. [JLiangLab/BenchmarkTransferLearning](https://github.com/JLiangLab/BenchmarkTransferLearning) (‚≠ê44 ¬∑ Python) Official PyTorch Implementation and Pre-trained Models for Benchmarking Transfer Learning for Medical Image Analysis ([35713581](https://pubmed.ncbi.nlm.nih.gov/35713581))
904. [LyonsLab/coge](https://github.com/LyonsLab/coge) (‚≠ê44 ¬∑ Perl) CoGe (Comparative Genomics) Platform ([28334338](https://pubmed.ncbi.nlm.nih.gov/28334338); [29474529](https://pubmed.ncbi.nlm.nih.gov/29474529))
905. [MedChaabane/deepRAM](https://github.com/MedChaabane/deepRAM) (‚≠ê44 ¬∑ Python) End-to-end deep learning toolkit for predicting protein binding sites and motifs. ([31510640](https://pubmed.ncbi.nlm.nih.gov/31510640))
906. [TheJacksonLaboratory/Picky](https://github.com/TheJacksonLaboratory/Picky) (‚≠ê44 ¬∑ Perl) Structural Variants Pipeline for Long Reads ([29713081](https://pubmed.ncbi.nlm.nih.gov/29713081))
907. [ZJUYH/radar-to-lidar-place-recognition](https://github.com/ZJUYH/radar-to-lidar-place-recognition) (‚≠ê44 ¬∑ Python) Radar-to-Lidar: Heterogeneous Place Recognition via Joint Learning ([34079825](https://pubmed.ncbi.nlm.nih.gov/34079825))
908. [blackrim/treePL](https://github.com/blackrim/treePL) (‚≠ê44 ¬∑ C++) Phylogenetic penalized likelihood ([22908216](https://pubmed.ncbi.nlm.nih.gov/22908216))
909. [bobeobibo/phigaro](https://github.com/bobeobibo/phigaro) (‚≠ê44 ¬∑ HTML) Phigaro is a scalable command-line tool for predicting phages and prophages ([32311023](https://pubmed.ncbi.nlm.nih.gov/32311023))
910. [dmnfarrell/epitopepredict](https://github.com/dmnfarrell/epitopepredict) (‚≠ê44 ¬∑ Jupyter Notebook) Python package  and command line tool for epitope prediction ([36824339](https://pubmed.ncbi.nlm.nih.gov/36824339))
911. [duolinwang/MusiteDeep](https://github.com/duolinwang/MusiteDeep) (‚≠ê44 ¬∑ Python) MusiteDeep provides a deep-learning method for general and kinase-specific phosphorylation site prediction. It is implemented by deep learning library Keras and Theano backend (the Keras2.0 and Tensorflow backend implementation were also provided under folder MusiteDeep_Keras2.0). At present, MusiteDeep only provides prediction of human phosphorylation sites; however, it also provides customized model training that enables users to train other PTM prediction models by using their own training data sets based on either CPU or GPU. ([29036382](https://pubmed.ncbi.nlm.nih.gov/29036382); [32324217](https://pubmed.ncbi.nlm.nih.gov/32324217))
912. [evidenceontology/evidenceontology](https://github.com/evidenceontology/evidenceontology) (‚≠ê44 ¬∑ HTML) Evidence & Conclusion Ontology development site:  Use ISSUES to request terms. See WIKI for how to request terms. See README for how to cite ECO. Visit our website for more project info. ([34986598](https://pubmed.ncbi.nlm.nih.gov/34986598))
913. [guopengf/FL-MRCM](https://github.com/guopengf/FL-MRCM) (‚≠ê44 ¬∑ Python) Multi-institutional Collaborations for Improving Deep Learning-based Magnetic Resonance Image Reconstruction Using Federated Learning ([35444379](https://pubmed.ncbi.nlm.nih.gov/35444379))
914. [ncclabsustech/DeepSeparator](https://github.com/ncclabsustech/DeepSeparator) (‚≠ê44 ¬∑ Python) Deep learning model for EEG artifact removal ([35378524](https://pubmed.ncbi.nlm.nih.gov/35378524))
915. [nluhmann/BlastFrost](https://github.com/nluhmann/BlastFrost) (‚≠ê44 ¬∑ C++) None ([33430919](https://pubmed.ncbi.nlm.nih.gov/33430919))
916. [perishky/meffil](https://github.com/perishky/meffil) (‚≠ê44 ¬∑ HTML) Efficient algorithms for analyzing DNA methylation data. ([29931280](https://pubmed.ncbi.nlm.nih.gov/29931280))
917. [quxiaojian/PGA](https://github.com/quxiaojian/PGA) (‚≠ê44 ¬∑ Perl) Plastid Genome Annotator ([31139240](https://pubmed.ncbi.nlm.nih.gov/31139240))
918. [BioinfoUNIBA/REDItools2](https://github.com/BioinfoUNIBA/REDItools2) (‚≠ê43 ¬∑ Python) None ([32838738](https://pubmed.ncbi.nlm.nih.gov/32838738))
919. [ELELAB/mutatex](https://github.com/ELELAB/mutatex) (‚≠ê43 ¬∑ Python) scripts and facilities for in-silico mutagenesis with FoldX ([35323860](https://pubmed.ncbi.nlm.nih.gov/35323860))
920. [UnixJunkie/FASMIFRA](https://github.com/UnixJunkie/FASMIFRA) (‚≠ê43 ¬∑ Python) Molecular Generation by Fast Assembly of SMILES Fragments ([34775976](https://pubmed.ncbi.nlm.nih.gov/34775976))
921. [andreasmang/claire](https://github.com/andreasmang/claire) (‚≠ê43 ¬∑ C++) Constrained Large Deformation Diffeomorphic Image Registration (CLAIRE) ([35295546](https://pubmed.ncbi.nlm.nih.gov/35295546))
922. [andreped/NoCodeSeg](https://github.com/andreped/NoCodeSeg) (‚≠ê43 ¬∑ Jupyter Notebook) üî¨ Code-free deep segmentation for computational pathology ([35155486](https://pubmed.ncbi.nlm.nih.gov/35155486))
923. [arttumiettinen/pi2](https://github.com/arttumiettinen/pi2) (‚≠ê43 ¬∑ C++) C++ library and command-line software for processing and analysis of terabyte-scale volume images locally or on a computing cluster. ([31116382](https://pubmed.ncbi.nlm.nih.gov/31116382))
924. [brentp/combined-pvalues](https://github.com/brentp/combined-pvalues) (‚≠ê43 ¬∑ Python) combining p-values using modified stouffer-liptak for spatially correlated results (probes) ([22954632](https://pubmed.ncbi.nlm.nih.gov/22954632))
925. [cellgeni/batchbench](https://github.com/cellgeni/batchbench) (‚≠ê43 ¬∑ R) None ([33524142](https://pubmed.ncbi.nlm.nih.gov/33524142))
926. [dayoonkwon/BioAge](https://github.com/dayoonkwon/BioAge) (‚≠ê43 ¬∑ R) Biological Age Calculations Using Several Biomarker Algorithms ([34725754](https://pubmed.ncbi.nlm.nih.gov/34725754))
927. [foxlf823/Multi-Filter-Residual-Convolutional-Neural-Network](https://github.com/foxlf823/Multi-Filter-Residual-Convolutional-Neural-Network) (‚≠ê43 ¬∑ Python) Multi-Filter Residual Convolutional Neural Network for Text Classification ([34322282](https://pubmed.ncbi.nlm.nih.gov/34322282))
928. [leylabmpi/Struo](https://github.com/leylabmpi/Struo) (‚≠ê43 ¬∑ Jupyter Notebook) Ley Lab MetaGenome Profiler DataBase generator ([31778148](https://pubmed.ncbi.nlm.nih.gov/31778148))
929. [microsoft/RespireNet](https://github.com/microsoft/RespireNet) (‚≠ê43 ¬∑ Python) This is the official implementation of the work RespireNet. ([34891348](https://pubmed.ncbi.nlm.nih.gov/34891348))
930. [psipred/DeepCov](https://github.com/psipred/DeepCov) (‚≠ê43 ¬∑ Python) Fully convolutional neural networks for protein residue-residue contact prediction ([29718112](https://pubmed.ncbi.nlm.nih.gov/29718112))
931. [vaquerizaslab/tadtool](https://github.com/vaquerizaslab/tadtool) (‚≠ê43 ¬∑ Python) TADtool is an interactive tool for the identification of meaningful parameters in TAD-calling algorithms for Hi-C data. ([27318199](https://pubmed.ncbi.nlm.nih.gov/27318199))
932. [AChavignon/PALA](https://github.com/AChavignon/PALA) (‚≠ê42 ¬∑ MATLAB) Sharing scripts and functions for OPUS-PALA article, and LOTUS Software. All functions are usable with agreement from their owner. ([35177778](https://pubmed.ncbi.nlm.nih.gov/35177778))
933. [BSGOxford/BrowseVCF](https://github.com/BSGOxford/BrowseVCF) (‚≠ê42 ¬∑ Python) BrowseVCF is a web-based application and workflow to quickly prioritise disease-causative variants in VCF files. ([27373737](https://pubmed.ncbi.nlm.nih.gov/27373737))
934. [CBIIT/LDlinkR](https://github.com/CBIIT/LDlinkR) (‚≠ê42 ¬∑ R) LDlinkR ([32180801](https://pubmed.ncbi.nlm.nih.gov/32180801))
935. [CDCgov/datasets-sars-cov-2](https://github.com/CDCgov/datasets-sars-cov-2) (‚≠ê42 ¬∑ Perl) Benchmark datasets for WGS analysis of SARS-CoV-2. (https://peerj.com/articles/13821/) ([36093336](https://pubmed.ncbi.nlm.nih.gov/36093336))
936. [GATB/simka](https://github.com/GATB/simka) (‚≠ê42 ¬∑ C++) Simka and SimkaMin are comparative metagenomics methods dedicated to NGS datasets. ([31504187](https://pubmed.ncbi.nlm.nih.gov/31504187))
937. [HaploKit/vechat](https://github.com/HaploKit/vechat) (‚≠ê42 ¬∑ C++) Correcting errors in noisy long reads using variation graphs ([36333324](https://pubmed.ncbi.nlm.nih.gov/36333324))
938. [OHNLP/MedTator](https://github.com/OHNLP/MedTator) (‚≠ê42 ¬∑ JavaScript) A Serverless Text Annotation Tool for Corpus Development ([34983060](https://pubmed.ncbi.nlm.nih.gov/34983060))
939. [PacificBiosciences/HiPhase](https://github.com/PacificBiosciences/HiPhase) (‚≠ê42 ¬∑ Rust) Small variant, structural variant, and short tandem repeat phasing tool for PacBio HiFi reads ([38269623](https://pubmed.ncbi.nlm.nih.gov/38269623))
940. [SouthGreenPlatform/panache](https://github.com/SouthGreenPlatform/panache) (‚≠ê42 ¬∑ Vue) Panache is a web-based interface designed for the visualization of linearized pangenomes. It can be used to show presence/absence information of pangenomic blocks of sequence or genes in a browser-like display. ([34601567](https://pubmed.ncbi.nlm.nih.gov/34601567))
941. [YuSugihara/MutMap](https://github.com/YuSugihara/MutMap) (‚≠ê42 ¬∑ Python) MutMap pipeline to identify causative mutations responsible for a phenotype ([35321412](https://pubmed.ncbi.nlm.nih.gov/35321412))
942. [ablab/TandemTools](https://github.com/ablab/TandemTools) (‚≠ê42 ¬∑ C++) Tool for assessing/improving assembly quality in extra-long tandem repeats ([32657355](https://pubmed.ncbi.nlm.nih.gov/32657355))
943. [alanqrwang/keymorph](https://github.com/alanqrwang/keymorph) (‚≠ê42 ¬∑ Python) Robust multimodal brain registration via keypoints ([37769550](https://pubmed.ncbi.nlm.nih.gov/37769550))
944. [boxiangliu/ParaMed](https://github.com/boxiangliu/ParaMed) (‚≠ê42 ¬∑ Python) Chinese to English medical translation ([34488734](https://pubmed.ncbi.nlm.nih.gov/34488734))
945. [caiyu6666/DDAD-ASR](https://github.com/caiyu6666/DDAD-ASR) (‚≠ê42 ¬∑ Python) [MedIA'2023] Dual-distribution discrepancy with self-supervised refinement for anomaly detection in medical images ([36934507](https://pubmed.ncbi.nlm.nih.gov/36934507))
946. [citiususc/BigSeqKit](https://github.com/citiususc/BigSeqKit) (‚≠ê42 ¬∑ Go) BigSeqKit: a parallel Big Data toolkit to process FASTA and FASTQ files at scale ([37522758](https://pubmed.ncbi.nlm.nih.gov/37522758))
947. [deweylab/metasra-pipeline](https://github.com/deweylab/metasra-pipeline) (‚≠ê42 ¬∑ Python) MetaSRA: normalized sample-specific metadata for the Sequence Read Archive ([28535296](https://pubmed.ncbi.nlm.nih.gov/28535296))
948. [dkoslicki/CMash](https://github.com/dkoslicki/CMash) (‚≠ê42 ¬∑ Python) Fast and accurate set similarity estimation via containment min hash ([35758788](https://pubmed.ncbi.nlm.nih.gov/35758788))
949. [dkoslicki/MetaPalette](https://github.com/dkoslicki/MetaPalette) (‚≠ê42 ¬∑ Python) Metagenomic profiling and phylogenetic distances via common kmers ([27822531](https://pubmed.ncbi.nlm.nih.gov/27822531))
950. [flatironinstitute/inferelator](https://github.com/flatironinstitute/inferelator) (‚≠ê42 ¬∑ Python) Task-based gene regulatory network inference using single-cell or bulk gene expression data conditioned on a prior network. ([35188184](https://pubmed.ncbi.nlm.nih.gov/35188184))
951. [hongwang01/InDuDoNet](https://github.com/hongwang01/InDuDoNet) (‚≠ê42 ¬∑ Python) „ÄêMICCAI 2021„ÄëAn Interpretable Dual Domain Network for CT Metal Artifact Reduction ([36623381](https://pubmed.ncbi.nlm.nih.gov/36623381))
952. [ikalatskaya/ISOWN](https://github.com/ikalatskaya/ISOWN) (‚≠ê42 ¬∑ Perl) None ([28659176](https://pubmed.ncbi.nlm.nih.gov/28659176))
953. [lanagarmire/cox-nnet](https://github.com/lanagarmire/cox-nnet) (‚≠ê42 ¬∑ Python) None ([29634719](https://pubmed.ncbi.nlm.nih.gov/29634719))
954. [lmweber/cytometry-clustering-comparison](https://github.com/lmweber/cytometry-clustering-comparison) (‚≠ê42 ¬∑ R) R scripts to reproduce analyses in our paper comparing clustering methods for high-dimensional cytometry data ([27992111](https://pubmed.ncbi.nlm.nih.gov/27992111))
955. [mahmoodlab/HistoFL](https://github.com/mahmoodlab/HistoFL) (‚≠ê42 ¬∑ Python) Federated Learning for Computational Pathology - Medical Image Analysis ([34911013](https://pubmed.ncbi.nlm.nih.gov/34911013))
956. [msraredon/NICHES](https://github.com/msraredon/NICHES) (‚≠ê42 ¬∑ HTML) Niche Interactions and Cellular Heterogeneity in Extracellular Signaling ([36458905](https://pubmed.ncbi.nlm.nih.gov/36458905))
957. [nitzanlab/biolord](https://github.com/nitzanlab/biolord) (‚≠ê42 ¬∑ Python) None ([38225466](https://pubmed.ncbi.nlm.nih.gov/38225466))
958. [schallerdavid/pyrod](https://github.com/schallerdavid/pyrod) (‚≠ê42 ¬∑ Python) PyRod - Tracing water molecules in molecular dynamics simulations ([31117512](https://pubmed.ncbi.nlm.nih.gov/31117512))
959. [shingocat/lrscaf](https://github.com/shingocat/lrscaf) (‚≠ê42 ¬∑ Java) TGS scaffolding ([31818249](https://pubmed.ncbi.nlm.nih.gov/31818249))
960. [ssolo/ALE](https://github.com/ssolo/ALE) (‚≠ê42 ¬∑ C++) Amalgamated likelihood estimation (ALE) is a probabilistic approach to exhaustively explore all reconciled gene trees that can be amalgamated as a combination of clades observed in a sample of gene trees. We implement the ALE approach in the context of a reconciliation model (cf. http://arxiv.org/abs/1211.4606 ), which allows for the duplication, transfer and loss of genes. We use ALE to efficiently approximate the sum of the joint likelihood over amalgamations and to find the reconciled gene tree that maximizes the joint likelihood among all such trees. ([23925510](https://pubmed.ncbi.nlm.nih.gov/23925510))
961. [svalkiers/clusTCR](https://github.com/svalkiers/clusTCR) (‚≠ê42 ¬∑ Python) CDR3 clustering module providing a new method for fast and accurate clustering of large data sets of CDR3 amino acid sequences, and offering functionalities for downstream analysis of clustering results. ([34132766](https://pubmed.ncbi.nlm.nih.gov/34132766))
962. [tbwxmu/SAMPN](https://github.com/tbwxmu/SAMPN) (‚≠ê42 ¬∑ Python) None ([33431047](https://pubmed.ncbi.nlm.nih.gov/33431047))
963. [vlothec/TRASH](https://github.com/vlothec/TRASH) (‚≠ê42 ¬∑ R) RepeatIdentifier ([37162382](https://pubmed.ncbi.nlm.nih.gov/37162382))
964. [wyang17/SQuIRE](https://github.com/wyang17/SQuIRE) (‚≠ê42 ¬∑ Python) Software for Quantifying Interspersed Repeat Expression ([30624635](https://pubmed.ncbi.nlm.nih.gov/30624635))
965. [yfukai/laptrack](https://github.com/yfukai/laptrack) (‚≠ê42 ¬∑ Python) Particle tracking by solving linear assignment problem. ([36495181](https://pubmed.ncbi.nlm.nih.gov/36495181))
966. [BackofenLab/IntaRNA](https://github.com/BackofenLab/IntaRNA) (‚≠ê41 ¬∑ C++) Efficient target prediction incorporating accessibility of interaction sites ([30590479](https://pubmed.ncbi.nlm.nih.gov/30590479))
967. [BergmannLab/MONET](https://github.com/BergmannLab/MONET) (‚≠ê41 ¬∑ GAP) MONET : MOdularising NEtwork Toolbox - https://doi.org/10.1093/bioinformatics/btaa236 ([32271874](https://pubmed.ncbi.nlm.nih.gov/32271874))
968. [CSB5/CaDRReS](https://github.com/CSB5/CaDRReS) (‚≠ê41 ¬∑ Python) Cancer Drug Response prediction using a Recommender System ([29868820](https://pubmed.ncbi.nlm.nih.gov/29868820))
969. [CSUBioGroup/DeepPPISP](https://github.com/CSUBioGroup/DeepPPISP) (‚≠ê41 ¬∑ Python) Protein-protein interaction sites prediction through combining local and global features with deep neural networks ([31593229](https://pubmed.ncbi.nlm.nih.gov/31593229))
970. [JiajinChen/shinyBN](https://github.com/JiajinChen/shinyBN) (‚≠ê41 ¬∑ R) shinyBN: An online application for interactive Bayesian network inference and visualization ([31842743](https://pubmed.ncbi.nlm.nih.gov/31842743))
971. [LouisFaure/scFates](https://github.com/LouisFaure/scFates) (‚≠ê41 ¬∑ Python) a scalable python suite for tree inference and advanced pseudotime analysis from scRNAseq data. ([36394263](https://pubmed.ncbi.nlm.nih.gov/36394263))
972. [MannLabs/alphapeptstats](https://github.com/MannLabs/alphapeptstats) (‚≠ê41 ¬∑ Jupyter Notebook) Python Package for the downstream analysis of mass-spectrometry-based proteomics data ([37527012](https://pubmed.ncbi.nlm.nih.gov/37527012))
973. [ShobiStassen/PARC](https://github.com/ShobiStassen/PARC) (‚≠ê41 ¬∑ Jupyter Notebook) None ([31971583](https://pubmed.ncbi.nlm.nih.gov/31971583))
974. [WGLab/Phen2Gene](https://github.com/WGLab/Phen2Gene) (‚≠ê41 ¬∑ Python) Phenotype driven gene prioritization for HPO ([32500119](https://pubmed.ncbi.nlm.nih.gov/32500119))
975. [YuSugihara/QTL-seq](https://github.com/YuSugihara/QTL-seq) (‚≠ê41 ¬∑ Python) QTL-seq pipeline to identify causative mutations responsible for a phenotype ([35321412](https://pubmed.ncbi.nlm.nih.gov/35321412))
976. [bircatmcri/MCIndoor20000](https://github.com/bircatmcri/MCIndoor20000) (‚≠ê41 ¬∑ None) None ([29876376](https://pubmed.ncbi.nlm.nih.gov/29876376))
977. [cbielow/PTXQC](https://github.com/cbielow/PTXQC) (‚≠ê41 ¬∑ HTML) A Quality Control (QC) pipeline for Proteomics (PTX) results generated by MaxQuant ([26653327](https://pubmed.ncbi.nlm.nih.gov/26653327))
978. [clinfo/SBMolGen](https://github.com/clinfo/SBMolGen) (‚≠ê41 ¬∑ Python) None ([34242036](https://pubmed.ncbi.nlm.nih.gov/34242036))
979. [ctlab/GADMA](https://github.com/ctlab/GADMA) (‚≠ê41 ¬∑ Python) Genetic Algorithm for Demographic Model Analysis ([32112099](https://pubmed.ncbi.nlm.nih.gov/32112099))
980. [geronimp/graftM](https://github.com/geronimp/graftM) (‚≠ê41 ¬∑ Python) GraftM - Rapid community profiles from metagenomes ([29562347](https://pubmed.ncbi.nlm.nih.gov/29562347))
981. [josuebarrera/GenEra](https://github.com/josuebarrera/GenEra) (‚≠ê41 ¬∑ Shell) genEra is a fast and easy-to-use command-line tool that estimates the age of the last common ancestor of protein-coding gene families. ([36964572](https://pubmed.ncbi.nlm.nih.gov/36964572))
982. [kundajelab/ChromDragoNN](https://github.com/kundajelab/ChromDragoNN) (‚≠ê41 ¬∑ Jupyter Notebook) Code for the paper "Integrating regulatory DNA sequence and gene expression to predict genome-wide chromatin accessibility across cellular contexts" ([31510655](https://pubmed.ncbi.nlm.nih.gov/31510655))
983. [mvollger/SDA](https://github.com/mvollger/SDA) (‚≠ê41 ¬∑ Python) Segmental Duplication Assembler (SDA). ([30559433](https://pubmed.ncbi.nlm.nih.gov/30559433))
984. [nrdg/fracridge](https://github.com/nrdg/fracridge) (‚≠ê41 ¬∑ Python) Fractional ridge regression ([33252656](https://pubmed.ncbi.nlm.nih.gov/33252656))
985. [portugueslab/stytra](https://github.com/portugueslab/stytra) (‚≠ê41 ¬∑ Python) A modular package to control stimulation and track behaviour ([30958870](https://pubmed.ncbi.nlm.nih.gov/30958870))
986. [randel/MIND](https://github.com/randel/MIND) (‚≠ê41 ¬∑ HTML) Using Bulk Gene Expression to Estimate Cell-Type-Specific Gene Expression via Deconvolution ([31400192](https://pubmed.ncbi.nlm.nih.gov/31400192))
987. [swsoyee/TCC-GUI](https://github.com/swsoyee/TCC-GUI) (‚≠ê41 ¬∑ R) üìä Graphical User Interface for TCC package ([30867032](https://pubmed.ncbi.nlm.nih.gov/30867032))
988. [xulabs/projects](https://github.com/xulabs/projects) (‚≠ê41 ¬∑ Python) None ([31510673](https://pubmed.ncbi.nlm.nih.gov/31510673); [36951805](https://pubmed.ncbi.nlm.nih.gov/36951805))
989. [CancerDataScience/NuCLS](https://github.com/CancerDataScience/NuCLS) (‚≠ê40 ¬∑ Jupyter Notebook) NuCLS: A scalable crowdsourcing, deep learning approach and dataset for nucleus classification, localization and segmentation ([34586355](https://pubmed.ncbi.nlm.nih.gov/34586355))
990. [ChuHan89/WSSS-Tissue](https://github.com/ChuHan89/WSSS-Tissue) (‚≠ê40 ¬∑ Python) None ([35671591](https://pubmed.ncbi.nlm.nih.gov/35671591))
991. [YinLiLin/KAML](https://github.com/YinLiLin/KAML) (‚≠ê40 ¬∑ R) :bicyclist: Kinship Adjusted Multi-Loci Best Linear Unbiased Prediction ([32552725](https://pubmed.ncbi.nlm.nih.gov/32552725))
992. [bcgsc/longstitch](https://github.com/bcgsc/longstitch) (‚≠ê40 ¬∑ Makefile) Correct and scaffold assemblies using long reads ([34717540](https://pubmed.ncbi.nlm.nih.gov/34717540))
993. [hashemifar/DPPI](https://github.com/hashemifar/DPPI) (‚≠ê40 ¬∑ Lua) A convolutional neural network to predict PPI interactions ([30423091](https://pubmed.ncbi.nlm.nih.gov/30423091))
994. [idruglab/hignn](https://github.com/idruglab/hignn) (‚≠ê40 ¬∑ Jupyter Notebook) Code for "HiGNN: A Hierarchical Informative Graph Neural Network for Molecular Property Prediction Equipped with Feature-Wise Attention" ([36519623](https://pubmed.ncbi.nlm.nih.gov/36519623))
995. [kdmurray91/kwip](https://github.com/kdmurray91/kwip) (‚≠ê40 ¬∑ C++) De novo estimates of genetic relatedness from next-gen sequencing data ([28873405](https://pubmed.ncbi.nlm.nih.gov/28873405))
996. [liutairan/eMolFrag](https://github.com/liutairan/eMolFrag) (‚≠ê40 ¬∑ Python) eMolFrag is a molecular fragmentation tool based on BRICS algorithm written in Python. ([28346786](https://pubmed.ncbi.nlm.nih.gov/28346786))
997. [natir/fpa](https://github.com/natir/fpa) (‚≠ê40 ¬∑ Rust) Filter of Pairwise Alignement ([32315402](https://pubmed.ncbi.nlm.nih.gov/32315402))
998. [neerakara/test-time-adaptable-neural-networks-for-domain-generalization](https://github.com/neerakara/test-time-adaptable-neural-networks-for-domain-generalization) (‚≠ê40 ¬∑ Python) Code for the paper "Test-time adaptable neural networks for robust medical image segmentation" ([33341496](https://pubmed.ncbi.nlm.nih.gov/33341496))
999. [oxpig/ABlooper](https://github.com/oxpig/ABlooper) (‚≠ê40 ¬∑ Python) Tool for modelling the CDRs of antibodies ([35099535](https://pubmed.ncbi.nlm.nih.gov/35099535))
1000. [phipsonlab/speckle](https://github.com/phipsonlab/speckle) (‚≠ê40 ¬∑ R) R package for analysing single cell data ([36005887](https://pubmed.ncbi.nlm.nih.gov/36005887))

