# PubMed-GitHub

A collection of GitHub repositories that in PubMed abstract.

## Top 20 Authors

| Rank | Author | Count | Stars | Score |
| ---: | -----: | ----: | ----: | ----: |
| 1 | [brentp](https://github.com/brentp) | 11 | 2506 | 81.61 |
| 2 | [lh3](https://github.com/lh3) | 11 | 2792 | 77.35 |
| 3 | [ncbi](https://github.com/ncbi) | 10 | 2299 | 71.85 |
| 4 | [theislab](https://github.com/theislab) | 9 | 3144 | 67.68 |
| 5 | [bcgsc](https://github.com/bcgsc) | 9 | 1026 | 59.22 |
| 6 | [OpenGene](https://github.com/OpenGene) | 6 | 2330 | 45.19 |
| 7 | [MolecularAI](https://github.com/MolecularAI) | 6 | 1249 | 42.96 |
| 8 | [Illumina](https://github.com/Illumina) | 5 | 1102 | 37.78 |
| 9 | [broadinstitute](https://github.com/broadinstitute) | 4 | 2302 | 32.46 |
| 10 | [rrwick](https://github.com/rrwick) | 4 | 1296 | 31.73 |
| 11 | [EpistasisLab](https://github.com/EpistasisLab) | 3 | 10468 | 30.59 |
| 12 | [MannLabs](https://github.com/MannLabs) | 5 | 339 | 30.25 |
| 13 | [marbl](https://github.com/marbl) | 4 | 892 | 29.90 |
| 14 | [choderalab](https://github.com/choderalab) | 4 | 637 | 28.48 |
| 15 | [oxpig](https://github.com/oxpig) | 4 | 352 | 25.61 |
| 16 | [nadeemlab](https://github.com/nadeemlab) | 4 | 373 | 25.59 |
| 17 | [microsoft](https://github.com/microsoft) | 4 | 322 | 24.94 |
| 18 | [galaxyproject](https://github.com/galaxyproject) | 3 | 1583 | 24.64 |
| 19 | [tanghaibao](https://github.com/tanghaibao) | 3 | 1449 | 24.64 |
| 20 | [pachterlab](https://github.com/pachterlab) | 3 | 1423 | 24.40 |


***Note: only count repositories with stars not less than top 1000 threshold (42).***

## Top 1000 Repositories

1. [CSSEGISandData/COVID-19](https://github.com/CSSEGISandData/COVID-19) (‚≠ê29186 ¬∑ None) Novel Coronavirus (COVID-19) Cases, provided by JHU CSSE (PMID: [32834919](https://pubmed.ncbi.nlm.nih.gov/32834919); [33121426](https://pubmed.ncbi.nlm.nih.gov/33121426); [34248208](https://pubmed.ncbi.nlm.nih.gov/34248208); [34790453](https://pubmed.ncbi.nlm.nih.gov/34790453))

2. [kubernetes/charts](https://github.com/kubernetes/charts) (‚≠ê15530 ¬∑ Go) ‚ö†Ô∏è(OBSOLETE) Curated applications for Kubernetes (PMID: [30101309](https://pubmed.ncbi.nlm.nih.gov/30101309))

3. [flairNLP/flair](https://github.com/flairNLP/flair) (‚≠ê13511 ¬∑ Python) A very simple framework for state-of-the-art Natural Language Processing (NLP) (PMID: [33508086](https://pubmed.ncbi.nlm.nih.gov/33508086))

4. [NVlabs/stylegan2](https://github.com/NVlabs/stylegan2) (‚≠ê10759 ¬∑ Python) StyleGAN2 - Official TensorFlow Implementation (PMID: [35871688](https://pubmed.ncbi.nlm.nih.gov/35871688))

5. [EpistasisLab/tpot](https://github.com/EpistasisLab/tpot) (‚≠ê9469 ¬∑ Python) A Python Automated Machine Learning tool that optimizes machine learning pipelines using genetic programming. (PMID: [31165141](https://pubmed.ncbi.nlm.nih.gov/31165141); [32998684](https://pubmed.ncbi.nlm.nih.gov/32998684))

6. [pachyderm/pachyderm](https://github.com/pachyderm/pachyderm) (‚≠ê6064 ¬∑ Go) Data-Centric Pipelines and Data Versioning (PMID: [30101309](https://pubmed.ncbi.nlm.nih.gov/30101309))

7. [owid/covid-19-data](https://github.com/owid/covid-19-data) (‚≠ê5628 ¬∑ Python) Data on COVID-19 (coronavirus) cases, deaths, hospitalizations, tests ‚Ä¢ All countries ‚Ä¢ Updated daily by Our World in Data (PMID: [34610042](https://pubmed.ncbi.nlm.nih.gov/34610042))

8. [tyiannak/pyAudioAnalysis](https://github.com/tyiannak/pyAudioAnalysis) (‚≠ê5626 ¬∑ Python) Python Audio Analysis Library: Feature Extraction, Classification, Segmentation and Applications (PMID: [26656189](https://pubmed.ncbi.nlm.nih.gov/26656189))

9. [deepchem/deepchem](https://github.com/deepchem/deepchem) (‚≠ê5033 ¬∑ Python) Democratizing Deep-Learning for Drug Discovery, Quantum Chemistry, Materials Science and Biology (PMID: [28470045](https://pubmed.ncbi.nlm.nih.gov/28470045))

10. [mdbloice/Augmentor](https://github.com/mdbloice/Augmentor) (‚≠ê5016 ¬∑ Python) Image augmentation library in Python for machine learning. (PMID: [30989173](https://pubmed.ncbi.nlm.nih.gov/30989173))

11. [google/deepvariant](https://github.com/google/deepvariant) (‚≠ê3055 ¬∑ Python) DeepVariant is an analysis pipeline that uses a deep neural network to call genetic variants from next-generation DNA sequencing data. (PMID: [33399819](https://pubmed.ncbi.nlm.nih.gov/33399819))

12. [OpenGene/fastp](https://github.com/OpenGene/fastp) (‚≠ê1735 ¬∑ C++) An ultra-fast all-in-one FASTQ preprocessor (QC/adapters/trimming/filtering/splitting/merging...) (PMID: [30423086](https://pubmed.ncbi.nlm.nih.gov/30423086))

13. [theislab/Scanpy](https://github.com/theislab/Scanpy) (‚≠ê1728 ¬∑ Python) Single-cell analysis in Python. Scales to >1M cells. (PMID: [29409532](https://pubmed.ncbi.nlm.nih.gov/29409532))

14. [sokrypton/ColabFold](https://github.com/sokrypton/ColabFold) (‚≠ê1676 ¬∑ Jupyter Notebook) Making Protein folding accessible to all! (PMID: [35637307](https://pubmed.ncbi.nlm.nih.gov/35637307))

15. [lh3/minimap2](https://github.com/lh3/minimap2) (‚≠ê1666 ¬∑ C) A versatile pairwise aligner for genomic and spliced nucleotide sequences (PMID: [29750242](https://pubmed.ncbi.nlm.nih.gov/29750242); [34623391](https://pubmed.ncbi.nlm.nih.gov/34623391))

16. [broadinstitute/gatk](https://github.com/broadinstitute/gatk) (‚≠ê1584 ¬∑ Java) Official code repository for GATK versions 4 and up (PMID: [29982281](https://pubmed.ncbi.nlm.nih.gov/29982281); [31830260](https://pubmed.ncbi.nlm.nih.gov/31830260))

17. [samtools/samtools](https://github.com/samtools/samtools) (‚≠ê1534 ¬∑ C) Tools (written in C using htslib) for manipulating next-generation sequencing data (PMID: [24974206](https://pubmed.ncbi.nlm.nih.gov/24974206))

18. [SmartDataAnalytics/PyKEEN](https://github.com/SmartDataAnalytics/PyKEEN) (‚≠ê1491 ¬∑ Python) ü§ñ A Python library for learning and evaluating knowledge graph embeddings (PMID: [30768158](https://pubmed.ncbi.nlm.nih.gov/30768158))

19. [galaxyproject/galaxy](https://github.com/galaxyproject/galaxy) (‚≠ê1297 ¬∑ Python) Data intensive science for everyone. (PMID: [31197310](https://pubmed.ncbi.nlm.nih.gov/31197310); [35669346](https://pubmed.ncbi.nlm.nih.gov/35669346))

20. [shenwei356/seqkit](https://github.com/shenwei356/seqkit) (‚≠ê1181 ¬∑ Go) A cross-platform and ultrafast toolkit for FASTA/Q file manipulation (PMID: [27706213](https://pubmed.ncbi.nlm.nih.gov/27706213))

21. [ewels/MultiQC](https://github.com/ewels/MultiQC) (‚≠ê1156 ¬∑ JavaScript) Aggregate results from bioinformatics analyses across many samples into a single report. (PMID: [35799367](https://pubmed.ncbi.nlm.nih.gov/35799367))

22. [glample/tagger](https://github.com/glample/tagger) (‚≠ê1151 ¬∑ Python) Named Entity Recognition Tool (PMID: [28881963](https://pubmed.ncbi.nlm.nih.gov/28881963))

23. [YosefLab/scVI](https://github.com/YosefLab/scVI) (‚≠ê1103 ¬∑ Python) Deep probabilistic analysis of single-cell omics data (PMID: [30504886](https://pubmed.ncbi.nlm.nih.gov/30504886); [32176273](https://pubmed.ncbi.nlm.nih.gov/32176273))

24. [fahadshamshad/awesome-transformers-in-medical-imaging](https://github.com/fahadshamshad/awesome-transformers-in-medical-imaging) (‚≠ê1081 ¬∑ None) A collection of resources on applications of Transformers in Medical Imaging. (PMID: [37315483](https://pubmed.ncbi.nlm.nih.gov/37315483))

25. [vlawhern/arl-eegmodels](https://github.com/vlawhern/arl-eegmodels) (‚≠ê1052 ¬∑ Python) This is the Army Research Laboratory (ARL) EEGModels Project: A Collection of Convolutional Neural Network (CNN) models for EEG signal classification, using Keras and Tensorflow (PMID: [29932424](https://pubmed.ncbi.nlm.nih.gov/29932424))

26. [vgteam/vg](https://github.com/vgteam/vg) (‚≠ê1050 ¬∑ C++) tools for working with genome variation graphs (PMID: [31406990](https://pubmed.ncbi.nlm.nih.gov/31406990); [32657356](https://pubmed.ncbi.nlm.nih.gov/32657356))

27. [ncbi/sra-tools](https://github.com/ncbi/sra-tools) (‚≠ê1039 ¬∑ C) SRA Tools (PMID: [36698146](https://pubmed.ncbi.nlm.nih.gov/36698146))

28. [tensorflow/gan](https://github.com/tensorflow/gan) (‚≠ê908 ¬∑ Jupyter Notebook) Tooling for GANs in TensorFlow (PMID: [33381813](https://pubmed.ncbi.nlm.nih.gov/33381813))

29. [kexinhuang12345/DeepPurpose](https://github.com/kexinhuang12345/DeepPurpose) (‚≠ê890 ¬∑ Jupyter Notebook) A Deep Learning Toolkit for DTI, Drug Property, PPI, DDI, Protein Function Prediction (Bioinformatics) (PMID: [33275143](https://pubmed.ncbi.nlm.nih.gov/33275143))

30. [pachterlab/gget](https://github.com/pachterlab/gget) (‚≠ê852 ¬∑ Python) üß¨ gget enables efficient querying of genomic reference databases (PMID: [36610989](https://pubmed.ncbi.nlm.nih.gov/36610989); [38377393](https://pubmed.ncbi.nlm.nih.gov/38377393))

31. [jdblischak/workflowr](https://github.com/jdblischak/workflowr) (‚≠ê800 ¬∑ R) Organize your project into a research website (PMID: [31723427](https://pubmed.ncbi.nlm.nih.gov/31723427))

32. [InsightSoftwareConsortium/SimpleITK-Notebooks](https://github.com/InsightSoftwareConsortium/SimpleITK-Notebooks) (‚≠ê798 ¬∑ Jupyter Notebook) Jupyter notebooks for learning how to use SimpleITK (PMID: [29181613](https://pubmed.ncbi.nlm.nih.gov/29181613))

33. [RainCloudPlots/RainCloudPlots](https://github.com/RainCloudPlots/RainCloudPlots) (‚≠ê787 ¬∑ HTML) Code and tutorials to visualise your data that is both beautiful *and* statistically valid (PMID: [31069261](https://pubmed.ncbi.nlm.nih.gov/31069261))

34. [EpistasisLab/pmlb](https://github.com/EpistasisLab/pmlb) (‚≠ê780 ¬∑ Python) PMLB: A large, curated repository of benchmark datasets for evaluating supervised machine learning algorithms. (PMID: [34677586](https://pubmed.ncbi.nlm.nih.gov/34677586))

35. [molecularsets/moses](https://github.com/molecularsets/moses) (‚≠ê777 ¬∑ Python) Molecular Sets (MOSES): A Benchmarking Platform for Molecular Generation Models (PMID: [33390943](https://pubmed.ncbi.nlm.nih.gov/33390943))

36. [arose/nglview](https://github.com/arose/nglview) (‚≠ê771 ¬∑ Jupyter Notebook) Jupyter widget to interactively view molecular structures and trajectories (PMID: [29236954](https://pubmed.ncbi.nlm.nih.gov/29236954))

37. [MrGiovanni/ModelsGenesis](https://github.com/MrGiovanni/ModelsGenesis) (‚≠ê719 ¬∑ Jupyter Notebook) [MICCAI 2019] [MEDIA 2020] Models Genesis (PMID: [32766570](https://pubmed.ncbi.nlm.nih.gov/32766570); [33188996](https://pubmed.ncbi.nlm.nih.gov/33188996))

38. [tanghaibao/goatools](https://github.com/tanghaibao/goatools) (‚≠ê710 ¬∑ Python) Python library to handle Gene Ontology (GO) terms (PMID: [30022098](https://pubmed.ncbi.nlm.nih.gov/30022098))

39. [deeptools/pyGenomeTracks](https://github.com/deeptools/pyGenomeTracks) (‚≠ê709 ¬∑ Python) python module to plot beautiful and highly customizable genome browser tracks (PMID: [32745185](https://pubmed.ncbi.nlm.nih.gov/32745185))

40. [tanghaibao/jcvi](https://github.com/tanghaibao/jcvi) (‚≠ê685 ¬∑ Python) Python library to facilitate genome assembly, annotation, and comparative genomics (PMID: [25583564](https://pubmed.ncbi.nlm.nih.gov/25583564))

41. [taoliu/MACS](https://github.com/taoliu/MACS) (‚≠ê679 ¬∑ Cython) MACS -- Model-based Analysis of ChIP-Seq (PMID: [24743991](https://pubmed.ncbi.nlm.nih.gov/24743991))

42. [awslabs/dgl-lifesci](https://github.com/awslabs/dgl-lifesci) (‚≠ê675 ¬∑ Python) Python package for graph neural networks in chemistry and biology (PMID: [34693143](https://pubmed.ncbi.nlm.nih.gov/34693143))

43. [volkamerlab/TeachOpenCADD](https://github.com/volkamerlab/TeachOpenCADD) (‚≠ê662 ¬∑ Jupyter Notebook) TeachOpenCADD: a teaching platform for computer-aided drug design (CADD) using open source packages and data (PMID: [30963287](https://pubmed.ncbi.nlm.nih.gov/30963287))

44. [beoutbreakprepared/nCoV2019](https://github.com/beoutbreakprepared/nCoV2019) (‚≠ê658 ¬∑ HTML) Location for summaries and analysis of data related to n-CoV 2019, first reported in Wuhan, China (PMID: [33051749](https://pubmed.ncbi.nlm.nih.gov/33051749))

45. [ablab/spades](https://github.com/ablab/spades) (‚≠ê657 ¬∑ C++) SPAdes Genome Assembler (PMID: [31510642](https://pubmed.ncbi.nlm.nih.gov/31510642); [32413137](https://pubmed.ncbi.nlm.nih.gov/32413137))

46. [brentp/mosdepth](https://github.com/brentp/mosdepth) (‚≠ê645 ¬∑ Nim) fast BAM/CRAM depth calculation for WGS, exome, or targeted sequencing (PMID: [29096012](https://pubmed.ncbi.nlm.nih.gov/29096012))

47. [arose/ngl](https://github.com/arose/ngl) (‚≠ê643 ¬∑ TypeScript) WebGL protein viewer (PMID: [29850778](https://pubmed.ncbi.nlm.nih.gov/29850778))

48. [naver/biobert-pretrained](https://github.com/naver/biobert-pretrained) (‚≠ê627 ¬∑ None) BioBERT: a pre-trained biomedical language representation model for biomedical text mining (PMID: [31501885](https://pubmed.ncbi.nlm.nih.gov/31501885))

49. [torognes/vsearch](https://github.com/torognes/vsearch) (‚≠ê625 ¬∑ C++) Versatile open-source tool for microbiome analysis (PMID: [27781170](https://pubmed.ncbi.nlm.nih.gov/27781170))

50. [davidemms/OrthoFinder](https://github.com/davidemms/OrthoFinder) (‚≠ê614 ¬∑ Python) Phylogenetic orthology inference for comparative genomics (PMID: [31727128](https://pubmed.ncbi.nlm.nih.gov/31727128))

51. [samtools/BCFtools](https://github.com/samtools/BCFtools) (‚≠ê613 ¬∑ C) This is the official development repository for BCFtools. See installation instructions and other documentation here http://samtools.github.io/bcftools/howtos/install.html (PMID: [26826718](https://pubmed.ncbi.nlm.nih.gov/26826718))

52. [igvteam/igv](https://github.com/igvteam/igv) (‚≠ê606 ¬∑ Java) Integrative Genomics Viewer. Fast, efficient, scalable visualization tool for genomics data and annotations (PMID: [36562559](https://pubmed.ncbi.nlm.nih.gov/36562559))

53. [NeuroTechX/moabb](https://github.com/NeuroTechX/moabb) (‚≠ê602 ¬∑ Python) Mother of All BCI Benchmarks (PMID: [30177583](https://pubmed.ncbi.nlm.nih.gov/30177583))

54. [KlugerLab/FIt-SNE](https://github.com/KlugerLab/FIt-SNE) (‚≠ê578 ¬∑ C++) Fast Fourier Transform-accelerated Interpolation-based  t-SNE (FIt-SNE) (PMID: [30742040](https://pubmed.ncbi.nlm.nih.gov/30742040))

55. [biojava/biojava](https://github.com/biojava/biojava) (‚≠ê572 ¬∑ Java) :book::microscope::coffee: BioJava is an open-source project dedicated to providing a Java library for processing biological data. (PMID: [25505094](https://pubmed.ncbi.nlm.nih.gov/25505094); [28334105](https://pubmed.ncbi.nlm.nih.gov/28334105); [30735498](https://pubmed.ncbi.nlm.nih.gov/30735498))

56. [ialbert/biostar-central](https://github.com/ialbert/biostar-central) (‚≠ê572 ¬∑ Python) Biostar Q&A (PMID: [32640986](https://pubmed.ncbi.nlm.nih.gov/32640986))

57. [molstar/molstar](https://github.com/molstar/molstar) (‚≠ê571 ¬∑ TypeScript) A comprehensive macromolecular library (PMID: [34864908](https://pubmed.ncbi.nlm.nih.gov/34864908))

58. [rrwick/Bandage](https://github.com/rrwick/Bandage) (‚≠ê554 ¬∑ C++) a Bioinformatics Application for Navigating De novo Assembly Graphs Easily (PMID: [26099265](https://pubmed.ncbi.nlm.nih.gov/26099265))

59. [broadinstitute/keras-rcnn](https://github.com/broadinstitute/keras-rcnn) (‚≠ê548 ¬∑ Python) Keras package for region-based convolutional neural networks (RCNNs) (PMID: [32652926](https://pubmed.ncbi.nlm.nih.gov/32652926))

60. [ZhaoZhibin/DL-based-Intelligent-Diagnosis-Benchmark](https://github.com/ZhaoZhibin/DL-based-Intelligent-Diagnosis-Benchmark) (‚≠ê545 ¬∑ Python) Source codes for the paper "Deep Learning Algorithms for Rotating Machinery Intelligent Diagnosis: An Open Source Benchmark Study" (PMID: [32854956](https://pubmed.ncbi.nlm.nih.gov/32854956))

61. [ThomasYeoLab/CBIG](https://github.com/ThomasYeoLab/CBIG) (‚≠ê543 ¬∑ MATLAB) None (PMID: [28981612](https://pubmed.ncbi.nlm.nih.gov/28981612); [29770530](https://pubmed.ncbi.nlm.nih.gov/29770530); [30974241](https://pubmed.ncbi.nlm.nih.gov/30974241); [31229658](https://pubmed.ncbi.nlm.nih.gov/31229658); [33942058](https://pubmed.ncbi.nlm.nih.gov/33942058); [36918136](https://pubmed.ncbi.nlm.nih.gov/36918136))

62. [gnina/gnina](https://github.com/gnina/gnina) (‚≠ê541 ¬∑ C++) A deep learning framework for molecular docking (PMID: [34108002](https://pubmed.ncbi.nlm.nih.gov/34108002))

63. [jerryji1993/DNABERT](https://github.com/jerryji1993/DNABERT) (‚≠ê533 ¬∑ Python) DNABERT: pre-trained Bidirectional Encoder Representations from Transformers model for DNA-language in genome (PMID: [33538820](https://pubmed.ncbi.nlm.nih.gov/33538820))

64. [MolecularAI/aizynthfinder](https://github.com/MolecularAI/aizynthfinder) (‚≠ê522 ¬∑ Python) A tool for retrosynthetic planning (PMID: [34342428](https://pubmed.ncbi.nlm.nih.gov/34342428))

65. [pachterlab/ffq](https://github.com/pachterlab/ffq) (‚≠ê521 ¬∑ Python) A tool to find sequencing data and metadata from public databases. (PMID: [36610997](https://pubmed.ncbi.nlm.nih.gov/36610997))

66. [rrwick/Unicycler](https://github.com/rrwick/Unicycler) (‚≠ê518 ¬∑ C++) hybrid assembly pipeline for bacterial genomes (PMID: [28594827](https://pubmed.ncbi.nlm.nih.gov/28594827))

67. [skovaka/UNCALLED](https://github.com/skovaka/UNCALLED) (‚≠ê512 ¬∑ C++) Raw nanopore signal mapper that enables real-time targeted sequencing (PMID: [33257863](https://pubmed.ncbi.nlm.nih.gov/33257863))

68. [fritzsedlazeck/Sniffles](https://github.com/fritzsedlazeck/Sniffles) (‚≠ê501 ¬∑ Python) Structural variation caller using third generation sequencing (PMID: [28456410](https://pubmed.ncbi.nlm.nih.gov/28456410); [29713083](https://pubmed.ncbi.nlm.nih.gov/29713083))

69. [zqfang/GSEApy](https://github.com/zqfang/GSEApy) (‚≠ê500 ¬∑ Python) Gene Set Enrichment Analysis in Python (PMID: [36426870](https://pubmed.ncbi.nlm.nih.gov/36426870))

70. [CERN/TIGRE](https://github.com/CERN/TIGRE) (‚≠ê498 ¬∑ MATLAB) TIGRE: Tomographic Iterative GPU-based Reconstruction Toolbox (PMID: [36088800](https://pubmed.ncbi.nlm.nih.gov/36088800))

71. [owkin/PyDESeq2](https://github.com/owkin/PyDESeq2) (‚≠ê498 ¬∑ Python) A Python implementation of the DESeq2 pipeline for bulk RNA-seq DEA. (PMID: [37669147](https://pubmed.ncbi.nlm.nih.gov/37669147))

72. [gamcil/clinker](https://github.com/gamcil/clinker) (‚≠ê497 ¬∑ Python) Gene cluster comparison figure generator (PMID: [33459763](https://pubmed.ncbi.nlm.nih.gov/33459763))

73. [theislab/anndata](https://github.com/theislab/anndata) (‚≠ê492 ¬∑ Python) Annotated data. (PMID: [29409532](https://pubmed.ncbi.nlm.nih.gov/29409532))

74. [ruanjue/wtdbg2](https://github.com/ruanjue/wtdbg2) (‚≠ê491 ¬∑ C) Redbean: A fuzzy Bruijn graph approach to long noisy reads assembly (PMID: [31819265](https://pubmed.ncbi.nlm.nih.gov/31819265))

75. [soedinglab/hh-suite](https://github.com/soedinglab/hh-suite) (‚≠ê491 ¬∑ C) Remote protein homology detection suite. (PMID: [31521110](https://pubmed.ncbi.nlm.nih.gov/31521110))

76. [YichiZhang98/SAM4MIS](https://github.com/YichiZhang98/SAM4MIS) (‚≠ê490 ¬∑ None) Segment Anything Model for Medical Image Segmentation:  paper list and open-source project summary (PMID: [38422961](https://pubmed.ncbi.nlm.nih.gov/38422961))

77. [biojs/biojs](https://github.com/biojs/biojs) (‚≠ê482 ¬∑ None) üî¨A library of JavaScript components to represent biological data (PMID: [24627804](https://pubmed.ncbi.nlm.nih.gov/24627804); [24715980](https://pubmed.ncbi.nlm.nih.gov/24715980); [24741440](https://pubmed.ncbi.nlm.nih.gov/24741440); [24860644](https://pubmed.ncbi.nlm.nih.gov/24860644); [25075287](https://pubmed.ncbi.nlm.nih.gov/25075287); [25075288](https://pubmed.ncbi.nlm.nih.gov/25075288); [25075290](https://pubmed.ncbi.nlm.nih.gov/25075290))

78. [Martinsos/edlib](https://github.com/Martinsos/edlib) (‚≠ê481 ¬∑ C++) Lightweight, super fast C/C++ (& Python) library for sequence alignment using edit (Levenshtein) distance. (PMID: [28453688](https://pubmed.ncbi.nlm.nih.gov/28453688))

79. [immunogenomics/harmony](https://github.com/immunogenomics/harmony) (‚≠ê480 ¬∑ R) Fast, sensitive and accurate integration of single-cell data with Harmony (PMID: [31740819](https://pubmed.ncbi.nlm.nih.gov/31740819))

80. [ncoudray/DeepPATH](https://github.com/ncoudray/DeepPATH) (‚≠ê480 ¬∑ Python) Classification of Lung cancer slide images using deep-learning (PMID: [30224757](https://pubmed.ncbi.nlm.nih.gov/30224757))

81. [ryanlayer/samplot](https://github.com/ryanlayer/samplot) (‚≠ê479 ¬∑ Python) Plot structural variant signals from many BAMs and CRAMs (PMID: [34034781](https://pubmed.ncbi.nlm.nih.gov/34034781))

82. [HazyResearch/hyena-dna](https://github.com/HazyResearch/hyena-dna) (‚≠ê476 ¬∑ Assembly) Official implementation for HyenaDNA, a long-range genomic foundation model built with Hyena (PMID: [37426456](https://pubmed.ncbi.nlm.nih.gov/37426456))

83. [Ildaron/ironbci](https://github.com/Ildaron/ironbci) (‚≠ê474 ¬∑ C) Open-Source Brain-Computer Interface, ADS1299 and STM32 (PMID: [34586477](https://pubmed.ncbi.nlm.nih.gov/34586477))

84. [BMIRDS/deepslide](https://github.com/BMIRDS/deepslide) (‚≠ê470 ¬∑ Python) Code for the Nature Scientific Reports paper "Pathologist-level classification of histologic patterns on resected lung adenocarcinoma slides with deep neural networks." A sliding window framework for classification of high resolution whole-slide images, often microscopy or histopathology images. (PMID: [30833650](https://pubmed.ncbi.nlm.nih.gov/30833650))

85. [lindenb/jvarkit](https://github.com/lindenb/jvarkit) (‚≠ê465 ¬∑ Java) Java utilities for Bioinformatics (PMID: [29186339](https://pubmed.ncbi.nlm.nih.gov/29186339))

86. [etal/cnvkit](https://github.com/etal/cnvkit) (‚≠ê463 ¬∑ Python) Copy number variant detection from targeted DNA sequencing (PMID: [27100738](https://pubmed.ncbi.nlm.nih.gov/27100738))

87. [OpenMS/OpenMS](https://github.com/OpenMS/OpenMS) (‚≠ê454 ¬∑ C++) The codebase of the OpenMS project (PMID: [25927999](https://pubmed.ncbi.nlm.nih.gov/25927999))

88. [seqan/seqan](https://github.com/seqan/seqan) (‚≠ê451 ¬∑ C++) SeqAn's official repository. (PMID: [29232861](https://pubmed.ncbi.nlm.nih.gov/29232861))

89. [benjjneb/dada2](https://github.com/benjjneb/dada2) (‚≠ê444 ¬∑ R) Accurate sample inference from amplicon data with single nucleotide resolution (PMID: [27214047](https://pubmed.ncbi.nlm.nih.gov/27214047))

90. [aiqm/torchani](https://github.com/aiqm/torchani) (‚≠ê429 ¬∑ Python) Accurate Neural Network Potential on PyTorch (PMID: [32568524](https://pubmed.ncbi.nlm.nih.gov/32568524))

91. [saeyslab/nichenetr](https://github.com/saeyslab/nichenetr) (‚≠ê428 ¬∑ R) NicheNet: predict active ligand-target links between interacting cells (PMID: [31819264](https://pubmed.ncbi.nlm.nih.gov/31819264))

92. [ecogenomics/gtdbtk](https://github.com/ecogenomics/gtdbtk) (‚≠ê414 ¬∑ Python) GTDB-Tk: a toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes. (PMID: [31730192](https://pubmed.ncbi.nlm.nih.gov/31730192); [36218463](https://pubmed.ncbi.nlm.nih.gov/36218463))

93. [pezmaster31/bamtools](https://github.com/pezmaster31/bamtools) (‚≠ê408 ¬∑ C++) C++ API & command-line toolkit for working with BAM data (PMID: [23620359](https://pubmed.ncbi.nlm.nih.gov/23620359))

94. [aristoteleo/dynamo-release](https://github.com/aristoteleo/dynamo-release) (‚≠ê395 ¬∑ Python) Inclusive model of expression dynamics with conventional or metabolic labeling based scRNA-seq / multiomics, vector field reconstruction and differential geometry analyses (PMID: [35108499](https://pubmed.ncbi.nlm.nih.gov/35108499))

95. [omnideconv/immunedeconv](https://github.com/omnideconv/immunedeconv) (‚≠ê394 ¬∑ R) A unified interface to immune deconvolution methods (CIBERSORT, EPIC, quanTIseq, TIMER, xCell, MCPcounter) and mouse deconvolution methods (PMID: [38464974](https://pubmed.ncbi.nlm.nih.gov/38464974))

96. [oddt/oddt](https://github.com/oddt/oddt) (‚≠ê392 ¬∑ Python) Open Drug Discovery Toolkit (PMID: [26101548](https://pubmed.ncbi.nlm.nih.gov/26101548); [30202917](https://pubmed.ncbi.nlm.nih.gov/30202917))

97. [agshumate/Liftoff](https://github.com/agshumate/Liftoff) (‚≠ê385 ¬∑ Python) An accurate GFF3/GTF lift over pipeline (PMID: [33320174](https://pubmed.ncbi.nlm.nih.gov/33320174))

98. [oschwengers/bakta](https://github.com/oschwengers/bakta) (‚≠ê383 ¬∑ Python) Rapid & standardized annotation of bacterial genomes, MAGs & plasmids (PMID: [34739369](https://pubmed.ncbi.nlm.nih.gov/34739369))

99. [Illumina/manta](https://github.com/Illumina/manta) (‚≠ê382 ¬∑ C++) Structural variant and indel caller for mapped sequencing data (PMID: [26647377](https://pubmed.ncbi.nlm.nih.gov/26647377))

100. [prody/ProDy](https://github.com/prody/ProDy) (‚≠ê380 ¬∑ Python) A Python Package for Protein Dynamics Analysis (PMID: [33822884](https://pubmed.ncbi.nlm.nih.gov/33822884); [34240100](https://pubmed.ncbi.nlm.nih.gov/34240100))

101. [cabani/MaskedFace-Net](https://github.com/cabani/MaskedFace-Net) (‚≠ê372 ¬∑ None) MaskedFace-Net is a dataset of human faces with a correctly and incorrectly worn mask based on the dataset Flickr-Faces-HQ (FFHQ). (PMID: [33521223](https://pubmed.ncbi.nlm.nih.gov/33521223))

102. [marbl/mash](https://github.com/marbl/mash) (‚≠ê371 ¬∑ C++) Fast genome and metagenome distance estimation using MinHash (PMID: [27323842](https://pubmed.ncbi.nlm.nih.gov/27323842))

103. [digitalmethodsinitiative/dmi-tcat](https://github.com/digitalmethodsinitiative/dmi-tcat) (‚≠ê365 ¬∑ PHP) Digital Methods Initiative - Twitter Capture and Analysis Toolset (PMID: [33665150](https://pubmed.ncbi.nlm.nih.gov/33665150))

104. [bxlab/metaWRAP](https://github.com/bxlab/metaWRAP) (‚≠ê363 ¬∑ Shell) MetaWRAP - a flexible pipeline for genome-resolved metagenomic data analysis (PMID: [30219103](https://pubmed.ncbi.nlm.nih.gov/30219103))

105. [nservant/HiC-Pro](https://github.com/nservant/HiC-Pro) (‚≠ê362 ¬∑ Python) HiC-Pro: An optimized and flexible pipeline for Hi-C data processing (PMID: [26619908](https://pubmed.ncbi.nlm.nih.gov/26619908))

106. [lishen/end2end-all-conv](https://github.com/lishen/end2end-all-conv) (‚≠ê356 ¬∑ Jupyter Notebook) Deep Learning to Improve Breast Cancer Detection on Screening Mammography (PMID: [31467326](https://pubmed.ncbi.nlm.nih.gov/31467326))

107. [brentp/cyvcf2](https://github.com/brentp/cyvcf2) (‚≠ê355 ¬∑ Cython) cython + htslib == fast VCF and BCF processing (PMID: [28165109](https://pubmed.ncbi.nlm.nih.gov/28165109))

108. [amkozlov/raxml-ng](https://github.com/amkozlov/raxml-ng) (‚≠ê353 ¬∑ C++) RAxML Next Generation: faster, easier-to-use and more flexible (PMID: [31070718](https://pubmed.ncbi.nlm.nih.gov/31070718); [31755898](https://pubmed.ncbi.nlm.nih.gov/31755898))

109. [wmayner/pyphi](https://github.com/wmayner/pyphi) (‚≠ê351 ¬∑ Python) A toolbox for integrated information theory. (PMID: [30048445](https://pubmed.ncbi.nlm.nih.gov/30048445))

110. [mlpen/Nystromformer](https://github.com/mlpen/Nystromformer) (‚≠ê350 ¬∑ Python) None (PMID: [34745767](https://pubmed.ncbi.nlm.nih.gov/34745767))

111. [haddocking/pdb-tools](https://github.com/haddocking/pdb-tools) (‚≠ê346 ¬∑ Python) A dependency-free cross-platform swiss army knife for PDB files. (PMID: [30705752](https://pubmed.ncbi.nlm.nih.gov/30705752))

112. [brentp/vcfanno](https://github.com/brentp/vcfanno) (‚≠ê345 ¬∑ Go) annotate a VCF with other VCFs/BEDs/tabixed files (PMID: [27250555](https://pubmed.ncbi.nlm.nih.gov/27250555))

113. [Illumina/strelka](https://github.com/Illumina/strelka) (‚≠ê343 ¬∑ C++) Strelka2 germline and somatic small variant caller (PMID: [30013048](https://pubmed.ncbi.nlm.nih.gov/30013048))

114. [fbreitwieser/pavian](https://github.com/fbreitwieser/pavian) (‚≠ê343 ¬∑ R) üåà Interactive analysis of metagenomics data (PMID: [31553437](https://pubmed.ncbi.nlm.nih.gov/31553437))

115. [pastas/pastas](https://github.com/pastas/pastas) (‚≠ê342 ¬∑ Python) :spaghetti: Pastas is an open-source Python framework for the analysis of groundwater time series. (PMID: [31347164](https://pubmed.ncbi.nlm.nih.gov/31347164))

116. [gpertea/stringtie](https://github.com/gpertea/stringtie) (‚≠ê335 ¬∑ C++) Transcript assembly and quantification for RNA-Seq (PMID: [35648784](https://pubmed.ncbi.nlm.nih.gov/35648784))

117. [MolecularAI/Reinvent](https://github.com/MolecularAI/Reinvent) (‚≠ê333 ¬∑ Python) None (PMID: [33118816](https://pubmed.ncbi.nlm.nih.gov/33118816))

118. [ncbi/GeneGPT](https://github.com/ncbi/GeneGPT) (‚≠ê332 ¬∑ Python) Code and data for GeneGPT. (PMID: [38341654](https://pubmed.ncbi.nlm.nih.gov/38341654))

119. [gpertea/gffread](https://github.com/gpertea/gffread) (‚≠ê330 ¬∑ C++) GFF/GTF utility providing format conversions, region filtering, FASTA sequence extraction and more (PMID: [32489650](https://pubmed.ncbi.nlm.nih.gov/32489650))

120. [jtamames/SqueezeMeta](https://github.com/jtamames/SqueezeMeta) (‚≠ê330 ¬∑ C) A complete pipeline for metagenomic analysis (PMID: [30733714](https://pubmed.ncbi.nlm.nih.gov/30733714))

121. [dstahlke/gnuplot-iostream](https://github.com/dstahlke/gnuplot-iostream) (‚≠ê319 ¬∑ C++) C++ interface to gnuplot (PMID: [27402903](https://pubmed.ncbi.nlm.nih.gov/27402903))

122. [taozh2017/RGBD-SODsurvey](https://github.com/taozh2017/RGBD-SODsurvey) (‚≠ê317 ¬∑ MATLAB) RGB-D Salient Object Detection: A Survey (PMID: [33432275](https://pubmed.ncbi.nlm.nih.gov/33432275))

123. [beringresearch/ivis](https://github.com/beringresearch/ivis) (‚≠ê315 ¬∑ Python) Dimensionality reduction in very large datasets using Siamese Networks (PMID: [31222035](https://pubmed.ncbi.nlm.nih.gov/31222035))

124. [TissueImageAnalytics/tiatoolbox](https://github.com/TissueImageAnalytics/tiatoolbox) (‚≠ê312 ¬∑ Python) Computational Pathology Toolbox developed by TIA Centre, University of Warwick. (PMID: [35640384](https://pubmed.ncbi.nlm.nih.gov/35640384))

125. [kqwang/phase-recovery](https://github.com/kqwang/phase-recovery) (‚≠ê311 ¬∑ None) Resources for phase recovery (also called phase imaging, phase retrieval, or phase reconstruction) (PMID: [38161203](https://pubmed.ncbi.nlm.nih.gov/38161203))

126. [stamatak/standard-RAxML](https://github.com/stamatak/standard-RAxML) (‚≠ê305 ¬∑ C) None (PMID: [34410861](https://pubmed.ncbi.nlm.nih.gov/34410861); [37337442](https://pubmed.ncbi.nlm.nih.gov/37337442))

127. [oushujun/EDTA](https://github.com/oushujun/EDTA) (‚≠ê302 ¬∑ Perl) Extensive de-novo TE Annotator (PMID: [31843001](https://pubmed.ncbi.nlm.nih.gov/31843001); [33900591](https://pubmed.ncbi.nlm.nih.gov/33900591))

128. [BBillot/SynthSeg](https://github.com/BBillot/SynthSeg) (‚≠ê301 ¬∑ Python) Contrast-agnostic segmentation of MRI scans (PMID: [34367472](https://pubmed.ncbi.nlm.nih.gov/34367472))

129. [jonathanking/sidechainnet](https://github.com/jonathanking/sidechainnet) (‚≠ê301 ¬∑ Python) An all-atom protein structure dataset for machine learning. (PMID: [34213059](https://pubmed.ncbi.nlm.nih.gov/34213059))

130. [arq5x/lumpy-sv](https://github.com/arq5x/lumpy-sv) (‚≠ê298 ¬∑ C) lumpy: a general probabilistic framework for structural variant discovery (PMID: [24970577](https://pubmed.ncbi.nlm.nih.gov/24970577))

131. [whatshap/whatshap](https://github.com/whatshap/whatshap) (‚≠ê298 ¬∑ C++) Read-based phasing of genomic variants, also called haplotype assembly (PMID: [29949989](https://pubmed.ncbi.nlm.nih.gov/29949989); [37928341](https://pubmed.ncbi.nlm.nih.gov/37928341))

132. [MarioniLab/miloR](https://github.com/MarioniLab/miloR) (‚≠ê296 ¬∑ HTML) R package implementation of Milo for testing for differential abundance in KNN graphs (PMID: [34594043](https://pubmed.ncbi.nlm.nih.gov/34594043))

133. [bcgsc/abyss](https://github.com/bcgsc/abyss) (‚≠ê294 ¬∑ C++) :microscope: Assemble large genomes using short reads (PMID: [26209068](https://pubmed.ncbi.nlm.nih.gov/26209068); [35729491](https://pubmed.ncbi.nlm.nih.gov/35729491))

134. [saketkc/pysradb](https://github.com/saketkc/pysradb) (‚≠ê293 ¬∑ Python) Package for fetching metadata and downloading data from SRA/ENA/GEO (PMID: [31114675](https://pubmed.ncbi.nlm.nih.gov/31114675))

135. [lh3/miniasm](https://github.com/lh3/miniasm) (‚≠ê288 ¬∑ TeX) Ultrafast de novo assembly for long noisy reads (though having no consensus step) (PMID: [27153593](https://pubmed.ncbi.nlm.nih.gov/27153593))

136. [williamgilpin/pypdb](https://github.com/williamgilpin/pypdb) (‚≠ê287 ¬∑ Python) A Python API for the RCSB Protein Data Bank (PDB) (PMID: [26369703](https://pubmed.ncbi.nlm.nih.gov/26369703))

137. [imagej/imglib](https://github.com/imagej/imglib) (‚≠ê284 ¬∑ Java) A generic next-generation Java library for image processing (PMID: [22962343](https://pubmed.ncbi.nlm.nih.gov/22962343))

138. [mengyao/Complete-Striped-Smith-Waterman-Library](https://github.com/mengyao/Complete-Striped-Smith-Waterman-Library) (‚≠ê284 ¬∑ C) None (PMID: [24324759](https://pubmed.ncbi.nlm.nih.gov/24324759))

139. [philres/ngmlr](https://github.com/philres/ngmlr) (‚≠ê283 ¬∑ C++) NGMLR is a long-read mapper designed to align PacBio or Oxford Nanopore (standard and ultra-long) to a reference genome with a focus on reads that span structural variations (PMID: [29713083](https://pubmed.ncbi.nlm.nih.gov/29713083))

140. [ncbi/pgap](https://github.com/ncbi/pgap) (‚≠ê274 ¬∑ Common Workflow Language) NCBI Prokaryotic Genome Annotation Pipeline (PMID: [37962425](https://pubmed.ncbi.nlm.nih.gov/37962425))

141. [usnistgov/jarvis](https://github.com/usnistgov/jarvis) (‚≠ê272 ¬∑ Python) JARVIS-Tools: an open-source software package for data-driven atomistic materials design. Publications: https://scholar.google.com/citations?user=3w6ej94AAAAJ (PMID: [30117438](https://pubmed.ncbi.nlm.nih.gov/30117438); [32165788](https://pubmed.ncbi.nlm.nih.gov/32165788); [32165790](https://pubmed.ncbi.nlm.nih.gov/32165790); [33850146](https://pubmed.ncbi.nlm.nih.gov/33850146))

142. [torchmd/torchmd-net](https://github.com/torchmd/torchmd-net) (‚≠ê270 ¬∑ Python) Neural network potentials (PMID: [38463504](https://pubmed.ncbi.nlm.nih.gov/38463504))

143. [PhanstielLab/plotgardener](https://github.com/PhanstielLab/plotgardener) (‚≠ê268 ¬∑ R) None (PMID: [35134826](https://pubmed.ncbi.nlm.nih.gov/35134826))

144. [uncbiag/registration](https://github.com/uncbiag/registration) (‚≠ê266 ¬∑ TeX) Image Registration (PMID: [32523327](https://pubmed.ncbi.nlm.nih.gov/32523327); [36081637](https://pubmed.ncbi.nlm.nih.gov/36081637))

145. [hafen/trelliscopejs](https://github.com/hafen/trelliscopejs) (‚≠ê261 ¬∑ R) TrelliscopeJS R Package (PMID: [36622218](https://pubmed.ncbi.nlm.nih.gov/36622218))

146. [dmcable/RCTD](https://github.com/dmcable/RCTD) (‚≠ê256 ¬∑ HTML) Spatial-eXpression-R: Cell type identification (including cell type mixtures) and cell type-specific differential expression for spatial transcriptomics (PMID: [33603203](https://pubmed.ncbi.nlm.nih.gov/33603203))

147. [dmcable/spacexr](https://github.com/dmcable/spacexr) (‚≠ê256 ¬∑ HTML) Spatial-eXpression-R: Cell type identification (including cell type mixtures) and cell type-specific differential expression for spatial transcriptomics (PMID: [36050488](https://pubmed.ncbi.nlm.nih.gov/36050488))

148. [marbl/MashMap](https://github.com/marbl/MashMap) (‚≠ê255 ¬∑ C++) A fast approximate aligner for long DNA sequences (PMID: [30423094](https://pubmed.ncbi.nlm.nih.gov/30423094); [37603771](https://pubmed.ncbi.nlm.nih.gov/37603771))

149. [spglib/spglib](https://github.com/spglib/spglib) (‚≠ê255 ¬∑ C) C library for finding and handling crystal symmetries (PMID: [37668050](https://pubmed.ncbi.nlm.nih.gov/37668050))

150. [schneebergerlab/plotsr](https://github.com/schneebergerlab/plotsr) (‚≠ê251 ¬∑ Python) Tool to plot synteny and structural rearrangements between genomes (PMID: [35561173](https://pubmed.ncbi.nlm.nih.gov/35561173))

151. [brentp/somalier](https://github.com/brentp/somalier) (‚≠ê250 ¬∑ Nim) fast sample-swap and relatedness checks on BAMs/CRAMs/VCFs/GVCFs... "like damn that is one smart wine guy" (PMID: [32664994](https://pubmed.ncbi.nlm.nih.gov/32664994))

152. [marianattestad/ribbon](https://github.com/marianattestad/ribbon) (‚≠ê249 ¬∑ JavaScript) A genome browser that shows long reads and complex variants better (PMID: [32766814](https://pubmed.ncbi.nlm.nih.gov/32766814))

153. [tSchlegl/f-AnoGAN](https://github.com/tSchlegl/f-AnoGAN) (‚≠ê243 ¬∑ Python) Code for reproducing f-AnoGAN training and anomaly scoring (PMID: [30831356](https://pubmed.ncbi.nlm.nih.gov/30831356))

154. [theislab/scgen](https://github.com/theislab/scgen) (‚≠ê243 ¬∑ Python) Single cell perturbation prediction (PMID: [31363220](https://pubmed.ncbi.nlm.nih.gov/31363220))

155. [sigven/pcgr](https://github.com/sigven/pcgr) (‚≠ê242 ¬∑ R) Personal Cancer Genome Reporter (PCGR) (PMID: [29272339](https://pubmed.ncbi.nlm.nih.gov/29272339))

156. [maickrau/GraphAligner](https://github.com/maickrau/GraphAligner) (‚≠ê240 ¬∑ C++) None (PMID: [30851095](https://pubmed.ncbi.nlm.nih.gov/30851095); [32972461](https://pubmed.ncbi.nlm.nih.gov/32972461))

157. [jtextor/dagitty](https://github.com/jtextor/dagitty) (‚≠ê239 ¬∑ JavaScript) Graphical analysis of structural causal models / graphical causal models. (PMID: [28089956](https://pubmed.ncbi.nlm.nih.gov/28089956))

158. [brentp/slivar](https://github.com/brentp/slivar) (‚≠ê236 ¬∑ Nim) genetic variant expressions, annotation, and filtering for great good. (PMID: [34267211](https://pubmed.ncbi.nlm.nih.gov/34267211))

159. [opencobra/cobratoolbox](https://github.com/opencobra/cobratoolbox) (‚≠ê236 ¬∑ MATLAB) The COnstraint-Based Reconstruction and Analysis Toolbox.              Documentation: (PMID: [27993782](https://pubmed.ncbi.nlm.nih.gov/27993782); [28158334](https://pubmed.ncbi.nlm.nih.gov/28158334); [30052768](https://pubmed.ncbi.nlm.nih.gov/30052768); [37697651](https://pubmed.ncbi.nlm.nih.gov/37697651))

160. [lmdu/pyfastx](https://github.com/lmdu/pyfastx) (‚≠ê235 ¬∑ C) a python package for fast random access to sequences from plain and gzipped FASTA/Q files (PMID: [33341884](https://pubmed.ncbi.nlm.nih.gov/33341884))

161. [Kinggerm/GetOrganelle](https://github.com/Kinggerm/GetOrganelle) (‚≠ê232 ¬∑ Python) Organelle Genome Assembly Toolkit (Chloroplast/Mitocondrial/ITS) (PMID: [32912315](https://pubmed.ncbi.nlm.nih.gov/32912315))

162. [jeffdaily/parasail](https://github.com/jeffdaily/parasail) (‚≠ê230 ¬∑ C) Pairwise Sequence Alignment Library (PMID: [26864881](https://pubmed.ncbi.nlm.nih.gov/26864881))

163. [lukasturcani/stk](https://github.com/lukasturcani/stk) (‚≠ê229 ¬∑ Python) A Python library which allows construction and manipulation of complex molecules, as well as automatic molecular design and the creation of molecular databases. (PMID: [34240979](https://pubmed.ncbi.nlm.nih.gov/34240979))

164. [biocommons/hgvs](https://github.com/biocommons/hgvs) (‚≠ê228 ¬∑ Python) Python library to parse, format, validate, normalize, and map sequence variants. `pip install hgvs` (PMID: [28967166](https://pubmed.ncbi.nlm.nih.gov/28967166))

165. [dasmoth/dalliance](https://github.com/dasmoth/dalliance) (‚≠ê226 ¬∑ JavaScript) Interactive web-based genome browser. (PMID: [26315906](https://pubmed.ncbi.nlm.nih.gov/26315906))

166. [gtonkinhill/panaroo](https://github.com/gtonkinhill/panaroo) (‚≠ê226 ¬∑ Python) An updated pipeline for pangenome investigation (PMID: [32698896](https://pubmed.ncbi.nlm.nih.gov/32698896))

167. [schatzlab/genomescope](https://github.com/schatzlab/genomescope) (‚≠ê226 ¬∑ JavaScript) Fast genome analysis from unassembled short reads (PMID: [28369201](https://pubmed.ncbi.nlm.nih.gov/28369201))

168. [RasmussenLab/vamb](https://github.com/RasmussenLab/vamb) (‚≠ê225 ¬∑ Python) Variational autoencoder for metagenomic binning (PMID: [33398153](https://pubmed.ncbi.nlm.nih.gov/33398153))

169. [adamewing/bamsurgeon](https://github.com/adamewing/bamsurgeon) (‚≠ê224 ¬∑ Python) tools for adding mutations to existing .bam files, used for testing mutation callers (PMID: [25984700](https://pubmed.ncbi.nlm.nih.gov/25984700); [30400818](https://pubmed.ncbi.nlm.nih.gov/30400818))

170. [alekseyzimin/masurca](https://github.com/alekseyzimin/masurca) (‚≠ê224 ¬∑ M4) None (PMID: [35120119](https://pubmed.ncbi.nlm.nih.gov/35120119))

171. [ssadedin/bpipe](https://github.com/ssadedin/bpipe) (‚≠ê223 ¬∑ Groovy) Bpipe - a tool for running and managing bioinformatics pipelines (PMID: [26460497](https://pubmed.ncbi.nlm.nih.gov/26460497))

172. [tjiangHIT/cuteSV](https://github.com/tjiangHIT/cuteSV) (‚≠ê220 ¬∑ Python) Long read based human genomic structural variation detection with cuteSV (PMID: [32746918](https://pubmed.ncbi.nlm.nih.gov/32746918); [35751813](https://pubmed.ncbi.nlm.nih.gov/35751813))

173. [EpistasisLab/Aliro](https://github.com/EpistasisLab/Aliro) (‚≠ê219 ¬∑ JavaScript) Aliro: AI-Driven Data Science (PMID: [37796839](https://pubmed.ncbi.nlm.nih.gov/37796839))

174. [smirarab/ASTRAL](https://github.com/smirarab/ASTRAL) (‚≠ê219 ¬∑ Java) Accurate Species TRee ALgorithm (PMID: [25161245](https://pubmed.ncbi.nlm.nih.gov/25161245); [26072508](https://pubmed.ncbi.nlm.nih.gov/26072508); [30903685](https://pubmed.ncbi.nlm.nih.gov/30903685))

175. [xiangyue9607/BioNEV](https://github.com/xiangyue9607/BioNEV) (‚≠ê218 ¬∑ Python) Graph Embedding Evaluation / Code and Datasets for  "Graph Embedding on Biomedical Networks: Methods, Applications, and Evaluations"  (Bioinformatics 2020) (PMID: [31584634](https://pubmed.ncbi.nlm.nih.gov/31584634))

176. [marbl/winnowmap](https://github.com/marbl/winnowmap) (‚≠ê217 ¬∑ C) Long read / genome alignment software (PMID: [32657365](https://pubmed.ncbi.nlm.nih.gov/32657365))

177. [qiicr/dcmqi](https://github.com/qiicr/dcmqi) (‚≠ê217 ¬∑ C++) dcmqi (DICOM for Quantitative Imaging) is a free, open source C++ library for conversion between imaging research formats and the standard DICOM representation for image analysis results (PMID: [29092948](https://pubmed.ncbi.nlm.nih.gov/29092948))

178. [ryanlayer/giggle](https://github.com/ryanlayer/giggle) (‚≠ê217 ¬∑ C) Interval data structure (PMID: [29309061](https://pubmed.ncbi.nlm.nih.gov/29309061))

179. [choderalab/pymbar](https://github.com/choderalab/pymbar) (‚≠ê216 ¬∑ Python) Python implementation of the multistate Bennett acceptance ratio (MBAR) (PMID: [25808134](https://pubmed.ncbi.nlm.nih.gov/25808134))

180. [RGLab/MAST](https://github.com/RGLab/MAST) (‚≠ê213 ¬∑ R) Tools and methods for analysis of single cell assay data in R (PMID: [26653891](https://pubmed.ncbi.nlm.nih.gov/26653891))

181. [debbiemarkslab/evcouplings](https://github.com/debbiemarkslab/evcouplings) (‚≠ê212 ¬∑ Jupyter Notebook) Evolutionary couplings from protein and RNA sequence alignments (PMID: [30304492](https://pubmed.ncbi.nlm.nih.gov/30304492))

182. [fbreitwieser/krakenuniq](https://github.com/fbreitwieser/krakenuniq) (‚≠ê212 ¬∑ C++) üêô KrakenUniq: Metagenomics classifier with unique k-mer counting for more specific results (PMID: [30445993](https://pubmed.ncbi.nlm.nih.gov/30445993))

183. [KamilSJaron/smudgeplot](https://github.com/KamilSJaron/smudgeplot) (‚≠ê211 ¬∑ R) Inference of ploidy and heterozygosity structure using whole genome sequencing data (PMID: [32188846](https://pubmed.ncbi.nlm.nih.gov/32188846))

184. [bcgsc/NanoSim](https://github.com/bcgsc/NanoSim) (‚≠ê211 ¬∑ Python) Nanopore sequence read simulator (PMID: [32520350](https://pubmed.ncbi.nlm.nih.gov/32520350); [36939007](https://pubmed.ncbi.nlm.nih.gov/36939007))

185. [taowenmicro/EasyMicrobiomeR](https://github.com/taowenmicro/EasyMicrobiomeR) (‚≠ê210 ¬∑ HTML) Using R language in microbiome analysis (PMID: [37128855](https://pubmed.ncbi.nlm.nih.gov/37128855))

186. [GregoryFaust/samblaster](https://github.com/GregoryFaust/samblaster) (‚≠ê209 ¬∑ C++) samblaster: a tool to mark duplicates and extract discordant and split reads from sam files. (PMID: [24812344](https://pubmed.ncbi.nlm.nih.gov/24812344))

187. [cambridgeltl/MTL-Bioinformatics-2016](https://github.com/cambridgeltl/MTL-Bioinformatics-2016) (‚≠ê209 ¬∑ Python) None (PMID: [32415963](https://pubmed.ncbi.nlm.nih.gov/32415963))

188. [brentp/goleft](https://github.com/brentp/goleft) (‚≠ê207 ¬∑ Go) goleft is a collection of bioinformatics tools distributed under MIT license in a single static binary (PMID: [29048539](https://pubmed.ncbi.nlm.nih.gov/29048539))

189. [rdk/p2rank](https://github.com/rdk/p2rank) (‚≠ê207 ¬∑ Groovy) P2Rank: Protein-ligand binding site prediction tool based on machine learning. Stand-alone command line program / Java library for predicting ligand binding pockets from protein structure. (PMID: [31114880](https://pubmed.ncbi.nlm.nih.gov/31114880))

190. [hkmztrk/DeepDTA](https://github.com/hkmztrk/DeepDTA) (‚≠ê206 ¬∑ Python) None (PMID: [30423097](https://pubmed.ncbi.nlm.nih.gov/30423097))

191. [labgem/PPanGGOLiN](https://github.com/labgem/PPanGGOLiN) (‚≠ê205 ¬∑ Python) Build a partitioned pangenome graph from microbial genomes (PMID: [32191703](https://pubmed.ncbi.nlm.nih.gov/32191703); [33381850](https://pubmed.ncbi.nlm.nih.gov/33381850))

192. [HajkD/biomartr](https://github.com/HajkD/biomartr) (‚≠ê204 ¬∑ R) Genomic Data Retrieval with R (PMID: [28110292](https://pubmed.ncbi.nlm.nih.gov/28110292))

193. [PacificBiosciences/FALCON](https://github.com/PacificBiosciences/FALCON) (‚≠ê204 ¬∑ Python) FALCON: experimental PacBio diploid assembler -- Out-of-date -- Please use a binary release: https://github.com/PacificBiosciences/FALCON_unzip/wiki/Binaries (PMID: [27749838](https://pubmed.ncbi.nlm.nih.gov/27749838))

194. [ucscGenomeBrowser/kent](https://github.com/ucscGenomeBrowser/kent) (‚≠ê204 ¬∑ C) UCSC Genome Browser source tree. Stable branch: "beta". (PMID: [28108446](https://pubmed.ncbi.nlm.nih.gov/28108446))

195. [iqtree/iqtree2](https://github.com/iqtree/iqtree2) (‚≠ê202 ¬∑ C++) NEW location of IQ-TREE software for efficient phylogenomic software by maximum likelihood http://www.iqtree.org (PMID: [36383168](https://pubmed.ncbi.nlm.nih.gov/36383168); [37656933](https://pubmed.ncbi.nlm.nih.gov/37656933))

196. [INCATools/ontology-development-kit](https://github.com/INCATools/ontology-development-kit) (‚≠ê201 ¬∑ Jinja) Bootstrap an OBO Library ontology (PMID: [36208225](https://pubmed.ncbi.nlm.nih.gov/36208225))

197. [ZJUFanLab/scCATCH](https://github.com/ZJUFanLab/scCATCH) (‚≠ê201 ¬∑ R) Automatic Annotation on Cell Types of Clusters from Single-Cell RNA Sequencing Data (PMID: [32062421](https://pubmed.ncbi.nlm.nih.gov/32062421))

198. [griffithlab/GenVisR](https://github.com/griffithlab/GenVisR) (‚≠ê201 ¬∑ R) Genome data visualizations (PMID: [27288499](https://pubmed.ncbi.nlm.nih.gov/27288499))

199. [jeetsukumaran/DendroPy](https://github.com/jeetsukumaran/DendroPy) (‚≠ê200 ¬∑ Python) A Python library for phylogenetic scripting, simulation, data processing and manipulation. (PMID: [20421198](https://pubmed.ncbi.nlm.nih.gov/20421198))

200. [choderalab/espaloma](https://github.com/choderalab/espaloma) (‚≠ê198 ¬∑ Python) Extensible Surrogate Potential of Ab initio Learned and Optimized by Message-passing Algorithm  üçπhttps://arxiv.org/abs/2010.01196 (PMID: [36349096](https://pubmed.ncbi.nlm.nih.gov/36349096))

201. [showteeth/ggcoverage](https://github.com/showteeth/ggcoverage) (‚≠ê198 ¬∑ R) Visualize and annotate genomic coverage with ggplot2 (PMID: [37559015](https://pubmed.ncbi.nlm.nih.gov/37559015))

202. [kormilitzin/med7](https://github.com/kormilitzin/med7) (‚≠ê196 ¬∑ None) None (PMID: [34412834](https://pubmed.ncbi.nlm.nih.gov/34412834))

203. [shervinea/enzynet](https://github.com/shervinea/enzynet) (‚≠ê196 ¬∑ Python) EnzyNet: enzyme classification using 3D convolutional neural networks on spatial representation (PMID: [29740518](https://pubmed.ncbi.nlm.nih.gov/29740518))

204. [MolecularAI/REINVENT4](https://github.com/MolecularAI/REINVENT4) (‚≠ê195 ¬∑ Python) AI molecular design tool for de novo design, scaffold hopping, R-group replacement, linker design and molecule optimization. (PMID: [38383444](https://pubmed.ncbi.nlm.nih.gov/38383444))

205. [TGAC/KAT](https://github.com/TGAC/KAT) (‚≠ê191 ¬∑ C++) The K-mer Analysis Toolkit (KAT) contains a number of tools that analyse and compare K-mer spectra. (PMID: [27797770](https://pubmed.ncbi.nlm.nih.gov/27797770))

206. [dariober/ASCIIGenome](https://github.com/dariober/ASCIIGenome) (‚≠ê191 ¬∑ Java) Text Only Genome Viewer! (PMID: [28119307](https://pubmed.ncbi.nlm.nih.gov/28119307))

207. [thinng/GraphDTA](https://github.com/thinng/GraphDTA) (‚≠ê191 ¬∑ Python) GraphDTA: Predicting drug-target binding affinity with graph neural networks (PMID: [33119053](https://pubmed.ncbi.nlm.nih.gov/33119053))

208. [wyp1125/MCScanX](https://github.com/wyp1125/MCScanX) (‚≠ê191 ¬∑ CAP CDS) MCScanX: Multiple Collinearity Scan toolkit X version. The most popular synteny analysis tool in the world! (PMID: [38491145](https://pubmed.ncbi.nlm.nih.gov/38491145))

209. [vibansal/HapCUT2](https://github.com/vibansal/HapCUT2) (‚≠ê190 ¬∑ C) software tools for haplotype assembly from sequence data (PMID: [36335497](https://pubmed.ncbi.nlm.nih.gov/36335497))

210. [CapsuleEndoscope/EndoSLAM](https://github.com/CapsuleEndoscope/EndoSLAM) (‚≠ê189 ¬∑ Python) EndoSLAM Dataset and an Unsupervised Monocular Visual Odometry and Depth Estimation Approach for Endoscopic Videos: Endo-SfMLearner (PMID: [33930829](https://pubmed.ncbi.nlm.nih.gov/33930829))

211. [atks/vt](https://github.com/atks/vt) (‚≠ê188 ¬∑ C) A tool set for short variant discovery in genetic sequence data. (PMID: [25701572](https://pubmed.ncbi.nlm.nih.gov/25701572))

212. [mirnylab/cooler](https://github.com/mirnylab/cooler) (‚≠ê188 ¬∑ Python) A cool place to store your Hi-C (PMID: [31290943](https://pubmed.ncbi.nlm.nih.gov/31290943); [33135074](https://pubmed.ncbi.nlm.nih.gov/33135074))

213. [pmelsted/bifrost](https://github.com/pmelsted/bifrost) (‚≠ê188 ¬∑ C++) Bifrost: Highly parallel construction and indexing of colored and compacted de Bruijn graphs (PMID: [32943081](https://pubmed.ncbi.nlm.nih.gov/32943081))

214. [rdkit/mmpdb](https://github.com/rdkit/mmpdb) (‚≠ê188 ¬∑ Python) A package to identify matched molecular pairs and use them to predict property changes. (PMID: [29770697](https://pubmed.ncbi.nlm.nih.gov/29770697))

215. [AstrobioMike/GToTree](https://github.com/AstrobioMike/GToTree) (‚≠ê187 ¬∑ Shell) A user-friendly workflow for phylogenomics (PMID: [30865266](https://pubmed.ncbi.nlm.nih.gov/30865266))

216. [cerr/CERR](https://github.com/cerr/CERR) (‚≠ê187 ¬∑ MATLAB) Matlab/Octave based platform for Radiological Research. (PMID: [34874302](https://pubmed.ncbi.nlm.nih.gov/34874302))

217. [noriakis/ggkegg](https://github.com/noriakis/ggkegg) (‚≠ê187 ¬∑ R) Analyzing and visualizing KEGG information using the grammar of graphics (PMID: [37846038](https://pubmed.ncbi.nlm.nih.gov/37846038))

218. [SajadMo/SleepEEGNet](https://github.com/SajadMo/SleepEEGNet) (‚≠ê186 ¬∑ Python) SleepEEGNet: Automated Sleep Stage Scoring with Sequence to Sequence Deep Learning Approach (PMID: [31063501](https://pubmed.ncbi.nlm.nih.gov/31063501))

219. [anjany/verse](https://github.com/anjany/verse) (‚≠ê186 ¬∑ Jupyter Notebook) Everything about the 'Large Scale Vertebrae Segmentation Challenge' @ MICCAI 2019-2020 (PMID: [34340104](https://pubmed.ncbi.nlm.nih.gov/34340104))

220. [danmcduff/iphys-toolbox](https://github.com/danmcduff/iphys-toolbox) (‚≠ê184 ¬∑ MATLAB) A MATLAB toolbox for iPPG analysis.  The toolbox includes implementations of commonly used methods. (PMID: [31947335](https://pubmed.ncbi.nlm.nih.gov/31947335))

221. [debbiemarkslab/DeepSequence](https://github.com/debbiemarkslab/DeepSequence) (‚≠ê184 ¬∑ Python) A generative latent variable model for biological sequence families. (PMID: [30250057](https://pubmed.ncbi.nlm.nih.gov/30250057))

222. [theislab/paga](https://github.com/theislab/paga) (‚≠ê183 ¬∑ Jupyter Notebook) Mapping out the coarse-grained connectivity structures of complex manifolds. (PMID: [30890159](https://pubmed.ncbi.nlm.nih.gov/30890159))

223. [wdecoster/nanopack](https://github.com/wdecoster/nanopack) (‚≠ê183 ¬∑ Python) An overview of all nanopack tools (PMID: [29547981](https://pubmed.ncbi.nlm.nih.gov/29547981); [37171891](https://pubmed.ncbi.nlm.nih.gov/37171891))

224. [openvax/mhcflurry](https://github.com/openvax/mhcflurry) (‚≠ê180 ¬∑ Python) Peptide-MHC I binding affinity prediction (PMID: [32124315](https://pubmed.ncbi.nlm.nih.gov/32124315); [32711842](https://pubmed.ncbi.nlm.nih.gov/32711842))

225. [Nextomics/NextPolish](https://github.com/Nextomics/NextPolish) (‚≠ê177 ¬∑ None)  Fast and accurately polish the genome generated by long reads. (PMID: [31778144](https://pubmed.ncbi.nlm.nih.gov/31778144))

226. [isovic/graphmap](https://github.com/isovic/graphmap) (‚≠ê177 ¬∑ C++) GraphMap - A highly sensitive and accurate mapper for long, error-prone reads http://www.nature.com/ncomms/2016/160415/ncomms11307/full/ncomms11307.html Note: This was the original repository which will no longer be officially maintained. Please use the new official repository here: (PMID: [27079541](https://pubmed.ncbi.nlm.nih.gov/27079541))

227. [pangenome/odgi](https://github.com/pangenome/odgi) (‚≠ê177 ¬∑ C++) Optimized Dynamic Genome/Graph Implementation: understanding pangenome graphs (PMID: [35552372](https://pubmed.ncbi.nlm.nih.gov/35552372))

228. [mortazavilab/PyWGCNA](https://github.com/mortazavilab/PyWGCNA) (‚≠ê176 ¬∑ Jupyter Notebook) PyWGCNA is a Python package designed to do Weighted Gene Correlation Network analysis (WGCNA) (PMID: [37399090](https://pubmed.ncbi.nlm.nih.gov/37399090))

229. [privefl/bigsnpr](https://github.com/privefl/bigsnpr) (‚≠ê175 ¬∑ R) R package for the analysis of massive SNP arrays. (PMID: [32415959](https://pubmed.ncbi.nlm.nih.gov/32415959))

230. [vsbuffalo/scidataflow](https://github.com/vsbuffalo/scidataflow) (‚≠ê175 ¬∑ Rust) Command line scientific data management tool (PMID: [38180848](https://pubmed.ncbi.nlm.nih.gov/38180848))

231. [ekimb/rust-mdbg](https://github.com/ekimb/rust-mdbg) (‚≠ê173 ¬∑ Rust) minimizer-space de Bruijn graphs (mdBG) for whole genome assembly (PMID: [34525345](https://pubmed.ncbi.nlm.nih.gov/34525345))

232. [gpertea/gffcompare](https://github.com/gpertea/gffcompare) (‚≠ê173 ¬∑ C++) classify, merge, tracking and annotation of GFF files by comparing to a reference annotation GFF (PMID: [32489650](https://pubmed.ncbi.nlm.nih.gov/32489650))

233. [HadrienG/InSilicoSeq](https://github.com/HadrienG/InSilicoSeq) (‚≠ê172 ¬∑ Python) :rocket: A sequencing simulator (PMID: [30016412](https://pubmed.ncbi.nlm.nih.gov/30016412))

234. [aquaskyline/Clairvoyante](https://github.com/aquaskyline/Clairvoyante) (‚≠ê172 ¬∑ Python) Clairvoyante: a multi-task convolutional deep neural network for variant calling in Single Molecule Sequencing (PMID: [30824707](https://pubmed.ncbi.nlm.nih.gov/30824707))

235. [choderalab/perses](https://github.com/choderalab/perses) (‚≠ê172 ¬∑ Python) Experiments with expanded ensembles to explore chemical space (PMID: [37450482](https://pubmed.ncbi.nlm.nih.gov/37450482))

236. [machinegun/SALSA](https://github.com/machinegun/SALSA) (‚≠ê171 ¬∑ Python) SALSA: A tool to scaffold long read assemblies with Hi-C data (PMID: [31433799](https://pubmed.ncbi.nlm.nih.gov/31433799))

237. [Genetalks/gtz](https://github.com/Genetalks/gtz) (‚≠ê170 ¬∑ None) A high performance and compression ratio compressor for genomic data, powered by GTXLab of Genetalks. (PMID: [29297296](https://pubmed.ncbi.nlm.nih.gov/29297296))

238. [KristapsE/DP4-AI](https://github.com/KristapsE/DP4-AI) (‚≠ê169 ¬∑ Python) Python workflow for DP5 and DP4 analysis of organic molecules (PMID: [34122893](https://pubmed.ncbi.nlm.nih.gov/34122893))

239. [jbkinney/logomaker](https://github.com/jbkinney/logomaker) (‚≠ê169 ¬∑ Jupyter Notebook) Software for the visualization of sequence-function relationships (PMID: [31821414](https://pubmed.ncbi.nlm.nih.gov/31821414))

240. [shenwei356/kmcp](https://github.com/shenwei356/kmcp) (‚≠ê169 ¬∑ Go) Accurate metagenomic profiling && Fast large-scale sequence/genome searching (PMID: [36579886](https://pubmed.ncbi.nlm.nih.gov/36579886))

241. [alexanderkroner/saliency](https://github.com/alexanderkroner/saliency) (‚≠ê168 ¬∑ Python) Contextual Encoder-Decoder Network for Visual Saliency Prediction [Neural Networks 2020] (PMID: [32563023](https://pubmed.ncbi.nlm.nih.gov/32563023))

242. [Illumina/ExpansionHunter](https://github.com/Illumina/ExpansionHunter) (‚≠ê167 ¬∑ C++) A tool for estimating repeat sizes (PMID: [31134279](https://pubmed.ncbi.nlm.nih.gov/31134279))

243. [malonge/RaGOO](https://github.com/malonge/RaGOO) (‚≠ê166 ¬∑ Python) RaGOO is no longer supported. Please use RagTag instead: https://github.com/malonge/RagTag (PMID: [31661016](https://pubmed.ncbi.nlm.nih.gov/31661016))

244. [vgteam/sequenceTubeMap](https://github.com/vgteam/sequenceTubeMap) (‚≠ê166 ¬∑ JavaScript) displays multiple genomic sequences in the form of a tube map (PMID: [31368484](https://pubmed.ncbi.nlm.nih.gov/31368484))

245. [kexinhuang12345/moltrans](https://github.com/kexinhuang12345/moltrans) (‚≠ê165 ¬∑ Jupyter Notebook) MolTrans: Molecular Interaction Transformer for Drug Target Interaction Prediction (Bioinformatics) (PMID: [33070179](https://pubmed.ncbi.nlm.nih.gov/33070179))

246. [soedinglab/metaeuk](https://github.com/soedinglab/metaeuk) (‚≠ê165 ¬∑ C) MetaEuk - sensitive, high-throughput gene discovery and annotation for large-scale eukaryotic metagenomics (PMID: [32245390](https://pubmed.ncbi.nlm.nih.gov/32245390))

247. [stephaneguindon/phyml](https://github.com/stephaneguindon/phyml) (‚≠ê165 ¬∑ C) PhyML -- Phylogenetic estimation using (Maximum) Likelihood (PMID: [30977781](https://pubmed.ncbi.nlm.nih.gov/30977781))

248. [alanwilter/acpype](https://github.com/alanwilter/acpype) (‚≠ê164 ¬∑ Python) OFFICIAL: AnteChamber PYthon Parser interfacE (PMID: [37252824](https://pubmed.ncbi.nlm.nih.gov/37252824))

249. [franciscozorrilla/metaGEM](https://github.com/franciscozorrilla/metaGEM) (‚≠ê164 ¬∑ Python) :gem: An easy-to-use workflow for generating context specific genome-scale metabolic models and predicting metabolic interactions within microbial communities directly from metagenomic data (PMID: [34614189](https://pubmed.ncbi.nlm.nih.gov/34614189))

250. [molgenis/systemsgenetics](https://github.com/molgenis/systemsgenetics) (‚≠ê164 ¬∑ Java) Generic Java genotype reader / writer, QTL mapping software, Strand alignment tool (PMID: [32532224](https://pubmed.ncbi.nlm.nih.gov/32532224); [34792549](https://pubmed.ncbi.nlm.nih.gov/34792549))

251. [ChiLiubio/microeco](https://github.com/ChiLiubio/microeco) (‚≠ê163 ¬∑ R) An R package for data analysis in microbial community ecology (PMID: [33332530](https://pubmed.ncbi.nlm.nih.gov/33332530))

252. [baoe/AlignGraph](https://github.com/baoe/AlignGraph) (‚≠ê163 ¬∑ C++) Algorithm for secondary de novo genome assembly guided by closely related references (PMID: [24932000](https://pubmed.ncbi.nlm.nih.gov/24932000))

253. [ndierckx/NOVOPlasty](https://github.com/ndierckx/NOVOPlasty) (‚≠ê163 ¬∑ Perl) NOVOPlasty - The organelle assembler and heteroplasmy caller (PMID: [28204566](https://pubmed.ncbi.nlm.nih.gov/28204566); [33575563](https://pubmed.ncbi.nlm.nih.gov/33575563))

254. [Superzchen/iFeature](https://github.com/Superzchen/iFeature) (‚≠ê162 ¬∑ Python) iFeature is a comprehensive Python-based toolkit for generating various numerical feature representation schemes from protein or peptide sequences. iFeature is capable of calculating and extracting a wide spectrum of 18 major sequence encoding schemes that encompass 53 different types of feature descriptors. Furthermore, iFeature also integrates five kinds of frequently used feature clustering algorithms, four feature selection algorithms and three dimensionality reduction algorithms. (PMID: [29528364](https://pubmed.ncbi.nlm.nih.gov/29528364))

255. [XinhaoLi74/SmilesPE](https://github.com/XinhaoLi74/SmilesPE) (‚≠ê160 ¬∑ Jupyter Notebook) SMILES Pair Encoding: A data-driven substructure representation of chemicals (PMID: [33715361](https://pubmed.ncbi.nlm.nih.gov/33715361))

256. [chrisamiller/fishplot](https://github.com/chrisamiller/fishplot) (‚≠ê160 ¬∑ R) Create timecourse "fish plots" that show changes in the clonal architecture of tumors (PMID: [27821060](https://pubmed.ncbi.nlm.nih.gov/27821060))

257. [UCLCheminformatics/ScaffoldGraph](https://github.com/UCLCheminformatics/ScaffoldGraph) (‚≠ê159 ¬∑ Python) ScaffoldGraph is an open-source cheminformatics library, built using RDKit and NetworkX, for the generation and analysis of scaffold networks and scaffold trees. (PMID: [32232438](https://pubmed.ncbi.nlm.nih.gov/32232438))

258. [mkirsche/Jasmine](https://github.com/mkirsche/Jasmine) (‚≠ê159 ¬∑ Java) Jasmine: SV Merging Across Samples (PMID: [36658279](https://pubmed.ncbi.nlm.nih.gov/36658279))

259. [yiluheihei/microbiomeMarker](https://github.com/yiluheihei/microbiomeMarker) (‚≠ê159 ¬∑ R) R package for microbiome biomarker discovery (PMID: [35771644](https://pubmed.ncbi.nlm.nih.gov/35771644))

260. [dnbaker/dashing](https://github.com/dnbaker/dashing) (‚≠ê158 ¬∑ C++) Fast and accurate genomic distances using HyperLogLog (PMID: [31801633](https://pubmed.ncbi.nlm.nih.gov/31801633))

261. [duartegroup/autodE](https://github.com/duartegroup/autodE) (‚≠ê158 ¬∑ Python) automated reaction profile generation (PMID: [33108028](https://pubmed.ncbi.nlm.nih.gov/33108028))

262. [galaxyproject/tools-iuc](https://github.com/galaxyproject/tools-iuc) (‚≠ê158 ¬∑ HTML) Tool Shed repositories maintained by the Intergalactic Utilities Commission (PMID: [34096994](https://pubmed.ncbi.nlm.nih.gov/34096994))

263. [sorgerlab/indra](https://github.com/sorgerlab/indra) (‚≠ê158 ¬∑ Python) INDRA (Integrated Network and Dynamical Reasoning Assembler) is an automated model assembly system interfacing with NLP systems and databases to collect knowledge, and through a process of assembly, produce causal graphs and dynamical models. (PMID: [31070726](https://pubmed.ncbi.nlm.nih.gov/31070726))

264. [nickabattista/IB2d](https://github.com/nickabattista/IB2d) (‚≠ê157 ¬∑ MATLAB) An easy to use immersed boundary method in 2D, with full implementations in MATLAB and Python that contains over 75 built-in examples, including multiple options for fiber-structure models and advection-diffusion, Boussinesq approximations, and/or artificial forcing. (PMID: [28355146](https://pubmed.ncbi.nlm.nih.gov/28355146))

265. [abyzovlab/CNVpytor](https://github.com/abyzovlab/CNVpytor) (‚≠ê156 ¬∑ Python) a python extension of CNVnator -- a tool for CNV analysis from depth-of-coverage by mapped reads (PMID: [34817058](https://pubmed.ncbi.nlm.nih.gov/34817058))

266. [BaselAbujamous/clust](https://github.com/BaselAbujamous/clust) (‚≠ê155 ¬∑ Python) Automatic and optimised consensus clustering of one or more heterogeneous datasets (PMID: [30359297](https://pubmed.ncbi.nlm.nih.gov/30359297))

267. [bjing2016/EigenFold](https://github.com/bjing2016/EigenFold) (‚≠ê155 ¬∑ Jupyter Notebook) EigenFold: Generative Protein Structure Prediction with Diffusion Models (PMID: [37064532](https://pubmed.ncbi.nlm.nih.gov/37064532))

268. [herrmannlab/highdicom](https://github.com/herrmannlab/highdicom) (‚≠ê154 ¬∑ Python) High-level DICOM abstractions for the Python programming language (PMID: [35995898](https://pubmed.ncbi.nlm.nih.gov/35995898))

269. [mfumagalli/ngsTools](https://github.com/mfumagalli/ngsTools) (‚≠ê154 ¬∑ R) Programs to analyse NGS data for population genetics purposes (PMID: [24458950](https://pubmed.ncbi.nlm.nih.gov/24458950))

270. [AnantharamanLab/METABOLIC](https://github.com/AnantharamanLab/METABOLIC) (‚≠ê153 ¬∑ Perl) A scalable high-throughput metabolic and biogeochemical functional trait profiler (PMID: [35172890](https://pubmed.ncbi.nlm.nih.gov/35172890))

271. [brentp/hts-nim](https://github.com/brentp/hts-nim) (‚≠ê153 ¬∑ Nim) nim wrapper for htslib for parsing genomics data files (PMID: [29718142](https://pubmed.ncbi.nlm.nih.gov/29718142))

272. [nygenome/lancet](https://github.com/nygenome/lancet) (‚≠ê153 ¬∑ C) Microassembly based somatic variant caller for NGS data (PMID: [30271907](https://pubmed.ncbi.nlm.nih.gov/30271907); [33241313](https://pubmed.ncbi.nlm.nih.gov/33241313))

273. [divelab/MoleculeX](https://github.com/divelab/MoleculeX) (‚≠ê152 ¬∑ Python) None (PMID: [35179547](https://pubmed.ncbi.nlm.nih.gov/35179547))

274. [dsehnal/LiteMol](https://github.com/dsehnal/LiteMol) (‚≠ê152 ¬∑ JavaScript) A library/plugin for handling 3D structural molecular data (not only) in the browser. (PMID: [30179493](https://pubmed.ncbi.nlm.nih.gov/30179493))

275. [huangnengCSU/compleasm](https://github.com/huangnengCSU/compleasm) (‚≠ê151 ¬∑ Python) A genome completeness evaluation tool based on miniprot (PMID: [37758247](https://pubmed.ncbi.nlm.nih.gov/37758247))

276. [pymzML/pymzML](https://github.com/pymzML/pymzML) (‚≠ê151 ¬∑ Python) pymzML - an interface between Python and mzML Mass spectrometry Files (PMID: [29394323](https://pubmed.ncbi.nlm.nih.gov/29394323))

277. [saezlab/decoupleR](https://github.com/saezlab/decoupleR) (‚≠ê151 ¬∑ R) R package to infer biological activities from omics data using a collection of methods. (PMID: [36699385](https://pubmed.ncbi.nlm.nih.gov/36699385))

278. [bjornwallner/DockQ](https://github.com/bjornwallner/DockQ) (‚≠ê150 ¬∑ Python) DockQ is a single continuous quality measure for protein docked models based on the CAPRI evaluation protocol (PMID: [27560519](https://pubmed.ncbi.nlm.nih.gov/27560519))

279. [marcelauliano/MitoHiFi](https://github.com/marcelauliano/MitoHiFi) (‚≠ê150 ¬∑ Python) Find, circularise and annotate mitogenome from PacBio assemblies (PMID: [37464285](https://pubmed.ncbi.nlm.nih.gov/37464285))

280. [open2c/bioframe](https://github.com/open2c/bioframe) (‚≠ê150 ¬∑ Python) Genomic Interval Operations on Pandas DataFrames (PMID: [38402507](https://pubmed.ncbi.nlm.nih.gov/38402507))

281. [crsl4/PhyloNetworks](https://github.com/crsl4/PhyloNetworks) (‚≠ê149 ¬∑ Julia) A Julia package for statistical inference, data manipulation and visualization of phylogenetic networks (PMID: [28961984](https://pubmed.ncbi.nlm.nih.gov/28961984))

282. [venyao/shinyCircos](https://github.com/venyao/shinyCircos) (‚≠ê148 ¬∑ HTML) an R/shiny application for creation of Circos plot interactively (PMID: [29186362](https://pubmed.ncbi.nlm.nih.gov/29186362))

283. [ComparativeGenomicsToolkit/hal](https://github.com/ComparativeGenomicsToolkit/hal) (‚≠ê147 ¬∑ C++) Hierarchical Alignment Format (PMID: [32463100](https://pubmed.ncbi.nlm.nih.gov/32463100))

284. [KorfLab/StochHMM](https://github.com/KorfLab/StochHMM) (‚≠ê147 ¬∑ C++) A Hidden Markov Model Application and C++ Library for Rapid and Flexible Development of HMMs (PMID: [24489371](https://pubmed.ncbi.nlm.nih.gov/24489371))

285. [glennhickey/hal](https://github.com/glennhickey/hal) (‚≠ê147 ¬∑ C++) Hierarchical Alignment Format (PMID: [23505295](https://pubmed.ncbi.nlm.nih.gov/23505295))

286. [statgen/locuszoom](https://github.com/statgen/locuszoom) (‚≠ê147 ¬∑ JavaScript) A Javascript/d3 embeddable plugin for interactively visualizing statistical genetic data from customizable sources. (PMID: [33734315](https://pubmed.ncbi.nlm.nih.gov/33734315))

287. [divonlan/genozip](https://github.com/divonlan/genozip) (‚≠ê146 ¬∑ C) A modern compressor for genomic files (FASTQ, SAM/BAM/CRAM, VCF, FASTA, GFF/GTF/GVF, 23andMe...), up to 5x better than gzip and faster too (PMID: [32407471](https://pubmed.ncbi.nlm.nih.gov/32407471))

288. [BenevolentAI/DeeplyTough](https://github.com/BenevolentAI/DeeplyTough) (‚≠ê145 ¬∑ Python) DeeplyTough: Learning Structural Comparison of Protein Binding Sites (PMID: [32023053](https://pubmed.ncbi.nlm.nih.gov/32023053))

289. [OpenGene/MutScan](https://github.com/OpenGene/MutScan) (‚≠ê145 ¬∑ C) Detect and visualize target mutations by scanning FastQ files directly (PMID: [29357822](https://pubmed.ncbi.nlm.nih.gov/29357822))

290. [neurodata/neuroparc](https://github.com/neurodata/neuroparc) (‚≠ê145 ¬∑ Jupyter Notebook) None (PMID: [33686079](https://pubmed.ncbi.nlm.nih.gov/33686079))

291. [ZJUDataIntelligence/TEMPO](https://github.com/ZJUDataIntelligence/TEMPO) (‚≠ê144 ¬∑ Python) None (PMID: [36535209](https://pubmed.ncbi.nlm.nih.gov/36535209))

292. [millanek/Dsuite](https://github.com/millanek/Dsuite) (‚≠ê144 ¬∑ C++) Fast calculation of Patterson's D (ABBA-BABA) and the f4-ratio statistics across many populations/species (PMID: [33012121](https://pubmed.ncbi.nlm.nih.gov/33012121))

293. [tsudalab/ChemTS](https://github.com/tsudalab/ChemTS) (‚≠ê144 ¬∑ Python) Molecule Design using Monte Carlo Tree Search with Neural Rollout (PMID: [29435094](https://pubmed.ncbi.nlm.nih.gov/29435094))

294. [mschubert/clustermq](https://github.com/mschubert/clustermq) (‚≠ê143 ¬∑ R) R package to send function calls as jobs on LSF, SGE, Slurm, PBS/Torque, or each via SSH (PMID: [31134271](https://pubmed.ncbi.nlm.nih.gov/31134271))

295. [ekg/seqwish](https://github.com/ekg/seqwish) (‚≠ê141 ¬∑ C++) alignment to variation graph inducer (PMID: [36448683](https://pubmed.ncbi.nlm.nih.gov/36448683))

296. [krishnanlab/PecanPy](https://github.com/krishnanlab/PecanPy) (‚≠ê141 ¬∑ Python) A fast, parallelized, memory efficient, and cache-optimized Python implementation of node2vec (PMID: [33760066](https://pubmed.ncbi.nlm.nih.gov/33760066))

297. [CellProfiler/CellProfiler-Analyst](https://github.com/CellProfiler/CellProfiler-Analyst) (‚≠ê140 ¬∑ Python) Open-source software for exploring and analyzing large, high-dimensional image-derived data. (PMID: [27354701](https://pubmed.ncbi.nlm.nih.gov/27354701); [34478488](https://pubmed.ncbi.nlm.nih.gov/34478488))

298. [LinearFold/LinearFold](https://github.com/LinearFold/LinearFold) (‚≠ê140 ¬∑ C++) The new source code of the LinearFold, linear-time prediction for RNA secondary structures (PMID: [31510672](https://pubmed.ncbi.nlm.nih.gov/31510672))

299. [eldariont/svim](https://github.com/eldariont/svim) (‚≠ê140 ¬∑ Python) Structural Variant Identification Method using Long Reads (PMID: [30668829](https://pubmed.ncbi.nlm.nih.gov/30668829))

300. [fenderglass/Ragout](https://github.com/fenderglass/Ragout) (‚≠ê140 ¬∑ Python) Chromosome-level scaffolding using multiple references (PMID: [24931998](https://pubmed.ncbi.nlm.nih.gov/24931998))

301. [ncbi/icn3d](https://github.com/ncbi/icn3d) (‚≠ê140 ¬∑ JavaScript) web-based protein structure viewer and analysis tool interactively or in batch mode (PMID: [31218344](https://pubmed.ncbi.nlm.nih.gov/31218344); [35252351](https://pubmed.ncbi.nlm.nih.gov/35252351))

302. [molML/MoleculeACE](https://github.com/molML/MoleculeACE) (‚≠ê139 ¬∑ Python) A tool for evaluating the predictive performance on activity cliff compounds of machine learning models (PMID: [36456532](https://pubmed.ncbi.nlm.nih.gov/36456532))

303. [sequana/sequana](https://github.com/sequana/sequana) (‚≠ê139 ¬∑ Jupyter Notebook) Sequana: a set of Snakemake NGS pipelines (PMID: [29361152](https://pubmed.ncbi.nlm.nih.gov/29361152))

304. [theislab/kBET](https://github.com/theislab/kBET) (‚≠ê139 ¬∑ HTML) An R package to test for batch effects in high-dimensional single-cell RNA sequencing data. (PMID: [30573817](https://pubmed.ncbi.nlm.nih.gov/30573817))

305. [getzlab/rnaseqc](https://github.com/getzlab/rnaseqc) (‚≠ê138 ¬∑ C++) Fast, efficient RNA-Seq metrics for quality control and process optimization (PMID: [33677499](https://pubmed.ncbi.nlm.nih.gov/33677499))

306. [jotech/gapseq](https://github.com/jotech/gapseq) (‚≠ê138 ¬∑ R) Informed prediction and analysis of bacterial metabolic pathways and genome-scale networks (PMID: [33691770](https://pubmed.ncbi.nlm.nih.gov/33691770))

307. [mlyg/unified-focal-loss](https://github.com/mlyg/unified-focal-loss) (‚≠ê138 ¬∑ Python) None (PMID: [34953431](https://pubmed.ncbi.nlm.nih.gov/34953431))

308. [stemangiola/tidyseurat](https://github.com/stemangiola/tidyseurat) (‚≠ê138 ¬∑ R) Seurat meets tidyverse. The best of both worlds. (PMID: [34028547](https://pubmed.ncbi.nlm.nih.gov/34028547))

309. [AdmiralenOla/Scoary](https://github.com/AdmiralenOla/Scoary) (‚≠ê137 ¬∑ Python) Pan-genome wide association studies (PMID: [27887642](https://pubmed.ncbi.nlm.nih.gov/27887642))

310. [lsyads/fisheye-ORB-SLAM](https://github.com/lsyads/fisheye-ORB-SLAM) (‚≠ê137 ¬∑ C++) A real-time robust monocular visual SLAM system based on ORB-SLAM for fisheye cameras, without rectifying or cropping the input images (PMID: [31623266](https://pubmed.ncbi.nlm.nih.gov/31623266))

311. [ncclabsustech/EEGdenoiseNet](https://github.com/ncclabsustech/EEGdenoiseNet) (‚≠ê137 ¬∑ Jupyter Notebook) EEGdenoiseNet, a benchmark dataset, that is suited for training and testing deep learning-based EEG denoising models, as well as for comparing the performance across different models. (PMID: [34596046](https://pubmed.ncbi.nlm.nih.gov/34596046))

312. [CBICA/GaNDLF](https://github.com/CBICA/GaNDLF) (‚≠ê136 ¬∑ Python) A generalizable application framework for segmentation, regression, and classification using PyTorch (PMID: [35465060](https://pubmed.ncbi.nlm.nih.gov/35465060))

313. [smarco/WFA](https://github.com/smarco/WFA) (‚≠ê136 ¬∑ C) WFA-lib: Wavefront alignment algorithm library v2 (PMID: [32915952](https://pubmed.ncbi.nlm.nih.gov/32915952))

314. [OpenBioLink/OpenBioLink](https://github.com/OpenBioLink/OpenBioLink) (‚≠ê135 ¬∑ Python) OpenBioLink is a resource and evaluation framework for evaluating link prediction models on heterogeneous biomedical graph data. (PMID: [32339214](https://pubmed.ncbi.nlm.nih.gov/32339214))

315. [benjjneb/decontam](https://github.com/benjjneb/decontam) (‚≠ê135 ¬∑ R) Simple statistical identification and removal of contaminants in marker-gene and metagenomics sequencing data (PMID: [30558668](https://pubmed.ncbi.nlm.nih.gov/30558668))

316. [Ensembl/ensembl-rest](https://github.com/Ensembl/ensembl-rest) (‚≠ê134 ¬∑ Perl) Language agnostic RESTful data access to Ensembl data over HTTP (PMID: [25236461](https://pubmed.ncbi.nlm.nih.gov/25236461))

317. [masqm/Faster-Mean-Shift](https://github.com/masqm/Faster-Mean-Shift) (‚≠ê134 ¬∑ Python) GPU accelerated Faster Mean-shift for cosine embedding clustering (PMID: [33872961](https://pubmed.ncbi.nlm.nih.gov/33872961))

318. [Teichlab/bbknn](https://github.com/Teichlab/bbknn) (‚≠ê133 ¬∑ Python) Batch balanced KNN (PMID: [31400197](https://pubmed.ncbi.nlm.nih.gov/31400197))

319. [deepmodeling/DMFF](https://github.com/deepmodeling/DMFF) (‚≠ê133 ¬∑ Python) DMFF (Differentiable Molecular Force Field) is a Jax-based python package that provides a full differentiable implementation of molecular force field models. (PMID: [37589304](https://pubmed.ncbi.nlm.nih.gov/37589304))

320. [healthylaife/MIMIC-IV-Data-Pipeline](https://github.com/healthylaife/MIMIC-IV-Data-Pipeline) (‚≠ê133 ¬∑ Jupyter Notebook) A customizable pipeline for data extraction from MIMIC-IV. (PMID: [36686986](https://pubmed.ncbi.nlm.nih.gov/36686986))

321. [ryanlayer/gqt](https://github.com/ryanlayer/gqt) (‚≠ê133 ¬∑ C) A genotype query interface. (PMID: [26550772](https://pubmed.ncbi.nlm.nih.gov/26550772))

322. [CapsuleEndoscope/VirtualCapsuleEndoscopy](https://github.com/CapsuleEndoscope/VirtualCapsuleEndoscopy) (‚≠ê132 ¬∑ C#) VR-Caps: A Virtual Environment for Active Capsule Endoscopy (PMID: [33609920](https://pubmed.ncbi.nlm.nih.gov/33609920))

323. [nadeemlab/DeepLIIF](https://github.com/nadeemlab/DeepLIIF) (‚≠ê132 ¬∑ Python) Deep Learning Inferred Multiplex ImmunoFluorescence for IHC Image Quantification (https://deepliif.org) [Nature Machine Intelligence'22, CVPR'22, MICCAI'23, Histopathology'23] (PMID: [37292462](https://pubmed.ncbi.nlm.nih.gov/37292462); [37841230](https://pubmed.ncbi.nlm.nih.gov/37841230))

324. [nadeemlab/deepliif](https://github.com/nadeemlab/deepliif) (‚≠ê132 ¬∑ Python) Deep Learning Inferred Multiplex ImmunoFluorescence for IHC Image Quantification (https://deepliif.org) [Nature Machine Intelligence'22, CVPR'22, MICCAI'23, Histopathology'23] (PMID: [36118303](https://pubmed.ncbi.nlm.nih.gov/36118303))

325. [royfrancis/pophelper](https://github.com/royfrancis/pophelper) (‚≠ê132 ¬∑ R) An R package to analyse and visualise admixture proportions from STRUCTURE, fastSTRUCTURE, TESS, ADMIXTURE etc. (PMID: [26850166](https://pubmed.ncbi.nlm.nih.gov/26850166))

326. [walaj/SeqLib](https://github.com/walaj/SeqLib) (‚≠ê132 ¬∑ C++) C++ htslib/bwa-mem/fermi interface for interrogating sequence data (PMID: [28011768](https://pubmed.ncbi.nlm.nih.gov/28011768))

327. [SheffieldML/GPmat](https://github.com/SheffieldML/GPmat) (‚≠ê131 ¬∑ Matlab) Matlab implementations of Gaussian processes and other machine learning tools. (PMID: [24618470](https://pubmed.ncbi.nlm.nih.gov/24618470))

328. [aertslab/cistopic](https://github.com/aertslab/cistopic) (‚≠ê131 ¬∑ R) cisTopic: Probabilistic modelling of cis-regulatory topics from single cell epigenomics data (PMID: [30962623](https://pubmed.ncbi.nlm.nih.gov/30962623))

329. [hewm2008/NGenomeSyn](https://github.com/hewm2008/NGenomeSyn) (‚≠ê130 ¬∑ Perl) Any Way to Show  Multi genomic Synteny (PMID: [36883694](https://pubmed.ncbi.nlm.nih.gov/36883694))

330. [Bioconductor/Contributions](https://github.com/Bioconductor/Contributions) (‚≠ê129 ¬∑ None) Contribute Packages to Bioconductor (PMID: [38407991](https://pubmed.ncbi.nlm.nih.gov/38407991))

331. [marianattestad/assemblytics](https://github.com/marianattestad/assemblytics) (‚≠ê129 ¬∑ JavaScript) Assemblytics is a bioinformatics tool to detect and analyze structural variants from a genome assembly by comparing it to a reference genome. (PMID: [27318204](https://pubmed.ncbi.nlm.nih.gov/27318204))

332. [stefpeschel/NetCoMi](https://github.com/stefpeschel/NetCoMi) (‚≠ê129 ¬∑ R) Network construction, analysis, and comparison for microbial compositional data (PMID: [33264391](https://pubmed.ncbi.nlm.nih.gov/33264391))

333. [theislab/scCODA](https://github.com/theislab/scCODA) (‚≠ê129 ¬∑ Jupyter Notebook) A Bayesian model for compositional single-cell data analysis (PMID: [34824236](https://pubmed.ncbi.nlm.nih.gov/34824236))

334. [galaxyproject/SARS-CoV-2](https://github.com/galaxyproject/SARS-CoV-2) (‚≠ê128 ¬∑ Jupyter Notebook) Ongoing analysis of COVID-19 using Galaxy, BioConda and public research infrastructures (PMID: [32790776](https://pubmed.ncbi.nlm.nih.gov/32790776))

335. [hakyimlab/MetaXcan](https://github.com/hakyimlab/MetaXcan) (‚≠ê128 ¬∑ Python) MetaXcan software and manuscript (PMID: [29752479](https://pubmed.ncbi.nlm.nih.gov/29752479))

336. [lanl001/halc](https://github.com/lanl001/halc) (‚≠ê128 ¬∑ C++) High Throughput Algorithm for Long Read Error Correction (PMID: [28381259](https://pubmed.ncbi.nlm.nih.gov/28381259))

337. [ATOMconsortium/AMPL](https://github.com/ATOMconsortium/AMPL) (‚≠ê127 ¬∑ Jupyter Notebook) The ATOM Modeling PipeLine (AMPL) is an open-source, modular, extensible software pipeline for building and sharing models to advance in silico drug discovery. (PMID: [32243153](https://pubmed.ncbi.nlm.nih.gov/32243153))

338. [brentp/echtvar](https://github.com/brentp/echtvar) (‚≠ê127 ¬∑ Rust) using all the bits for echt rapid variant annotation and filtering (PMID: [36300617](https://pubmed.ncbi.nlm.nih.gov/36300617))

339. [churchmanlab/genewalk](https://github.com/churchmanlab/genewalk) (‚≠ê127 ¬∑ Python) GeneWalk identifies relevant gene functions for a biological context using network representation learning (PMID: [33526072](https://pubmed.ncbi.nlm.nih.gov/33526072))

340. [gabraham/flashpca](https://github.com/gabraham/flashpca) (‚≠ê127 ¬∑ C++) Fast Principal Component Analysis of Large-Scale Genome-Wide Data (PMID: [28475694](https://pubmed.ncbi.nlm.nih.gov/28475694))

341. [gbouras13/pharokka](https://github.com/gbouras13/pharokka) (‚≠ê127 ¬∑ Python) fast phage annotation program (PMID: [36453861](https://pubmed.ncbi.nlm.nih.gov/36453861))

342. [hasindu2008/f5c](https://github.com/hasindu2008/f5c) (‚≠ê127 ¬∑ C) Ultra-fast methylation calling and event alignment tool for nanopore sequencing data (supports CUDA acceleration) (PMID: [32758139](https://pubmed.ncbi.nlm.nih.gov/32758139))

343. [imgag/ngs-bits](https://github.com/imgag/ngs-bits) (‚≠ê127 ¬∑ C++) Short-read sequencing tools (PMID: [27161244](https://pubmed.ncbi.nlm.nih.gov/27161244); [28130233](https://pubmed.ncbi.nlm.nih.gov/28130233))

344. [lh3/pangene](https://github.com/lh3/pangene) (‚≠ê127 ¬∑ C) Constructing a pangenome gene graph (PMID: [38463499](https://pubmed.ncbi.nlm.nih.gov/38463499))

345. [xlucpu/MOVICS](https://github.com/xlucpu/MOVICS) (‚≠ê127 ¬∑ R) Multi-Omics integration and VIsualization in Cancer Subtyping (PMID: [33315104](https://pubmed.ncbi.nlm.nih.gov/33315104))

346. [SGDDNB/ShinyCell](https://github.com/SGDDNB/ShinyCell) (‚≠ê126 ¬∑ R) Shiny Interactive Web Apps for Single-Cell Data (PMID: [33774659](https://pubmed.ncbi.nlm.nih.gov/33774659))

347. [antigenomics/vdjdb-db](https://github.com/antigenomics/vdjdb-db) (‚≠ê126 ¬∑ Groovy) üóÇÔ∏è [vdjdb.cdr3.net is up and running] Git-based TCR database storage & management. Submissions welcome! (PMID: [28977646](https://pubmed.ncbi.nlm.nih.gov/28977646))

348. [mikessh/vdjtools](https://github.com/mikessh/vdjtools) (‚≠ê126 ¬∑ Groovy) Post-analysis of immune repertoire sequencing data (PMID: [26606115](https://pubmed.ncbi.nlm.nih.gov/26606115))

349. [saezlab/decoupler-py](https://github.com/saezlab/decoupler-py) (‚≠ê126 ¬∑ Python) Python package to perform enrichment analysis from omics data. (PMID: [36699385](https://pubmed.ncbi.nlm.nih.gov/36699385))

350. [vals/umis](https://github.com/vals/umis) (‚≠ê126 ¬∑ Python) Tools for processing UMI RNA-tag data (PMID: [28263961](https://pubmed.ncbi.nlm.nih.gov/28263961))

351. [zhanxw/rvtests](https://github.com/zhanxw/rvtests) (‚≠ê125 ¬∑ C++) Rare variant test software for next generation sequencing data (PMID: [27153000](https://pubmed.ncbi.nlm.nih.gov/27153000))

352. [cruizperez/MicrobeAnnotator](https://github.com/cruizperez/MicrobeAnnotator) (‚≠ê124 ¬∑ Python) Pipeline for metabolic annotation of microbial genomes (PMID: [33407081](https://pubmed.ncbi.nlm.nih.gov/33407081))

353. [theislab/sc-pert](https://github.com/theislab/sc-pert) (‚≠ê124 ¬∑ Jupyter Notebook) Models and datasets for perturbational single-cell omics (PMID: [34139164](https://pubmed.ncbi.nlm.nih.gov/34139164))

354. [BBillot/SynthSR](https://github.com/BBillot/SynthSR) (‚≠ê123 ¬∑ Python) A framework for joint super-resolution and image synthesis, without requiring real training data (PMID: [34048902](https://pubmed.ncbi.nlm.nih.gov/34048902))

355. [ksahlin/strobealign](https://github.com/ksahlin/strobealign) (‚≠ê123 ¬∑ C++) Aligns short reads using dynamic seed size with strobemers (PMID: [36522758](https://pubmed.ncbi.nlm.nih.gov/36522758))

356. [lh3/bedtk](https://github.com/lh3/bedtk) (‚≠ê123 ¬∑ C) A simple toolset for BED files (warning: CLI may change before bedtk becomes stable) (PMID: [32966548](https://pubmed.ncbi.nlm.nih.gov/32966548))

357. [Merck/BioPhi](https://github.com/Merck/BioPhi) (‚≠ê122 ¬∑ Python) BioPhi is an open-source antibody design platform. It features methods for automated antibody humanization (Sapiens), humanness evaluation (OASis) and an interface for computer-assisted antibody sequence design. (PMID: [35133949](https://pubmed.ncbi.nlm.nih.gov/35133949))

358. [bigdatagenomics/mango](https://github.com/bigdatagenomics/mango) (‚≠ê122 ¬∑ Scala) A scalable genome browser. Apache 2 licensed. (PMID: [29308454](https://pubmed.ncbi.nlm.nih.gov/29308454))

359. [ncbi/TPMCalculator](https://github.com/ncbi/TPMCalculator) (‚≠ê122 ¬∑ C++) TPMCalculator quantifies mRNA abundance directly from the alignments by parsing BAM files (PMID: [30379987](https://pubmed.ncbi.nlm.nih.gov/30379987))

360. [rrwick/Deepbinner](https://github.com/rrwick/Deepbinner) (‚≠ê122 ¬∑ Python) a signal-level demultiplexer for Oxford Nanopore reads (PMID: [30458005](https://pubmed.ncbi.nlm.nih.gov/30458005))

361. [BlueBrain/NeuroMorphoVis](https://github.com/BlueBrain/NeuroMorphoVis) (‚≠ê121 ¬∑ Python) A lightweight, interactive, extensible and cross-platform framework for building, visualizing and analyzing digital reconstructions of neuronal morphology skeletons extracted from microscopy stacks. (PMID: [29949998](https://pubmed.ncbi.nlm.nih.gov/29949998))

362. [Gabaldonlab/redundans](https://github.com/Gabaldonlab/redundans) (‚≠ê121 ¬∑ C++) Redundans is a pipeline that assists an assembly of heterozygous/polymorphic genomes. (PMID: [27131372](https://pubmed.ncbi.nlm.nih.gov/27131372))

363. [xulabs/aitom](https://github.com/xulabs/aitom) (‚≠ê121 ¬∑ Python) AI for tomography (PMID: [32722755](https://pubmed.ncbi.nlm.nih.gov/32722755); [33620460](https://pubmed.ncbi.nlm.nih.gov/33620460); [34897387](https://pubmed.ncbi.nlm.nih.gov/34897387))

364. [echen102/us-pres-elections-2020](https://github.com/echen102/us-pres-elections-2020) (‚≠ê120 ¬∑ Python) The repository contains a collection of tweets IDs associated with the 2020 U.S. Presidential Elections through 6 months post-inauguration. (PMID: [33824934](https://pubmed.ncbi.nlm.nih.gov/33824934))

365. [microsoft/frame-flow](https://github.com/microsoft/frame-flow) (‚≠ê120 ¬∑ Python) Fast protein backbone generation with SE(3) flow matching. (PMID: [38259348](https://pubmed.ncbi.nlm.nih.gov/38259348))

366. [muhammedtalo/COVID-19](https://github.com/muhammedtalo/COVID-19) (‚≠ê120 ¬∑ Jupyter Notebook) Automated Detection of COVID-19 Cases Using Deep Neural Networks with X-Ray Images (PMID: [32568675](https://pubmed.ncbi.nlm.nih.gov/32568675))

367. [songc001/remilo](https://github.com/songc001/remilo) (‚≠ê120 ¬∑ C++) Reference Assisted Misassembly Detection Algorithm  Using Short and Long Reads (PMID: [28961789](https://pubmed.ncbi.nlm.nih.gov/28961789))

368. [spetti/SMURF](https://github.com/spetti/SMURF) (‚≠ê120 ¬∑ Python) None (PMID: [36355460](https://pubmed.ncbi.nlm.nih.gov/36355460))

369. [Illumina/canvas](https://github.com/Illumina/canvas) (‚≠ê119 ¬∑ C#) Canvas - Copy number variant (CNV) calling from DNA sequencing data (PMID: [27153601](https://pubmed.ncbi.nlm.nih.gov/27153601); [29028893](https://pubmed.ncbi.nlm.nih.gov/29028893))

370. [OpenGene/repaq](https://github.com/OpenGene/repaq) (‚≠ê119 ¬∑ C) A fast lossless FASTQ compressor with ultra-high compression ratio (PMID: [37811144](https://pubmed.ncbi.nlm.nih.gov/37811144))

371. [PoisonAlien/trackplot](https://github.com/PoisonAlien/trackplot) (‚≠ê118 ¬∑ R) Generate IGV style locus tracks from bigWig files in R (PMID: [38476298](https://pubmed.ncbi.nlm.nih.gov/38476298))

372. [baoe/flas](https://github.com/baoe/flas) (‚≠ê118 ¬∑ C++) Fast and High Throughput Algorithm for PacBio Long Read Self-Correction (PMID: [30895306](https://pubmed.ncbi.nlm.nih.gov/30895306))

373. [parklab/NGSCheckMate](https://github.com/parklab/NGSCheckMate) (‚≠ê118 ¬∑ Python) Software program for checking sample matching for NGS data (PMID: [28369524](https://pubmed.ncbi.nlm.nih.gov/28369524))

374. [torognes/swarm](https://github.com/torognes/swarm) (‚≠ê117 ¬∑ C++) A robust and fast clustering method for amplicon-based studies (PMID: [34244702](https://pubmed.ncbi.nlm.nih.gov/34244702))

375. [yatisht/usher](https://github.com/yatisht/usher) (‚≠ê117 ¬∑ C++) Ultrafast Sample Placement on Existing Trees (PMID: [34469548](https://pubmed.ncbi.nlm.nih.gov/34469548); [35731204](https://pubmed.ncbi.nlm.nih.gov/35731204))

376. [AndersenLab/VCF-kit](https://github.com/AndersenLab/VCF-kit) (‚≠ê116 ¬∑ Python) VCF-kit: Assorted utilities for the variant call format (PMID: [28093408](https://pubmed.ncbi.nlm.nih.gov/28093408))

377. [JiekaiLab/scDIOR](https://github.com/JiekaiLab/scDIOR) (‚≠ê116 ¬∑ HTML) scDIOR: Single cell data IO softwaRe (PMID: [34991457](https://pubmed.ncbi.nlm.nih.gov/34991457))

378. [MRCIEU/PHESANT](https://github.com/MRCIEU/PHESANT) (‚≠ê116 ¬∑ R) PHESANT - PHEnome Scan ANalysis Tool (pheWAS, Mendelian randomisation (MR)-pheWAS etc.) in UK Biobank (PMID: [29040602](https://pubmed.ncbi.nlm.nih.gov/29040602))

379. [dzhang32/ggtranscript](https://github.com/dzhang32/ggtranscript) (‚≠ê116 ¬∑ R) Visualizing transcript structure and annotation using ggplot2 (PMID: [35751589](https://pubmed.ncbi.nlm.nih.gov/35751589))

380. [jdidion/atropos](https://github.com/jdidion/atropos) (‚≠ê116 ¬∑ Python) An NGS read trimming tool that is specific, sensitive, and speedy. (production) (PMID: [28875074](https://pubmed.ncbi.nlm.nih.gov/28875074))

381. [DiltheyLab/HLA-LA](https://github.com/DiltheyLab/HLA-LA) (‚≠ê115 ¬∑ C++) Fast HLA type inference from whole-genome data (PMID: [30942877](https://pubmed.ncbi.nlm.nih.gov/30942877))

382. [biojava/biojava-tutorial](https://github.com/biojava/biojava-tutorial) (‚≠ê114 ¬∑ Python) :book::books:An introductory tutorial for BioJava (PMID: [30735498](https://pubmed.ncbi.nlm.nih.gov/30735498))

383. [spacegraphcats/spacegraphcats](https://github.com/spacegraphcats/spacegraphcats) (‚≠ê114 ¬∑ Standard ML) Indexing & querying large assembly graphs -- in space, no one can hear you miao! (PMID: [32631445](https://pubmed.ncbi.nlm.nih.gov/32631445))

384. [wanderine/BROCCOLI](https://github.com/wanderine/BROCCOLI) (‚≠ê114 ¬∑ C++) BROCCOLI: Software for Fast fMRI Analysis on Many-Core CPUs and GPUs (PMID: [24672471](https://pubmed.ncbi.nlm.nih.gov/24672471))

385. [OpenGene/GeneFuse](https://github.com/OpenGene/GeneFuse) (‚≠ê113 ¬∑ C) Gene fusion detection and visualization (PMID: [29989075](https://pubmed.ncbi.nlm.nih.gov/29989075))

386. [XiaoleiLiuBio/G2P](https://github.com/XiaoleiLiuBio/G2P) (‚≠ê113 ¬∑ None) G2P: A Genome-Wide-Association-Study Simulation Tool for Genotype Simulation, Phenotype Simulation, and Power Evaluation (PMID: [30848784](https://pubmed.ncbi.nlm.nih.gov/30848784))

387. [hammerlab/cytokit](https://github.com/hammerlab/cytokit) (‚≠ê113 ¬∑ Jupyter Notebook) Microscopy Image Cytometry Toolkit (PMID: [31477013](https://pubmed.ncbi.nlm.nih.gov/31477013))

388. [oxpig/DeLinker](https://github.com/oxpig/DeLinker) (‚≠ê113 ¬∑ Python) None (PMID: [32195587](https://pubmed.ncbi.nlm.nih.gov/32195587))

389. [neuropoly/axondeepseg](https://github.com/neuropoly/axondeepseg) (‚≠ê112 ¬∑ Python) Axon/Myelin segmentation using Deep Learning (PMID: [29491478](https://pubmed.ncbi.nlm.nih.gov/29491478))

390. [thomas0809/MolScribe](https://github.com/thomas0809/MolScribe) (‚≠ê112 ¬∑ Python) Robust Molecular Structure Recognition with Image-to-Graph Generation (PMID: [36971363](https://pubmed.ncbi.nlm.nih.gov/36971363))

391. [ChaissonLab/LRA](https://github.com/ChaissonLab/LRA) (‚≠ê111 ¬∑ C++) Long read aligner (PMID: [34153026](https://pubmed.ncbi.nlm.nih.gov/34153026))

392. [DeepRank/DeepRank-GNN](https://github.com/DeepRank/DeepRank-GNN) (‚≠ê111 ¬∑ Python) Graph Network for protein-protein interface (PMID: [36420989](https://pubmed.ncbi.nlm.nih.gov/36420989))

393. [timolassmann/kalign](https://github.com/timolassmann/kalign) (‚≠ê111 ¬∑ C) A fast multiple sequence alignment program. (PMID: [31665271](https://pubmed.ncbi.nlm.nih.gov/31665271))

394. [guigolab/ggsashimi](https://github.com/guigolab/ggsashimi) (‚≠ê110 ¬∑ Python) Command-line tool for the visualization of splicing events across multiple samples (PMID: [30118475](https://pubmed.ncbi.nlm.nih.gov/30118475))

395. [jameshadfield/phandango](https://github.com/jameshadfield/phandango) (‚≠ê110 ¬∑ JavaScript) an interactive viewer for populations of bacterial genomes linked by a phylogeny (PMID: [29028899](https://pubmed.ncbi.nlm.nih.gov/29028899))

396. [mcveanlab/mccortex](https://github.com/mcveanlab/mccortex) (‚≠ê110 ¬∑ C) De novo genome assembly and multisample variant calling (PMID: [29554215](https://pubmed.ncbi.nlm.nih.gov/29554215))

397. [OpenGene/fastv](https://github.com/OpenGene/fastv) (‚≠ê109 ¬∑ C++) An ultra-fast tool for identification of SARS-CoV-2 and other microbes from sequencing data. This tool can be used to detect viral infectious diseases, like COVID-19. (PMID: [33003197](https://pubmed.ncbi.nlm.nih.gov/33003197))

398. [OpenGene/gencore](https://github.com/OpenGene/gencore) (‚≠ê109 ¬∑ C++) Generate duplex/single consensus reads to reduce sequencing noises and remove duplications (PMID: [31881822](https://pubmed.ncbi.nlm.nih.gov/31881822))

399. [biocore/mmvec](https://github.com/biocore/mmvec) (‚≠ê109 ¬∑ Python) Neural networks for microbe-metabolite interaction analysis (PMID: [31686038](https://pubmed.ncbi.nlm.nih.gov/31686038))

400. [carmonalab/UCell](https://github.com/carmonalab/UCell) (‚≠ê109 ¬∑ R) An R package for single-cell gene signature scoring (PMID: [34285779](https://pubmed.ncbi.nlm.nih.gov/34285779))

401. [igvteam/igv-webapp](https://github.com/igvteam/igv-webapp) (‚≠ê109 ¬∑ JavaScript) IGV Web App (PMID: [36562559](https://pubmed.ncbi.nlm.nih.gov/36562559))

402. [lingluodlut/Att-ChemdNER](https://github.com/lingluodlut/Att-ChemdNER) (‚≠ê109 ¬∑ Python) Att-ChemdNER (PMID: [29186323](https://pubmed.ncbi.nlm.nih.gov/29186323))

403. [linzhi2013/MitoZ](https://github.com/linzhi2013/MitoZ) (‚≠ê109 ¬∑ Shell) MitoZ: A toolkit for assembly, annotation, and visualization of animal mitochondrial genomes (PMID: [30864657](https://pubmed.ncbi.nlm.nih.gov/30864657))

404. [lykaust15/DeepSimulator](https://github.com/lykaust15/DeepSimulator) (‚≠ê109 ¬∑ C) The first deep learning based Nanopore simulator which can simulate the process of Nanopore sequencing. (PMID: [29659695](https://pubmed.ncbi.nlm.nih.gov/29659695); [31913436](https://pubmed.ncbi.nlm.nih.gov/31913436))

405. [PheWAS/PheWAS](https://github.com/PheWAS/PheWAS) (‚≠ê108 ¬∑ R) The PheWAS R package (PMID: [29912272](https://pubmed.ncbi.nlm.nih.gov/29912272))

406. [juglab/EmbedSeg](https://github.com/juglab/EmbedSeg) (‚≠ê108 ¬∑ Python) Code Implementation for EmbedSeg, an Instance Segmentation Method for Microscopy Images (PMID: [35926335](https://pubmed.ncbi.nlm.nih.gov/35926335))

407. [lh3/fermikit](https://github.com/lh3/fermikit) (‚≠ê108 ¬∑ TeX) De novo assembly based variant calling pipeline for Illumina short reads (PMID: [26220959](https://pubmed.ncbi.nlm.nih.gov/26220959))

408. [molgenis/molgenis](https://github.com/molgenis/molgenis) (‚≠ê108 ¬∑ Java) MOLGENIS - for scientific data: management, exploration, integration and analysis. (PMID: [27153686](https://pubmed.ncbi.nlm.nih.gov/27153686); [29036577](https://pubmed.ncbi.nlm.nih.gov/29036577); [29599418](https://pubmed.ncbi.nlm.nih.gov/29599418))

409. [OpenTreeOfLife/opentree](https://github.com/OpenTreeOfLife/opentree) (‚≠ê107 ¬∑ JavaScript) Opentree browsing and curation web site.  For overarching or cross-repo concerns, please see the 'germinator' repo. (PMID: [25940563](https://pubmed.ncbi.nlm.nih.gov/25940563))

410. [Xinglab/DARTS](https://github.com/Xinglab/DARTS) (‚≠ê107 ¬∑ Python) Deep-learning Augmented RNA-seq analysis of Transcript Splicing (PMID: [30923373](https://pubmed.ncbi.nlm.nih.gov/30923373))

411. [althonos/pyhmmer](https://github.com/althonos/pyhmmer) (‚≠ê107 ¬∑ Cython) Cython bindings and Python interface to HMMER3. (PMID: [37074928](https://pubmed.ncbi.nlm.nih.gov/37074928))

412. [evolbioinfo/gotree](https://github.com/evolbioinfo/gotree) (‚≠ê107 ¬∑ Go) Gotree is a set of command line tools and an API to manipulate phylogenetic trees. It is implemented in Go language. (PMID: [34396097](https://pubmed.ncbi.nlm.nih.gov/34396097))

413. [mikevoets/jama16-retina-replication](https://github.com/mikevoets/jama16-retina-replication) (‚≠ê107 ¬∑ Python) JAMA 2016; 316(22) Replication Study (PMID: [31170223](https://pubmed.ncbi.nlm.nih.gov/31170223))

414. [shubhamchandak94/SPRING](https://github.com/shubhamchandak94/SPRING) (‚≠ê107 ¬∑ C) FASTQ compression (PMID: [30535063](https://pubmed.ncbi.nlm.nih.gov/30535063))

415. [ycwu1997/MC-Net](https://github.com/ycwu1997/MC-Net) (‚≠ê107 ¬∑ Python) Official Code for our MedIA paper "Mutual Consistency Learning for Semi-supervised Medical Image Segmentation" (PMID: [35839737](https://pubmed.ncbi.nlm.nih.gov/35839737))

416. [ChristofferFlensburg/SuperFreq](https://github.com/ChristofferFlensburg/SuperFreq) (‚≠ê106 ¬∑ R) Analysis pipeline for cancer sequencing data (PMID: [34132781](https://pubmed.ncbi.nlm.nih.gov/34132781))

417. [YaqiangCao/cLoops](https://github.com/YaqiangCao/cLoops) (‚≠ê106 ¬∑ Python) Accurate and flexible loops calling tool for 3D genomic data. (PMID: [31504161](https://pubmed.ncbi.nlm.nih.gov/31504161))

418. [bioinfomaticsCSU/deepsignal](https://github.com/bioinfomaticsCSU/deepsignal) (‚≠ê106 ¬∑ Python) Detecting methylation using signal-level features from Nanopore sequencing reads (PMID: [30994904](https://pubmed.ncbi.nlm.nih.gov/30994904))

419. [cansyl/DEEPscreen](https://github.com/cansyl/DEEPscreen) (‚≠ê106 ¬∑ Python) DEEPScreen: Virtual Screening with Deep Convolutional Neural Networks Using Compound Images (PMID: [33209251](https://pubmed.ncbi.nlm.nih.gov/33209251))

420. [cozygene/FEAST](https://github.com/cozygene/FEAST) (‚≠ê106 ¬∑ R) Fast expectation maximization for microbial source tracking (PMID: [31182859](https://pubmed.ncbi.nlm.nih.gov/31182859))

421. [hellloxiaotian/ADNet](https://github.com/hellloxiaotian/ADNet) (‚≠ê106 ¬∑ Python) Attention-guided CNN for image denoising(Neural Networks,2020) (PMID: [31991307](https://pubmed.ncbi.nlm.nih.gov/31991307))

422. [lh3/minimap](https://github.com/lh3/minimap) (‚≠ê106 ¬∑ C) This repo is DEPRECATED. Please use minimap2, the successor of minimap. (PMID: [27153593](https://pubmed.ncbi.nlm.nih.gov/27153593))

423. [morrislab/phylowgs](https://github.com/morrislab/phylowgs) (‚≠ê106 ¬∑ Python) Application for inferring subclonal composition and evolution from whole-genome sequencing data. (PMID: [25786235](https://pubmed.ncbi.nlm.nih.gov/25786235))

424. [vanessajurtz/lasagne4bio](https://github.com/vanessajurtz/lasagne4bio) (‚≠ê106 ¬∑ Jupyter Notebook) None (PMID: [28961695](https://pubmed.ncbi.nlm.nih.gov/28961695))

425. [xiaochuanle/MECAT](https://github.com/xiaochuanle/MECAT) (‚≠ê106 ¬∑ C++) MECAT: an ultra-fast mapping, error correction and de novo assembly tool for single-molecule sequencing reads (PMID: [28945707](https://pubmed.ncbi.nlm.nih.gov/28945707))

426. [broadinstitute/ml4h](https://github.com/broadinstitute/ml4h) (‚≠ê105 ¬∑ Jupyter Notebook) None (PMID: [35157695](https://pubmed.ncbi.nlm.nih.gov/35157695))

427. [labsyspharm/ashlar](https://github.com/labsyspharm/ashlar) (‚≠ê105 ¬∑ Python) ASHLAR: Alignment by Simultaneous Harmonization of Layer/Adjacency Registration (PMID: [35972352](https://pubmed.ncbi.nlm.nih.gov/35972352))

428. [parklab/bamsnap](https://github.com/parklab/bamsnap) (‚≠ê105 ¬∑ HTML) None (PMID: [33416869](https://pubmed.ncbi.nlm.nih.gov/33416869))

429. [thiviyanT/torch-rgcn](https://github.com/thiviyanT/torch-rgcn) (‚≠ê105 ¬∑ Python) A PyTorch implementation of the Relational Graph Convolutional Network (RGCN). (PMID: [36426239](https://pubmed.ncbi.nlm.nih.gov/36426239))

430. [whitead/molbloom](https://github.com/whitead/molbloom) (‚≠ê105 ¬∑ Python) Molecular bloom filter tool (PMID: [37828615](https://pubmed.ncbi.nlm.nih.gov/37828615))

431. [MASILab/SLANTbrainSeg](https://github.com/MASILab/SLANTbrainSeg) (‚≠ê104 ¬∑ Python) Deep Whole Brain Segmentation Using SLANT Method (PMID: [30910724](https://pubmed.ncbi.nlm.nih.gov/30910724); [34999162](https://pubmed.ncbi.nlm.nih.gov/34999162))

432. [RabadanLab/arcasHLA](https://github.com/RabadanLab/arcasHLA) (‚≠ê104 ¬∑ Python) Fast and accurate in silico inference of HLA genotypes from RNA-seq (PMID: [31173059](https://pubmed.ncbi.nlm.nih.gov/31173059))

433. [bvieth/powsimR](https://github.com/bvieth/powsimR) (‚≠ê104 ¬∑ HTML) Power analysis is essential to optimize the design of RNA-seq experiments and to assess and compare the power to detect differentially expressed genes. PowsimR is a flexible tool to simulate and evaluate differential expression from bulk and especially single-cell RNA-seq data making it suitable for a priori and posterior power analyses. (PMID: [29036287](https://pubmed.ncbi.nlm.nih.gov/29036287))

434. [chrishah/MITObim](https://github.com/chrishah/MITObim) (‚≠ê104 ¬∑ Perl) MITObim - mitochondrial baiting and iterative mapping (PMID: [23661685](https://pubmed.ncbi.nlm.nih.gov/23661685))

435. [yangao07/abPOA](https://github.com/yangao07/abPOA) (‚≠ê104 ¬∑ C) abPOA: an SIMD-based C library for fast partial order alignment using adaptive band (PMID: [33165528](https://pubmed.ncbi.nlm.nih.gov/33165528))

436. [BaderLab/saber](https://github.com/BaderLab/saber) (‚≠ê103 ¬∑ Python) Saber is a deep-learning based tool for information extraction in the biomedical domain. Pull requests are welcome! Note: this is a work in progress. Many things are broken, and the codebase is not stable. (PMID: [31218364](https://pubmed.ncbi.nlm.nih.gov/31218364))

437. [FePhyFoFum/phyx](https://github.com/FePhyFoFum/phyx) (‚≠ê103 ¬∑ C++) phylogenetics tools for linux (and other mostly posix compliant) computers (PMID: [28174903](https://pubmed.ncbi.nlm.nih.gov/28174903))

438. [csyben/PYRO-NN](https://github.com/csyben/PYRO-NN) (‚≠ê103 ¬∑ Python) Python Reconstruction Operators in Neural Networks. High level python API for PYRO-NN-Layers (PMID: [31389023](https://pubmed.ncbi.nlm.nih.gov/31389023))

439. [louiejtaylor/grabseqs](https://github.com/louiejtaylor/grabseqs) (‚≠ê103 ¬∑ Python) A utility for easy downloading of reads from next-gen sequencing repositories like NCBI SRA (PMID: [32154830](https://pubmed.ncbi.nlm.nih.gov/32154830))

440. [adamewing/methylartist](https://github.com/adamewing/methylartist) (‚≠ê102 ¬∑ Python) Tools for plotting methylation data in various ways (PMID: [35482479](https://pubmed.ncbi.nlm.nih.gov/35482479))

441. [mabessa/F3DAS](https://github.com/mabessa/F3DAS) (‚≠ê102 ¬∑ Python) Framework for Data-Driven Design & Analysis of Structures & Materials (F3DASM) (PMID: [31608516](https://pubmed.ncbi.nlm.nih.gov/31608516))

442. [rrwick/perfect-bacterial-genome-tutorial](https://github.com/rrwick/perfect-bacterial-genome-tutorial) (‚≠ê102 ¬∑ Python) None (PMID: [36862631](https://pubmed.ncbi.nlm.nih.gov/36862631))

443. [yanwu2014/swne](https://github.com/yanwu2014/swne) (‚≠ê102 ¬∑ R) Similarity Weighted Nonnegative Embedding (SWNE), a method for visualizing high dimensional datasets (PMID: [30528274](https://pubmed.ncbi.nlm.nih.gov/30528274))

444. [zhjohnchan/M3AE](https://github.com/zhjohnchan/M3AE) (‚≠ê102 ¬∑ Python) [MICCAI-2022] This is the official implementation of Multi-Modal Masked Autoencoders for Medical Vision-and-Language Pre-Training. (PMID: [37976867](https://pubmed.ncbi.nlm.nih.gov/37976867))

445. [biopragmatics/bioregistry](https://github.com/biopragmatics/bioregistry) (‚≠ê101 ¬∑ Python) üìÆ An integrative registry of biological databases, ontologies, and nomenclatures. (PMID: [36402838](https://pubmed.ncbi.nlm.nih.gov/36402838))

446. [ncbi/SKESA](https://github.com/ncbi/SKESA) (‚≠ê101 ¬∑ C++) SKESA assembler (PMID: [30286803](https://pubmed.ncbi.nlm.nih.gov/30286803))

447. [openPupil/Open-PupilEXT](https://github.com/openPupil/Open-PupilEXT) (‚≠ê101 ¬∑ C++) Official implementation of a free pupillometry platform called PupilEXT proposed in the article "PupilEXT: flexible open-source platform for high resolution pupillometry in vision research". (PMID: [34220432](https://pubmed.ncbi.nlm.nih.gov/34220432))

448. [oxpig/AbLang](https://github.com/oxpig/AbLang) (‚≠ê101 ¬∑ Python)  AbLang: A language model for antibodies (PMID: [36699403](https://pubmed.ncbi.nlm.nih.gov/36699403))

449. [ratschlab/spladder](https://github.com/ratschlab/spladder) (‚≠ê101 ¬∑ Python) Tool for the detection and quantification of alternative splicing events from RNA-Seq data. (PMID: [26873928](https://pubmed.ncbi.nlm.nih.gov/26873928))

450. [rnabioco/clustifyr](https://github.com/rnabioco/clustifyr) (‚≠ê101 ¬∑ R) Infer cell types in scRNA-seq data using bulk RNA-seq or gene sets (PMID: [32765839](https://pubmed.ncbi.nlm.nih.gov/32765839))

451. [szpiech/selscan](https://github.com/szpiech/selscan) (‚≠ê101 ¬∑ C) Haplotype based scans for selection (PMID: [25015648](https://pubmed.ncbi.nlm.nih.gov/25015648); [38180866](https://pubmed.ncbi.nlm.nih.gov/38180866))

452. [zhxiaokang/RASflow](https://github.com/zhxiaokang/RASflow) (‚≠ê101 ¬∑ HTML) RNA-Seq analysis workflow (PMID: [32183729](https://pubmed.ncbi.nlm.nih.gov/32183729))

453. [CostaLab/reg-gen](https://github.com/CostaLab/reg-gen) (‚≠ê100 ¬∑ Python) Regulatory Genomics Toolbox: Python library and set of tools for the integrative analysis of high throughput regulatory genomics data. (PMID: [36879236](https://pubmed.ncbi.nlm.nih.gov/36879236))

454. [RagnarGrootKoerkamp/astar-pairwise-aligner](https://github.com/RagnarGrootKoerkamp/astar-pairwise-aligner) (‚≠ê100 ¬∑ Rust) A pairwise sequence aligner written in Rust (PMID: [38265119](https://pubmed.ncbi.nlm.nih.gov/38265119))

455. [SBRG/ssbio](https://github.com/SBRG/ssbio) (‚≠ê100 ¬∑ Python) A Python framework for structural systems biology (PMID: [29444205](https://pubmed.ncbi.nlm.nih.gov/29444205))

456. [hakyimlab/PrediXcan](https://github.com/hakyimlab/PrediXcan) (‚≠ê100 ¬∑ Python) PrediXcan Project (PMID: [27835642](https://pubmed.ncbi.nlm.nih.gov/27835642))

457. [3DGenomes/tadbit](https://github.com/3DGenomes/tadbit) (‚≠ê99 ¬∑ Python) TADbit is a complete Python library to deal with all steps to analyze, model and explore 3C-based data. With TADbit the user can map FASTQ files to obtain raw interaction binned matrices (Hi-C like matrices), normalize and correct interaction matrices, identify and compare the so-called Topologically Associating Domains (TADs), build 3D models from the interaction matrices, and finally, extract structural properties from the models. TADbit is complemented by TADkit for visualizing 3D models (PMID: [28723903](https://pubmed.ncbi.nlm.nih.gov/28723903))

458. [SunPengChuan/wgdi](https://github.com/SunPengChuan/wgdi) (‚≠ê99 ¬∑ Python) WGDI: A user-friendly toolkit for evolutionary analyses of whole-genome duplications and ancestral karyotypes (PMID: [36307977](https://pubmed.ncbi.nlm.nih.gov/36307977))

459. [zeeev/wham](https://github.com/zeeev/wham) (‚≠ê99 ¬∑ C++) Structural variant detection and association testing (PMID: [26625158](https://pubmed.ncbi.nlm.nih.gov/26625158))

460. [brentp/duphold](https://github.com/brentp/duphold) (‚≠ê98 ¬∑ Nim) don't get DUP'ed or DEL'ed by your putative SVs. (PMID: [31222198](https://pubmed.ncbi.nlm.nih.gov/31222198))

461. [evoldoers/biomake](https://github.com/evoldoers/biomake) (‚≠ê98 ¬∑ Prolog) GNU-Make-like utility for managing builds and complex workflows (PMID: [28486579](https://pubmed.ncbi.nlm.nih.gov/28486579))

462. [hahnlab/CAFE5](https://github.com/hahnlab/CAFE5) (‚≠ê98 ¬∑ C++) Version 5 of the CAFE phylogenetics software (PMID: [33325502](https://pubmed.ncbi.nlm.nih.gov/33325502))

463. [hugheslab/snpgenie](https://github.com/hugheslab/snpgenie) (‚≠ê98 ¬∑ Perl) Program for estimating œÄN/œÄS, dN/dS, and other diversity measures from next-generation sequencing data (PMID: [26227143](https://pubmed.ncbi.nlm.nih.gov/26227143))

464. [vaquerizaslab/fanc](https://github.com/vaquerizaslab/fanc) (‚≠ê98 ¬∑ Python) FAN-C: Framework for the ANalysis of C-like data (PMID: [33334380](https://pubmed.ncbi.nlm.nih.gov/33334380))

465. [MolecularAI/DockStream](https://github.com/MolecularAI/DockStream) (‚≠ê97 ¬∑ Python) DockStream: A Docking Wrapper to Enhance De Novo Molecular Design (PMID: [34789335](https://pubmed.ncbi.nlm.nih.gov/34789335))

466. [verilylifesciences/genomewarp](https://github.com/verilylifesciences/genomewarp) (‚≠ê97 ¬∑ Java) GenomeWarp translates genetic variants from one genome assembly version to another. (PMID: [30916319](https://pubmed.ncbi.nlm.nih.gov/30916319))

467. [GATB/bcalm](https://github.com/GATB/bcalm) (‚≠ê96 ¬∑ Python) compacted de Bruijn graph construction in low memory (PMID: [27307618](https://pubmed.ncbi.nlm.nih.gov/27307618))

468. [MathOnco/NeoPredPipe](https://github.com/MathOnco/NeoPredPipe) (‚≠ê96 ¬∑ Python) Neoantigens prediction pipeline for multi- or single-region vcf files using ANNOVAR and netMHCpan. (PMID: [31117948](https://pubmed.ncbi.nlm.nih.gov/31117948))

469. [abhi4ssj/few-shot-segmentation](https://github.com/abhi4ssj/few-shot-segmentation) (‚≠ê96 ¬∑ Python) PyTorch implementation of 'Squeeze and Excite' Guided Few Shot Segmentation of Volumetric Scans (PMID: [31630012](https://pubmed.ncbi.nlm.nih.gov/31630012))

470. [arseha/peakonly](https://github.com/arseha/peakonly) (‚≠ê96 ¬∑ Python) None (PMID: [31841624](https://pubmed.ncbi.nlm.nih.gov/31841624))

471. [grimmlab/MicrobiomeBestPracticeReview](https://github.com/grimmlab/MicrobiomeBestPracticeReview) (‚≠ê96 ¬∑ Shell) Current Challenges and Best Practice Protocols for Microbiome Analysis using Amplicon and Metagenomic Sequencing (PMID: [31848574](https://pubmed.ncbi.nlm.nih.gov/31848574))

472. [j3xugit/RaptorX-3DModeling](https://github.com/j3xugit/RaptorX-3DModeling) (‚≠ê96 ¬∑ C) Note that current version does not include search of very large metagenome data. For some proteins, metagenome data is important. We will update this as soon as possible. (PMID: [33961022](https://pubmed.ncbi.nlm.nih.gov/33961022))

473. [makism/dyconnmap](https://github.com/makism/dyconnmap) (‚≠ê96 ¬∑ Python) A dynamic connectome mapping module in python. (PMID: [34250674](https://pubmed.ncbi.nlm.nih.gov/34250674))

474. [CDDLeiden/DrugEx](https://github.com/CDDLeiden/DrugEx) (‚≠ê95 ¬∑ Jupyter Notebook) De Novo Drug Design with RNNs and Transformers (PMID: [37272707](https://pubmed.ncbi.nlm.nih.gov/37272707))

475. [ZJUFanLab/bulk2space](https://github.com/ZJUFanLab/bulk2space) (‚≠ê95 ¬∑ Jupyter Notebook) a spatial deconvolution method based on deep learning frameworks, which converts bulk transcriptomes into spatially resolved single-cell expression profiles (PMID: [36310179](https://pubmed.ncbi.nlm.nih.gov/36310179))

476. [gaocong13/Projective-Spatial-Transformers](https://github.com/gaocong13/Projective-Spatial-Transformers) (‚≠ê95 ¬∑ Python) Generalizing Spatial Transformers to Projective Geometry with Applications to 2D/3D Registration (PMID: [33135014](https://pubmed.ncbi.nlm.nih.gov/33135014))

477. [jlianglab/Fixed-Point-GAN](https://github.com/jlianglab/Fixed-Point-GAN) (‚≠ê95 ¬∑ Python) Official implementation of Fixed-Point GAN - ICCV 2019 (PMID: [32612486](https://pubmed.ncbi.nlm.nih.gov/32612486))

478. [ncbi/MedCPT](https://github.com/ncbi/MedCPT) (‚≠ê95 ¬∑ Python) Code for MedCPT, a model for zero-shot biomedical information retrieval. (PMID: [37930897](https://pubmed.ncbi.nlm.nih.gov/37930897))

479. [BEETL/BEETL](https://github.com/BEETL/BEETL) (‚≠ê94 ¬∑ C++) BEETL (PMID: [22556365](https://pubmed.ncbi.nlm.nih.gov/22556365); [24950811](https://pubmed.ncbi.nlm.nih.gov/24950811))

480. [BigDataBiology/SemiBin](https://github.com/BigDataBiology/SemiBin) (‚≠ê94 ¬∑ Python) SemiBin: metagenomics binning with self-supervised deep learning (PMID: [37387171](https://pubmed.ncbi.nlm.nih.gov/37387171))

481. [ZJUFanLab/scDeepSort](https://github.com/ZJUFanLab/scDeepSort) (‚≠ê94 ¬∑ Python) Cell-type Annotation for Single-cell Transcriptomics using Deep Learning with a Weighted Graph Neural Network (PMID: [34500471](https://pubmed.ncbi.nlm.nih.gov/34500471))

482. [aistairc/DeepEventMine](https://github.com/aistairc/DeepEventMine) (‚≠ê94 ¬∑ Python) DeepEventMine: End-to-end Neural Nested Event Extraction from Biomedical Texts (PMID: [33141147](https://pubmed.ncbi.nlm.nih.gov/33141147))

483. [bbci/wyrm](https://github.com/bbci/wyrm) (‚≠ê94 ¬∑ Python) Python toolbox for Brain-Computer Interfacing (BCI) (PMID: [26001643](https://pubmed.ncbi.nlm.nih.gov/26001643))

484. [lh3/bgt](https://github.com/lh3/bgt) (‚≠ê94 ¬∑ C) Flexible genotype query among 30,000+ samples whole-genome (PMID: [26500154](https://pubmed.ncbi.nlm.nih.gov/26500154))

485. [nmquijada/tormes](https://github.com/nmquijada/tormes) (‚≠ê94 ¬∑ Shell) Making whole bacterial genome sequencing data analysis easy (PMID: [30957837](https://pubmed.ncbi.nlm.nih.gov/30957837))

486. [AlexandrovLab/SigProfilerMatrixGenerator](https://github.com/AlexandrovLab/SigProfilerMatrixGenerator) (‚≠ê93 ¬∑ Python) SigProfilerMatrixGenerator creates mutational matrices for all types of somatic mutations. It allows downsizing the generated mutations only to parts for the genome (e.g., exome or a custom BED file). The tool seamlessly integrates with other SigProfiler tools. (PMID: [31470794](https://pubmed.ncbi.nlm.nih.gov/31470794); [37605126](https://pubmed.ncbi.nlm.nih.gov/37605126))

487. [mgalardini/pyseer](https://github.com/mgalardini/pyseer) (‚≠ê93 ¬∑ Python) SEER, reimplemented in python üêçüîÆ (PMID: [30535304](https://pubmed.ncbi.nlm.nih.gov/30535304))

488. [roblanf/sangeranalyseR](https://github.com/roblanf/sangeranalyseR) (‚≠ê93 ¬∑ R) functions to analyse sanger sequencing reads in R (PMID: [33591316](https://pubmed.ncbi.nlm.nih.gov/33591316))

489. [treangen/MetAMOS](https://github.com/treangen/MetAMOS) (‚≠ê93 ¬∑ Roff) A metagenomic and isolate assembly and analysis pipeline built with AMOS (PMID: [23320958](https://pubmed.ncbi.nlm.nih.gov/23320958))

490. [SysBioChalmers/RAVEN](https://github.com/SysBioChalmers/RAVEN) (‚≠ê92 ¬∑ MATLAB) The RAVEN Toolbox for genome scale model reconstruction, curation and analysis. (PMID: [30335785](https://pubmed.ncbi.nlm.nih.gov/30335785))

491. [immunogenomics/symphony](https://github.com/immunogenomics/symphony) (‚≠ê92 ¬∑ Jupyter Notebook) Efficient and precise single-cell reference atlas mapping with Symphony (PMID: [34620862](https://pubmed.ncbi.nlm.nih.gov/34620862))

492. [mossmatters/HybPiper](https://github.com/mossmatters/HybPiper) (‚≠ê92 ¬∑ Python) Recovering genes from targeted sequence capture data (PMID: [27437175](https://pubmed.ncbi.nlm.nih.gov/27437175))

493. [pulimeng/DeepDrug3D](https://github.com/pulimeng/DeepDrug3D) (‚≠ê92 ¬∑ Python) A convolutional neural network for classifying binding pockets based on spatial and chemical information extracted from the pockets. (PMID: [30716081](https://pubmed.ncbi.nlm.nih.gov/30716081))

494. [ropensci/nasapower](https://github.com/ropensci/nasapower) (‚≠ê92 ¬∑ R) API Client for NASA POWER Global Meteorology, Surface Solar Energy and Climatology in R (PMID: [34039412](https://pubmed.ncbi.nlm.nih.gov/34039412))

495. [zstephens/neat-genreads](https://github.com/zstephens/neat-genreads) (‚≠ê92 ¬∑ Python) NEAT read simulation tools (PMID: [27893777](https://pubmed.ncbi.nlm.nih.gov/27893777))

496. [Illumina/Pisces](https://github.com/Illumina/Pisces) (‚≠ê91 ¬∑ C#) Somatic and germline variant caller for amplicon data.  Recommended caller for tumor-only workflows. (PMID: [30304370](https://pubmed.ncbi.nlm.nih.gov/30304370))

497. [JLiangLab/DiRA](https://github.com/JLiangLab/DiRA) (‚≠ê91 ¬∑ Python) Official PyTorch Implementation for DiRA: Discriminative, Restorative, and Adversarial Learning for Self-supervised Medical Image Analysis - CVPR 2022 (PMID: [36313959](https://pubmed.ncbi.nlm.nih.gov/36313959); [38537414](https://pubmed.ncbi.nlm.nih.gov/38537414))

498. [bcgsc/ntHash](https://github.com/bcgsc/ntHash) (‚≠ê91 ¬∑ C++) Fast hash function for DNA/RNA sequences (PMID: [36000872](https://pubmed.ncbi.nlm.nih.gov/36000872))

499. [caitiecollins/treeWAS](https://github.com/caitiecollins/treeWAS) (‚≠ê91 ¬∑ R) treeWAS: A Phylogenetic Tree-Based Tool for Genome-Wide Association Studies in Microbes (PMID: [29401456](https://pubmed.ncbi.nlm.nih.gov/29401456))

500. [genomicsITER/NanoCLUST](https://github.com/genomicsITER/NanoCLUST) (‚≠ê91 ¬∑ Nextflow) NanoCLUST is an analysis pipeline for UMAP-based classification of amplicon-based full-length 16S rRNA nanopore reads (PMID: [33079990](https://pubmed.ncbi.nlm.nih.gov/33079990))

501. [jbohnslav/deepethogram](https://github.com/jbohnslav/deepethogram) (‚≠ê91 ¬∑ Python) None (PMID: [34473051](https://pubmed.ncbi.nlm.nih.gov/34473051))

502. [microsoft/genomicsnotebook](https://github.com/microsoft/genomicsnotebook) (‚≠ê91 ¬∑ Jupyter Notebook) Jupyter Notebooks on Azure for Genomics Data Analysis (PMID: [36726729](https://pubmed.ncbi.nlm.nih.gov/36726729))

503. [qubekit/QUBEKit](https://github.com/qubekit/QUBEKit) (‚≠ê91 ¬∑ Python)   Quantum Mechanical Bespoke Force Field Derivation Toolkit (PMID: [35792069](https://pubmed.ncbi.nlm.nih.gov/35792069))

504. [bcgsc/ARCS](https://github.com/bcgsc/ARCS) (‚≠ê90 ¬∑ C++) üåàScaffold genome sequence assemblies using linked or long read sequencing data (PMID: [29069293](https://pubmed.ncbi.nlm.nih.gov/29069293))

505. [cytoscape/cyjShiny](https://github.com/cytoscape/cyjShiny) (‚≠ê90 ¬∑ JavaScript) An R/shiny widget for cytoscape.js (PMID: [37585474](https://pubmed.ncbi.nlm.nih.gov/37585474))

506. [labsquare/cutevariant](https://github.com/labsquare/cutevariant) (‚≠ê89 ¬∑ Python) A standalone and free application to explore genetics variations from VCF file (PMID: [36699349](https://pubmed.ncbi.nlm.nih.gov/36699349))

507. [mr7495/COVID-CTset](https://github.com/mr7495/COVID-CTset) (‚≠ê89 ¬∑ Python) Large Covid-19 CT scans dataset from the paper: https://doi.org/10.1016/j.bspc.2021.102588 (PMID: [33821166](https://pubmed.ncbi.nlm.nih.gov/33821166))

508. [nawrockie/vadr](https://github.com/nawrockie/vadr) (‚≠ê89 ¬∑ Perl) Viral Annotation DefineR: classification and annotation of viral sequences based on RefSeq annotation (PMID: [32448124](https://pubmed.ncbi.nlm.nih.gov/32448124))

509. [theosanderson/taxonium](https://github.com/theosanderson/taxonium) (‚≠ê89 ¬∑ JavaScript) A tool for exploring very large trees in the browser (PMID: [36377483](https://pubmed.ncbi.nlm.nih.gov/36377483); [36453872](https://pubmed.ncbi.nlm.nih.gov/36453872))

510. [MannLabs/alphapeptdeep](https://github.com/MannLabs/alphapeptdeep) (‚≠ê88 ¬∑ Jupyter Notebook) Deep learning framework for proteomics (PMID: [36433986](https://pubmed.ncbi.nlm.nih.gov/36433986))

511. [biocore/deblur](https://github.com/biocore/deblur) (‚≠ê88 ¬∑ Python) Deblur is a greedy deconvolution algorithm based on known read error profiles. (PMID: [28289731](https://pubmed.ncbi.nlm.nih.gov/28289731))

512. [FastTrackOrg/FastTrack](https://github.com/FastTrackOrg/FastTrack) (‚≠ê87 ¬∑ C++) FastTrack is a cross-platform application designed to track multiple objects in video recording. (PMID: [33571205](https://pubmed.ncbi.nlm.nih.gov/33571205))

513. [KatherLab/HIA](https://github.com/KatherLab/HIA) (‚≠ê87 ¬∑ Python) Histopathology Image Analysis (PMID: [35588568](https://pubmed.ncbi.nlm.nih.gov/35588568))

514. [cpockrandt/genmap](https://github.com/cpockrandt/genmap) (‚≠ê87 ¬∑ C++) GenMap - Fast and Exact Computation of Genome Mappability (PMID: [32246826](https://pubmed.ncbi.nlm.nih.gov/32246826))

515. [giesselmann/nanopype](https://github.com/giesselmann/nanopype) (‚≠ê87 ¬∑ Python) Snakemake pipelines for nanopore sequencing data archiving and processing (PMID: [31192347](https://pubmed.ncbi.nlm.nih.gov/31192347))

516. [meren/illumina-utils](https://github.com/meren/illumina-utils) (‚≠ê87 ¬∑ Python) A library and collection of scripts to work with Illumina paired-end data (for CASAVA 1.7+ pipeline). (PMID: [23799126](https://pubmed.ncbi.nlm.nih.gov/23799126))

517. [mozack/abra2](https://github.com/mozack/abra2) (‚≠ê87 ¬∑ Java) ABRA2 (PMID: [30649250](https://pubmed.ncbi.nlm.nih.gov/30649250))

518. [mrzResearchArena/PyFeat](https://github.com/mrzResearchArena/PyFeat) (‚≠ê87 ¬∑ Python) A Python-based Effective Feature Generation Tool from DNA, RNA, and Protein Sequences (PMID: [30850831](https://pubmed.ncbi.nlm.nih.gov/30850831))

519. [orbkit/orbkit](https://github.com/orbkit/orbkit) (‚≠ê87 ¬∑ Python) A Toolbox for Post-Processing Quantum Chemical Wavefunction Data (PMID: [27043934](https://pubmed.ncbi.nlm.nih.gov/27043934))

520. [CogStack/SemEHR](https://github.com/CogStack/SemEHR) (‚≠ê86 ¬∑ JavaScript) Surfacing Semantic Data from Clinical Notes in Electronic Health Records for Tailored Care, Trial Recruitment and Clinical Research (PMID: [29361077](https://pubmed.ncbi.nlm.nih.gov/29361077))

521. [bigchem/transformer-cnn](https://github.com/bigchem/transformer-cnn) (‚≠ê86 ¬∑ Python) Transformer CNN for QSAR/QSPR modelling (PMID: [33431004](https://pubmed.ncbi.nlm.nih.gov/33431004))

522. [blab/nextflu](https://github.com/blab/nextflu) (‚≠ê86 ¬∑ JavaScript) Real-time tracking of influenza evolution (PMID: [26115986](https://pubmed.ncbi.nlm.nih.gov/26115986))

523. [budach/pysster](https://github.com/budach/pysster) (‚≠ê86 ¬∑ Python) pysster: Learning Sequence And Structure Motifs In Biological Sequences Using Convolutional Neural Networks (PMID: [29659719](https://pubmed.ncbi.nlm.nih.gov/29659719))

524. [jminnier/STARTapp](https://github.com/jminnier/STARTapp) (‚≠ê86 ¬∑ HTML) The START App: R Shiny Transcriptome Analysis Resource Tool (PMID: [28171615](https://pubmed.ncbi.nlm.nih.gov/28171615))

525. [modeloriented/survex](https://github.com/modeloriented/survex) (‚≠ê86 ¬∑ R) Explainable Machine Learning in Survival Analysis (PMID: [38039146](https://pubmed.ncbi.nlm.nih.gov/38039146))

526. [oxpig/ImmuneBuilder](https://github.com/oxpig/ImmuneBuilder) (‚≠ê86 ¬∑ Jupyter Notebook) Predict the structure of immune receptor proteins (PMID: [37248282](https://pubmed.ncbi.nlm.nih.gov/37248282))

527. [parashardhapola/scarf](https://github.com/parashardhapola/scarf) (‚≠ê86 ¬∑ Python) Toolkit for highly memory efficient analysis of single-cell RNA-Seq, scATAC-Seq and CITE-Seq data. Analyze atlas scale datasets with millions of cells on laptop. (PMID: [35941103](https://pubmed.ncbi.nlm.nih.gov/35941103))

528. [romanhaa/Cerebro](https://github.com/romanhaa/Cerebro) (‚≠ê86 ¬∑ HTML) Visualization of scRNA-seq data. (PMID: [31764967](https://pubmed.ncbi.nlm.nih.gov/31764967))

529. [vanderschaarlab/AutoPrognosis](https://github.com/vanderschaarlab/AutoPrognosis) (‚≠ê86 ¬∑ Python) A system for automating the design of predictive modeling pipelines tailored for clinical prognosis. (PMID: [37347752](https://pubmed.ncbi.nlm.nih.gov/37347752))

530. [BUStools/bustools](https://github.com/BUStools/bustools) (‚≠ê85 ¬∑ C) Tools for working with BUS files (PMID: [37129540](https://pubmed.ncbi.nlm.nih.gov/37129540))

531. [ChristopherWilks/megadepth](https://github.com/ChristopherWilks/megadepth) (‚≠ê85 ¬∑ C++) BigWig and BAM utilities (PMID: [33693500](https://pubmed.ncbi.nlm.nih.gov/33693500))

532. [ShuweiShao/AF-SfMLearner](https://github.com/ShuweiShao/AF-SfMLearner) (‚≠ê85 ¬∑ Python) [MedIA2022 & ICRA2021] Self-Supervised Monocular Depth and Ego-Motion Estimation in Endoscopy: Appearance Flow to the Rescue (PMID: [35016079](https://pubmed.ncbi.nlm.nih.gov/35016079))

533. [bergmanlab/mcclintock](https://github.com/bergmanlab/mcclintock) (‚≠ê85 ¬∑ Python) Meta-pipeline to identify transposable element insertions using next generation sequencing data (PMID: [28637810](https://pubmed.ncbi.nlm.nih.gov/28637810); [37452430](https://pubmed.ncbi.nlm.nih.gov/37452430))

534. [llnl/fast](https://github.com/llnl/fast) (‚≠ê85 ¬∑ Python) Fusion models for Atomic and molecular STructures (FAST) (PMID: [33754707](https://pubmed.ncbi.nlm.nih.gov/33754707))

535. [parklab/xTea](https://github.com/parklab/xTea) (‚≠ê85 ¬∑ Python) Comprehensive TE insertion identification with WGS/WES data from multiple sequencing technics (PMID: [34158502](https://pubmed.ncbi.nlm.nih.gov/34158502))

536. [ccipd/MRQy](https://github.com/ccipd/MRQy) (‚≠ê84 ¬∑ JavaScript) MRQy is a quality assurance and checking tool for quantitative assessment of magnetic resonance imaging (MRI) data. (PMID: [33176026](https://pubmed.ncbi.nlm.nih.gov/33176026))

537. [gui11aume/starcode](https://github.com/gui11aume/starcode) (‚≠ê84 ¬∑ C) All pairs search and sequence clustering (PMID: [25638815](https://pubmed.ncbi.nlm.nih.gov/25638815))

538. [htseq/htseq](https://github.com/htseq/htseq) (‚≠ê84 ¬∑ Python) HTSeq is a Python library to facilitate processing and analysis of data from  high-throughput sequencing (HTS) experiments. (PMID: [35561197](https://pubmed.ncbi.nlm.nih.gov/35561197))

539. [jodyphelan/TBProfiler](https://github.com/jodyphelan/TBProfiler) (‚≠ê84 ¬∑ Python) Profiling tool for Mycobacterium tuberculosis to detect ressistance and strain type from WGS data (PMID: [33317631](https://pubmed.ncbi.nlm.nih.gov/33317631))

540. [lanagarmire/DeepImpute](https://github.com/lanagarmire/DeepImpute) (‚≠ê84 ¬∑ Python) An accurate and efficient deep learning method for single-cell RNA-seq data imputation (PMID: [31627739](https://pubmed.ncbi.nlm.nih.gov/31627739))

541. [twolinin/LongPhase](https://github.com/twolinin/LongPhase) (‚≠ê84 ¬∑ C) None (PMID: [35104333](https://pubmed.ncbi.nlm.nih.gov/35104333))

542. [BioJulia/BioStructures](https://github.com/BioJulia/BioStructures) (‚≠ê83 ¬∑ Julia) A Julia package to read, write and manipulate macromolecular structures (particularly proteins) (PMID: [32407511](https://pubmed.ncbi.nlm.nih.gov/32407511))

543. [carmonalab/scGate](https://github.com/carmonalab/scGate) (‚≠ê83 ¬∑ R) marker-based purification of cell types from single-cell RNA-seq datasets (PMID: [35258562](https://pubmed.ncbi.nlm.nih.gov/35258562))

544. [catavallejos/BASiCS](https://github.com/catavallejos/BASiCS) (‚≠ê83 ¬∑ R) BASiCS: Bayesian Analysis of Single-Cell Sequencing Data. This is an unstable experimental version. Please see http://bioconductor.org/packages/BASiCS/ for the official release version (PMID: [27083558](https://pubmed.ncbi.nlm.nih.gov/27083558))

545. [cdcgov/microbetrace](https://github.com/cdcgov/microbetrace) (‚≠ê83 ¬∑ TypeScript) The Visualization Multitool for Molecular Epidemiology and Bioinformatics (PMID: [34492010](https://pubmed.ncbi.nlm.nih.gov/34492010))

546. [citiususc/veryfasttree](https://github.com/citiususc/veryfasttree) (‚≠ê83 ¬∑ C++) Efficient phylogenetic tree inference for massive taxonomic datasets (PMID: [32573652](https://pubmed.ncbi.nlm.nih.gov/32573652))

547. [fwhelan/coinfinder](https://github.com/fwhelan/coinfinder) (‚≠ê83 ¬∑ C++) A tool for the identification of coincident (associating and dissociating) genes in pangenomes. (PMID: [32100706](https://pubmed.ncbi.nlm.nih.gov/32100706))

548. [openbiox/UCSCXenaShiny](https://github.com/openbiox/UCSCXenaShiny) (‚≠ê83 ¬∑ R) üìä An R package for interactively exploring UCSC Xena https://xenabrowser.net/datapages; Book: https://lishensuo.github.io/UCSCXenaShiny_Book; App online: https://shiny.hiplot.cn/ucsc-xena-shiny/, https://shiny.zhoulab.ac.cn/UCSCXenaShiny (PMID: [34323947](https://pubmed.ncbi.nlm.nih.gov/34323947))

549. [uwrit/leaf](https://github.com/uwrit/leaf) (‚≠ê83 ¬∑ TypeScript) Leaf Clinical Data Explorer (PMID: [31592524](https://pubmed.ncbi.nlm.nih.gov/31592524))

550. [0liliulei/Mem3D](https://github.com/0liliulei/Mem3D) (‚≠ê82 ¬∑ Python) None (PMID: [36327652](https://pubmed.ncbi.nlm.nih.gov/36327652))

551. [CenterForMedicalGeneticsGhent/WisecondorX](https://github.com/CenterForMedicalGeneticsGhent/WisecondorX) (‚≠ê82 ¬∑ Python) WisecondorX ‚Äî An evolved WISECONDOR (PMID: [30566647](https://pubmed.ncbi.nlm.nih.gov/30566647))

552. [Vini2/GraphBin](https://github.com/Vini2/GraphBin) (‚≠ê82 ¬∑ Python) ‚ú®üß¨ Refined binning of metagenomic contigs using assembly graphs (PMID: [32167528](https://pubmed.ncbi.nlm.nih.gov/32167528))

553. [nunofonseca/irap](https://github.com/nunofonseca/irap) (‚≠ê82 ¬∑ R) integrated RNA-seq Analysis Pipeline (PMID: [28369191](https://pubmed.ncbi.nlm.nih.gov/28369191))

554. [shervinmin/DeepCovid](https://github.com/shervinmin/DeepCovid) (‚≠ê82 ¬∑ Python) None (PMID: [32781377](https://pubmed.ncbi.nlm.nih.gov/32781377))

555. [splatlab/squeakr](https://github.com/splatlab/squeakr) (‚≠ê82 ¬∑ C++) Squeakr: An Exact and Approximate k -mer Counting System (PMID: [29444235](https://pubmed.ncbi.nlm.nih.gov/29444235))

556. [vanallenlab/comut](https://github.com/vanallenlab/comut) (‚≠ê82 ¬∑ Python) CoMut is a Python library for visualizing genomic and phenotypic information via comutation plots (PMID: [32502231](https://pubmed.ncbi.nlm.nih.gov/32502231))

557. [EtienneCmb/tensorpac](https://github.com/EtienneCmb/tensorpac) (‚≠ê81 ¬∑ Python) Phase-Amplitude Coupling under Python (PMID: [33119593](https://pubmed.ncbi.nlm.nih.gov/33119593))

558. [Superzchen/iLearnPlus](https://github.com/Superzchen/iLearnPlus) (‚≠ê81 ¬∑ Python) iLearnPlus is the first machine-learning platform with both graphical- and web-based user interface that enables the construction of automated machine-learning pipelines for computational analysis and predictions using nucleic acid and protein sequences. (PMID: [33660783](https://pubmed.ncbi.nlm.nih.gov/33660783))

559. [balbasty/nitorch](https://github.com/balbasty/nitorch) (‚≠ê81 ¬∑ Python) Neuroimaging in PyTorch (PMID: [34271531](https://pubmed.ncbi.nlm.nih.gov/34271531))

560. [bcgsc/RNA-Bloom](https://github.com/bcgsc/RNA-Bloom) (‚≠ê81 ¬∑ Java) :hibiscus: reference-free transcriptome assembly for short and long reads (PMID: [37217540](https://pubmed.ncbi.nlm.nih.gov/37217540))

561. [futianfan/clinical-trial-outcome-prediction](https://github.com/futianfan/clinical-trial-outcome-prediction) (‚≠ê81 ¬∑ Python) benchmark dataset and Deep learning method (Hierarchical Interaction Network, HINT) for clinical trial approval probability prediction, published in Cell Patterns 2022. (PMID: [35465223](https://pubmed.ncbi.nlm.nih.gov/35465223))

562. [ramiromagno/gwasrapidd](https://github.com/ramiromagno/gwasrapidd) (‚≠ê81 ¬∑ R) gwasrapidd: an R package to query, download and wrangle GWAS Catalog data (PMID: [31373609](https://pubmed.ncbi.nlm.nih.gov/31373609))

563. [smirarab/pasta](https://github.com/smirarab/pasta) (‚≠ê81 ¬∑ Python) PASTA (Practical Alignment using SATe and Transitivity) (PMID: [29931282](https://pubmed.ncbi.nlm.nih.gov/29931282))

564. [xie186/ViewBS](https://github.com/xie186/ViewBS) (‚≠ê81 ¬∑ Perl) ViewBS - a powerful toolkit for visualization of high-throughput bisulfite sequencing data (PMID: [29087450](https://pubmed.ncbi.nlm.nih.gov/29087450))

565. [gamcil/cblaster](https://github.com/gamcil/cblaster) (‚≠ê80 ¬∑ HTML) Find clustered hits from a BLAST search (PMID: [36700093](https://pubmed.ncbi.nlm.nih.gov/36700093))

566. [splatlab/mantis](https://github.com/splatlab/mantis) (‚≠ê80 ¬∑ C++) Mantis: A Fast, Small, and Exact Large-Scale Sequence-Search Index (PMID: [35325039](https://pubmed.ncbi.nlm.nih.gov/35325039))

567. [wdecoster/methplotlib](https://github.com/wdecoster/methplotlib) (‚≠ê80 ¬∑ Python) Plotting tools for nanopore methylation data (PMID: [32053166](https://pubmed.ncbi.nlm.nih.gov/32053166))

568. [CoSMoMVPA/CoSMoMVPA](https://github.com/CoSMoMVPA/CoSMoMVPA) (‚≠ê79 ¬∑ MATLAB) A lightweight multivariate pattern analysis (MVPA) toolbox in Matlab / Octave (PMID: [27499741](https://pubmed.ncbi.nlm.nih.gov/27499741))

569. [MICCoM/SSAGES-public](https://github.com/MICCoM/SSAGES-public) (‚≠ê79 ¬∑ C++) Software Suite for Advanced General Ensemble Simulations (PMID: [29390830](https://pubmed.ncbi.nlm.nih.gov/29390830))

570. [PayneLab/cptac](https://github.com/PayneLab/cptac) (‚≠ê79 ¬∑ Jupyter Notebook) Python packaging for CPTAC data (PMID: [33560848](https://pubmed.ncbi.nlm.nih.gov/33560848))

571. [crazyhottommy/scclusteval](https://github.com/crazyhottommy/scclusteval) (‚≠ê79 ¬∑ R) Single Cell Cluster Evaluation (PMID: [33165513](https://pubmed.ncbi.nlm.nih.gov/33165513))

572. [freeseek/score](https://github.com/freeseek/score) (‚≠ê79 ¬∑ C) Tools to work with GWAS-VCF summary statistics files (PMID: [38261650](https://pubmed.ncbi.nlm.nih.gov/38261650))

573. [nstrodt/UDSMProt](https://github.com/nstrodt/UDSMProt) (‚≠ê79 ¬∑ Python) Protein sequence classification with self-supervised pretraining (PMID: [31913448](https://pubmed.ncbi.nlm.nih.gov/31913448))

574. [yjx1217/simuG](https://github.com/yjx1217/simuG) (‚≠ê79 ¬∑ Perl) simuG: a general-purpose genome simulator (PMID: [31116378](https://pubmed.ncbi.nlm.nih.gov/31116378))

575. [zzstefan/BrainNetClass](https://github.com/zzstefan/BrainNetClass) (‚≠ê79 ¬∑ MATLAB) None (PMID: [32163221](https://pubmed.ncbi.nlm.nih.gov/32163221))

576. [DlutMedimgGroup/AnatomySketch-Software](https://github.com/DlutMedimgGroup/AnatomySketch-Software) (‚≠ê78 ¬∑ C++) A medical image segmentation software supporting user interventions and user-defined funtion extention (PMID: [35768752](https://pubmed.ncbi.nlm.nih.gov/35768752))

577. [KeenThera/SECSE](https://github.com/KeenThera/SECSE) (‚≠ê78 ¬∑ Python) Systemic Evolutionary Chemical Space Exploration for Drug Discovery (PMID: [35365231](https://pubmed.ncbi.nlm.nih.gov/35365231))

578. [MeHelmy/princess](https://github.com/MeHelmy/princess) (‚≠ê78 ¬∑ Python) None (PMID: [34521442](https://pubmed.ncbi.nlm.nih.gov/34521442))

579. [Vaa3D/vaa3d](https://github.com/Vaa3D/vaa3d) (‚≠ê78 ¬∑ None) www.vaa3d.org (PMID: [37213868](https://pubmed.ncbi.nlm.nih.gov/37213868))

580. [bcgsc/ntjoin](https://github.com/bcgsc/ntjoin) (‚≠ê78 ¬∑ Python) üîóGenome assembly scaffolder using minimizer graphs (PMID: [32311025](https://pubmed.ncbi.nlm.nih.gov/32311025))

581. [glennhickey/progressiveCactus](https://github.com/glennhickey/progressiveCactus) (‚≠ê78 ¬∑ Python) Distribution package for the Prgressive Cactus multiple genome aligner.  Dependencies are linked as submodules (PMID: [25138168](https://pubmed.ncbi.nlm.nih.gov/25138168))

582. [moiexpositoalonsolab/grenepipe](https://github.com/moiexpositoalonsolab/grenepipe) (‚≠ê78 ¬∑ Python) A flexible, scalable, and reproducible pipeline to automate variant calling from sequence reads. (PMID: [36053180](https://pubmed.ncbi.nlm.nih.gov/36053180))

583. [sjspielman/pyvolve](https://github.com/sjspielman/pyvolve) (‚≠ê78 ¬∑ Python) Python library to simulate evolutionary sequence data (PMID: [26397960](https://pubmed.ncbi.nlm.nih.gov/26397960))

584. [smirarab/sepp](https://github.com/smirarab/sepp) (‚≠ê78 ¬∑ Python) Ensemble of HMM methods (SEPP, TIPP, UPP) (PMID: [26076734](https://pubmed.ncbi.nlm.nih.gov/26076734); [28185571](https://pubmed.ncbi.nlm.nih.gov/28185571); [33471121](https://pubmed.ncbi.nlm.nih.gov/33471121); [34791036](https://pubmed.ncbi.nlm.nih.gov/34791036))

585. [arzwa/wgd](https://github.com/arzwa/wgd) (‚≠ê77 ¬∑ Python) Python package and CLI for whole-genome duplication related analyses. This package is deprecated in favor of https://github.com/heche-psb/wgd. (PMID: [30398564](https://pubmed.ncbi.nlm.nih.gov/30398564))

586. [calico/solo](https://github.com/calico/solo) (‚≠ê77 ¬∑ Python) software to detect doublets (PMID: [32592658](https://pubmed.ncbi.nlm.nih.gov/32592658))

587. [changwn/scFEA](https://github.com/changwn/scFEA) (‚≠ê77 ¬∑ Jupyter Notebook) single cell Flux Estimation Analysis (scFEA) Try the below web server! (PMID: [37216602](https://pubmed.ncbi.nlm.nih.gov/37216602))

588. [eldariont/svim-asm](https://github.com/eldariont/svim-asm) (‚≠ê77 ¬∑ Python) Structural Variant Identification Method using Genome Assemblies (PMID: [33346817](https://pubmed.ncbi.nlm.nih.gov/33346817))

589. [genome/gms](https://github.com/genome/gms) (‚≠ê77 ¬∑ Perl) The Genome Modeling System installer (PMID: [26158448](https://pubmed.ncbi.nlm.nih.gov/26158448))

590. [jermp/sshash](https://github.com/jermp/sshash) (‚≠ê77 ¬∑ C++) A compressed, associative, exact, and weighted dictionary for k-mers. (PMID: [35758794](https://pubmed.ncbi.nlm.nih.gov/35758794))

591. [mdsufz/MuDoGeR](https://github.com/mdsufz/MuDoGeR) (‚≠ê77 ¬∑ Shell) MuDoGeR makes the recovery of genomes from prokaryotes, viruses, and eukaryotes from metagenomes easy. (PMID: [37994269](https://pubmed.ncbi.nlm.nih.gov/37994269))

592. [scvae/scvae](https://github.com/scvae/scvae) (‚≠ê77 ¬∑ Python) Deep learning for single-cell transcript counts (PMID: [32415966](https://pubmed.ncbi.nlm.nih.gov/32415966))

593. [seqan/flexbar](https://github.com/seqan/flexbar) (‚≠ê77 ¬∑ C++) flexible barcode and adapter removal (PMID: [28541403](https://pubmed.ncbi.nlm.nih.gov/28541403))

594. [stephens999/ashr](https://github.com/stephens999/ashr) (‚≠ê77 ¬∑ R) An R package for adaptive shrinkage (PMID: [27756721](https://pubmed.ncbi.nlm.nih.gov/27756721))

595. [xavierdidelot/BactDating](https://github.com/xavierdidelot/BactDating) (‚≠ê77 ¬∑ R) Bayesian inference of ancestral dates on bacterial phylogenetic trees (PMID: [30184106](https://pubmed.ncbi.nlm.nih.gov/30184106))

596. [arthurpessa/ordpy](https://github.com/arthurpessa/ordpy) (‚≠ê76 ¬∑ Python) A Python package for data analysis with permutation entropy and ordinal network methods. (PMID: [34241315](https://pubmed.ncbi.nlm.nih.gov/34241315))

597. [gevaertlab/MultimodalPrognosis](https://github.com/gevaertlab/MultimodalPrognosis) (‚≠ê76 ¬∑ Python) Deep Learning with Multimodal Representation for Pancancer Prognosis Prediction (PMID: [31510656](https://pubmed.ncbi.nlm.nih.gov/31510656))

598. [ncbi/fcs](https://github.com/ncbi/fcs) (‚≠ê76 ¬∑ None) Foreign Contamination Screening caller scripts and documentation (PMID: [38409096](https://pubmed.ncbi.nlm.nih.gov/38409096))

599. [s312569/clj-biosequence](https://github.com/s312569/clj-biosequence) (‚≠ê76 ¬∑ Clojure) A Clojure library designed to make the manipulation of biological sequence data easier. (PMID: [24794932](https://pubmed.ncbi.nlm.nih.gov/24794932))

600. [bcgsc/ntCard](https://github.com/bcgsc/ntCard) (‚≠ê75 ¬∑ C++) Estimating k-mer coverage histogram of genomics data (PMID: [28453674](https://pubmed.ncbi.nlm.nih.gov/28453674))

601. [cafferychen777/ggpicrust2](https://github.com/cafferychen777/ggpicrust2) (‚≠ê75 ¬∑ R) Make Picrust2 Output Analysis and Visualization Easier (PMID: [37527009](https://pubmed.ncbi.nlm.nih.gov/37527009))

602. [jokergoo/rGREAT](https://github.com/jokergoo/rGREAT) (‚≠ê75 ¬∑ R) GREAT Analysis - Functional Enrichment on Genomic Regions (PMID: [36394265](https://pubmed.ncbi.nlm.nih.gov/36394265))

603. [rezacsedu/XAI-for-bioinformatics](https://github.com/rezacsedu/XAI-for-bioinformatics) (‚≠ê75 ¬∑ Jupyter Notebook) Explainable AI for Bioinformatics (PMID: [37478371](https://pubmed.ncbi.nlm.nih.gov/37478371))

604. [JLiangLab/SemanticGenesis](https://github.com/JLiangLab/SemanticGenesis) (‚≠ê74 ¬∑ Python) Official Keras & PyTorch Implementation and Pre-trained Models for Semantic Genesis - MICCAI 2020 (PMID: [35695848](https://pubmed.ncbi.nlm.nih.gov/35695848))

605. [MannLabs/alphatims](https://github.com/MannLabs/alphatims) (‚≠ê74 ¬∑ Python) An open-source Python package for efficient accession and visualization of Bruker TimsTOF raw data from the Mann Labs at the Max Planck Institute of Biochemistry. (PMID: [34543758](https://pubmed.ncbi.nlm.nih.gov/34543758))

606. [Oshlack/JAFFA](https://github.com/Oshlack/JAFFA) (‚≠ê74 ¬∑ C++) JAFFA is a multi-step pipeline that takes either raw RNA-Seq reads, or pre-assembled transcripts, then searches for gene fusions (PMID: [26019724](https://pubmed.ncbi.nlm.nih.gov/26019724); [34991664](https://pubmed.ncbi.nlm.nih.gov/34991664))

607. [asad/ReactionDecoder](https://github.com/asad/ReactionDecoder) (‚≠ê74 ¬∑ Java) Reaction Decoder Tool (RDT) - Atom Atom Mapping Tool (PMID: [27153692](https://pubmed.ncbi.nlm.nih.gov/27153692))

608. [frankvogt/vcf2gwas](https://github.com/frankvogt/vcf2gwas) (‚≠ê74 ¬∑ Python) Python API for comprehensive GWAS analysis using GEMMA (PMID: [34636840](https://pubmed.ncbi.nlm.nih.gov/34636840))

609. [lh3/fermi](https://github.com/lh3/fermi) (‚≠ê74 ¬∑ C) A WGS de novo assembler based on the FMD-index for large genomes (PMID: [22569178](https://pubmed.ncbi.nlm.nih.gov/22569178))

610. [seqan/lambda](https://github.com/seqan/lambda) (‚≠ê74 ¬∑ C++) LAMBDA ‚Äì the Local Aligner for Massive Biological DatA (PMID: [38485699](https://pubmed.ncbi.nlm.nih.gov/38485699))

611. [HRGV/phyloFlash](https://github.com/HRGV/phyloFlash) (‚≠ê73 ¬∑ Perl) phyloFlash - A pipeline to rapidly reconstruct the SSU rRNAs and explore phylogenetic composition of an illumina (meta)genomic dataset. (PMID: [33109753](https://pubmed.ncbi.nlm.nih.gov/33109753))

612. [PapenfussLab/bionix](https://github.com/PapenfussLab/bionix) (‚≠ê73 ¬∑ Nix) Functional highly reproducible bioinformatics pipelines (PMID: [33205815](https://pubmed.ncbi.nlm.nih.gov/33205815))

613. [compgenome365/TCGA-Assembler-2](https://github.com/compgenome365/TCGA-Assembler-2) (‚≠ê73 ¬∑ R) TCGA-Assembler 2 (PMID: [29272348](https://pubmed.ncbi.nlm.nih.gov/29272348))

614. [grailbio/bio](https://github.com/grailbio/bio) (‚≠ê73 ¬∑ Go) Bioinformatic infrastructure libraries (PMID: [31510681](https://pubmed.ncbi.nlm.nih.gov/31510681))

615. [jwalabroad/VariantBam](https://github.com/jwalabroad/VariantBam) (‚≠ê73 ¬∑ Makefile) Filtering and profiling of next-generational sequencing data using region-specific rules (PMID: [27153727](https://pubmed.ncbi.nlm.nih.gov/27153727))

616. [netZoo/netZooPy](https://github.com/netZoo/netZooPy) (‚≠ê73 ¬∑ Jupyter Notebook) netZooPy is a network biology package implemented in Python. (PMID: [35156023](https://pubmed.ncbi.nlm.nih.gov/35156023))

617. [BioEasy/EasyCodeML](https://github.com/BioEasy/EasyCodeML) (‚≠ê72 ¬∑ C)  EasyCodeML: A visual tool for analysis of selection using CodeML (PMID: [31015974](https://pubmed.ncbi.nlm.nih.gov/31015974))

618. [FangpingWan/NeoDTI](https://github.com/FangpingWan/NeoDTI) (‚≠ê72 ¬∑ Python) NeoDTI: Neural integration of neighbor information from a heterogeneous network for discovering new drug-target interactions (PMID: [30561548](https://pubmed.ncbi.nlm.nih.gov/30561548))

619. [TinkerTools/tinker-hp](https://github.com/TinkerTools/tinker-hp) (‚≠ê72 ¬∑ C) Tinker-HP: High-Performance Massively Parallel Evolution of Tinker on CPUs & GPUs (PMID: [33173801](https://pubmed.ncbi.nlm.nih.gov/33173801); [33755446](https://pubmed.ncbi.nlm.nih.gov/33755446))

620. [diazlab/CONICS](https://github.com/diazlab/CONICS) (‚≠ê72 ¬∑ R) CONICS: COpy-Number analysis In single-Cell RNA-Sequencing (PMID: [29897414](https://pubmed.ncbi.nlm.nih.gov/29897414))

621. [fkong7/MeshDeformNet](https://github.com/fkong7/MeshDeformNet) (‚≠ê72 ¬∑ Python) A deep-learning approach for direct whole-heart mesh reconstruction (PMID: [34543913](https://pubmed.ncbi.nlm.nih.gov/34543913))

622. [montilab/hypeR](https://github.com/montilab/hypeR) (‚≠ê72 ¬∑ R) An R Package for Geneset Enrichment Workflows (PMID: [31498385](https://pubmed.ncbi.nlm.nih.gov/31498385))

623. [tderrien/FEELnc](https://github.com/tderrien/FEELnc) (‚≠ê72 ¬∑ Perl) FEELnc : FlExible Extraction of LncRNA (PMID: [28053114](https://pubmed.ncbi.nlm.nih.gov/28053114))

624. [MannLabs/alphamap](https://github.com/MannLabs/alphamap) (‚≠ê71 ¬∑ Jupyter Notebook) An open-source Python package for the visual annotation of proteomics data with sequence specific knowledge. (PMID: [34586352](https://pubmed.ncbi.nlm.nih.gov/34586352))

625. [Phlya/coolpuppy](https://github.com/Phlya/coolpuppy) (‚≠ê71 ¬∑ Python) A versatile tool to perform pile-up analysis on Hi-C data in .cool format. (PMID: [32003791](https://pubmed.ncbi.nlm.nih.gov/32003791))

626. [SaskiaFreytag/schex](https://github.com/SaskiaFreytag/schex) (‚≠ê71 ¬∑ R) None (PMID: [31794001](https://pubmed.ncbi.nlm.nih.gov/31794001))

627. [canlab/MediationToolbox](https://github.com/canlab/MediationToolbox) (‚≠ê71 ¬∑ HTML) Single-level and multi-level mediation analyses for any kind of data, with bootstrap-based significance testing. Neuroimaging-oriented functions allow for mediation effect parametric mapping (mapping of mediation effects across the brain) and multivariate mediation. (PMID: [36569614](https://pubmed.ncbi.nlm.nih.gov/36569614))

628. [chentao1999/MedicalRelationExtraction](https://github.com/chentao1999/MedicalRelationExtraction) (‚≠ê71 ¬∑ Python) The depository support training and testing BERT-CNN model on three medical relation extraction corpora: BioCreative V CDR task corpus, traditional Chinese medicine  literature corpus, and i2b2 temporal relation corpus. (PMID: [31800044](https://pubmed.ncbi.nlm.nih.gov/31800044))

629. [ddarriba/modeltest](https://github.com/ddarriba/modeltest) (‚≠ê71 ¬∑ C++) Best-fit model selection (PMID: [31432070](https://pubmed.ncbi.nlm.nih.gov/31432070))

630. [instadeepai/manyfold](https://github.com/instadeepai/manyfold) (‚≠ê71 ¬∑ Python) üß¨ ManyFold: An efficient and flexible library for training and validating protein folding models (PMID: [36495196](https://pubmed.ncbi.nlm.nih.gov/36495196))

631. [mateidavid/nanocall](https://github.com/mateidavid/nanocall) (‚≠ê71 ¬∑ C++) An Oxford Nanopore Basecaller (PMID: [27614348](https://pubmed.ncbi.nlm.nih.gov/27614348))

632. [yuhoo0302/Segment-Anything-Model-for-Medical-Images](https://github.com/yuhoo0302/Segment-Anything-Model-for-Medical-Images) (‚≠ê71 ¬∑ Python) Codes and models for Medical Image Analysis (MIA) 2023 paper. Segment Anything Model for Medical Images?. (PMID: [38086235](https://pubmed.ncbi.nlm.nih.gov/38086235))

633. [DerKevinRiehl/TransposonUltimate](https://github.com/DerKevinRiehl/TransposonUltimate) (‚≠ê70 ¬∑ None) TransposonUltimate - a holistic set of tools for transposon identification (PMID: [35234904](https://pubmed.ncbi.nlm.nih.gov/35234904))

634. [LiLabAtVT/DeepTE](https://github.com/LiLabAtVT/DeepTE) (‚≠ê70 ¬∑ Python) Neural network classification of TE (PMID: [32415954](https://pubmed.ncbi.nlm.nih.gov/32415954))

635. [Liuxg16/GeoPPI](https://github.com/Liuxg16/GeoPPI) (‚≠ê70 ¬∑ Python) None (PMID: [34347784](https://pubmed.ncbi.nlm.nih.gov/34347784))

636. [RemiAllio/MitoFinder](https://github.com/RemiAllio/MitoFinder) (‚≠ê70 ¬∑ Python) MitoFinder: efficient automated large-scale extraction of mitogenomic data from high throughput sequencing data (PMID: [32243090](https://pubmed.ncbi.nlm.nih.gov/32243090))

637. [bigbio/proteomics-metadata-standard](https://github.com/bigbio/proteomics-metadata-standard) (‚≠ê70 ¬∑ Python) The Proteomics Experimental Design file format: Standard for experimental design annotation (PMID: [32786688](https://pubmed.ncbi.nlm.nih.gov/32786688))

638. [cosconatilab/PyRMD](https://github.com/cosconatilab/PyRMD) (‚≠ê70 ¬∑ Python) AI-powered Virtual Screening (PMID: [34270903](https://pubmed.ncbi.nlm.nih.gov/34270903); [37552222](https://pubmed.ncbi.nlm.nih.gov/37552222))

639. [jltsiren/gcsa2](https://github.com/jltsiren/gcsa2) (‚≠ê70 ¬∑ C++) BWT-based index for graphs (PMID: [31406990](https://pubmed.ncbi.nlm.nih.gov/31406990))

640. [lh3/dna-nn](https://github.com/lh3/dna-nn) (‚≠ê70 ¬∑ C) Model and predict short DNA sequence features with neural networks (PMID: [30989183](https://pubmed.ncbi.nlm.nih.gov/30989183))

641. [lmmpf/PyAutoFEP](https://github.com/lmmpf/PyAutoFEP) (‚≠ê70 ¬∑ Python) PyAutoFEP: an automated FEP workflow for GROMACS integrating enhanced sampling methods (PMID: [34142828](https://pubmed.ncbi.nlm.nih.gov/34142828))

642. [GfellerLab/EPIC](https://github.com/GfellerLab/EPIC) (‚≠ê69 ¬∑ R) Repository for the R package EPIC, to Estimate the Proportion of Immune and Cancer cells from bulk gene expression data. (PMID: [32124324](https://pubmed.ncbi.nlm.nih.gov/32124324))

643. [VoiceBeer/MS-MDA](https://github.com/VoiceBeer/MS-MDA) (‚≠ê69 ¬∑ Jupyter Notebook) Code of paper MS-MDA: Multisource Marginal Distribution Adaptation for Cross-subject and Cross-session EEG Emotion Recognition (PMID: [34949983](https://pubmed.ncbi.nlm.nih.gov/34949983))

644. [gnina/models](https://github.com/gnina/models) (‚≠ê69 ¬∑ Python) Trained caffe models (PMID: [32865404](https://pubmed.ncbi.nlm.nih.gov/32865404))

645. [illumina/REViewer](https://github.com/illumina/REViewer) (‚≠ê69 ¬∑ C++) A tool for visualizing alignments of reads in regions containing tandem repeats (PMID: [35948990](https://pubmed.ncbi.nlm.nih.gov/35948990))

646. [marekborowiec/AMAS](https://github.com/marekborowiec/AMAS) (‚≠ê69 ¬∑ Python) Calculate summary statistics and manipulate multiple sequence alignments (PMID: [26835189](https://pubmed.ncbi.nlm.nih.gov/26835189))

647. [mozack/abra](https://github.com/mozack/abra) (‚≠ê69 ¬∑ Java) Assembly Based ReAligner (PMID: [24907369](https://pubmed.ncbi.nlm.nih.gov/24907369))

648. [vertgenlab/gonomics](https://github.com/vertgenlab/gonomics) (‚≠ê69 ¬∑ Go) A collection of genomics software tools written in Go (golang). (PMID: [37624924](https://pubmed.ncbi.nlm.nih.gov/37624924))

649. [xiezhq/ISEScan](https://github.com/xiezhq/ISEScan) (‚≠ê69 ¬∑ Python) A python pipeline to identify IS (Insertion Sequence) elements in genome and metagenome (PMID: [29077810](https://pubmed.ncbi.nlm.nih.gov/29077810))

650. [yukiteruono/pbsim2](https://github.com/yukiteruono/pbsim2) (‚≠ê69 ¬∑ C++) PBSIM2: a simulator for long read sequencers with a novel generative model of quality scores (PMID: [32976553](https://pubmed.ncbi.nlm.nih.gov/32976553))

651. [zuoxinian/CCS](https://github.com/zuoxinian/CCS) (‚≠ê69 ¬∑ MATLAB) Connectome Computation System (PMID: [36738058](https://pubmed.ncbi.nlm.nih.gov/36738058))

652. [Genentech/pviz](https://github.com/Genentech/pviz) (‚≠ê68 ¬∑ JavaScript) Pviz (PMID: [25147360](https://pubmed.ncbi.nlm.nih.gov/25147360))

653. [MaterialsDiscovery/PyChemia](https://github.com/MaterialsDiscovery/PyChemia) (‚≠ê68 ¬∑ Python) Python Materials Discovery Framework (PMID: [27232694](https://pubmed.ncbi.nlm.nih.gov/27232694))

654. [ShellyCoder/cellcall](https://github.com/ShellyCoder/cellcall) (‚≠ê68 ¬∑ R) inferring cell-cell communication from scRNA-seq of ligand-receptor (PMID: [34331449](https://pubmed.ncbi.nlm.nih.gov/34331449))

655. [TheWebMonks/meshmonk](https://github.com/TheWebMonks/meshmonk) (‚≠ê68 ¬∑ C++) Open mesh-to-mesh registration framework (PMID: [32746977](https://pubmed.ncbi.nlm.nih.gov/32746977))

656. [chainsawriot/rang](https://github.com/chainsawriot/rang) (‚≠ê68 ¬∑ R) üê∂ (Re)constructing R computational environments (PMID: [37289797](https://pubmed.ncbi.nlm.nih.gov/37289797))

657. [citiususc/SparkBWA](https://github.com/citiususc/SparkBWA) (‚≠ê68 ¬∑ C) SparkBWA is a new tool that exploits the capabilities of a Big Data technology as Apache Spark to boost the performance of one of the most widely adopted sequence aligner, the Burrows-Wheeler Aligner (BWA). (PMID: [27182962](https://pubmed.ncbi.nlm.nih.gov/27182962))

658. [eturro/mmseq](https://github.com/eturro/mmseq) (‚≠ê68 ¬∑ C++) Haplotype, isoform and gene level expression analysis using multi-mapping RNA-seq reads (PMID: [24281695](https://pubmed.ncbi.nlm.nih.gov/24281695))

659. [lh3/bfc](https://github.com/lh3/bfc) (‚≠ê68 ¬∑ TeX) High-performance error correction for Illumina resequencing data (PMID: [25953801](https://pubmed.ncbi.nlm.nih.gov/25953801))

660. [lh3/ropebwt2](https://github.com/lh3/ropebwt2) (‚≠ê68 ¬∑ TeX) Incremental construction of FM-index for DNA sequences (PMID: [25107872](https://pubmed.ncbi.nlm.nih.gov/25107872))

661. [natir/yacrd](https://github.com/natir/yacrd) (‚≠ê68 ¬∑ Rust) Yet Another Chimeric Read Detector (PMID: [32315402](https://pubmed.ncbi.nlm.nih.gov/32315402))

662. [scwatts/FastSpar](https://github.com/scwatts/FastSpar) (‚≠ê68 ¬∑ C++) Rapid and scalable correlation estimation for compositional data (PMID: [30169561](https://pubmed.ncbi.nlm.nih.gov/30169561))

663. [sebwouters/chemps2](https://github.com/sebwouters/chemps2) (‚≠ê68 ¬∑ C++) CheMPS2: a spin-adapted implementation of DMRG for ab initio quantum chemistry (PMID: [27497552](https://pubmed.ncbi.nlm.nih.gov/27497552))

664. [chaoszhang/ASTER](https://github.com/chaoszhang/ASTER) (‚≠ê67 ¬∑ C++) Accurate Species Tree EstimatoR series: a family of optimation algorithms for species tree inference implemented in C++ (including ASTRAL-Pro & Weighted ASTRAL) (PMID: [36094339](https://pubmed.ncbi.nlm.nih.gov/36094339))

665. [comprna/METEORE](https://github.com/comprna/METEORE) (‚≠ê67 ¬∑ R) Automatic DNA methylation detection from nanopore tools and their consensus model (PMID: [34103501](https://pubmed.ncbi.nlm.nih.gov/34103501))

666. [hdaoud/ORBSLAMM](https://github.com/hdaoud/ORBSLAMM) (‚≠ê67 ¬∑ C++) None (PMID: [29702697](https://pubmed.ncbi.nlm.nih.gov/29702697))

667. [histocartography/hact-net](https://github.com/histocartography/hact-net) (‚≠ê67 ¬∑ Python) None (PMID: [34781160](https://pubmed.ncbi.nlm.nih.gov/34781160))

668. [pymodproject/pymod](https://github.com/pymodproject/pymod) (‚≠ê67 ¬∑ Python) PyMod 3 - sequence similarity searches, multiple sequence/structure alignments, and homology modeling within PyMOL. (PMID: [28158668](https://pubmed.ncbi.nlm.nih.gov/28158668); [33010156](https://pubmed.ncbi.nlm.nih.gov/33010156))

669. [simonvh/fluff](https://github.com/simonvh/fluff) (‚≠ê67 ¬∑ Python) Fluff is a Python package that contains several scripts to produce pretty, publication-quality figures for next-generation sequencing experiments. (PMID: [27547532](https://pubmed.ncbi.nlm.nih.gov/27547532))

670. [zoecastillo/microbiomeExplorer](https://github.com/zoecastillo/microbiomeExplorer) (‚≠ê67 ¬∑ R) The Microbiome Explorer provides methods to analyze and visualize microbial community sequencing data either from the R command line or through a Shiny application. Written by Janina Reeder and Joseph N. Paulson. (PMID: [32960962](https://pubmed.ncbi.nlm.nih.gov/32960962))

671. [AllonKleinLab/SPRING](https://github.com/AllonKleinLab/SPRING) (‚≠ê66 ¬∑ OpenEdge ABL) None (PMID: [29228172](https://pubmed.ncbi.nlm.nih.gov/29228172))

672. [GanjinZero/CODER](https://github.com/GanjinZero/CODER) (‚≠ê66 ¬∑ Python) CODER: Knowledge infused cross-lingual medical term embedding for term normalization. [JBI, ACL-BioNLP 2022] (PMID: [34990838](https://pubmed.ncbi.nlm.nih.gov/34990838))

673. [Pbdas/epa-ng](https://github.com/Pbdas/epa-ng) (‚≠ê66 ¬∑ C++) Massively parallel phylogenetic placement of genetic sequences (PMID: [30165689](https://pubmed.ncbi.nlm.nih.gov/30165689))

674. [brinkmanlab/GenomeD3Plot](https://github.com/brinkmanlab/GenomeD3Plot) (‚≠ê66 ¬∑ JavaScript) SVG based genome viewer written in javascript using D3 (PMID: [26093150](https://pubmed.ncbi.nlm.nih.gov/26093150))

675. [czodrowskilab/Machine-learning-meets-pKa](https://github.com/czodrowskilab/Machine-learning-meets-pKa) (‚≠ê66 ¬∑ Jupyter Notebook) None (PMID: [32226607](https://pubmed.ncbi.nlm.nih.gov/32226607))

676. [dereneaton/ipyrad](https://github.com/dereneaton/ipyrad) (‚≠ê66 ¬∑ Jupyter Notebook) Interactive assembly and analysis of RAD-seq data sets (PMID: [31904816](https://pubmed.ncbi.nlm.nih.gov/31904816))

677. [epiviz/epiviz](https://github.com/epiviz/epiviz) (‚≠ê66 ¬∑ JavaScript) EpiViz is a scientific information visualization tool for genetic and epigenetic data, used to aid in the exploration and understanding of correlations between various genome features. (PMID: [31782758](https://pubmed.ncbi.nlm.nih.gov/31782758))

678. [microsoft/IGT-Intermolecular-Graph-Transformer](https://github.com/microsoft/IGT-Intermolecular-Graph-Transformer) (‚≠ê66 ¬∑ None) None (PMID: [35514186](https://pubmed.ncbi.nlm.nih.gov/35514186))

679. [tlemane/kmtricks](https://github.com/tlemane/kmtricks) (‚≠ê66 ¬∑ C++) modular k-mer count matrix and Bloom filter construction for large read collections (PMID: [36699393](https://pubmed.ncbi.nlm.nih.gov/36699393))

680. [broadinstitute/oncotator](https://github.com/broadinstitute/oncotator) (‚≠ê65 ¬∑ Python) None (PMID: [25703262](https://pubmed.ncbi.nlm.nih.gov/25703262))

681. [lanagarmire/DeepProg](https://github.com/lanagarmire/DeepProg) (‚≠ê65 ¬∑ Python) Deep-Learning framework for multi-omic and survival data integration (PMID: [34261540](https://pubmed.ncbi.nlm.nih.gov/34261540))

682. [lidaof/eg-react](https://github.com/lidaof/eg-react) (‚≠ê65 ¬∑ JavaScript) WashU Epigenome Browser (PMID: [32941613](https://pubmed.ncbi.nlm.nih.gov/32941613))

683. [WatsonLab/MAGpy](https://github.com/WatsonLab/MAGpy) (‚≠ê64 ¬∑ Python) Snakemake pipeline for downstream analysis of metagenome-assembled genomes (MAGs) (pronounced mag-pie) (PMID: [30418481](https://pubmed.ncbi.nlm.nih.gov/30418481))

684. [aatapa/RLScore](https://github.com/aatapa/RLScore) (‚≠ê64 ¬∑ Python) None (PMID: [30329015](https://pubmed.ncbi.nlm.nih.gov/30329015))

685. [amcrisan/Adjutant](https://github.com/amcrisan/Adjutant) (‚≠ê64 ¬∑ R) Runs a pubmed query, returns results and allows user to explore high-level structure of returned documents (PMID: [30875428](https://pubmed.ncbi.nlm.nih.gov/30875428))

686. [deprekate/PHANOTATE](https://github.com/deprekate/PHANOTATE) (‚≠ê64 ¬∑ Python) PHANOTATE: a tool to annotate phage genomes. (PMID: [31329826](https://pubmed.ncbi.nlm.nih.gov/31329826))

687. [dphansti/Sushi](https://github.com/dphansti/Sushi) (‚≠ê64 ¬∑ R) Tools for visualizing genomics data (PMID: [24903420](https://pubmed.ncbi.nlm.nih.gov/24903420))

688. [fanglab/nanodisco](https://github.com/fanglab/nanodisco) (‚≠ê64 ¬∑ R) nanodisco: a toolbox for discovering and exploiting multiple types of DNA methylation from individual bacteria and microbiomes using nanopore sequencing. (PMID: [33820988](https://pubmed.ncbi.nlm.nih.gov/33820988))

689. [huangzhii/SALMON](https://github.com/huangzhii/SALMON) (‚≠ê64 ¬∑ Python) SALMON: Survival Analysis Learning with Multi-Omics Neural Networks (PMID: [30906311](https://pubmed.ncbi.nlm.nih.gov/30906311))

690. [jwintersinger/kablammo](https://github.com/jwintersinger/kablammo) (‚≠ê64 ¬∑ JavaScript) Interactive, web-based BLAST results visualization tool. Accessible at https://kablammo.wasmuthlab.org. (PMID: [25481007](https://pubmed.ncbi.nlm.nih.gov/25481007))

691. [meringlab/FlashWeave](https://github.com/meringlab/FlashWeave) (‚≠ê64 ¬∑ Julia) Inference of microbial interaction networks from large-scale heterogeneous abundance data (PMID: [31542415](https://pubmed.ncbi.nlm.nih.gov/31542415))

692. [niemasd/TreeCluster](https://github.com/niemasd/TreeCluster) (‚≠ê64 ¬∑ Python) Efficient phylogenetic clustering of viral sequences (PMID: [31437182](https://pubmed.ncbi.nlm.nih.gov/31437182))

693. [powellgenomicslab/scPred](https://github.com/powellgenomicslab/scPred) (‚≠ê64 ¬∑ R) scPred package for cell type prediction from scRNA-seq data (PMID: [31829268](https://pubmed.ncbi.nlm.nih.gov/31829268))

694. [robotoD/GenoVi](https://github.com/robotoD/GenoVi) (‚≠ê64 ¬∑ Python) GenoVi, an automated customizable circular genome visualizer for bacteria and archaea (PMID: [37014908](https://pubmed.ncbi.nlm.nih.gov/37014908))

695. [OpenWaterAnalytics/epanet-dev](https://github.com/OpenWaterAnalytics/epanet-dev) (‚≠ê63 ¬∑ C++) Development Repository for the object-oriented implementation of EPANET (PMID: [31096428](https://pubmed.ncbi.nlm.nih.gov/31096428))

696. [kweimann/ecg-transfer-learning](https://github.com/kweimann/ecg-transfer-learning) (‚≠ê63 ¬∑ Python) None (PMID: [33664343](https://pubmed.ncbi.nlm.nih.gov/33664343))

697. [neurogenomics/rworkflows](https://github.com/neurogenomics/rworkflows) (‚≠ê63 ¬∑ HTML) Continuous integration for R packages. üîÄ Automates testing ‚úÖ, documentation website building üì¶, & containerised deployment üê≥. (PMID: [38167858](https://pubmed.ncbi.nlm.nih.gov/38167858))

698. [ocxtal/libgaba](https://github.com/ocxtal/libgaba) (‚≠ê63 ¬∑ C) Adaptive semi-global banded alignment on string graphs (PMID: [29504909](https://pubmed.ncbi.nlm.nih.gov/29504909))

699. [samsledje/D-SCRIPT](https://github.com/samsledje/D-SCRIPT) (‚≠ê63 ¬∑ Jupyter Notebook)  A structure-aware interpretable deep learning model for sequence-based prediction of protein-protein interactions (PMID: [37897686](https://pubmed.ncbi.nlm.nih.gov/37897686))

700. [superlouis/GATGNN](https://github.com/superlouis/GATGNN) (‚≠ê63 ¬∑ Python) Pytorch Repository for our work: Graph convolutional neural networks with global attention for improved materials property prediction (PMID: [32766627](https://pubmed.ncbi.nlm.nih.gov/32766627))

701. [ParkerLab/ataqv](https://github.com/ParkerLab/ataqv) (‚≠ê62 ¬∑ C++) A toolkit for QC and visualization of ATAC-seq results. (PMID: [32213349](https://pubmed.ncbi.nlm.nih.gov/32213349))

702. [carmonalab/STACAS](https://github.com/carmonalab/STACAS) (‚≠ê62 ¬∑ R) R package for semi-supervised single-cell data integration (PMID: [32845323](https://pubmed.ncbi.nlm.nih.gov/32845323))

703. [danielhuson/megan-ce](https://github.com/danielhuson/megan-ce) (‚≠ê62 ¬∑ Java) MEGAN Community Edition (PMID: [27327495](https://pubmed.ncbi.nlm.nih.gov/27327495))

704. [genesofeve/projectR](https://github.com/genesofeve/projectR) (‚≠ê62 ¬∑ HTML) None (PMID: [32167521](https://pubmed.ncbi.nlm.nih.gov/32167521))

705. [ggonnella/gfaviz](https://github.com/ggonnella/gfaviz) (‚≠ê62 ¬∑ C++) Graphical interactive tool for the visualization of sequence graphs in GFA format. (PMID: [30596893](https://pubmed.ncbi.nlm.nih.gov/30596893))

706. [hellloxiaotian/ESRGCNN](https://github.com/hellloxiaotian/ESRGCNN) (‚≠ê62 ¬∑ Python) Image Super-resolution with An Enhanced Group Convolutional Neural Network (Neural Networks, 2022) (PMID: [35779445](https://pubmed.ncbi.nlm.nih.gov/35779445))

707. [husonlab/megan-ce](https://github.com/husonlab/megan-ce) (‚≠ê62 ¬∑ Java) MEGAN Community Edition (PMID: [36825821](https://pubmed.ncbi.nlm.nih.gov/36825821))

708. [nadeemlab/CEP](https://github.com/nadeemlab/CEP) (‚≠ê62 ¬∑ Python) Computational Endoscopy Platform (advanced deep learning toolset for analyzing endoscopy videos) [MICCAI'22, MICCAI'21, ISBI'21, CVPR'20] (PMID: [34642595](https://pubmed.ncbi.nlm.nih.gov/34642595); [35403172](https://pubmed.ncbi.nlm.nih.gov/35403172); [36178456](https://pubmed.ncbi.nlm.nih.gov/36178456))

709. [ncbi/BAMscale](https://github.com/ncbi/BAMscale) (‚≠ê62 ¬∑ C) BAMscale is a one-step tool for either 1) quantifying and normalizing the coverage of peaks or 2) generated scaled BigWig files for easy visualization of commonly used DNA-seq capture based methods. (PMID: [32321568](https://pubmed.ncbi.nlm.nih.gov/32321568))

710. [nikhilroxtomar/TGANet](https://github.com/nikhilroxtomar/TGANet) (‚≠ê62 ¬∑ Python) TGANet: Text-guided attention for improved polyp segmentation [Early Accepted & Student Travel Award at MICCAI 2022] (PMID: [36780239](https://pubmed.ncbi.nlm.nih.gov/36780239))

711. [qiyunzhu/woltka](https://github.com/qiyunzhu/woltka) (‚≠ê62 ¬∑ Python) Woltka: a versatile meta'omic data classifier (PMID: [36073806](https://pubmed.ncbi.nlm.nih.gov/36073806))

712. [simonhmartin/twisst](https://github.com/simonhmartin/twisst) (‚≠ê62 ¬∑ Python) Topology weighting by iterative sampling of sub-trees (PMID: [28341652](https://pubmed.ncbi.nlm.nih.gov/28341652))

713. [tk2/RetroSeq](https://github.com/tk2/RetroSeq) (‚≠ê62 ¬∑ Perl) RetroSeq is a bioinformatics tool that searches for mobile element insertions from aligned reads in a BAM file and a library of reference transposable elements. Please read the wiki page (link below) for usage instructions. Also, there is a page on the wiki describing how the 1000 genomes CEU trio was carried out with the files and parameters used for the various steps. (PMID: [23233656](https://pubmed.ncbi.nlm.nih.gov/23233656))

714. [tripal/tripal](https://github.com/tripal/tripal) (‚≠ê62 ¬∑ PHP) The Tripal package is  a suite of Drupal modules for creating biological (genomic, genetic, breeding) websites. Visit the Tripal homepage at http://tripal.info for documentation, support, and other information.  The Drupal project page is at http://drupal.org/project/tripal. (PMID: [29220446](https://pubmed.ncbi.nlm.nih.gov/29220446))

715. [zqgao22/HIGH-PPI](https://github.com/zqgao22/HIGH-PPI) (‚≠ê62 ¬∑ Python) None (PMID: [36841846](https://pubmed.ncbi.nlm.nih.gov/36841846))

716. [KCCG/ClinSV](https://github.com/KCCG/ClinSV) (‚≠ê61 ¬∑ Perl) Robust detection of clinically relevant structural and copy number variation from whole genome sequencing data (PMID: [33632298](https://pubmed.ncbi.nlm.nih.gov/33632298))

717. [RitchieLab/eQTpLot](https://github.com/RitchieLab/eQTpLot) (‚≠ê61 ¬∑ R)  Visualization of Colocalization Between eQTL and GWAS Data (PMID: [34273980](https://pubmed.ncbi.nlm.nih.gov/34273980))

718. [dewyman/TranscriptClean](https://github.com/dewyman/TranscriptClean) (‚≠ê61 ¬∑ Python) Correct mismatches, microindels, and noncanonical splice junctions in long reads that have been mapped to the genome (PMID: [29912287](https://pubmed.ncbi.nlm.nih.gov/29912287))

719. [KailiWang1/DeepDTAF](https://github.com/KailiWang1/DeepDTAF) (‚≠ê60 ¬∑ Jupyter Notebook)  a deep learning architecture for protein-ligand binding affinity prediction (PMID: [33834190](https://pubmed.ncbi.nlm.nih.gov/33834190))

720. [MannLabs/ProteomicsVisualization](https://github.com/MannLabs/ProteomicsVisualization) (‚≠ê60 ¬∑ Jupyter Notebook) This is a collection of minimal code to visualize various aspects of proteomics data. (PMID: [35107884](https://pubmed.ncbi.nlm.nih.gov/35107884))

721. [Oshlack/STRetch](https://github.com/Oshlack/STRetch) (‚≠ê60 ¬∑ Python) Method for detecting STR expansions from short-read sequencing data (PMID: [30129428](https://pubmed.ncbi.nlm.nih.gov/30129428))

722. [VCCRI/CIDR](https://github.com/VCCRI/CIDR) (‚≠ê60 ¬∑ R) Clustering through Imputation and Dimensionality Reduction (PMID: [28351406](https://pubmed.ncbi.nlm.nih.gov/28351406))

723. [bede/hostile](https://github.com/bede/hostile) (‚≠ê60 ¬∑ Python) Accurate host read removal (PMID: [38039142](https://pubmed.ncbi.nlm.nih.gov/38039142))

724. [ggonnella/gfapy](https://github.com/ggonnella/gfapy) (‚≠ê60 ¬∑ Python) Gfapy: a flexible and extensible software library for handling sequence graphs in Python (PMID: [28645150](https://pubmed.ncbi.nlm.nih.gov/28645150))

725. [griffithlab/pVAC-Seq](https://github.com/griffithlab/pVAC-Seq) (‚≠ê60 ¬∑ Python) DEPRECATED. This tool has been superseded by https://github.com/griffithlab/pVACtools (PMID: [26825632](https://pubmed.ncbi.nlm.nih.gov/26825632))

726. [kimmo1019/DeepCDR](https://github.com/kimmo1019/DeepCDR) (‚≠ê60 ¬∑ Python) Cancer Drug Response Prediction via a Hybrid Graph Convolutional Network (PMID: [33381841](https://pubmed.ncbi.nlm.nih.gov/33381841))

727. [lrkrol/SEREEGA](https://github.com/lrkrol/SEREEGA) (‚≠ê60 ¬∑ MATLAB) SEREEGA: Simulating Event-Related EEG Activity (PMID: [30114381](https://pubmed.ncbi.nlm.nih.gov/30114381))

728. [ygidtu/trackplot](https://github.com/ygidtu/trackplot) (‚≠ê60 ¬∑ Python) trackplot is a tool for visualizing various next-generation sequencing (NGS) data, including DNA-seq, RNA-seq, single-cell RNA-seq and full-length sequencing datasets. (PMID: [37669275](https://pubmed.ncbi.nlm.nih.gov/37669275))

729. [HALFpipe/HALFpipe](https://github.com/HALFpipe/HALFpipe) (‚≠ê59 ¬∑ Python) ENIGMA HALFpipe is a user-friendly software that facilitates reproducible analysis of fMRI data (PMID: [35305030](https://pubmed.ncbi.nlm.nih.gov/35305030))

730. [SAMtoBAM/MUMandCo](https://github.com/SAMtoBAM/MUMandCo) (‚≠ê59 ¬∑ Shell) MUM&Co is a simple bash script that uses Whole Genome Alignment information provided by MUMmer (only v4) to detect Structural Variation (PMID: [32096823](https://pubmed.ncbi.nlm.nih.gov/32096823))

731. [Vini2/phables](https://github.com/Vini2/phables) (‚≠ê59 ¬∑ Python) ü´ßüß¨ From fragmented assemblies to high-quality bacteriophage genomes (PMID: [37738590](https://pubmed.ncbi.nlm.nih.gov/37738590))

732. [ablab/viralVerify](https://github.com/ablab/viralVerify) (‚≠ê59 ¬∑ Python) viralVerify: viral contig verification tool (PMID: [32413137](https://pubmed.ncbi.nlm.nih.gov/32413137))

733. [dtamayo/spock](https://github.com/dtamayo/spock) (‚≠ê59 ¬∑ Python) üññ A package to determine whether planetary orbital configurations will live long and prosper (PMID: [34599094](https://pubmed.ncbi.nlm.nih.gov/34599094))

734. [iprada/Circle-Map](https://github.com/iprada/Circle-Map) (‚≠ê59 ¬∑ Python) A method for circular DNA detection based on probabilistic mapping of ultrashort reads (PMID: [31830908](https://pubmed.ncbi.nlm.nih.gov/31830908))

735. [ksahlin/ultra](https://github.com/ksahlin/ultra) (‚≠ê59 ¬∑ Python) Long-read splice alignment with high accuracy (PMID: [34302453](https://pubmed.ncbi.nlm.nih.gov/34302453))

736. [mourisl/Rcorrector](https://github.com/mourisl/Rcorrector) (‚≠ê59 ¬∑ C++) Error correction for Illumina RNA-seq reads (PMID: [26500767](https://pubmed.ncbi.nlm.nih.gov/26500767))

737. [paiardin/DockingPie](https://github.com/paiardin/DockingPie) (‚≠ê59 ¬∑ Python) A Consensus Docking Plugin for PyMOL (PMID: [35792827](https://pubmed.ncbi.nlm.nih.gov/35792827))

738. [ymcui/expmrc](https://github.com/ymcui/expmrc) (‚≠ê59 ¬∑ Python) ExpMRC: Explainability Evaluation for Machine Reading Comprehension (PMID: [35497046](https://pubmed.ncbi.nlm.nih.gov/35497046))

739. [BenoitMorel/GeneRax](https://github.com/BenoitMorel/GeneRax) (‚≠ê58 ¬∑ C++) None (PMID: [32502238](https://pubmed.ncbi.nlm.nih.gov/32502238); [35021210](https://pubmed.ncbi.nlm.nih.gov/35021210))

740. [NBISweden/EMBLmyGFF3](https://github.com/NBISweden/EMBLmyGFF3) (‚≠ê58 ¬∑ Python) An efficient way to convert gff3 annotation files into EMBL format ready to submit. (PMID: [30103816](https://pubmed.ncbi.nlm.nih.gov/30103816))

741. [RVanDamme/MUFFIN](https://github.com/RVanDamme/MUFFIN) (‚≠ê58 ¬∑ Nextflow) hybrid assembly and differential binning workflow for metagenomics, transcriptomics and pathway analysis (PMID: [33561126](https://pubmed.ncbi.nlm.nih.gov/33561126))

742. [YifanDengWHU/DDIMDL](https://github.com/YifanDengWHU/DDIMDL) (‚≠ê58 ¬∑ Python) None (PMID: [32407508](https://pubmed.ncbi.nlm.nih.gov/32407508))

743. [bcgsc/ntedit](https://github.com/bcgsc/ntedit) (‚≠ê58 ¬∑ C++) ‚úèÔ∏èultra fast, scalable genome assembly polishing and snv detection (PMID: [31095290](https://pubmed.ncbi.nlm.nih.gov/31095290); [35567771](https://pubmed.ncbi.nlm.nih.gov/35567771))

744. [brettc/partitionfinder](https://github.com/brettc/partitionfinder) (‚≠ê58 ¬∑ Python) PartitionFinder discovers optimal partitioning schemes for DNA sequences. (PMID: [28013191](https://pubmed.ncbi.nlm.nih.gov/28013191))

745. [dantaki/SV2](https://github.com/dantaki/SV2) (‚≠ê58 ¬∑ Python) Support Vector Structural Variation Genotyper (PMID: [29300834](https://pubmed.ncbi.nlm.nih.gov/29300834))

746. [eliberis/parapred](https://github.com/eliberis/parapred) (‚≠ê58 ¬∑ Python) Paratope Prediction using Deep Learning (PMID: [29672675](https://pubmed.ncbi.nlm.nih.gov/29672675))

747. [ncbi/SSDraw](https://github.com/ncbi/SSDraw) (‚≠ê58 ¬∑ Jupyter Notebook) None (PMID: [37953705](https://pubmed.ncbi.nlm.nih.gov/37953705))

748. [BTheDragonMaster/pikachu](https://github.com/BTheDragonMaster/pikachu) (‚≠ê57 ¬∑ Python) Python-based Informatics Kit for Analysing Chemical Units (PMID: [35672769](https://pubmed.ncbi.nlm.nih.gov/35672769))

749. [Genotek/ClassifyCNV](https://github.com/Genotek/ClassifyCNV) (‚≠ê57 ¬∑ Python) ClassifyCNV: a tool for clinical annotation of copy-number variants (PMID: [33230148](https://pubmed.ncbi.nlm.nih.gov/33230148))

750. [QuKunLab/ATAC-pipe](https://github.com/QuKunLab/ATAC-pipe) (‚≠ê57 ¬∑ Python) Analysis pipeline for ATAC-seq Data (PMID: [29982337](https://pubmed.ncbi.nlm.nih.gov/29982337))

751. [RyanCook94/inphared](https://github.com/RyanCook94/inphared) (‚≠ê57 ¬∑ Perl) Providing up-to-date phage genome databases, metrics and useful input files for a number of bioinformatic pipelines. (PMID: [36159887](https://pubmed.ncbi.nlm.nih.gov/36159887))

752. [SRKH/S4NN](https://github.com/SRKH/S4NN) (‚≠ê57 ¬∑ Jupyter Notebook) Temporal backpropagation for spiking neural networks with one spike per neuron, by S. R. Kheradpisheh and T. Masquelier, International Journal of Neural Systems (2020), doi: 10.1142/S0129065720500276 (PMID: [32466691](https://pubmed.ncbi.nlm.nih.gov/32466691))

753. [harbourlab/UPhyloplot2](https://github.com/harbourlab/UPhyloplot2) (‚≠ê57 ¬∑ Python) Draw phylogenetic trees of tumor evolution (PMID: [34090344](https://pubmed.ncbi.nlm.nih.gov/34090344))

754. [jltsiren/gbwt](https://github.com/jltsiren/gbwt) (‚≠ê57 ¬∑ C++) Substring index for paths in a graph (PMID: [31406990](https://pubmed.ncbi.nlm.nih.gov/31406990))

755. [ksahlin/BESST](https://github.com/ksahlin/BESST) (‚≠ê57 ¬∑ Python) BESST - scaffolder for genomic assemblies (PMID: [27153683](https://pubmed.ncbi.nlm.nih.gov/27153683))

756. [lczech/gappa](https://github.com/lczech/gappa) (‚≠ê57 ¬∑ C++) A toolkit for analyzing and visualizing phylogenetic (placement) data (PMID: [30169747](https://pubmed.ncbi.nlm.nih.gov/30169747); [32016344](https://pubmed.ncbi.nlm.nih.gov/32016344))

757. [rnacentral/R2DT](https://github.com/rnacentral/R2DT) (‚≠ê57 ¬∑ PostScript) Visualise RNA secondary structure in consistent, reproducible and recognisable layouts (PMID: [34108470](https://pubmed.ncbi.nlm.nih.gov/34108470))

758. [rundherum/pymia](https://github.com/rundherum/pymia) (‚≠ê57 ¬∑ Python) pymia: A Python package for data handling and evaluation in deep learning-based medical image analysis (PMID: [33137700](https://pubmed.ncbi.nlm.nih.gov/33137700))

759. [telatin/BamToCov](https://github.com/telatin/BamToCov) (‚≠ê57 ¬∑ Nim) üèî  coverage extraction from BAM/CRAM files, supporting targets üìä (PMID: [35199151](https://pubmed.ncbi.nlm.nih.gov/35199151))

760. [volkamerlab/KinFragLib](https://github.com/volkamerlab/KinFragLib) (‚≠ê57 ¬∑ Jupyter Notebook) Kinase-focused fragment library (PMID: [33155465](https://pubmed.ncbi.nlm.nih.gov/33155465))

761. [JhuangLab/BioInstaller](https://github.com/JhuangLab/BioInstaller) (‚≠ê56 ¬∑ R) A comprehensive R package to construct interactive and reproducible biological data analysis applications based on the R platform (PMID: [30402350](https://pubmed.ncbi.nlm.nih.gov/30402350))

762. [OpenNFT/OpenNFT](https://github.com/OpenNFT/OpenNFT) (‚≠ê56 ¬∑ MATLAB) OpenNFT is an integrated software package designed for neurofeedback training and automatic real-time fMRI quality assessment (rtQA). (PMID: [28795112](https://pubmed.ncbi.nlm.nih.gov/28795112))

763. [PapenfussLab/bioshake](https://github.com/PapenfussLab/bioshake) (‚≠ê56 ¬∑ Haskell) Bioinformatics pipelines with Haskell and Shake (PMID: [31328031](https://pubmed.ncbi.nlm.nih.gov/31328031))

764. [RuleWorld/bionetgen](https://github.com/RuleWorld/bionetgen) (‚≠ê56 ¬∑ Perl) Rule-based modeling framework (PMID: [27402907](https://pubmed.ncbi.nlm.nih.gov/27402907))

765. [SofieVG/FlowSOM](https://github.com/SofieVG/FlowSOM) (‚≠ê56 ¬∑ R) Using self-organizing maps for visualization and interpretation of cytometry data (PMID: [25573116](https://pubmed.ncbi.nlm.nih.gov/25573116))

766. [XiaoTaoWang/HiCLift](https://github.com/XiaoTaoWang/HiCLift) (‚≠ê56 ¬∑ Python) A fast and efficient tool for converting chromatin interaction data between genome assemblies (PMID: [37335863](https://pubmed.ncbi.nlm.nih.gov/37335863))

767. [abehr/pong](https://github.com/abehr/pong) (‚≠ê56 ¬∑ Python) Fast analysis and visualization of latent clusters in population genetic data (PMID: [27283948](https://pubmed.ncbi.nlm.nih.gov/27283948))

768. [bene51/3Dscript](https://github.com/bene51/3Dscript) (‚≠ê56 ¬∑ Java) Create 3D/4D animations using a natural language-based syntax (PMID: [34152405](https://pubmed.ncbi.nlm.nih.gov/34152405))

769. [cyang03/CHECKED](https://github.com/cyang03/CHECKED) (‚≠ê56 ¬∑ Jupyter Notebook) None (PMID: [34178179](https://pubmed.ncbi.nlm.nih.gov/34178179))

770. [ddofer/ProFET](https://github.com/ddofer/ProFET) (‚≠ê56 ¬∑ Python) ProFET: Protein Feature Engineering Toolkit for Machine Learning (PMID: [26130574](https://pubmed.ncbi.nlm.nih.gov/26130574))

771. [lczech/genesis](https://github.com/lczech/genesis) (‚≠ê56 ¬∑ C++) A library for working with phylogenetic and population genetic data. (PMID: [32016344](https://pubmed.ncbi.nlm.nih.gov/32016344))

772. [materialsintelligence/matscholar](https://github.com/materialsintelligence/matscholar) (‚≠ê56 ¬∑ Python) Code to access the Matscholar public API. (PMID: [31361962](https://pubmed.ncbi.nlm.nih.gov/31361962))

773. [vembrane/vembrane](https://github.com/vembrane/vembrane) (‚≠ê56 ¬∑ Python) vembrane filters VCF records using python expressions (PMID: [36519840](https://pubmed.ncbi.nlm.nih.gov/36519840))

774. [xavierdidelot/TransPhylo](https://github.com/xavierdidelot/TransPhylo) (‚≠ê56 ¬∑ R) Reconstruction of transmission trees using genomic data (PMID: [28100788](https://pubmed.ncbi.nlm.nih.gov/28100788))

775. [10XGenomics/scHLAcount](https://github.com/10XGenomics/scHLAcount) (‚≠ê55 ¬∑ TeX) Count HLA alleles in single-cell RNA-seq data (PMID: [32330223](https://pubmed.ncbi.nlm.nih.gov/32330223))

776. [Data2Dynamics/d2d](https://github.com/Data2Dynamics/d2d) (‚≠ê55 ¬∑ MATLAB) a modeling environment tailored to parameter estimation in dynamical systems (PMID: [32176768](https://pubmed.ncbi.nlm.nih.gov/32176768))

777. [Graylab/GeoDock](https://github.com/Graylab/GeoDock) (‚≠ê55 ¬∑ Python) Flexible Protein-Protein Docking with a Multi-Track Iterative Transformer. (PMID: [38148272](https://pubmed.ncbi.nlm.nih.gov/38148272))

778. [Proteomicslab57357/UniprotR](https://github.com/Proteomicslab57357/UniprotR) (‚≠ê55 ¬∑ R) Retrieving Information of Proteins from Uniprot (PMID: [31843688](https://pubmed.ncbi.nlm.nih.gov/31843688))

779. [giannimonaco/ABIS](https://github.com/giannimonaco/ABIS) (‚≠ê55 ¬∑ R) None (PMID: [30726743](https://pubmed.ncbi.nlm.nih.gov/30726743))

780. [immunedynamics/spectre](https://github.com/immunedynamics/spectre) (‚≠ê55 ¬∑ HTML) A computational toolkit in R for the integration, exploration, and analysis of high-dimensional single-cell cytometry and imaging data. (PMID: [33840138](https://pubmed.ncbi.nlm.nih.gov/33840138))

781. [muellan/metacache](https://github.com/muellan/metacache) (‚≠ê55 ¬∑ C++) memory efficient, fast & precise taxnomomic classification system for metagenomic read mapping (PMID: [28961782](https://pubmed.ncbi.nlm.nih.gov/28961782))

782. [pratas/cryfa](https://github.com/pratas/cryfa) (‚≠ê55 ¬∑ C++) A secure encryption tool for genomic data (PMID: [30020420](https://pubmed.ncbi.nlm.nih.gov/30020420))

783. [smfleming/HMeta-d](https://github.com/smfleming/HMeta-d) (‚≠ê55 ¬∑ MATLAB) Hierarchical meta-d' model (PMID: [29877507](https://pubmed.ncbi.nlm.nih.gov/29877507))

784. [twestbrookunh/paladin](https://github.com/twestbrookunh/paladin) (‚≠ê55 ¬∑ C) Protein Alignment and Detection Interface (PMID: [28158639](https://pubmed.ncbi.nlm.nih.gov/28158639))

785. [biocore/gneiss](https://github.com/biocore/gneiss) (‚≠ê54 ¬∑ Jupyter Notebook) compositional data analysis toolbox (PMID: [28144630](https://pubmed.ncbi.nlm.nih.gov/28144630))

786. [charite/jannovar](https://github.com/charite/jannovar) (‚≠ê54 ¬∑ Java) Annotation of VCF variants with functional impact and from databases (executable+library) (PMID: [24677618](https://pubmed.ncbi.nlm.nih.gov/24677618))

787. [eskederks/eMAGMA-tutorial](https://github.com/eskederks/eMAGMA-tutorial) (‚≠ê54 ¬∑ None) A step by step guide on how to use eMAGMA, an approach to conducting eQTL informed gene-based tests. (PMID: [33624746](https://pubmed.ncbi.nlm.nih.gov/33624746))

788. [insilichem/pychimera](https://github.com/insilichem/pychimera) (‚≠ê54 ¬∑ Python) Use UCSF Chimera Python API in a standard interpreter (PMID: [29340616](https://pubmed.ncbi.nlm.nih.gov/29340616))

789. [jag1g13/pycgtool](https://github.com/jag1g13/pycgtool) (‚≠ê54 ¬∑ Python) Generate coarse-grained molecular dynamics models from atomistic trajectories. (PMID: [28345910](https://pubmed.ncbi.nlm.nih.gov/28345910))

790. [jpquast/protti](https://github.com/jpquast/protti) (‚≠ê54 ¬∑ R) Picotti lab data analysis package. (PMID: [36699412](https://pubmed.ncbi.nlm.nih.gov/36699412))

791. [kiharalab/DOVE](https://github.com/kiharalab/DOVE) (‚≠ê54 ¬∑ Python) A Deep-learning based dOcking decoy eValuation mEthod (PMID: [31746961](https://pubmed.ncbi.nlm.nih.gov/31746961))

792. [lcnhappe/happe](https://github.com/lcnhappe/happe) (‚≠ê54 ¬∑ Matlab) Harvard Automated Pre-Processing Pipeline for EEG (PMID: [29535597](https://pubmed.ncbi.nlm.nih.gov/29535597))

793. [mlin/spVCF](https://github.com/mlin/spVCF) (‚≠ê54 ¬∑ C++) Sparse Project VCF: evolution of VCF to encode population genotype matrices efficiently (PMID: [33300997](https://pubmed.ncbi.nlm.nih.gov/33300997))

794. [morispi/CONSENT](https://github.com/morispi/CONSENT) (‚≠ê54 ¬∑ C++) Scalable long read self-correction and assembly polishing with multiple sequence alignment (PMID: [33436980](https://pubmed.ncbi.nlm.nih.gov/33436980))

795. [rhysf/Synima](https://github.com/rhysf/Synima) (‚≠ê54 ¬∑ Perl) Synteny Imager (PMID: [29162056](https://pubmed.ncbi.nlm.nih.gov/29162056))

796. [sequencing/NxTrim](https://github.com/sequencing/NxTrim) (‚≠ê54 ¬∑ C++) Adapter trimming and virtual library creation for Illumina Nextera Mate Pair libraries. (PMID: [25661542](https://pubmed.ncbi.nlm.nih.gov/25661542))

797. [tanghaibao/quota-alignment](https://github.com/tanghaibao/quota-alignment) (‚≠ê54 ¬∑ Python) Guided synteny alignment between duplicated genomes (within specified quota constraint) (PMID: [21501495](https://pubmed.ncbi.nlm.nih.gov/21501495))

798. [ushaham/BatchEffectRemoval](https://github.com/ushaham/BatchEffectRemoval) (‚≠ê54 ¬∑ Python) None (PMID: [28419223](https://pubmed.ncbi.nlm.nih.gov/28419223))

799. [RWilton/Arioc](https://github.com/RWilton/Arioc) (‚≠ê53 ¬∑ None) Arioc:  GPU-accelerated DNA short-read alignment (PMID: [25780763](https://pubmed.ncbi.nlm.nih.gov/25780763); [29554207](https://pubmed.ncbi.nlm.nih.gov/29554207))

800. [clwgg/nQuire](https://github.com/clwgg/nQuire) (‚≠ê53 ¬∑ C) A statistical framework for ploidy estimation using NGS short-read data (PMID: [29618319](https://pubmed.ncbi.nlm.nih.gov/29618319))

801. [gtonkinhill/fastbaps](https://github.com/gtonkinhill/fastbaps) (‚≠ê53 ¬∑ R) A fast approximation to a Dirichlet Process Mixture model (DPM) for clustering genetic data (PMID: [31076776](https://pubmed.ncbi.nlm.nih.gov/31076776))

802. [haifangong/TRFE-Net-for-thyroid-nodule-segmentation](https://github.com/haifangong/TRFE-Net-for-thyroid-nodule-segmentation) (‚≠ê53 ¬∑ Python) [ISBI'21, CBM'22] TRFE-Net for thyroid nodule segmentation (PMID: [36812810](https://pubmed.ncbi.nlm.nih.gov/36812810))

803. [hillerlab/GenomeAlignmentTools](https://github.com/hillerlab/GenomeAlignmentTools) (‚≠ê53 ¬∑ C) Tools for improving the sensitivity and specificity of genome alignments (PMID: [31742600](https://pubmed.ncbi.nlm.nih.gov/31742600))

804. [lasigeBioTM/MER](https://github.com/lasigeBioTM/MER) (‚≠ê53 ¬∑ Shell) Minimal Named-Entity Recognizer (MER) (PMID: [30519990](https://pubmed.ncbi.nlm.nih.gov/30519990))

805. [maickrau/MBG](https://github.com/maickrau/MBG) (‚≠ê53 ¬∑ C++) None (PMID: [33475133](https://pubmed.ncbi.nlm.nih.gov/33475133))

806. [niemasd/ViralMSA](https://github.com/niemasd/ViralMSA) (‚≠ê53 ¬∑ Python) Reference-guided multiple sequence alignment of viral genomes (PMID: [32814953](https://pubmed.ncbi.nlm.nih.gov/32814953))

807. [theislab/AutoGeneS](https://github.com/theislab/AutoGeneS) (‚≠ê53 ¬∑ Jupyter Notebook) None (PMID: [34293324](https://pubmed.ncbi.nlm.nih.gov/34293324))

808. [theislab/trvae](https://github.com/theislab/trvae) (‚≠ê53 ¬∑ Python) Conditional out-of-distribution prediction (PMID: [33381839](https://pubmed.ncbi.nlm.nih.gov/33381839))

809. [vibbits/phyd3](https://github.com/vibbits/phyd3) (‚≠ê53 ¬∑ JavaScript) Phylogenetic tree viewer based on D3.js (PMID: [28525531](https://pubmed.ncbi.nlm.nih.gov/28525531))

810. [xiaoyeye/GCNG](https://github.com/xiaoyeye/GCNG) (‚≠ê53 ¬∑ Python) using graph convolutional neural network and spaital transcriptomics data to infer cell-cell interactions (PMID: [33303016](https://pubmed.ncbi.nlm.nih.gov/33303016))

811. [zhangxaochen/CuFusion](https://github.com/zhangxaochen/CuFusion) (‚≠ê53 ¬∑ C++) None (PMID: [28974030](https://pubmed.ncbi.nlm.nih.gov/28974030))

812. [BioinfoUNIBA/REDItools](https://github.com/BioinfoUNIBA/REDItools) (‚≠ê52 ¬∑ Python) REDItools are python scripts to investigate RNA editing at genomic scale. (PMID: [31996844](https://pubmed.ncbi.nlm.nih.gov/31996844))

813. [BorgwardtLab/S3M](https://github.com/BorgwardtLab/S3M) (‚≠ê52 ¬∑ C++) A software package for statistically significant shapelet mining (PMID: [29949972](https://pubmed.ncbi.nlm.nih.gov/29949972); [33381811](https://pubmed.ncbi.nlm.nih.gov/33381811))

814. [MolecularAI/Icolos](https://github.com/MolecularAI/Icolos) (‚≠ê52 ¬∑ Python) Icolos: A workflow manager for structure based post-processing of de novo generated small molecules (PMID: [36073898](https://pubmed.ncbi.nlm.nih.gov/36073898))

815. [RASpicer/MetabolomicsTools](https://github.com/RASpicer/MetabolomicsTools) (‚≠ê52 ¬∑ None) None (PMID: [28890673](https://pubmed.ncbi.nlm.nih.gov/28890673))

816. [UCLOrengoGroup/cath-tools](https://github.com/UCLOrengoGroup/cath-tools) (‚≠ê52 ¬∑ C++) Protein structure comparison tools such as SSAP and SNAP (PMID: [30295745](https://pubmed.ncbi.nlm.nih.gov/30295745))

817. [acadTags/Explainable-Automated-Medical-Coding](https://github.com/acadTags/Explainable-Automated-Medical-Coding) (‚≠ê52 ¬∑ Python) Implementation and demo of explainable coding of clinical notes with Hierarchical Label-wise Attention Networks (HLAN) (PMID: [33711543](https://pubmed.ncbi.nlm.nih.gov/33711543))

818. [cramerlab/warp](https://github.com/cramerlab/warp) (‚≠ê52 ¬∑ C#) None (PMID: [31591575](https://pubmed.ncbi.nlm.nih.gov/31591575))

819. [daquang/YAMDA](https://github.com/daquang/YAMDA) (‚≠ê52 ¬∑ Python) Yet Another Motif Discovery Algorithm (PMID: [29790915](https://pubmed.ncbi.nlm.nih.gov/29790915))

820. [frankMusacchia/Annocript](https://github.com/frankMusacchia/Annocript) (‚≠ê52 ¬∑ Perl) Annocript is a pipeline for the annotation of de-novo generated  transcriptomes. It executes BLAST analysis with UniProt, NCBI Conserved Domain Database and Nucleotide divisions, Gene Ontology, UniPathways and the Enzyme Commission. It gives information about the longest ORF (using DNA2PEP) and non-coding potential of the sequences (using Portrait). A final heuristic makes Annocript able to identify putative long non-coding RNAs among your transcripts. Like our facebook page to be always updated: https://www.facebook.com/annocript (PMID: [25701574](https://pubmed.ncbi.nlm.nih.gov/25701574))

821. [hkaneko1985/dcekit](https://github.com/hkaneko1985/dcekit) (‚≠ê52 ¬∑ Python) DCEKit (Data Chemical Engineering toolKit) (PMID: [35309472](https://pubmed.ncbi.nlm.nih.gov/35309472); [37360490](https://pubmed.ncbi.nlm.nih.gov/37360490); [38496944](https://pubmed.ncbi.nlm.nih.gov/38496944))

822. [kamimrcht/REINDEER](https://github.com/kamimrcht/REINDEER) (‚≠ê52 ¬∑ C++) REINDEER  REad Index for abuNDancE quERy (PMID: [32657392](https://pubmed.ncbi.nlm.nih.gov/32657392))

823. [kundajelab/dfim](https://github.com/kundajelab/dfim) (‚≠ê52 ¬∑ Python) Deep Feature Interaction Maps (DFIM) (PMID: [30423062](https://pubmed.ncbi.nlm.nih.gov/30423062))

824. [luoyuanlab/Clinical-Longformer](https://github.com/luoyuanlab/Clinical-Longformer) (‚≠ê52 ¬∑ Python) None (PMID: [36451266](https://pubmed.ncbi.nlm.nih.gov/36451266))

825. [minoh0201/DeepMicro](https://github.com/minoh0201/DeepMicro) (‚≠ê52 ¬∑ Python) Deep representation learning for disease prediction based on microbiome data (PMID: [32265477](https://pubmed.ncbi.nlm.nih.gov/32265477))

826. [nitzanlab/biolord](https://github.com/nitzanlab/biolord) (‚≠ê52 ¬∑ Python) None (PMID: [38225466](https://pubmed.ncbi.nlm.nih.gov/38225466))

827. [nygenome/conpair](https://github.com/nygenome/conpair) (‚≠ê52 ¬∑ Python) Concordance and contamination estimator for tumor‚Äìnormal pairs (PMID: [27354699](https://pubmed.ncbi.nlm.nih.gov/27354699))

828. [oxpig/DEVELOP](https://github.com/oxpig/DEVELOP) (‚≠ê52 ¬∑ Python) None (PMID: [34881010](https://pubmed.ncbi.nlm.nih.gov/34881010))

829. [sigven/cpsr](https://github.com/sigven/cpsr) (‚≠ê52 ¬∑ R) Cancer Predisposition Sequencing Reporter (CPSR) (PMID: [34310709](https://pubmed.ncbi.nlm.nih.gov/34310709))

830. [stevemussmann/admixturePipeline](https://github.com/stevemussmann/admixturePipeline) (‚≠ê52 ¬∑ Python) A pipeline that accepts a VCF file to run through Admixture (PMID: [32727359](https://pubmed.ncbi.nlm.nih.gov/32727359); [38046098](https://pubmed.ncbi.nlm.nih.gov/38046098))

831. [BojarLab/glycowork](https://github.com/BojarLab/glycowork) (‚≠ê51 ¬∑ Jupyter Notebook) Package for processing and analyzing glycans and their role in biology. (PMID: [34192308](https://pubmed.ncbi.nlm.nih.gov/34192308))

832. [ETHmodlab/BIMODAL](https://github.com/ETHmodlab/BIMODAL) (‚≠ê51 ¬∑ Python) Supporting code for the paper "Bidirectional Molecule Generation with Recurrent Neural Networks" (J. Chem. Inf. 2020, 60, 3). (PMID: [31904964](https://pubmed.ncbi.nlm.nih.gov/31904964))

833. [PedroMTQ/mantis](https://github.com/PedroMTQ/mantis) (‚≠ê51 ¬∑ Python) A package to annotate protein sequences (PMID: [34076241](https://pubmed.ncbi.nlm.nih.gov/34076241))

834. [RealityBending/Pyllusion](https://github.com/RealityBending/Pyllusion) (‚≠ê51 ¬∑ Python) A Parametric Framework to Generate Visual Illusions using Python (PMID: [34841973](https://pubmed.ncbi.nlm.nih.gov/34841973))

835. [alfonsosemeraro/pyplutchik](https://github.com/alfonsosemeraro/pyplutchik) (‚≠ê51 ¬∑ Python) Python visualisation for Plutchik annotated corpora. (PMID: [34469455](https://pubmed.ncbi.nlm.nih.gov/34469455))

836. [choderalab/ensembler](https://github.com/choderalab/ensembler) (‚≠ê51 ¬∑ Python) Automated omics-scale protein modeling and simulation setup. (PMID: [27337644](https://pubmed.ncbi.nlm.nih.gov/27337644))

837. [chopralab/lemon](https://github.com/chopralab/lemon) (‚≠ê51 ¬∑ C++) A framework for rapidly mining structural information from the Protein Data Bank (PMID: [30873531](https://pubmed.ncbi.nlm.nih.gov/30873531); [32639154](https://pubmed.ncbi.nlm.nih.gov/32639154))

838. [irycisBioinfo/PATO](https://github.com/irycisBioinfo/PATO) (‚≠ê51 ¬∑ R) PATO is a R package designed to analyze pangenomes (set of genomes) intra or inter species. (PMID: [34623430](https://pubmed.ncbi.nlm.nih.gov/34623430))

839. [kassonlab/gmxapi](https://github.com/kassonlab/gmxapi) (‚≠ê51 ¬∑ C++) (outdated) fork of https://gitlab.com/gromacs/gromacs (PMID: [29912282](https://pubmed.ncbi.nlm.nih.gov/29912282))

840. [mlbendall/telescope](https://github.com/mlbendall/telescope) (‚≠ê51 ¬∑ Python) Quantification of transposable element expression using RNA-seq (PMID: [31568525](https://pubmed.ncbi.nlm.nih.gov/31568525))

841. [noriakis/CBNplot](https://github.com/noriakis/CBNplot) (‚≠ê51 ¬∑ R) Bayesian network plot for the enrichment analysis results (PMID: [35561164](https://pubmed.ncbi.nlm.nih.gov/35561164))

842. [psathyrella/partis](https://github.com/psathyrella/partis) (‚≠ê51 ¬∑ C++) B- and T-cell receptor sequence annotation, simulation, clonal family and germline inference, and affinity prediction (PMID: [26751373](https://pubmed.ncbi.nlm.nih.gov/26751373); [31329576](https://pubmed.ncbi.nlm.nih.gov/31329576); [33175831](https://pubmed.ncbi.nlm.nih.gov/33175831))

843. [xq141839/DCSAU-Net](https://github.com/xq141839/DCSAU-Net) (‚≠ê51 ¬∑ Python) Elsevier-CIBM-2023: A deeper and more compact split-attention U-Net for medical image segmentation (PMID: [36736096](https://pubmed.ncbi.nlm.nih.gov/36736096))

844. [MolecularAI/Lib-INVENT](https://github.com/MolecularAI/Lib-INVENT) (‚≠ê50 ¬∑ Jupyter Notebook) None (PMID: [34460269](https://pubmed.ncbi.nlm.nih.gov/34460269))

845. [aldro61/kover](https://github.com/aldro61/kover) (‚≠ê50 ¬∑ Python) Learn interpretable computational phenotyping models from k-merized genomic data (PMID: [27671088](https://pubmed.ncbi.nlm.nih.gov/27671088))

846. [compbiomed/animalcules](https://github.com/compbiomed/animalcules) (‚≠ê50 ¬∑ R) R shiny app for interactive microbiome analysis (PMID: [33775256](https://pubmed.ncbi.nlm.nih.gov/33775256))

847. [fgcz/rawrr](https://github.com/fgcz/rawrr) (‚≠ê50 ¬∑ R) Access Orbitrap data in R lang using C# mono assembly - bioconductor package (PMID: [33686856](https://pubmed.ncbi.nlm.nih.gov/33686856))

848. [insilicomedicine/BiAAE](https://github.com/insilicomedicine/BiAAE) (‚≠ê50 ¬∑ Python) Molecular Generation for Desired Transcriptome Changes with Adversarial Autoencoders (PMID: [32362822](https://pubmed.ncbi.nlm.nih.gov/32362822))

849. [pachterlab/metakallisto](https://github.com/pachterlab/metakallisto) (‚≠ê50 ¬∑ Python) Using kallisto for metagenomic analysis (PMID: [28334086](https://pubmed.ncbi.nlm.nih.gov/28334086))

850. [psipred/cgdms](https://github.com/psipred/cgdms) (‚≠ê50 ¬∑ Python) Differentiable molecular simulation of proteins with a coarse-grained potential (PMID: [34473813](https://pubmed.ncbi.nlm.nih.gov/34473813))

851. [rcs333/VAPiD](https://github.com/rcs333/VAPiD) (‚≠ê50 ¬∑ Shell) VAPiD: Viral Annotation and Identification Pipeline (PMID: [30674273](https://pubmed.ncbi.nlm.nih.gov/30674273))

852. [songweizhi/MetaCHIP](https://github.com/songweizhi/MetaCHIP) (‚≠ê50 ¬∑ Python) Horizontal gene transfer (HGT) identification pipeline (PMID: [30832740](https://pubmed.ncbi.nlm.nih.gov/30832740))

853. [stephaniehicks/qsmooth](https://github.com/stephaniehicks/qsmooth) (‚≠ê50 ¬∑ R) Smooth quantile normalization (qsmooth) is a generalization of quantile normalization, which is an average of the two types of assumptions about the data generation process: quantile normalization and quantile normalization between groups. (PMID: [29036413](https://pubmed.ncbi.nlm.nih.gov/29036413))

854. [yvquanli/trimnet](https://github.com/yvquanli/trimnet) (‚≠ê50 ¬∑ Python) Code for paper "TrimNet: learning molecular representation from triplet messages for biomedicine " (PMID: [33147620](https://pubmed.ncbi.nlm.nih.gov/33147620))

855. [zhaoxiaofei/BinDash](https://github.com/zhaoxiaofei/BinDash) (‚≠ê50 ¬∑ C++) Fast and precise comparison of genomes and metagenomes (in the order of terabytes) on a typical personal laptop (PMID: [30052763](https://pubmed.ncbi.nlm.nih.gov/30052763))

856. [BigDataBiology/macrel](https://github.com/BigDataBiology/macrel) (‚≠ê49 ¬∑ Python) Predict AMPs in (meta)genomes and peptides (PMID: [33384902](https://pubmed.ncbi.nlm.nih.gov/33384902))

857. [HannesStark/protein-localization](https://github.com/HannesStark/protein-localization) (‚≠ê49 ¬∑ Jupyter Notebook) Using Transformer protein embeddings with a linear attention mechanism to make SOTA de-novo predictions for the subcellular location of proteins :microscope: (PMID: [36700108](https://pubmed.ncbi.nlm.nih.gov/36700108))

858. [IBM/fold2seq](https://github.com/IBM/fold2seq) (‚≠ê49 ¬∑ Python) Code for Fold2Seq paper from ICML 2021 (PMID: [34423306](https://pubmed.ncbi.nlm.nih.gov/34423306))

859. [KirillKryukov/naf](https://github.com/KirillKryukov/naf) (‚≠ê49 ¬∑ C) Nucleotide Archival Format - Compressed file format for DNA/RNA/protein sequences (PMID: [30799504](https://pubmed.ncbi.nlm.nih.gov/30799504))

860. [Merck/Matcher](https://github.com/Merck/Matcher) (‚≠ê49 ¬∑ Python) Matcher is a tool for understanding how chemical structure optimization problems have been solved. Matcher enables deep control over searching structure/activity relationships (SAR) derived from large datasets, and takes the form of an accessible web application with simple deployment. Matcher is built around the mmpdb platform. (PMID: [36977316](https://pubmed.ncbi.nlm.nih.gov/36977316))

861. [TheJacksonLaboratory/SVE](https://github.com/TheJacksonLaboratory/SVE) (‚≠ê49 ¬∑ Python) None (PMID: [29559002](https://pubmed.ncbi.nlm.nih.gov/29559002))

862. [WGLab/bioformer](https://github.com/WGLab/bioformer) (‚≠ê49 ¬∑ None) Bioformer: an efficient BERT model for biomedical text mining (PMID: [36945685](https://pubmed.ncbi.nlm.nih.gov/36945685))

863. [aehrc/cvt2distilgpt2](https://github.com/aehrc/cvt2distilgpt2) (‚≠ê49 ¬∑ Python) Improving Chest X-Ray Report Generation by Leveraging Warm-Starting (PMID: [37783533](https://pubmed.ncbi.nlm.nih.gov/37783533))

864. [dayoonkwon/BioAge](https://github.com/dayoonkwon/BioAge) (‚≠ê49 ¬∑ R) Biological Age Calculations Using Several Biomarker Algorithms (PMID: [34725754](https://pubmed.ncbi.nlm.nih.gov/34725754))

865. [edawson/gfakluge](https://github.com/edawson/gfakluge) (‚≠ê49 ¬∑ C++) A C++ library and utilities for manipulating the Graphical Fragment Assembly format. (PMID: [31535074](https://pubmed.ncbi.nlm.nih.gov/31535074))

866. [horsepurve/DeepRTplus](https://github.com/horsepurve/DeepRTplus) (‚≠ê49 ¬∑ Python) Deep (Transfer) Learning for Peptide Retention Time Prediction (PMID: [30114359](https://pubmed.ncbi.nlm.nih.gov/30114359))

867. [marbl/harvest](https://github.com/marbl/harvest) (‚≠ê49 ¬∑ None) None (PMID: [25410596](https://pubmed.ncbi.nlm.nih.gov/25410596))

868. [mrazvan22/brain-coloring](https://github.com/mrazvan22/brain-coloring) (‚≠ê49 ¬∑ Python) BrainPainter - Brain Visualisation Software (PMID: [33537683](https://pubmed.ncbi.nlm.nih.gov/33537683))

869. [mstsuite/MuST](https://github.com/mstsuite/MuST) (‚≠ê49 ¬∑ Fortran) Multiple Scattering Theory code for first principles calculations (PMID: [33344416](https://pubmed.ncbi.nlm.nih.gov/33344416))

870. [schmeing/ReSeq](https://github.com/schmeing/ReSeq) (‚≠ê49 ¬∑ C++) More realistic simulator for genomic DNA sequences from Illumina machines that achieves a similar k-mer spectrum as the original (PMID: [33608040](https://pubmed.ncbi.nlm.nih.gov/33608040))

871. [wenqi006/SlideGraph](https://github.com/wenqi006/SlideGraph) (‚≠ê49 ¬∑ Python) None (PMID: [35640384](https://pubmed.ncbi.nlm.nih.gov/35640384))

872. [IFMlab/ChemFlow](https://github.com/IFMlab/ChemFlow) (‚≠ê48 ¬∑ Python) Computational Chemistry Workflows (PMID: [36603846](https://pubmed.ncbi.nlm.nih.gov/36603846); [37620503](https://pubmed.ncbi.nlm.nih.gov/37620503))

873. [LaboratorioBioinformatica/MARVEL](https://github.com/LaboratorioBioinformatica/MARVEL) (‚≠ê48 ¬∑ Python) MARVEL: Metagenomic Analyses and Retrieval of Viral ELements (PMID: [30131825](https://pubmed.ncbi.nlm.nih.gov/30131825))

874. [bcgsc/orca](https://github.com/bcgsc/orca) (‚≠ê48 ¬∑ R) :whale: Genomics Research Container Architecture (PMID: [31004474](https://pubmed.ncbi.nlm.nih.gov/31004474))

875. [hosseinshn/MOLI](https://github.com/hosseinshn/MOLI) (‚≠ê48 ¬∑ Jupyter Notebook) MOLI: Multi-Omics Late Integration with deep neural networks for drug response prediction (PMID: [31510700](https://pubmed.ncbi.nlm.nih.gov/31510700))

876. [jyangfsu/WQChartPy](https://github.com/jyangfsu/WQChartPy) (‚≠ê48 ¬∑ Python) Python Package for Visualization of Water Geochemistry Data (PMID: [35157303](https://pubmed.ncbi.nlm.nih.gov/35157303))

877. [msproteomicstools/msproteomicstools](https://github.com/msproteomicstools/msproteomicstools) (‚≠ê48 ¬∑ Python) Some hopefully useful tools for mass spectrometry applied to proteomics (PMID: [25788625](https://pubmed.ncbi.nlm.nih.gov/25788625))

878. [precimed/mixer](https://github.com/precimed/mixer) (‚≠ê48 ¬∑ C) Causal Mixture Model for GWAS summary statistics (PMID: [32539089](https://pubmed.ncbi.nlm.nih.gov/32539089))

879. [sigven/oncoEnrichR](https://github.com/sigven/oncoEnrichR) (‚≠ê48 ¬∑ R) Cancer-dedicated gene set interpretation (PMID: [37551617](https://pubmed.ncbi.nlm.nih.gov/37551617))

880. [soedinglab/wish](https://github.com/soedinglab/wish) (‚≠ê48 ¬∑ C++) Predict prokaryotic host for phage metagenomic sequences (PMID: [28957499](https://pubmed.ncbi.nlm.nih.gov/28957499))

881. [stephenslab/flashr](https://github.com/stephenslab/flashr) (‚≠ê48 ¬∑ R) R package for Empirical Bayes Factor Analysis. (PMID: [37920532](https://pubmed.ncbi.nlm.nih.gov/37920532))

882. [willtownes/nsf-paper](https://github.com/willtownes/nsf-paper) (‚≠ê48 ¬∑ Jupyter Notebook) Nonnegative spatial factorization for multivariate count data (PMID: [36587187](https://pubmed.ncbi.nlm.nih.gov/36587187))

883. [yuzhimanhua/lm-lstm-crf](https://github.com/yuzhimanhua/lm-lstm-crf) (‚≠ê48 ¬∑ None) None (PMID: [30307536](https://pubmed.ncbi.nlm.nih.gov/30307536))

884. [ADicksonLab/wepy](https://github.com/ADicksonLab/wepy) (‚≠ê47 ¬∑ Python) Weighted Ensemble simulation framework in Python (PMID: [33344813](https://pubmed.ncbi.nlm.nih.gov/33344813))

885. [CBIIT/LDlinkR](https://github.com/CBIIT/LDlinkR) (‚≠ê47 ¬∑ R) LDlinkR (PMID: [32180801](https://pubmed.ncbi.nlm.nih.gov/32180801))

886. [IARCbioinfo/needlestack](https://github.com/IARCbioinfo/needlestack) (‚≠ê47 ¬∑ R) Multi-sample somatic variant caller (PMID: [32363341](https://pubmed.ncbi.nlm.nih.gov/32363341))

887. [JLSteenwyk/PhyKIT](https://github.com/JLSteenwyk/PhyKIT) (‚≠ê47 ¬∑ Python) a UNIX shell toolkit for processing and analyzing multiple sequence alignments and phylogenies (PMID: [33560364](https://pubmed.ncbi.nlm.nih.gov/33560364))

888. [KlugerLab/t-SNE-Heatmaps](https://github.com/KlugerLab/t-SNE-Heatmaps) (‚≠ê47 ¬∑ R) Beta version of 1D t-SNE heatmaps to visualize expression patterns of hundreds of genes simultaneously in scRNA-seq (PMID: [30742040](https://pubmed.ncbi.nlm.nih.gov/30742040))

889. [VCCRI/Sierra](https://github.com/VCCRI/Sierra) (‚≠ê47 ¬∑ R) Discover differential transcript usage from polyA-captured single cell RNA-seq data (PMID: [32641141](https://pubmed.ncbi.nlm.nih.gov/32641141))

890. [algbio/themisto](https://github.com/algbio/themisto) (‚≠ê47 ¬∑ C++) Space-efficient pseudoalignment with a colored de Bruijn graph (PMID: [37387143](https://pubmed.ncbi.nlm.nih.gov/37387143))

891. [baranwa2/Struct2Graph](https://github.com/baranwa2/Struct2Graph) (‚≠ê47 ¬∑ Python) A PyTorch implementation of GCN with mutual attention for protein-protein interaction prediction (PMID: [36088285](https://pubmed.ncbi.nlm.nih.gov/36088285))

892. [brentp/hts-nim-tools](https://github.com/brentp/hts-nim-tools) (‚≠ê47 ¬∑ Nim) useful command-line tools written to showcase hts-nim (PMID: [29718142](https://pubmed.ncbi.nlm.nih.gov/29718142))

893. [ignaciorlando/red-lesion-detection](https://github.com/ignaciorlando/red-lesion-detection) (‚≠ê47 ¬∑ MATLAB) This code implements a red lesion detection method based on a combination of hand-crafted features and CNN based descriptors. Our paper is under revision now, so please do not use this repository until we release the paper. (PMID: [29157445](https://pubmed.ncbi.nlm.nih.gov/29157445))

894. [joshr17/IFM](https://github.com/joshr17/IFM) (‚≠ê47 ¬∑ Python) Code for paper "Can contrastive learning avoid shortcut solutions?" NeurIPS 2021. (PMID: [35546903](https://pubmed.ncbi.nlm.nih.gov/35546903))

895. [nadeemlab/Physics-ArX](https://github.com/nadeemlab/Physics-ArX) (‚≠ê47 ¬∑ Python) Physics-based data augmentation library for quantifying CT and CBCT images in radiotherapy [PMB'23, PMB'21, Medical Physics'21, AAPM'21] (PMID: [34245012](https://pubmed.ncbi.nlm.nih.gov/34245012))

896. [ncclabsustech/DeepSeparator](https://github.com/ncclabsustech/DeepSeparator) (‚≠ê47 ¬∑ Python) Deep learning model for EEG artifact removal (PMID: [35378524](https://pubmed.ncbi.nlm.nih.gov/35378524))

897. [rothlab/chromozoom](https://github.com/rothlab/chromozoom) (‚≠ê47 ¬∑ JavaScript) ChromoZoom is a fast, fluid web-based genome browser (PMID: [23220575](https://pubmed.ncbi.nlm.nih.gov/23220575))

898. [variani/lme4qtl](https://github.com/variani/lme4qtl) (‚≠ê47 ¬∑ R) Mixed models @lme4 + custom covariances + parameter constraints (PMID: [29486711](https://pubmed.ncbi.nlm.nih.gov/29486711))

899. [BimberLab/DISCVRSeq](https://github.com/BimberLab/DISCVRSeq) (‚≠ê46 ¬∑ HTML) A collection of command line tools for working with sequencing data (PMID: [31309221](https://pubmed.ncbi.nlm.nih.gov/31309221))

900. [HaploKit/vechat](https://github.com/HaploKit/vechat) (‚≠ê46 ¬∑ C++) Correcting errors in noisy long reads using variation graphs (PMID: [36333324](https://pubmed.ncbi.nlm.nih.gov/36333324))

901. [JLiangLab/BenchmarkTransferLearning](https://github.com/JLiangLab/BenchmarkTransferLearning) (‚≠ê46 ¬∑ Python) Official PyTorch Implementation and Pre-trained Models for Benchmarking Transfer Learning for Medical Image Analysis (PMID: [35713581](https://pubmed.ncbi.nlm.nih.gov/35713581))

902. [LooseLab/bulkvis](https://github.com/LooseLab/bulkvis) (‚≠ê46 ¬∑ Python) None (PMID: [30462145](https://pubmed.ncbi.nlm.nih.gov/30462145))

903. [MannLabs/alphapeptstats](https://github.com/MannLabs/alphapeptstats) (‚≠ê46 ¬∑ Jupyter Notebook) Python Package for the downstream analysis of mass-spectrometry-based proteomics data (PMID: [37527012](https://pubmed.ncbi.nlm.nih.gov/37527012))

904. [PacificBiosciences/HiPhase](https://github.com/PacificBiosciences/HiPhase) (‚≠ê46 ¬∑ Rust) Small variant, structural variant, and short tandem repeat phasing tool for PacBio HiFi reads (PMID: [38269623](https://pubmed.ncbi.nlm.nih.gov/38269623))

905. [PayamDiba/SERGIO](https://github.com/PayamDiba/SERGIO) (‚≠ê46 ¬∑ Python) A simulator for single-cell expression data  guided by gene regulatory networks (PMID: [32871105](https://pubmed.ncbi.nlm.nih.gov/32871105))

906. [aquaskyline/LRSIM](https://github.com/aquaskyline/LRSIM) (‚≠ê46 ¬∑ C) 10x Genomics Reads Simulator (PMID: [29213995](https://pubmed.ncbi.nlm.nih.gov/29213995))

907. [arttumiettinen/pi2](https://github.com/arttumiettinen/pi2) (‚≠ê46 ¬∑ C++) C++ library and command-line software for processing and analysis of terabyte-scale volume images locally or on a computing cluster. (PMID: [31116382](https://pubmed.ncbi.nlm.nih.gov/31116382))

908. [biopragmatics/biomappings](https://github.com/biopragmatics/biomappings) (‚≠ê46 ¬∑ Jupyter Notebook) üó∫Ô∏è Community curated and predicted equivalences and related mappings between named biological entities that are not available from primary sources. (PMID: [36916735](https://pubmed.ncbi.nlm.nih.gov/36916735))

909. [blaxterlab/blobology](https://github.com/blaxterlab/blobology) (‚≠ê46 ¬∑ Perl) Tools for making blobplots or Taxon-Annotated-GC-Coverage plots (TAGC plots) to visualise the contents of genome assembly data sets as a QC step (NO LONGER MANTAINED) (PMID: [24348509](https://pubmed.ncbi.nlm.nih.gov/24348509))

910. [bobeobibo/phigaro](https://github.com/bobeobibo/phigaro) (‚≠ê46 ¬∑ HTML) Phigaro is a scalable command-line tool for predicting phages and prophages (PMID: [32311023](https://pubmed.ncbi.nlm.nih.gov/32311023))

911. [citiususc/BigSeqKit](https://github.com/citiususc/BigSeqKit) (‚≠ê46 ¬∑ Go) BigSeqKit: a parallel Big Data toolkit to process FASTA and FASTQ files at scale (PMID: [37522758](https://pubmed.ncbi.nlm.nih.gov/37522758))

912. [dmnfarrell/epitopepredict](https://github.com/dmnfarrell/epitopepredict) (‚≠ê46 ¬∑ Jupyter Notebook) Python package  and command line tool for epitope prediction (PMID: [36824339](https://pubmed.ncbi.nlm.nih.gov/36824339))

913. [genomicsITER/NanoDJ](https://github.com/genomicsITER/NanoDJ) (‚≠ê46 ¬∑ Jupyter Notebook) NanoDJ: A Dockerized Jupyter Notebook for Interactive Oxford Nanopore MinION Sequence Manipulation and Genome Assembly (PMID: [31072312](https://pubmed.ncbi.nlm.nih.gov/31072312))

914. [perishky/meffil](https://github.com/perishky/meffil) (‚≠ê46 ¬∑ HTML) Efficient algorithms for analyzing DNA methylation data. (PMID: [29931280](https://pubmed.ncbi.nlm.nih.gov/29931280))

915. [rinikerlab/Ensembler](https://github.com/rinikerlab/Ensembler) (‚≠ê46 ¬∑ Python) Ensembler is a python package that provides fast and easy access to 1D and 2D model system simulations. It can be used for method development or to deepen understanding of a broad spectrum of modeling methods, from basic sampling techniques to enhanced sampling and free energy calculations. It is easy to install, fast, increases shareability, comparability, and reproducibility of scientific code developments. (PMID: [33512157](https://pubmed.ncbi.nlm.nih.gov/33512157))

916. [stephenslab/rss](https://github.com/stephenslab/rss) (‚≠ê46 ¬∑ MATLAB) Regression with Summary Statistics. (PMID: [29399241](https://pubmed.ncbi.nlm.nih.gov/29399241))

917. [wgmao/PLIER](https://github.com/wgmao/PLIER) (‚≠ê46 ¬∑ R) Pathway-Level Information Extractor (PLIER): a generative model for gene expression data (PMID: [31249421](https://pubmed.ncbi.nlm.nih.gov/31249421))

918. [ChuHan89/WSSS-Tissue](https://github.com/ChuHan89/WSSS-Tissue) (‚≠ê45 ¬∑ Python) None (PMID: [35671591](https://pubmed.ncbi.nlm.nih.gov/35671591))

919. [ELELAB/mutatex](https://github.com/ELELAB/mutatex) (‚≠ê45 ¬∑ Python) scripts and facilities for in-silico mutagenesis with FoldX (PMID: [35323860](https://pubmed.ncbi.nlm.nih.gov/35323860))

920. [OHNLP/MedTator](https://github.com/OHNLP/MedTator) (‚≠ê45 ¬∑ JavaScript) A Serverless Text Annotation Tool for Corpus Development (PMID: [34983060](https://pubmed.ncbi.nlm.nih.gov/34983060))

921. [PingChang818/TDSTF](https://github.com/PingChang818/TDSTF) (‚≠ê45 ¬∑ Python) None (PMID: [38350189](https://pubmed.ncbi.nlm.nih.gov/38350189))

922. [PoonLab/covizu](https://github.com/PoonLab/covizu) (‚≠ê45 ¬∑ JavaScript) Rapid analysis and visualization of coronavirus genome variation (PMID: [37124703](https://pubmed.ncbi.nlm.nih.gov/37124703))

923. [SmartDataAnalytics/BioKEEN](https://github.com/SmartDataAnalytics/BioKEEN) (‚≠ê45 ¬∑ Jupyter Notebook) A computational library for learning and evaluating biological knowledge graph embeddings - please see the main PyKEEN repo at https://github.com/pykeen/pykeen/ (PMID: [30768158](https://pubmed.ncbi.nlm.nih.gov/30768158))

924. [UnixJunkie/FASMIFRA](https://github.com/UnixJunkie/FASMIFRA) (‚≠ê45 ¬∑ Python) Molecular Generation by Fast Assembly of SMILES Fragments (PMID: [34775976](https://pubmed.ncbi.nlm.nih.gov/34775976))

925. [gerberlab/mitre](https://github.com/gerberlab/mitre) (‚≠ê45 ¬∑ Python) The Microbiome Interpretable Temporal Rule Engine (PMID: [31477162](https://pubmed.ncbi.nlm.nih.gov/31477162))

926. [jordisr/poreover](https://github.com/jordisr/poreover) (‚≠ê45 ¬∑ Python) Nanopore basecalling and consensus decoding (PMID: [33468205](https://pubmed.ncbi.nlm.nih.gov/33468205))

927. [kshulgina/codetta](https://github.com/kshulgina/codetta) (‚≠ê45 ¬∑ Python) None (PMID: [36511586](https://pubmed.ncbi.nlm.nih.gov/36511586))

928. [microsoft/RespireNet](https://github.com/microsoft/RespireNet) (‚≠ê45 ¬∑ Python) This is the official implementation of the work RespireNet. (PMID: [34891348](https://pubmed.ncbi.nlm.nih.gov/34891348))

929. [petmri/ROCKETSHIP](https://github.com/petmri/ROCKETSHIP) (‚≠ê45 ¬∑ MATLAB) Dynamic Contrast Enhanced (DCE) MRI Analysis Software (PMID: [26076957](https://pubmed.ncbi.nlm.nih.gov/26076957))

930. [qingyuzhao/br-net](https://github.com/qingyuzhao/br-net) (‚≠ê45 ¬∑ Jupyter Notebook) None (PMID: [33243992](https://pubmed.ncbi.nlm.nih.gov/33243992))

931. [quxiaojian/PGA](https://github.com/quxiaojian/PGA) (‚≠ê45 ¬∑ Perl) Plastid Genome Annotator (PMID: [31139240](https://pubmed.ncbi.nlm.nih.gov/31139240))

932. [shilpagarg/WHdenovo](https://github.com/shilpagarg/WHdenovo) (‚≠ê45 ¬∑ Python) None (PMID: [31860070](https://pubmed.ncbi.nlm.nih.gov/31860070))

933. [tjhwangxiong/TCGAplot](https://github.com/tjhwangxiong/TCGAplot) (‚≠ê45 ¬∑ R) A number of functions were generated to perform pan-cancer DEG analysis, correlation analysis between gene expression and TMB, MSI, TIME, and promoter methylation. Methods for visualization  were provided  in order to easily perform integrative pan-cancer multi-omics analysis. (PMID: [38105215](https://pubmed.ncbi.nlm.nih.gov/38105215))

934. [yijingru/ANCIS-Pytorch](https://github.com/yijingru/ANCIS-Pytorch) (‚≠ê45 ¬∑ Python) [Medical Image Analysis 2019] Attentive Neural Cell Instance Segmentation (PMID: [31103790](https://pubmed.ncbi.nlm.nih.gov/31103790))

935. [BioinfoUNIBA/REDItools2](https://github.com/BioinfoUNIBA/REDItools2) (‚≠ê44 ¬∑ Python) None (PMID: [32838738](https://pubmed.ncbi.nlm.nih.gov/32838738))

936. [CSUBioGroup/DeepPPISP](https://github.com/CSUBioGroup/DeepPPISP) (‚≠ê44 ¬∑ Python) Protein-protein interaction sites prediction through combining local and global features with deep neural networks (PMID: [31593229](https://pubmed.ncbi.nlm.nih.gov/31593229))

937. [GATB/simka](https://github.com/GATB/simka) (‚≠ê44 ¬∑ C++) Simka and SimkaMin are comparative metagenomics methods dedicated to NGS datasets. (PMID: [31504187](https://pubmed.ncbi.nlm.nih.gov/31504187))

938. [LyonsLab/coge](https://github.com/LyonsLab/coge) (‚≠ê44 ¬∑ Perl) CoGe (Comparative Genomics) Platform (PMID: [28334338](https://pubmed.ncbi.nlm.nih.gov/28334338); [29474529](https://pubmed.ncbi.nlm.nih.gov/29474529))

939. [MedChaabane/deepRAM](https://github.com/MedChaabane/deepRAM) (‚≠ê44 ¬∑ Python) End-to-end deep learning toolkit for predicting protein binding sites and motifs. (PMID: [31510640](https://pubmed.ncbi.nlm.nih.gov/31510640))

940. [TheJacksonLaboratory/Picky](https://github.com/TheJacksonLaboratory/Picky) (‚≠ê44 ¬∑ Perl) Structural Variants Pipeline for Long Reads (PMID: [29713081](https://pubmed.ncbi.nlm.nih.gov/29713081))

941. [ZJUYH/radar-to-lidar-place-recognition](https://github.com/ZJUYH/radar-to-lidar-place-recognition) (‚≠ê44 ¬∑ Python) Radar-to-Lidar: Heterogeneous Place Recognition via Joint Learning (PMID: [34079825](https://pubmed.ncbi.nlm.nih.gov/34079825))

942. [alanqrwang/keymorph](https://github.com/alanqrwang/keymorph) (‚≠ê44 ¬∑ Python) Robust multimodal brain registration via keypoints (PMID: [37769550](https://pubmed.ncbi.nlm.nih.gov/37769550))

943. [andreped/NoCodeSeg](https://github.com/andreped/NoCodeSeg) (‚≠ê44 ¬∑ Jupyter Notebook) üî¨ Code-free deep segmentation for computational pathology (PMID: [35155486](https://pubmed.ncbi.nlm.nih.gov/35155486))

944. [blackrim/treePL](https://github.com/blackrim/treePL) (‚≠ê44 ¬∑ C++) Phylogenetic penalized likelihood (PMID: [22908216](https://pubmed.ncbi.nlm.nih.gov/22908216))

945. [boxiangliu/ParaMed](https://github.com/boxiangliu/ParaMed) (‚≠ê44 ¬∑ Python) Chinese to English medical translation (PMID: [34488734](https://pubmed.ncbi.nlm.nih.gov/34488734))

946. [caiyu6666/DDAD-ASR](https://github.com/caiyu6666/DDAD-ASR) (‚≠ê44 ¬∑ Python) [MedIA'2023] Dual-distribution discrepancy with self-supervised refinement for anomaly detection in medical images (PMID: [36934507](https://pubmed.ncbi.nlm.nih.gov/36934507))

947. [duolinwang/MusiteDeep](https://github.com/duolinwang/MusiteDeep) (‚≠ê44 ¬∑ Python) MusiteDeep provides a deep-learning method for general and kinase-specific phosphorylation site prediction. It is implemented by deep learning library Keras and Theano backend (the Keras2.0 and Tensorflow backend implementation were also provided under folder MusiteDeep_Keras2.0). At present, MusiteDeep only provides prediction of human phosphorylation sites; however, it also provides customized model training that enables users to train other PTM prediction models by using their own training data sets based on either CPU or GPU. (PMID: [29036382](https://pubmed.ncbi.nlm.nih.gov/29036382); [32324217](https://pubmed.ncbi.nlm.nih.gov/32324217))

948. [evidenceontology/evidenceontology](https://github.com/evidenceontology/evidenceontology) (‚≠ê44 ¬∑ HTML) Evidence & Conclusion Ontology development site:  Use ISSUES to request terms. See WIKI for how to request terms. See README for how to cite ECO. Visit our website for more project info. (PMID: [34986598](https://pubmed.ncbi.nlm.nih.gov/34986598))

949. [hongwang01/InDuDoNet](https://github.com/hongwang01/InDuDoNet) (‚≠ê44 ¬∑ Python) „ÄêMICCAI 2021„ÄëAn Interpretable Dual Domain Network for CT Metal Artifact Reduction (PMID: [36623381](https://pubmed.ncbi.nlm.nih.gov/36623381))

950. [kdmurray91/kwip](https://github.com/kdmurray91/kwip) (‚≠ê44 ¬∑ C++) De novo estimates of genetic relatedness from next-gen sequencing data (PMID: [28873405](https://pubmed.ncbi.nlm.nih.gov/28873405))

951. [keiserlab/LUNA](https://github.com/keiserlab/LUNA) (‚≠ê44 ¬∑ Jupyter Notebook) None (PMID: [36102784](https://pubmed.ncbi.nlm.nih.gov/36102784))

952. [kundajelab/ChromDragoNN](https://github.com/kundajelab/ChromDragoNN) (‚≠ê44 ¬∑ Jupyter Notebook) Code for the paper "Integrating regulatory DNA sequence and gene expression to predict genome-wide chromatin accessibility across cellular contexts" (PMID: [31510655](https://pubmed.ncbi.nlm.nih.gov/31510655))

953. [lanagarmire/cox-nnet](https://github.com/lanagarmire/cox-nnet) (‚≠ê44 ¬∑ Python) None (PMID: [29634719](https://pubmed.ncbi.nlm.nih.gov/29634719))

954. [msraredon/NICHES](https://github.com/msraredon/NICHES) (‚≠ê44 ¬∑ HTML) Niche Interactions and Cellular Heterogeneity in Extracellular Signaling (PMID: [36458905](https://pubmed.ncbi.nlm.nih.gov/36458905))

955. [nluhmann/BlastFrost](https://github.com/nluhmann/BlastFrost) (‚≠ê44 ¬∑ C++) None (PMID: [33430919](https://pubmed.ncbi.nlm.nih.gov/33430919))

956. [oddt/rfscorevs](https://github.com/oddt/rfscorevs) (‚≠ê44 ¬∑ Jupyter Notebook) RF-Score-VS - Random forest based protein-ligand scoring function for Virtual Screening (PMID: [28440302](https://pubmed.ncbi.nlm.nih.gov/28440302))

957. [psipred/DeepCov](https://github.com/psipred/DeepCov) (‚≠ê44 ¬∑ Python) Fully convolutional neural networks for protein residue-residue contact prediction (PMID: [29718112](https://pubmed.ncbi.nlm.nih.gov/29718112))

958. [AChavignon/PALA](https://github.com/AChavignon/PALA) (‚≠ê43 ¬∑ MATLAB) Sharing scripts and functions for OPUS-PALA article, and LOTUS Software. All functions are usable with agreement from their owner. (PMID: [35177778](https://pubmed.ncbi.nlm.nih.gov/35177778))

959. [AnantharamanLab/ViWrap](https://github.com/AnantharamanLab/ViWrap) (‚≠ê43 ¬∑ Python) A wrapper to identify, bin, classify, and predict host-viral relationship for viruses (PMID: [38152703](https://pubmed.ncbi.nlm.nih.gov/38152703))

960. [BackofenLab/IntaRNA](https://github.com/BackofenLab/IntaRNA) (‚≠ê43 ¬∑ C++) Efficient target prediction incorporating accessibility of interaction sites (PMID: [30590479](https://pubmed.ncbi.nlm.nih.gov/30590479))

961. [CDCgov/datasets-sars-cov-2](https://github.com/CDCgov/datasets-sars-cov-2) (‚≠ê43 ¬∑ Perl) Benchmark datasets for WGS analysis of SARS-CoV-2. (https://peerj.com/articles/13821/) (PMID: [36093336](https://pubmed.ncbi.nlm.nih.gov/36093336))

962. [LouisFaure/scFates](https://github.com/LouisFaure/scFates) (‚≠ê43 ¬∑ Python) a scalable python suite for tree inference and advanced pseudotime analysis from scRNAseq data. (PMID: [36394263](https://pubmed.ncbi.nlm.nih.gov/36394263))

963. [YuSugihara/MutMap](https://github.com/YuSugihara/MutMap) (‚≠ê43 ¬∑ Python) MutMap pipeline to identify causative mutations responsible for a phenotype (PMID: [35321412](https://pubmed.ncbi.nlm.nih.gov/35321412))

964. [ablab/TandemTools](https://github.com/ablab/TandemTools) (‚≠ê43 ¬∑ C++) Tool for assessing/improving assembly quality in extra-long tandem repeats (PMID: [32657355](https://pubmed.ncbi.nlm.nih.gov/32657355))

965. [andreasmang/claire](https://github.com/andreasmang/claire) (‚≠ê43 ¬∑ C++) Constrained Large Deformation Diffeomorphic Image Registration (CLAIRE) (PMID: [35295546](https://pubmed.ncbi.nlm.nih.gov/35295546))

966. [brentp/combined-pvalues](https://github.com/brentp/combined-pvalues) (‚≠ê43 ¬∑ Python) combining p-values using modified stouffer-liptak for spatially correlated results (probes) (PMID: [22954632](https://pubmed.ncbi.nlm.nih.gov/22954632))

967. [cellgeni/batchbench](https://github.com/cellgeni/batchbench) (‚≠ê43 ¬∑ R) None (PMID: [33524142](https://pubmed.ncbi.nlm.nih.gov/33524142))

968. [ctlab/GADMA](https://github.com/ctlab/GADMA) (‚≠ê43 ¬∑ Python) Genetic Algorithm for Demographic Model Analysis (PMID: [32112099](https://pubmed.ncbi.nlm.nih.gov/32112099))

969. [flatironinstitute/inferelator](https://github.com/flatironinstitute/inferelator) (‚≠ê43 ¬∑ Python) Task-based gene regulatory network inference using single-cell or bulk gene expression data conditioned on a prior network. (PMID: [35188184](https://pubmed.ncbi.nlm.nih.gov/35188184))

970. [foxlf823/Multi-Filter-Residual-Convolutional-Neural-Network](https://github.com/foxlf823/Multi-Filter-Residual-Convolutional-Neural-Network) (‚≠ê43 ¬∑ Python) Multi-Filter Residual Convolutional Neural Network for Text Classification (PMID: [34322282](https://pubmed.ncbi.nlm.nih.gov/34322282))

971. [gbouras13/plassembler](https://github.com/gbouras13/plassembler) (‚≠ê43 ¬∑ Python) Program to quickly and accurately assemble plasmids in hybrid and long-only sequenced bacterial isolates (PMID: [37369026](https://pubmed.ncbi.nlm.nih.gov/37369026))

972. [guopengf/FL-MRCM](https://github.com/guopengf/FL-MRCM) (‚≠ê43 ¬∑ Python) Multi-institutional Collaborations for Improving Deep Learning-based Magnetic Resonance Image Reconstruction Using Federated Learning (PMID: [35444379](https://pubmed.ncbi.nlm.nih.gov/35444379))

973. [leylabmpi/Struo](https://github.com/leylabmpi/Struo) (‚≠ê43 ¬∑ Jupyter Notebook) Ley Lab MetaGenome Profiler DataBase generator (PMID: [31778148](https://pubmed.ncbi.nlm.nih.gov/31778148))

974. [liutairan/eMolFrag](https://github.com/liutairan/eMolFrag) (‚≠ê43 ¬∑ Python) eMolFrag is a molecular fragmentation tool based on BRICS algorithm written in Python. (PMID: [28346786](https://pubmed.ncbi.nlm.nih.gov/28346786))

975. [lmweber/cytometry-clustering-comparison](https://github.com/lmweber/cytometry-clustering-comparison) (‚≠ê43 ¬∑ R) R scripts to reproduce analyses in our paper comparing clustering methods for high-dimensional cytometry data (PMID: [27992111](https://pubmed.ncbi.nlm.nih.gov/27992111))

976. [svalkiers/clusTCR](https://github.com/svalkiers/clusTCR) (‚≠ê43 ¬∑ Python) CDR3 clustering module providing a new method for fast and accurate clustering of large data sets of CDR3 amino acid sequences, and offering functionalities for downstream analysis of clustering results. (PMID: [34132766](https://pubmed.ncbi.nlm.nih.gov/34132766))

977. [tbwxmu/SAMPN](https://github.com/tbwxmu/SAMPN) (‚≠ê43 ¬∑ Python) None (PMID: [33431047](https://pubmed.ncbi.nlm.nih.gov/33431047))

978. [vaquerizaslab/tadtool](https://github.com/vaquerizaslab/tadtool) (‚≠ê43 ¬∑ Python) TADtool is an interactive tool for the identification of meaningful parameters in TAD-calling algorithms for Hi-C data. (PMID: [27318199](https://pubmed.ncbi.nlm.nih.gov/27318199))

979. [vlothec/TRASH](https://github.com/vlothec/TRASH) (‚≠ê43 ¬∑ R) RepeatIdentifier (PMID: [37162382](https://pubmed.ncbi.nlm.nih.gov/37162382))

980. [yfukai/laptrack](https://github.com/yfukai/laptrack) (‚≠ê43 ¬∑ Python) Particle tracking by solving linear assignment problem. (PMID: [36495181](https://pubmed.ncbi.nlm.nih.gov/36495181))

981. [BSGOxford/BrowseVCF](https://github.com/BSGOxford/BrowseVCF) (‚≠ê42 ¬∑ Python) BrowseVCF is a web-based application and workflow to quickly prioritise disease-causative variants in VCF files. (PMID: [27373737](https://pubmed.ncbi.nlm.nih.gov/27373737))

982. [CancerDataScience/NuCLS](https://github.com/CancerDataScience/NuCLS) (‚≠ê42 ¬∑ Jupyter Notebook) NuCLS: A scalable crowdsourcing, deep learning approach and dataset for nucleus classification, localization and segmentation (PMID: [34586355](https://pubmed.ncbi.nlm.nih.gov/34586355))

983. [JiajinChen/shinyBN](https://github.com/JiajinChen/shinyBN) (‚≠ê42 ¬∑ R) shinyBN: An online application for interactive Bayesian network inference and visualization (PMID: [31842743](https://pubmed.ncbi.nlm.nih.gov/31842743))

984. [YuSugihara/QTL-seq](https://github.com/YuSugihara/QTL-seq) (‚≠ê42 ¬∑ Python) QTL-seq pipeline to identify causative mutations responsible for a phenotype (PMID: [35321412](https://pubmed.ncbi.nlm.nih.gov/35321412))

985. [cbielow/PTXQC](https://github.com/cbielow/PTXQC) (‚≠ê42 ¬∑ HTML) A Quality Control (QC) pipeline for Proteomics (PTX) results generated by MaxQuant (PMID: [26653327](https://pubmed.ncbi.nlm.nih.gov/26653327))

986. [deweylab/metasra-pipeline](https://github.com/deweylab/metasra-pipeline) (‚≠ê42 ¬∑ Python) MetaSRA: normalized sample-specific metadata for the Sequence Read Archive (PMID: [28535296](https://pubmed.ncbi.nlm.nih.gov/28535296))

987. [dkoslicki/CMash](https://github.com/dkoslicki/CMash) (‚≠ê42 ¬∑ Python) Fast and accurate set similarity estimation via containment min hash (PMID: [35758788](https://pubmed.ncbi.nlm.nih.gov/35758788))

988. [dkoslicki/MetaPalette](https://github.com/dkoslicki/MetaPalette) (‚≠ê42 ¬∑ Python) Metagenomic profiling and phylogenetic distances via common kmers (PMID: [27822531](https://pubmed.ncbi.nlm.nih.gov/27822531))

989. [idruglab/hignn](https://github.com/idruglab/hignn) (‚≠ê42 ¬∑ Jupyter Notebook) Code for "HiGNN: A Hierarchical Informative Graph Neural Network for Molecular Property Prediction Equipped with Feature-Wise Attention" (PMID: [36519623](https://pubmed.ncbi.nlm.nih.gov/36519623))

990. [ikalatskaya/ISOWN](https://github.com/ikalatskaya/ISOWN) (‚≠ê42 ¬∑ Perl) None (PMID: [28659176](https://pubmed.ncbi.nlm.nih.gov/28659176))

991. [josuebarrera/GenEra](https://github.com/josuebarrera/GenEra) (‚≠ê42 ¬∑ Shell) genEra is a fast and easy-to-use command-line tool that estimates the age of the last common ancestor of protein-coding gene families. (PMID: [36964572](https://pubmed.ncbi.nlm.nih.gov/36964572))

992. [mahmoodlab/HistoFL](https://github.com/mahmoodlab/HistoFL) (‚≠ê42 ¬∑ Python) Federated Learning for Computational Pathology - Medical Image Analysis (PMID: [34911013](https://pubmed.ncbi.nlm.nih.gov/34911013))

993. [schallerdavid/pyrod](https://github.com/schallerdavid/pyrod) (‚≠ê42 ¬∑ Python) PyRod - Tracing water molecules in molecular dynamics simulations (PMID: [31117512](https://pubmed.ncbi.nlm.nih.gov/31117512))

994. [shingocat/lrscaf](https://github.com/shingocat/lrscaf) (‚≠ê42 ¬∑ Java) TGS scaffolding (PMID: [31818249](https://pubmed.ncbi.nlm.nih.gov/31818249))

995. [ssolo/ALE](https://github.com/ssolo/ALE) (‚≠ê42 ¬∑ C++) Amalgamated likelihood estimation (ALE) is a probabilistic approach to exhaustively explore all reconciled gene trees that can be amalgamated as a combination of clades observed in a sample of gene trees. We implement the ALE approach in the context of a reconciliation model (cf. http://arxiv.org/abs/1211.4606 ), which allows for the duplication, transfer and loss of genes. We use ALE to efficiently approximate the sum of the joint likelihood over amalgamations and to find the reconciled gene tree that maximizes the joint likelihood among all such trees. (PMID: [23925510](https://pubmed.ncbi.nlm.nih.gov/23925510))

996. [thomas0809/RxnScribe](https://github.com/thomas0809/RxnScribe) (‚≠ê42 ¬∑ Jupyter Notebook) A Sequence Generation Model for Reaction Diagram Parsing (PMID: [37368970](https://pubmed.ncbi.nlm.nih.gov/37368970))

997. [venyao/shinyChromosome](https://github.com/venyao/shinyChromosome) (‚≠ê42 ¬∑ R) an R/Shiny application for interactive creation of non-circular plots of whole genomes (PMID: [31931182](https://pubmed.ncbi.nlm.nih.gov/31931182))

998. [wyang17/SQuIRE](https://github.com/wyang17/SQuIRE) (‚≠ê42 ¬∑ Python) Software for Quantifying Interspersed Repeat Expression (PMID: [30624635](https://pubmed.ncbi.nlm.nih.gov/30624635))

999. [xulabs/projects](https://github.com/xulabs/projects) (‚≠ê42 ¬∑ Python) None (PMID: [31510673](https://pubmed.ncbi.nlm.nih.gov/31510673); [36951805](https://pubmed.ncbi.nlm.nih.gov/36951805))

1000. [zhqingit/giremi](https://github.com/zhqingit/giremi) (‚≠ê42 ¬∑ R) GIREMI is a method that can identify RNA editing sites using one RNA-seq data set without requiring genome sequence data. (PMID: [29508292](https://pubmed.ncbi.nlm.nih.gov/29508292))

